<html xmlns="http://www.w3.org/1999/xhtml"><head> <style type="text/css">
         	     		 
         	    a                          {background : #ffffff; }
         	    article                    {border-style : dotted; border-width : 2px; }
         		 	
         		 	div                        {background : #ffffcc;}
         		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
         		 	div.ack                    {border-style : solid ; border-color : red; margin
         : 2em; }
         		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.boxed-text             {margin : 5em; border-style : solid;}
         		 	div.contrib-group          {margin : 1em; }
         		 	div.fig                    {border-style : solid; border-width : 2px; margin :
         2em; }
         		 	div.funding                {font-weight : bold ; font-size : 16pt;}
         		 	div.given-names            {font-style : italic;}
         		 	div.intro                  {border-style : inset; margin : 5px;}
         		 	div.introduction           {border-style : inset; margin : 5px;}
         		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.journal-title-group    {background : #ffeeee;}
         		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
         		 	div.kwd                    {font-style : italic}
         		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
         		 	div.name                   {font-weight : bold;}
         		 	div.alt-title              {font-style : italic; font-size : 12px;}
         		 	
         		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
         		 	div.title                  {font-family : courier; font-weight : bold;
         		 	                            font-size : 16pt; margin : 5px;}
         		 	
         		 	div.materials_methods      {border-style : double; margin : 5px; }
         		 	div.methods                {border-style : double; margin : 5px; }
         		 	
         		 	div.results                {border-style : solid; margin : 5px;}
         		 	div.background             {border-style : dotted; margin : 5px;}
         		 	
         		 	div.discussion             {border-style : groove; margin : 5px;}
         		 	div.conclusion             {border-style : ridge; margin : 5px;}
         		 	div.conclusions             {border-style : ridge; margin : 5px;}
         		 	div.supplementary-material {border-style : inset; margin : 5px;}
         		 	div.abbreviations          {border-style : double; border-color : red; margin
         : 5px;}
         		 	div.competinginterests     {border-style : double; border-color : blue; margin
         : 5px;}
         		 	div.acknowledgements       {border-style : double; border-color : green; margin
         : 5px;}
         		 	div.authors_contributions   {border-style : double; border-color : purple; margin
         : 5px;}
         		 	
         		 	div.publisher              {border-style : outset; margin : 5px;}
         		 	div.fn-type-conflict       {background : #f88; }
         		 	div.fn-type-con            {background : #ddf; }
         		 	div.fn-type-other          {background : #ddd; }
         		 	
         		 	div.unknown                {background : #ffd;
         									 	  border-style : solid;
         									 	  border-width : 1px;
         									 	  padding : 2 px;}
         		 	  
         	    table                      {background : #ffffdd;}
         		 	tr                         {background : #ddddff; padding : 1px;}
         		 	
         		 	span                       {background : #ffcccc;}
         		 	
         		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
         		 	span.collab                {background : #ddffff; }
         		 	span.comment               {font-family : courier; font-size : 6px; background
         : #ffaaff;}
         		 	span.contrib               {background : #ffffff;}
         		 	span.corresp               {background : #ddffdd; }
         		 	span.doi                   {background : #ffffff;}
         		 	span.email                 {font-family : courier; }
         		 	span.etal                  {font-style : italic;}
         		 	span.fpage                 {font-family : courier;}
         		 	span.given-names           {background : #ffffff;}
         		 	span.iso-abbrev            {background : #ffffff;}
         		 	span.issn-epub             {background : #ffffff;}
         		 	span.issn-ppub             {background : #ffffff;}
         		 	span.journal-title         {background : #ffffff;}
         		 	span.lpage                 {font-family : courier;}
         		 	span.mixed-article-title   {font-style : italic ;}
         		 	span.nlm-ta                {background : #ffffff;}
         		 	span.pmc                   {background : #ffffff;}
         		 	span.pmcid                 {background : #ffffff;}
         		 	span.pmid                  {background : #ffffff;}
         		 	span.publisher             {background : #ffffff;}
         		 	span.publisher-id          {background : #ffffff;}
         		 	span.publisher-name        {background : #ffffff;}
         		 	span.source                {background : #ffffff;}
         		 	span.subject               {background : #ffffff;}
         		 	span.surname               {background : #ffffff;}
         		 	span.volume                {font-family : courier; font-weight : bold;}
         		 	span.year                  {font-family : courier ; font-style : italic;}
         			
      </style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"> <span class="nlm-ta" title="nlm-ta">Molecules</span> <span class="iso-abbrev" title="iso-abbrev">Molecules</span> <span class="publisher-id" title="publisher-id">molecules</span> <div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"> <span class="journal-title" tagx="journal-title" title="journal-title">Molecules</span> </div> <span class="issn-epub" tagx="issn" title="issn-epub">1420-3049</span> <div class="publisher" tagx="publisher" title="publisher"> <span class="publisher-name" tagx="publisher-name" title="publisher-name">MDPI</span> </div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"> <span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/6514702">6514702</a> </span> <span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.3390/molecules24081593">10.3390/molecules24081593</a> </span> <span class="publisher-id" title="publisher-id">molecules-24-01593</span> <div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Article</span> </div> <div class="title-group"> <div class="article-title" title="article-title">
High-Throughput Biochemical Fingerprinting of Oleaginous <i>Aurantiochytrium</i> sp. Strains by Fourier Transform Infrared Spectroscopy (FT-IR) for Lipid and Carbohydrate Productions </div> </div> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yu</span> <span class="given-names" tagx="given-names" title="given-names">Xin-Jun</span> </span> <a href="#af1-molecules-24-01593">1</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Huang</span> <span class="given-names" tagx="given-names" title="given-names">Chang-Yi</span> </span> <a href="#af1-molecules-24-01593">1</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">Hong</span> </span> <a href="#af1-molecules-24-01593">1</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">Dong-Sheng</span> </span> <a href="#af2-molecules-24-01593">2</a> <a href="#c1-molecules-24-01593">*</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">Jing-Liang</span> </span> <a href="#af1-molecules-24-01593">1</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">Hui-Juan</span> </span> <a href="#af3-molecules-24-01593">3</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">Xiao-Yan</span> </span> <a href="#af4-molecules-24-01593">4</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">Zhao</span> </span> <a href="#af1-molecules-24-01593">1</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sun</span> <span class="given-names" tagx="given-names" title="given-names">Jie</span> </span> <a href="#af1-molecules-24-01593">1</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">Zhi-Peng</span> </span> <a href="#af5-molecules-24-01593">5</a> </span> </div> <span class="citation_author_institution" id="af1-molecules-24-01593">[1], <span class="email" tagx="email" title="email">xjyu@zjut.edu.cn</span> <span class="email" tagx="email" title="email">hcy929390419@gmail.com</span> <span class="email" tagx="email" title="email">hzchenhong1016@163.com</span> <span class="email" tagx="email" title="email">jlchen2018@163.com</span> <span class="email" tagx="email" title="email">hzwangzhao@sina.cn</span> <span class="email" tagx="email" title="email">jsun@zjut.edu.cn</span> </span> <span class="citation_author_institution" id="af2-molecules-24-01593">[2], </span> <span class="citation_author_institution" id="af3-molecules-24-01593">[3], <span class="email" tagx="email" title="email">lihuijuan611@126.com</span> </span> <span class="citation_author_institution" id="af4-molecules-24-01593">[4], <span class="email" tagx="email" title="email">catty5082003@163.com</span> </span> <span class="citation_author_institution" id="af5-molecules-24-01593">[5], <span class="email" tagx="email" title="email">wangzp@ysfri.ac.cn</span> </span> <div class="author-notes" title="author-notes"> <div class="corresp" title="corresp"> <span class="label" tagx="label" title="label">*</span>Correspondence: <span class="email" tagx="email" title="email">w_d_sh@126.com</span>; Tel./Fax: +86-0791-88175720 </div> </div> <span class="pub-date-epub" title="pub-date-epub">epub: <span>2019-4-4</span> </span> <span class="pub-date-collection" title="pub-date-collection">collection: <span>2019-4</span> </span> <span class="volume" tagx="volume" title="volume">24</span> <span class="issue" tagx="issue" title="issue">8</span> <span class="elocation-id" tagx="elocation-id" title="elocation-id">1593</span> <span class="history" title="history"> <span class="received" title="received">received: 2019-3-20</span> <span class="accepted" title="accepted">accepted: 2019-4-19</span> </span> <div class="permissions"> <span class="copyright" title="copyright">(C) , <span class="copyright-year" tagx="copyright-year" title="copyright-year">2019</span> </span> <span class="license" title="license"> <span class="license-p" title="license-p">Licensee MDPI, Basel, Switzerland. This article is an open access article distributed under the terms and conditions of the Creative Commons Attribution (CC BY) license ( <a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>). </span> </span> </div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>The traditional biochemical methods for analyzing cellular composition of oleaginous microorganisms are time-consuming, polluting, and expensive. In the present study, an FT-IR method was used to analyze the cellular composition of the marine oleaginous protist <i>Aurantiochytrium</i> sp. during various research processes, such as strains screening, medium optimization, and fermentation, and was evaluated as a green, low-cost, high throughput, and accurate method compared with the traditional methods. A total of 109 <i>Aurantiochytrium</i> sp. strains were screened for lipid and carbohydrate production and the best results were found for the strains No. 6 and No. 32. The yields and productivities could reach up to 47.2 g/L and 0.72 g/L/h for lipid, 21.6 g/L and 0.33 g/L/h for docosahexaenoic acid (DHA) in the strain No. 6, and 15.4 g/L and 0.18 g/L/h for carbohydrate in the strain No. 32, under the optimal conditions, respectively. These results confirmed potentials of the two <i>Aurantiochytrium</i> sp. strains for lipid, DHA, and carbohydrate productions at industrial scales. The FT-IR method in this study will facilitate research on the oleaginous <i>Aurantiochytrium</i> sp., and the obtained two strains for lipid and carbohydrate productions will provide the foundations for their applications in medical, food, and feed industries. </p> </div> <div class="kwd-group"> <div class="kwd" title="kwd"> <i>Aurantiochytrium</i> sp., carbohydrate </div> <div class="kwd" title="kwd">
DHA</div> <div class="kwd" title="kwd">
feed</div> <div class="kwd" title="kwd">
FT-IR</div> <div class="kwd" title="kwd">
high-throughput</div> </div> </div> </div> <div class="body" title="body"> <div class="intro" title="intro"> <div class="title" tagx="title" title="title">
1. Introduction</div> <p>Docosahexaenoic acid (DHA), a kind of polyunsaturated fatty acids (PUFAs), plays essential roles in alleviating cardiovascular diseases, hypertension, diabetes, neuropsychiatric disorders, and cancer in humans [ <a href="#B1-molecules-24-01593">1</a>]. Moreover, aquatic animals and poultries also require DHA for their development and survival [ <a href="#B2-molecules-24-01593">2</a>]. <i>Aurantiochytrium</i> sp. is a kind of <i>Thraustochytrids</i>, which belong to a group of heterotrophic, unicellular, marine protists, and <i>Aurantiochytrium</i>, <i>Oblongichytrium</i>, and <i>Schizochytrium</i> in <i>Thraustochytrids</i> are considered to form one genus [ <a href="#B3-molecules-24-01593">3</a>]. As a well-known oleaginous microorganism, <i>Aurantiochytrium</i> sp. can accumulate a large amount of lipid, rich in DHA in its cell. Thus, <i>Aurantiochytrium</i> sp. is a DHA producer and a substitute for the traditional fish oil resource as a new DHA resource [ <a href="#B4-molecules-24-01593">4</a>]. Moreover, cells of <i>Aurantiochytrium</i> sp. are rich in valuable metabolites, such as terpenoids, pigment, carbohydrate, and protein, which have high nutritional values and multiple physiological functions. Thus, <i>Aurantiochytrium</i> sp. has been widely applied in the medical, food, and feed industries [ <a href="#B5-molecules-24-01593">5</a>, <a href="#B6-molecules-24-01593">6</a>]. </p> <p>A large number of biochemical methods are essential for the analysis of the cellular compositions of <i>Aurantiochytrium</i> sp. in numerous types of researches, such as strain screening, medium optimization and fermentation monitoring for productions of the high-value metabolites. For examples, lipid content in the cell was analyzed through organic solvent extraction, evaporation and weighing [ <a href="#B7-molecules-24-01593">7</a>, <a href="#B8-molecules-24-01593">8</a>]. Some cellular dyes, such as Sudan Black B and Nile red, were also used to dye lipid in the cell, and the fluorescence intensity was analyzed for lipid quantification [ <a href="#B9-molecules-24-01593">9</a>, <a href="#B10-molecules-24-01593">10</a>]. A wet extraction method was directly exerted on the wet cells of <i>Aurantiochytrium</i> sp. for lipid extraction and analysis through a high shear mixer (HSM) [ <a href="#B11-molecules-24-01593">11</a>]. Otherwise, pyrolysis of cell based on the thermogravimetric analysis and ultrasound-assisted transesterification in wet <i>Aurantiochytrium</i> sp. cells using potassium carbonate were also used to treat cell for analysis of bio-macromolecules [ <a href="#B12-molecules-24-01593">12</a>, <a href="#B13-molecules-24-01593">13</a>]. The phenol-sulfuric acid method, DNS method, folin phenol method, sulfuric acid-anthrone method, Bradford method, etc., which relies on biomass extraction and needs a large amount of the biomass sample, were used for protein or carbohydrate quantification [ <a href="#B14-molecules-24-01593">14</a>, <a href="#B15-molecules-24-01593">15</a>, <a href="#B16-molecules-24-01593">16</a>]. </p> <p>Moreover, large numbers of procedures, such as cell lyophilization and disruption, treatments using organic or inorganic reagents, the calibration standard, etc., were all needed in these traditional biochemical methods. In summary, these traditional biochemical methods described above are time-consuming, heavily polluting and require a large amount of equipment. Thus, there is a pressing need for developing reliable, environment-friendly and fast methods to analyze cellular composition in numerous researches for <i>Aurantiochytrium</i> sp. </p> <p>FT-IR (Fourier Transform Infrared Spectroscopy) is molecular vibrational spectroscopy that dissects chemical functional groups in the mid-infrared zone between 4000 and 400 cm <sup>−1</sup>. The typical functional chemical groups in lipid, protein, and carbohydrate, such as C–H stretching vibration acyl chains, ester C=O stretching band, C=O stretching vibration, N–H bending vibration, C–O and C–O–C stretching band, and so on, have characteristic absorbance in different frequency regions of the mid-infrared zone. Moreover, FT-IR can directly scan the cultured microalgal cells for cellular composition analysis without complex procedures and organic or inorganic reagents which are necessities in the traditional biochemical methods [ <a href="#B17-molecules-24-01593">17</a>]. Thus, FT-IR has been a rapid, green, high throughput and cost-effective tool for monitoring and analysis of the microalgal cellular composition [ <a href="#B17-molecules-24-01593">17</a>, <a href="#B18-molecules-24-01593">18</a>]. The lipid content of <i>Thraustochytrium</i> sp., belonging to the <i>Thraustochytrids</i> same as <i>Aurantiochytrium</i> sp., has been analyzed by the FT-IR technology for comparing with the marine oleaginous yeast [ <a href="#B19-molecules-24-01593">19</a>]. The <i>Schizochytrium</i> sp. cell was pyrolyzed, and the cellular lysates were analyzed by FT-IR method for cellular composition characterization [ <a href="#B20-molecules-24-01593">20</a>]. The cellular composition of the <i>Aurantiochytrium</i> sp. KRS101, including protein, lipid, carbohydrate, and ash, was been analyzed through FT-IR method [ <a href="#B21-molecules-24-01593">21</a>]. However, the feasibility and reliability of FT-IR method for analyzing cellular composition in <i>Aurantiochytrium</i> sp. have rarely evaluated comparing with the traditional biochemical methods. </p> <p>In the present study, an analytic method based on the FT-IR technology for the three main bio-macromolecules, including lipid, carbohydrate, and protein, will be constructed to analyze the cellular composition of the oleaginous protist <i>Aurantiochytrium</i> sp. This method will be applied in strain screening, medium optimization and fermentation of <i>Aurantiochytrium</i> sp. The feasibility and reliability of the FT-IR method and the potentials of <i>Aurantiochytrium</i> sp. strains for lipid and carbohydrate productions will be evaluated, which accelerates the research development of <i>Aurantiochytrium</i> sp., and improves its applications in medical, food and feed industries. </p> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
2. Results and Discussion</div> <div class="2.1.ft-irinaurantiochytriumsp.strainsscreening" title="sec"> <div class="title" tagx="title" title="title">
2.1. FT-IR in Aurantiochytrium sp. Strains Screening</div> <p>Screening <i>Aurantiochytrium</i> sp. strains with high-level content of lipid or carbohydrate is the first and most essential step for achieving their significant industrial value. However, traditional biochemical analysis of lipid and carbohydrate need a long time, a large amount of cell culture and multi-instruments, leading to time and space limitations for screening. FT-IR is a high-throughput method for cellular biomolecular analysis, and needs a trace amount of cell culture, enabling it to be used for the rapid and high- throughput screening of <i>Aurantiochytrium</i> sp. strains. </p> <p>Protein, carbohydrate, and lipid are three major bio-macromolecules of the cell and play different roles in the cell. Protein is closely related to basic biosynthesis and cell division, while lipids and carbohydrate serve as intracellular reservoirs of carbon and energy. Cells preferentially synthesize carbohydrate or lipid rather than protein for reserving carbon and energy to answer the poor growth condition. To screen <i>Aurantiochytrium</i> sp. strains synthesizing high level of carbohydrate and lipid, the medium with nitrogen limitation condition, which might increase the carbohydrate and lipid synthesis and lead to a reduced synthesis of the protein [ <a href="#B22-molecules-24-01593">22</a>], was used in this study. A total of 109 strains in our <i>Aurantiochytrium</i> sp. strain library were analyzed by FT-IR. Based on the cellular contents of lipid, carbohydrate, and protein, a total of 18 strains were picked up among the 109 strains for further analysis after the 1st screening round (Data not shown). The FT-IR absorption profile for the <i>Aurantiochytrium</i> sp. strain was shown in <a href="#molecules-24-01593-f001">Figure 1</a>A. Lipid was the bio-macromolecule with the highest content in <i>Aurantiochytrium</i> sp. cell, followed by protein and carbohydrate in proper order, consistent with its oleaginous characteristic and similar to other <i>Aurantiochytrium</i> sp. strain [ <a href="#B22-molecules-24-01593">22</a>]. Absorption areas of representing peaks were used to quantify the contents of protein, carbohydrate, and lipid. As shown in <a href="#molecules-24-01593-f002">Figure 2</a>A,B, the highest content of lipid was obtained in <i>Aurantiochytrium</i> sp. strain 6, while the lowest content of protein was obtained in this strain. The strain No. 32, with the highest content of carbohydrate, is a potential carbohydrate producer. Oppositely, the <i>Aurantiochytrium</i> sp. strain 103 had the highest content of protein in its cell, with the lowest content of lipid and carbohydrate. These results indicated that the contents of carbohydrate and lipid, and the content of protein were negatively correlated in <i>Aurantiochytrium</i> sp. cell. The traditional biochemical methods for analyzing lipid, carbohydrate, and protein in the <i>Aurantiochytrium</i> sp. cells were applied to validate the accuracy of FT-IR analysis ( <a href="#molecules-24-01593-f002">Figure 2</a>B). To correct the effect of deuterated metabolic products on contents of three biomolecules in the <i>Aurantiochytrium</i> sp. cells, the <i>Aurantiochytrium</i> sp. strain No. 6 was cultured in the medium prepared with D <sub>2</sub>O for cell deuteration (D group). Meanwhile, the cells cultured in the medium prepared with H <sub>2</sub>O were set as the H group. As shown in <a href="#molecules-24-01593-f001">Figure 1</a>, the FT-IR profiles of the D- and H-group cells were very similar. Further quantification analysis deduced by the FT-IR showed that the contents of three biomolecules, including lipid, protein and carbohydrate, were similar for the D- and H-group cells, suggesting that deuteration of cell or deuterated metabolic products distributed in a cell cannot affect the FT-IR results ( <a href="#molecules-24-01593-f002">Figure 2</a>C). Biomass composition of <i>Aurantiochytrium</i> sp. varied greatly among the 18 strains ( <a href="#molecules-24-01593-f003">Figure 3</a>). The lipid made up 35.9–67.0% of DCW, the protein made up 8.0–14.0% of DCW, and the carbohydrate made up 3.8–8.4% of DCW. The FT-IR absorption area and the corresponding protein, carbohydrate and lipid content, were analyzed respectively. As shown in <a href="#molecules-24-01593-f003">Figure 3</a>, good linear correlations among FT-IR absorption area versus protein (R <sup>2</sup> = 0.9381), carbohydrate (R <sup>2</sup> = 0.9226) and lipid (R <sup>2</sup> = 0.9164) contents, respectively, were observed in 18 <i>Aurantiochytrium</i> sp. strains. In this study, two normalization strategies, with amide I (1690–1650 cm <sup>−1</sup>, associated with biomass) and C–H stretching region (3000–2800 cm <sup>−1</sup>, associated with lipid) as internal reference peaks for assessments of the relative contents of protein, lipid and carbohydrate, respectively, were applied to minimize the peak absorption’s fluctuation caused by the inhomogeneous thickness of sample tablet [ <a href="#B18-molecules-24-01593">18</a>, <a href="#B23-molecules-24-01593">23</a>]. However, weak linear correlations were obtained through these two normalization methods, indicating that the two normalization methods are highly species-specific among microorganism strains, and not suitable for <i>Aurantiochytrium</i> sp. strains (data not shown). As the Lambert-Beer’s law described, the lipid or carbohydrate content (% DCW) is proportional to the corresponding FT-IR areas normalized by the amide I band (A <sub>lipid or carbohydrate</sub>/A <sub>amide I</sub>) only when the protein content keeps unchanged if the ratio of the molar absorptive coefficients of protein and lipid or carbohydrate is a constant [ <a href="#B24-molecules-24-01593">24</a>]. This law is not suitable in <i>Aurantiochytrium</i> sp. strain due to the high variations of protein content in its cellular composition ( <a href="#molecules-24-01593-f002">Figure 2</a>). In the present study, we normalized the OD <sub>600</sub> of every cell sample to 0.1 and then analyzed its FT-IR profile. There were good linear correlations between lipid, carbohydrate and protein content (%DCW) and the corresponding FT-IR areas normalized by the OD <sub>600</sub> (A <sub>lipid, carbohydrate or protein</sub>/0.1 OD <sub>600</sub>), indicating that FT-IR analysis normalized by the OD <sub>600</sub> can be used to analyze cellular composition of <i>Aurantiochytrium</i> sp. strains. The same method was also applied to normalize the FT-IR peaks to analyze the lipid, carbohydrate and protein content in algae, yeast, and bacteria cell [ <a href="#B25-molecules-24-01593">25</a>]. </p> </div> <div class="2.2.optimizationsofmediaforlipidandcarbohydrateproductions" title="sec"> <div class="title" tagx="title" title="title">
2.2. Optimizations of Media for Lipid and Carbohydrate Productions</div> <p> <i>Aurantiochytrium</i> sp. strain 6 had been screened for lipid and DHA productions; however, the media composition needed to be further optimized to obtain the optimal yields. As shown in <a href="#molecules-24-01593-f004">Figure 4</a>, the optimal lipid content and DHA yield of <i>Aurantiochytrium</i> sp. strain No. 6 were 73.6% DCW and 8 g/L when the concentrations of glucose, peptone and yeast extract were 120, 15 and 10 g/L, 35.1% and 25% higher than those in the initial condition. <i>Aurantiochytrium</i> sp. strain 6 could endure up to 120 g/L glucose, and did not exhibit the glucose inhibition until the concentration of glucose reached up to 140 g/L, which was higher than those of the other <i>Aurantiochytrium</i> sp. strains, and characterized the <i>Aurantiochytrium</i> sp. strain 6 as an ideal lipid and DHA producer needing less fed-batch times with lower potential pollution risk [ <a href="#B26-molecules-24-01593">26</a>, <a href="#B27-molecules-24-01593">27</a>]. C/N ratio in the medium is a key factor regulating lipid synthesis in oleaginous microorganisms, and the high C/N ratio condition (Nitrogen limitation) induces improvement of lipid production for carbon and energy reservations as an anti-stress response. Under nitrogen limitation condition, residual carbon source in medium flow to the lipid synthesis by producing acetoacetyl coenzyme A, which was the structural unit for lipid and DHA synthesis. However, a high C/N ratio is a disadvantage to cell growth and protein synthesis [ <a href="#B28-molecules-24-01593">28</a>]. The optimal C/N ratio of the cultivation medium for lipid and DHA synthesis in strain 6 was up to 4.8 (glucose as carbon resource, peptone and yeast extract as nitrogen resource), which induced lipid and DHA synthesis significantly and had no obvious negative effect on cell growth ( <a href="#molecules-24-01593-f004">Figure 4</a>). Huang et al. found that a low C/N ratio (1.25) was optimal for <i>Aurantiochytrium limacinum</i> SR21 to produce lipid and DHA at a balance condition between biomass and lipid synthesis [ <a href="#B28-molecules-24-01593">28</a>]. Results reported by Ryu et al. indicated that the highest DHA productivity (38.8% of total lipid) was observed in <i>Aurantiochytrium</i> sp. KRS101 when a C/N ratio in the medium was up to 20 [ <a href="#B29-molecules-24-01593">29</a>]. These results validated that the C/N ratio in medium plays a significant role in regulating biomass, lipid and DHA productions in <i>Aurantiochytrium</i> sp., and showed strain-specific among different strains. </p> <p> <i>Aurantiochytrium</i> sp. strain No. 32 had been screened for the high carbohydrate content in its cells, and the medium composition was further optimized for carbohydrate production. As shown in <a href="#molecules-24-01593-f005">Figure 5</a>, the highest carbohydrate content and yield (34.2% DCW, 7.56 g/L) were obtained under the optimal medium with 100 g/L glucose, 20 g/L peptone and 30 g/L yeast extract, 307% and 367% higher than those in the initial condition. As a heterotrophic marine protist, <i>Aurantiochytrium</i> sp. strain No. 32 could utilize up to 100 g/L of glucose to grow fast (22.1 g/L of biomass in 4 days) without glucose effect and inhibition, making it an ideal strain for industrial fermentation purposes. Meanwhile, the carbohydrate content and yield in <i>Aurantiochytrium</i> sp. strain No. 32 cell could reach up to 34.2% of DCW and 7.56 g/L, which were higher than those in <i>A. limacinum</i> SR21 cell (33.38% DCW, 3.84 g/L) and those in <i>Schizochytrium</i> sp. HX-308 (13.58% DCW, 6.63 g/L) [ <a href="#B22-molecules-24-01593">22</a>, <a href="#B30-molecules-24-01593">30</a>]. Under nutrient limitations, microbial cell preferentially synthesized carbohydrate and lipid for the carbon and energy reservation. However, accumulation of lipid was enhanced more quickly than that of carbohydrate in <i>Aurantiochytrium</i> sp. under different nutrient limitation conditions [ <a href="#B22-molecules-24-01593">22</a>]. Thus, a relatively low C/N ratio of 2 was the optimal value for balancing carbohydrate synthesis and biomass in <i>Aurantiochytrium</i> sp. strain No. 32 to avoid over-accumulation of lipid, lower than that (4.8) of <i>Aurantiochytrium</i> sp. strain 6 for lipid production. Meanwhile, many micro-elements, such as metal elements and vitamins, were essential for <i>Aurantiochytrium</i> sp. strains for cell growth and metabolite synthesis, increasing the production cost significantly [ <a href="#B31-molecules-24-01593">31</a>]. Unlike the other strains, <i>Aurantiochytrium</i> sp. strain No. 32 did not need any micro-elements to obtain a high level of biomass and carbohydrate in its cell. This result indicates that the <i>Aurantiochytrium</i> sp. strain No. 32 is an ideal potential candidate for producing carbohydrate. </p> <p>Moreover, the FT-IR method was also used for lipid and carbohydrate contents analysis in optimization performances, and the results (% DCW) from the FT-IR method and the traditional biochemical method had high linear correlations with R <sup>2</sup> values of 0.9006 for lipid analysis and 0.9575 for carbohydrate analysis, respectively ( <a href="#molecules-24-01593-f004">Figure 4</a> and <a href="#molecules-24-01593-f005">Figure 5</a>). The linear equations representing lipid and carbohydrate contents with the FT-IR method (x-axis) vs the traditional biochemical method (y-axis) were y = 1.5168x − 26.143 for lipid and y = 1.0231x − 0.1715 for carbohydrate ( <a href="#molecules-24-01593-f004">Figure 4</a> and <a href="#molecules-24-01593-f005">Figure 5</a>). The area of the 1740 cm <sup>−1</sup> in FT-IR, representing the ester C=O stretching band, was used to quantify the lipid content in <i>Thraustochytrium</i> sp. cells [ <a href="#B19-molecules-24-01593">19</a>]. Meanwhile, the 3000–2800 cm <sup>−1</sup> area of FT-IR, representing the C–H stretching vibration acyl chains, was also used to analyze the lipid contents in marine microalgae with the amino I area (1650 cm <sup>−1</sup>) as the internal standard. However, this range of area in FT-IR could not be used to quantify the lipid content in some microalgae due to its high variability in this area of FT-IR caused by the overlapping of OH bond absorption and therefore leading to misinterpretations [ <a href="#B32-molecules-24-01593">32</a>]. However, these two areas in FT-IR, representing two important chemical bonds in lipid, were firstly considered in combination for lipid analysis in <i>Aurantiochytrium</i> sp. cells due to its high lipid content and relative stable profile in the 3000–2800 cm <sup>−1</sup> of FT-IR, and a high accuracy for lipid analysis (R <sup>2</sup> = 0.9006) was obtained through this analysis method. However, these results above were obtained at an approximate lipid content range of 45–73% DCW, cells with the lipid content under 26% DCW might not be analyzed by this FT-IR method. This result indicated that the FT-IR method for lipid analysis used in this study is specific for the oleaginous microorganism, such as <i>Aurantiochytrium</i> sp. (with higher lipid content, up to 70% in cells), while not fit for the microorganisms with low lipid content. </p> <p>For carbohydrate analysis, absorption signals around 1180–1133 cm <sup>−1</sup> in FT-IR characterized over a C–O, and the C–O–C stretching band area was used to quantify the total carbohydrate content in the cell. The C–O and C–O–C stretching bands were the typical bonds in carbohydrate, and a high coefficient of determination (0.9575) was obtained through this method, indicating that the 1200–900 cm <sup>−1</sup> in FT-IR response accurately to the carbohydrate content in the <i>Aurantiochytrium</i> sp. cells. Absorption area of around the 1200–900 cm <sup>−1</sup> band in FT-IR was also used to analyze carbohydrate content in <i>Chlorella pyrenoedosa</i>, <i>Nannochloropsis</i> sp., <i>Botryococcus braunii</i> and <i>Microcystis aeruginosa</i> [ <a href="#B33-molecules-24-01593">33</a>]. </p> </div> <div class="2.3.assessmentsoftheft-irandtraditionalmethods" title="sec"> <div class="title" tagx="title" title="title">
2.3. Assessments of the FT-IR and Traditional Methods</div> <p>Assessments of the FT-IR and traditional methods for analysis of bio-macromolecule were summarized in <a href="#molecules-24-01593-t001">Table 1</a>. At least 1 g of the dried cell is needed for analysis using the traditional biochemical method; thus, at least 50 mL of liquid sample was cultured and lyophilized. The time consumption of this method for analysis is at least 72 h, including cell lyophilization and grinding, lipid extraction and weighing, carbohydrate extraction, reaction with sulfuric acid and anthrone, and analysis of carbohydrate content. A large number of reagents, such as liquid nitrogen, chloroform, methanol, sulfuric acid, and anthrone, which have severe corrosivity, irritation, and pollution, have to be used in the traditional biochemical method. Moreover, numerous pieces of equipment, such as the centrifuge, freeze dryer, mortar, pestle, evaporimeter, and balance, are also essential for the traditional method. Thus, the traditional biochemical method for bio-macromolecules is time-consuming, expensive, equipment-limited and severely polluted. The 96 well microplates are used for analysis in FT-IR. Therefore, only approximately 200 μL of the liquid sample is needed for every sample analysis through FT-IR. Without cell lyophilization, bio-macromolecule extraction and analysis, only approximate 4 h is required for FT-IR analysis, saving approximate 17 times than that of the traditional biochemical method. Moreover, due to the simple process, only the FT-IR spectrometer and spectrophotometer are needed in FT-IR, and no chemical reagent is required for analysis. Unlike the traditional biochemical methods, which can detect only one kind of bio-macromolecule at a time, the FT-IR can simultaneously detect the content of lipid, carbohydrate and protein components. However, the overlap of the stretching vibration peaks of different biochemical components made FT-IR analysis a certain degree of quantity inaccuracy. In this study, various combinations of peaks-integration and normalization were analyzed and compared with the results from the traditional biochemical methods, and high coefficients of determination (R = 0.9006 for lipid and R = 0.9575 for carbohydrate) were obtained. Thus, FT-IR analysis is a very economical and green method with less pollution and high accuracy for bio-macromolecule analysis in <i>Aurantiochytrium</i> sp. cell. </p> </div> <div class="2.4.ft-irformonitoringfermentativeparametersduringlipidandcarbohydrateproductions" title="sec"> <div class="title" tagx="title" title="title">
2.4. FT-IR for Monitoring Fermentative Parameters during Lipid and Carbohydrate Productions</div> <p>Based on the optimized media for the <i>Aurantiochytrium</i> sp. strains No. 6 and No. 32, respectively, 5 L fermentations were performed for lipid and carbohydrate productions under the FT-IR monitoring. As shown in <a href="#molecules-24-01593-f006">Figure 6</a>A, a total of 119.1 g/L reducing sugar was consumed, and a maximum DCW of up to 66.4 g/L was obtained in 60 h. The cell of <i>Aurantiochytrium</i> sp. strains No. 6 entered the stable stage of growth at 54 h and was kept until the end of fermentation at 72 h. At the later period of fermentation, the maximum lipid content and DHA yield, up to 72.4% DCW and 21.6 g/L, respectively, were obtained at 66 h, suggesting that the carbon resource was first consumed for cell growth of <i>Aurantiochytrium</i> sp. strains No. 6, and then for lipid synthesis when the carbon resource was consumed to deficiency. The DCW, lipid and DHA productivities of <i>Aurantiochytrium</i> sp. strain No. 6 could reach up to 1.11 g/L/h, 0.72 g/L/h and 0.33 g/L/h, respectively, higher than those of <i>Aurantiochytrium</i> sp. strains YLH70, SR21, and <i>Japonochytrium marinum</i> AN-4 [ <a href="#B26-molecules-24-01593">26</a>, <a href="#B34-molecules-24-01593">34</a>]. As an essential fermentative parameter of oleaginous <i>Aurantiochytrium</i> sp., lipid content was monitored through FT-IR method synchronously, guiding for lipid and DHA production of <i>Aurantiochytrium</i> sp. with high speed and accuracy. </p> <p>FT-IR method was also used for carbohydrate analysis during the fermentation by <i>Aurantiochytrium</i> sp. strain No. 32. As shown in <a href="#molecules-24-01593-f006">Figure 6</a>B, a total of 99.9 g/L of reducing sugar was consumed during the fermentation process (96 h), with a consumption rate of up to 1.04 g/L/h, indicating an ideal fermentative characteristic of <i>Aurantiochytrium</i> sp. strain No. 32 without apparent glucose inhibition effect. The cell of <i>Aurantiochytrium</i> sp. strain No. 32 reached the stable stage of growth at approximate 78 h, and the maximum biomass of 49.1 g/L was obtained at 84 h. The biomass productivity for <i>Aurantiochytrium</i> sp. strain No. 32 was 0.58 g/L/h. Meanwhile, a maximum carbohydrate production of 31.3% DCW, reaching up to 15.4 g/L, was obtained at 84 h with carbohydrate productivity of up to 0.18 g/L/h. It was found that the carbohydrate content in the cell of <i>Aurantiochytrium</i> sp. strain No. 32 begun to be improved when the concentration of reducing sugar dropped to approximate 20 g/L and reached the highest level during the stable stage of the cell growth. This result suggested that the reducing sugar was first inflowed into the metabolic pathway for cell growth, and then into the carbohydrate synthesis for energy reservation when the reducing sugar was consumed to deficiency. Moreover, although microbial cell preferentially synthesizes lipid and carbohydrate under the nutrition conditions, lipid was accumulated more rapidly than carbohydrate. This finding could explain why the emergence of the maximum carbohydrate content in <i>Aurantiochytrium</i> sp. strain No. 32 (at 66 h) was later than that of the maximum lipid content in <i>Aurantiochytrium</i> sp. strain No. 6 (at 84 h), and carbohydrate fermentation needed more time for higher productivity [ <a href="#B22-molecules-24-01593">22</a>, <a href="#B35-molecules-24-01593">35</a>]. This carbohydrate content in the <i>Aurantiochytrium</i> sp. strain No. 32 cell was comparable with or far higher than those of <i>Schizochytrium</i> sp. Strains [ <a href="#B14-molecules-24-01593">14</a>, <a href="#B22-molecules-24-01593">22</a>, <a href="#B36-molecules-24-01593">36</a>]. <i>Aurantiochytrium</i> sp. has been widely applied as feed and nutrient-enriching composition in aquatic and livestock industries, due to the multiple nutrition in its cell, such as lipid-rich in PUFAs, carbohydrate, protein, and micronutrients [ <a href="#B2-molecules-24-01593">2</a>, <a href="#B36-molecules-24-01593">36</a>]. Among these nutrients, carbohydrate is essential for animal cultivations, such as <i>Artemia franciscana</i>, oyster, scallop and cow, et al. [ <a href="#B2-molecules-24-01593">2</a>, <a href="#B36-molecules-24-01593">36</a>, <a href="#B37-molecules-24-01593">37</a>]. Our result indicated that the <i>Aurantiochytrium</i> sp. strain No. 32, with a high growth rate and carbohydrate content, is fit for feed supplement in aquatic and livestock industries. </p> </div> </div> <div class="3.materialsandmethods" title="sec"> <div class="title" tagx="title" title="title">
3. Materials and Methods</div> <div class="3.1.materials" title="sec"> <div class="title" tagx="title" title="title">
3.1. Materials</div> <p>The nonadecanoicacid methyl ester used as the internal standard was purchased from Sigma-Aldrich (St. Louis, MO, USA). The BF <sub>3</sub>-methanol solution was purchased from ANPEL Laboratory Technologies Inc. (Shanghai, China). All other chemicals used in this study were of analytical grade and purchased from Aladdin Industrial Inc. (Shanghai, China). </p> </div> <div class="3.2.strains,mediumandcultureconditions" title="sec"> <div class="title" tagx="title" title="title">
3.2. Strains, Medium and Culture Conditions</div> <p>A total of 109 <i>Aurantiochytrium</i> sp. strains from our <i>Aurantiochytrium</i> library, which was isolated from the mangrove ecosystem in Yueqing Bay (Zhejiang, China), were used in this study [ <a href="#B26-molecules-24-01593">26</a>]. Each strain was preserved in 20% ( <i>v</i>/ <i>v</i>) glycerol and at −80 °C before it was used. The strain was transferred onto the GPY plate (2% glucose, 1% peptone, 0.5% yeast extract, 20% sea salt and 20% agar powder) for purification and recovering of strains. For screenings of strains with high levels of lipid, protein and carbohydrate contents, respectively, the single colony was transferred into the 250 mL flask containing 50 mL screening medium and cultured at 28 °C and 150 rpm for 3 days. The screening medium was composed of 8% glucose, 1% polypeptone, 0.5% yeast extract and 20% sea salt ( <i>m</i>/ <i>v</i>). To analyze the effect of deuteration distributed in cell on cellular contents of three biomacromolecules, the screening medium was prepared with H <sub>2</sub>O and 50% D <sub>2</sub>O, as the H <sub>2</sub>O medium and the D <sub>2</sub>O medium, respectively. The <i>Aurantiochytrium</i> sp. strain 6 was selected as the analyzed strain, and it was transferred into the D <sub>2</sub>O medium and cultured for deuteration of cell. The cultured cells under the D <sub>2</sub>O medium were washed and diluted with D <sub>2</sub>O and was set as the D <sub>2</sub>O-treated group (D group). Meanwhile, the cells cultured in the H <sub>2</sub>O medium and prepared with H <sub>2</sub>O were set as the H <sub>2</sub>O-treated group (H and the control group). The cells of two groups were analyzed by the FT-IR based on the method described below. The results from the two groups were compared and analyzed. </p> </div> <div class="3.3.ft-irspectroscopyanalysis" title="sec"> <div class="title" tagx="title" title="title">
3.3. FT-IR Spectroscopy Analysis</div> <p>The FT-IR analysis was based on the method described by Giordano et al. with minor modifications [ <a href="#B17-molecules-24-01593">17</a>]. The optical density (OD) of the cell under 600 nm for every culture sample was measured and calculated. A proper proportion of dilution with the blank medium was performed to make sure the OD <sub>600</sub> of every sample as 0.1. The culture with OD <sub>600</sub> value of 0.1 was harvested by centrifugation at 6000× <i>g</i> for 5 min. The supernatant was removed, and the pellet was then washed twice with sterile isotonic saline. After the final wash, a total of 50 µL isotonic saline was left, and the cell pellet was re-suspended in this volume. For FT-IR spectroscopy analysis, 30 µL of cell suspension for every sample was homogenized using pipettor in 96 well microplates and dried at 60 °C for 90 min. Spectra were acquired with a TENSOR 27 FT-IR spectrometer (Bruker Optics, Ettlingen, Germany) equipped with a High Throughput Screening eX-Tension (HTS-XT) unit. The spectral region was set 4000 to 600 cm <sup>−1</sup> with a resolution of 4 cm <sup>−1</sup>, an aperture of 5.0 mm, taking 64 scans that were subsequently averaged. The spectra were resorted to baseline correction to minimize variations between spectra caused by the baseline shift. The OPUS 6.5 software (Bruker Optics, Billerica, MA, USA) package was used for quantitive and qualitative analysis of target peaks. </p> <p>Peak qualitative analysis was performed according to the method from Giordano et al. with minor modifications [ <a href="#B17-molecules-24-01593">17</a>]. Absorption signals around 3000–2800 cm <sup>−1</sup> and 1740 cm <sup>−1</sup>, representing the C–H stretching vibration acyl chains and the ester C=O stretching band, corresponding to the lipid content (Peaks 1, 2 and 3 in <a href="#molecules-24-01593-f001">Figure 1</a>). Absorption signals around 1690–1650 cm <sup>−1</sup> and 1540–1500 cm <sup>−1</sup> representing amide I and amide II, corresponds to protein C=O stretching vibration and N–H bending vibrations, respectively, were characterized as protein content (Peaks 4 and 5 in <a href="#molecules-24-01593-f001">Figure 1</a>). Absorption signals around 1200–900 cm <sup>−1</sup> characterized over a C–O and C–O–C stretching band area, were used to quantify the total carbohydrate content in the cell (Peaks 6 and 7 in <a href="#molecules-24-01593-f001">Figure 1</a>). OD <sub>600</sub> (0.1–0.3) of every sample was used for normalization of the results derived from the FT-IR. </p> </div> <div class="3.4.determinationofbiomass" title="sec"> <div class="title" tagx="title" title="title">
3.4. Determination of Biomass</div> <p>The biomass was determined as the dried cell weight (DCW) or the optical density under 600 nm (OD <sub>600</sub>). For DCW analysis, an aliquot of 50 mL of culture was centrifuged at 4 °C and 10,000× <i>g</i> for 5 min and then washed twice with distilled water. The cell pellet was lyophilized to a constant weight at 50 °C for 2 days. If needed, the dried cells were pestled into a fine powder with mortar under liquid nitrogen for further analysis. For OD <sub>600</sub> analysis, proper dilution time of cell culture with a blank medium was performed to make sure the reading of spectrophotometer was at a range of 0.1–0.3. The OD <sub>600</sub> of cell culture was then calculated. </p> </div> <div class="3.5.traditionalbiochemicalanalysisofcellularcomposition" title="sec"> <div class="title" tagx="title" title="title">
3.5. Traditional Biochemical Analysis of Cellular Composition</div> <p>Lipid, protein, and carbohydrate were analyzed using traditional biochemical methods to validate the FT-IR results. Lipid content analysis was performed according to our previous method [ <a href="#B7-molecules-24-01593">7</a>]. The lyophilized cell was ground by mortar and pestle into a fine powder under liquid nitrogen. The cell powder was extracted into 100 mL of chloroform/methanol (2:1, <i>v</i>/ <i>v</i>) at room temperature. The lipid extract was dried by evaporation and weighed. Carbohydrate content was analyzed using sulfuric acid-anthrone method [ <a href="#B16-molecules-24-01593">16</a>]. Absorbance was measured at 620 nm with Ultrospec 2100 pro Spectrophotometer (GE, Boston, MA, USA). A calibration curve was prepared using glucose as a standard. Protein content was analyzed using the Bradford method [ <a href="#B15-molecules-24-01593">15</a>]. Fatty acids analysis was performed based on our previous method [ <a href="#B7-molecules-24-01593">7</a>]. </p> </div> <div class="3.6.mediaoptimizationforlipidorcarbohydrateproductions" title="sec"> <div class="title" tagx="title" title="title">
3.6. Media Optimization for Lipid or Carbohydrate Productions</div> <p>To optimize media composition for lipid and carbohydrate productions, glucose, peptone and yeast extract, representing carbon and nitrogen resources, were optimized through the "One at a time" method. Different concentrations of glucose (40, 60, 80, 100, 120 and 140 g/L), peptone (0.5, 1, 1.5, 2, 3 and 4 g/L) and yeast extract (0, 0.5, 1, 2, 3 and 4 g/L) were tested, and one factor was optimized when the other factors were fixed. The biomass, lipid, and carbohydrate, measured by the traditional method and FT-IR method, respectively, were responses for optimizations. </p> </div> <div class="3.7.fermentationin5lbioreactor" title="sec"> <div class="title" tagx="title" title="title">
3.7. Fermentation in 5 L Bioreactor </div> <p>The RALF 5 l bioreactor (Bioengineering, Switzerland) was used to perform the fed-batch bioreactor cultivations using the optimal media obtained from above optimizations. The pre-culture was prepared in a 500 mL flask and then transferred into the 5 L bioreactor containing 4 l media optimized for lipid and carbohydrate production, respectively. For lipid production, the fermentation was set based on our previous method [ <a href="#B26-molecules-24-01593">26</a>]. The temperature was set at 25 °C with aeration and agitation rate kept at 2 vvm and 600 rpm, respectively. For carbohydrate production, the fermentation temperature was set at 28 °C with aeration and agitation rate kept at 1 vvm and 400 rpm, respectively. Fifty milliliters of sample was taken at 6 h intervals over the entire cultivation (72 h for lipid fermentation and 96 h for carbohydrate fermentation) for DCW, reducing sugar, lipid, fatty acid, carbohydrate, and FT-IR analyses. </p> </div> <div class="3.8.statisticalanalysis" title="sec"> <div class="title" tagx="title" title="title">
3.8. Statistical Analysis</div> <p>All the experiments above were repeated three times, and the average values were taken as the results. </p> </div> </div> <div class="conclusions" title="conclusions"> <div class="title" tagx="title" title="title">
4. Conclusions</div> <p>A bio-macromolecule analysis based on the FT-IR, specific for the oleaginous <i>Aurantiochytrium</i> sp., was firstly constructed in this study. This method was economic, green, and accurate, and was applied in strain screenings, medium optimization, and fermentation. Two <i>Aurantiochytrium</i> sp. strains were screened for lipid and carbohydrate productions, respectively. Lipid and carbohydrate accumulated rapidly when the concentration of the reducing sugar in medium consumed to deficiency, and accumulation of lipid was ahead of that of carbohydrate. The 5-l fermentation performance of the two strains confirmed their potentials for lipid, DHA, and carbohydrate production, and broad applications in medical, food, and feed industries. </p> </div> </div> <div class="back" title="back"> <div class="fn-group" title="fn-group"> <div class="fn-type-" title=""> <p> <b>Sample Availability:</b> Samples of the compounds microalgal lipid and carbohydrate are available from the authors. </p> </div> </div> <div class="title" tagx="title" title="title">
Author Contributions</div> <p>Conceptualization, X.-J.Y. and D.-S.W.; methodology, C.-Y.H., H.C., and J.-L.C.; software, C.-Y.H., H.C. and Z.-P.W.; validation, X.-J.Y., H.-J.L., X.-Y.L. and J.S.; formal analysis, X.-J.Y., C.-Y.H., and Z.-P.W.; investigation, X.-J.Y., D.-S.W., H.-J.L. and Z.W.; resources, X.-J.Y. and Z.W.; data curation, C.-Y.H., H.C. and Z.-P.W.; writing—original draft preparation, X.-J.Y., C.-Y.H., X.-Y.L., J.S. and Z.-P.W.; writing—review and editing, X.-J.Y., D.-S.W., C.-Y.H., X.-Y.L., J.S. and Z.-P.W.; visualization, C.-Y.H., J.-L.C. and J.S.; supervision, X.-J.Y., D.-S.W. and Z.W.; project administration, X.-J.Y., D.-S.W. and Z.W.; funding acquisition, X.-J.Y. and D.-S.W. </p> <div class="title" tagx="title" title="title">
Funding</div> <p>This study was financially supported by the Zhejiang Provincial Natural Science Foundation of China [No. LY18C010004], the National Natural Science Foundation of China [No. 31560024] and the Jiangxi provincial key research projects [No. 20161BBF60101, No. S2018ZPYFE0461 and No. 20142BDH80013]. </p> <div class="title" tagx="title" title="title">
Conflicts of Interest</div> <p>The authors declare no conflict of interest. The funders had no role in the design of the study; in the collection, analyses, or interpretation of data; in the writing of the manuscript, or in the decision to publish the results. </p> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"> <a name="B1-molecules-24-01593" /> <span class="label" tagx="label" title="label">1.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Horrocks</span> <span class="given-names" tagx="given-names" title="given-names">L.A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yeo</span> <span class="given-names" tagx="given-names" title="given-names">Y.K.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Health benefits of docosahexaenoic acid (DHA)</span> <span class="source" tagx="source" title="source">Pharmacol. Res.</span> <span class="year" tagx="year" title="year">1999</span> <span class="volume" tagx="volume" title="volume">40</span> <span class="fpage" tagx="fpage" title="fpage">211</span> <span class="lpage" tagx="lpage" title="lpage">225</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1006/phrs.1999.0495">10.1006/phrs.1999.0495</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10479465">10479465</a> </span> </span> </li> <li tag="ref"> <a name="B2-molecules-24-01593" /> <span class="label" tagx="label" title="label">2.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Madeira</span> <span class="given-names" tagx="given-names" title="given-names">M.S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cardoso</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lopes</span> <span class="given-names" tagx="given-names" title="given-names">P.A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Coelho</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Afonso</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bandarra</span> <span class="given-names" tagx="given-names" title="given-names">N.M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Prates</span> <span class="given-names" tagx="given-names" title="given-names">J.A.M.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Microalgae as feed ingredients for livestock production and meat quality: A review</span> <span class="source" tagx="source" title="source">Livest. Sci.</span> <span class="year" tagx="year" title="year">2017</span> <span class="volume" tagx="volume" title="volume">205</span> <span class="fpage" tagx="fpage" title="fpage">111</span> <span class="lpage" tagx="lpage" title="lpage">121</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.livsci.2017.09.020">10.1016/j.livsci.2017.09.020</a> </span> </span> </li> <li tag="ref"> <a name="B3-molecules-24-01593" /> <span class="label" tagx="label" title="label">3.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yokoyama</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Salleh</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Honda</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Taxonomic rearrangement of the genus Ulkenia sensu lato based on morphology, chemotaxonomical characteristics, and 18S rRNA gene phylogeny (Thraustochytriaceae, Labyrinthulomycetes): Emendation for Ulkenia and erection of <i>Botryochytrium</i>, <i>Parietichytrium</i>, and <i>Sicyoidochytrium</i> gen. nov </span> <span class="source" tagx="source" title="source">Mycoscience</span> <span class="year" tagx="year" title="year">2007</span> <span class="volume" tagx="volume" title="volume">48</span> <span class="fpage" tagx="fpage" title="fpage">329</span> <span class="lpage" tagx="lpage" title="lpage">341</span> </span> </li> <li tag="ref"> <a name="B4-molecules-24-01593" /> <span class="label" tagx="label" title="label">4.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Barclay</span> <span class="given-names" tagx="given-names" title="given-names">W.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Weaver</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Metz</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hansen</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">4-Development of a docosahexaenoic acid production technology using <i>Schizochytrium</i>: Historical perspective and update </span> <span class="source" tagx="source" title="source">Single Cell Oils</span> <span class="edition" tagx="edition" title="edition">2nd ed.</span> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cohen</span> <span class="given-names" tagx="given-names" title="given-names">Z.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ratledge</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> </span> <span class="publisher-name" tagx="publisher-name" title="publisher-name">AOCS Press</span> <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Urbana, IL, USA</span> <span class="year" tagx="year" title="year">2010</span> <span class="fpage" tagx="fpage" title="fpage">75</span> <span class="lpage" tagx="lpage" title="lpage">96</span> </span> </li> <li tag="ref"> <a name="B5-molecules-24-01593" /> <span class="label" tagx="label" title="label">5.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xie</span> <span class="given-names" tagx="given-names" title="given-names">Y.X.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sen</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">G.Y.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Mining terpenoids production and biosynthetic pathway in thraustochytrids</span> <span class="source" tagx="source" title="source">Bioresour. Technol.</span> <span class="year" tagx="year" title="year">2017</span> <span class="volume" tagx="volume" title="volume">244</span> <span class="fpage" tagx="fpage" title="fpage">1269</span> <span class="lpage" tagx="lpage" title="lpage">1280</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.biortech.2017.05.002">10.1016/j.biortech.2017.05.002</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/28549813">28549813</a> </span> </span> </li> <li tag="ref"> <a name="B6-molecules-24-01593" /> <span class="label" tagx="label" title="label">6.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fan</span> <span class="given-names" tagx="given-names" title="given-names">K.W.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jiang</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ho</span> <span class="given-names" tagx="given-names" title="given-names">L.T.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">F.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Differentiation in fatty acid profiles of pigmented and nonpigmented <i>Aurantiochytrium</i> isolated from Hong Kong mangroves </span> <span class="source" tagx="source" title="source">J. Agric. Food Chem.</span> <span class="year" tagx="year" title="year">2009</span> <span class="volume" tagx="volume" title="volume">57</span> <span class="fpage" tagx="fpage" title="fpage">6334</span> <span class="lpage" tagx="lpage" title="lpage">6341</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1021/jf901117z">10.1021/jf901117z</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19534536">19534536</a> </span> </span> </li> <li tag="ref"> <a name="B7-molecules-24-01593" /> <span class="label" tagx="label" title="label">7.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yu</span> <span class="given-names" tagx="given-names" title="given-names">X.J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">J.H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sun</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zheng</span> <span class="given-names" tagx="given-names" title="given-names">J.Y.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">Y.J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">Z.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Docosahexaenoic acid production from the acidic hydrolysate of <i>Jerusalem artichoke</i> by an efficient sugar-utilizing <i>Aurantiochytrium</i> sp. YLH70 </span> <span class="source" tagx="source" title="source">Ind. Crop. Prod.</span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">83</span> <span class="fpage" tagx="fpage" title="fpage">372</span> <span class="lpage" tagx="lpage" title="lpage">378</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.indcrop.2016.01.013">10.1016/j.indcrop.2016.01.013</a> </span> </span> </li> <li tag="ref"> <a name="B8-molecules-24-01593" /> <span class="label" tagx="label" title="label">8.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yu</span> <span class="given-names" tagx="given-names" title="given-names">X.J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sun</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sun</span> <span class="given-names" tagx="given-names" title="given-names">Y.Q.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zheng</span> <span class="given-names" tagx="given-names" title="given-names">J.Y.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">Z.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Metabolomics analysis of phytohormone gibberellin improving lipid and DHA accumulation in <i>Aurantiochytrium</i> sp. </span> <span class="source" tagx="source" title="source">Biochem. Eng. J.</span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">112</span> <span class="fpage" tagx="fpage" title="fpage">258</span> <span class="lpage" tagx="lpage" title="lpage">268</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.bej.2016.05.002">10.1016/j.bej.2016.05.002</a> </span> </span> </li> <li tag="ref"> <a name="B9-molecules-24-01593" /> <span class="label" tagx="label" title="label">9.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Manikan</span> <span class="given-names" tagx="given-names" title="given-names">V.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nazir</span> <span class="given-names" tagx="given-names" title="given-names">M.Y.M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kalil</span> <span class="given-names" tagx="given-names" title="given-names">M.S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Isa</span> <span class="given-names" tagx="given-names" title="given-names">M.H.M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kader</span> <span class="given-names" tagx="given-names" title="given-names">A.J.A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yusoff</span> <span class="given-names" tagx="given-names" title="given-names">W.M.W.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hamid</span> <span class="given-names" tagx="given-names" title="given-names">A.A.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">A new strain of docosahexaenoic acid producing microalga from Malaysian coastal waters</span> <span class="source" tagx="source" title="source">Algal. Res.</span> <span class="year" tagx="year" title="year">2015</span> <span class="volume" tagx="volume" title="volume">9</span> <span class="fpage" tagx="fpage" title="fpage">40</span> <span class="lpage" tagx="lpage" title="lpage">47</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.algal.2015.02.023">10.1016/j.algal.2015.02.023</a> </span> </span> </li> <li tag="ref"> <a name="B10-molecules-24-01593" /> <span class="label" tagx="label" title="label">10.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Montalbo-Lomboy</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kantekin</span> <span class="given-names" tagx="given-names" title="given-names">M.N.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Lipid Estimation of surfactant-extracted microalgae oil using Nile Red</span> <span class="source" tagx="source" title="source">J. Am. Oil. Chem. Soc.</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">91</span> <span class="fpage" tagx="fpage" title="fpage">665</span> <span class="lpage" tagx="lpage" title="lpage">680</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s11746-013-2395-9">10.1007/s11746-013-2395-9</a> </span> </span> </li> <li tag="ref"> <a name="B11-molecules-24-01593" /> <span class="label" tagx="label" title="label">11.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kwak</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kang</span> <span class="given-names" tagx="given-names" title="given-names">S.G.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hong</span> <span class="given-names" tagx="given-names" title="given-names">W.K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Han</span> <span class="given-names" tagx="given-names" title="given-names">J.I.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chang</span> <span class="given-names" tagx="given-names" title="given-names">Y.K.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Simultaneous cell disruption and lipid extraction of wet <i>Aurantiochytrium</i> sp. KRS101 using a high shear mixer </span> <span class="source" tagx="source" title="source">Bioprocess Biosyst. Eng.</span> <span class="year" tagx="year" title="year">2018</span> <span class="volume" tagx="volume" title="volume">41</span> <span class="fpage" tagx="fpage" title="fpage">671</span> <span class="lpage" tagx="lpage" title="lpage">678</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s00449-018-1901-8">10.1007/s00449-018-1901-8</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/29453513">29453513</a> </span> </span> </li> <li tag="ref"> <a name="B12-molecules-24-01593" /> <span class="label" tagx="label" title="label">12.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sung</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Han</span> <span class="given-names" tagx="given-names" title="given-names">J.I.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Ultrasound-assisted in-situ transesterification of wet <i>Aurantiochytrium</i> sp. KRS 101 using potassium carbonate </span> <span class="source" tagx="source" title="source">Bioresour. Technol.</span> <span class="year" tagx="year" title="year">2018</span> <span class="volume" tagx="volume" title="volume">261</span> <span class="fpage" tagx="fpage" title="fpage">117</span> <span class="lpage" tagx="lpage" title="lpage">121</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.biortech.2018.03.099">10.1016/j.biortech.2018.03.099</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/29654996">29654996</a> </span> </span> </li> <li tag="ref"> <a name="B13-molecules-24-01593" /> <span class="label" tagx="label" title="label">13.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kang</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Honda</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Okano</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Aki</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Omasa</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ohtake</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Thermal analysis for differentiating between oleaginous and non-oleaginous microorganisms</span> <span class="source" tagx="source" title="source">Biochem. Eng. J.</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">57</span> <span class="fpage" tagx="fpage" title="fpage">23</span> <span class="lpage" tagx="lpage" title="lpage">29</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.bej.2011.08.002">10.1016/j.bej.2011.08.002</a> </span> </span> </li> <li tag="ref"> <a name="B14-molecules-24-01593" /> <span class="label" tagx="label" title="label">14.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Arney</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">W.S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Forster</span> <span class="given-names" tagx="given-names" title="given-names">I.P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">McKinley</span> <span class="given-names" tagx="given-names" title="given-names">R.S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pearce</span> <span class="given-names" tagx="given-names" title="given-names">C.M.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Feasibility of dietary substitution of live microalgae with spray-dried <i>Schizochytrium</i> sp. or <i>Spirulina</i> in the hatchery culture of juveniles of the Pacific geoduck clam ( <i>Panopea generosa</i>) </span> <span class="source" tagx="source" title="source">Aquaculture</span> <span class="year" tagx="year" title="year">2015</span> <span class="volume" tagx="volume" title="volume">444</span> <span class="fpage" tagx="fpage" title="fpage">117</span> <span class="lpage" tagx="lpage" title="lpage">133</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.aquaculture.2015.02.014">10.1016/j.aquaculture.2015.02.014</a> </span> </span> </li> <li tag="ref"> <a name="B15-molecules-24-01593" /> <span class="label" tagx="label" title="label">15.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bradford</span> <span class="given-names" tagx="given-names" title="given-names">M.M.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">A rapid and sensitive method for the quantitation of microgram quantities of protein utilizing the principle of protein-dye binding </span> <span class="source" tagx="source" title="source">Anal. Biochem.</span> <span class="year" tagx="year" title="year">1976</span> <span class="volume" tagx="volume" title="volume">72</span> <span class="fpage" tagx="fpage" title="fpage">248</span> <span class="lpage" tagx="lpage" title="lpage">254</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/0003-2697(76)90527-3">10.1016/0003-2697(76)90527-3</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/942051">942051</a> </span> </span> </li> <li tag="ref"> <a name="B16-molecules-24-01593" /> <span class="label" tagx="label" title="label">16.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Somani</span> <span class="given-names" tagx="given-names" title="given-names">B.L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Khanade</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sinha</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">A modified anthrone-sulfuric acid method for the determination of fructose in the presence of certain proteins </span> <span class="source" tagx="source" title="source">Anal. Biochem.</span> <span class="year" tagx="year" title="year">1987</span> <span class="volume" tagx="volume" title="volume">167</span> <span class="fpage" tagx="fpage" title="fpage">327</span> <span class="lpage" tagx="lpage" title="lpage">330</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/0003-2697(87)90172-2">10.1016/0003-2697(87)90172-2</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/3442328">3442328</a> </span> </span> </li> <li tag="ref"> <a name="B17-molecules-24-01593" /> <span class="label" tagx="label" title="label">17.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Giordano</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kansiz</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Heraud</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Beardall</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wood</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">McNaughton</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Fourier transform infrared spectroscopy as a novel tool to investigate changes in intracellular macromolecular pools in the marine microalga <i>Chaetoceros muellerii</i> (Bacillariophyceae) </span> <span class="source" tagx="source" title="source">J. Phycol.</span> <span class="year" tagx="year" title="year">2001</span> <span class="volume" tagx="volume" title="volume">37</span> <span class="fpage" tagx="fpage" title="fpage">271</span> <span class="lpage" tagx="lpage" title="lpage">279</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1046/j.1529-8817.2001.037002271.x">10.1046/j.1529-8817.2001.037002271.x</a> </span> </span> </li> <li tag="ref"> <a name="B18-molecules-24-01593" /> <span class="label" tagx="label" title="label">18.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Meng</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yao</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xue</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yang</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Application of Fourier transform infrared (FT-IR) spectroscopy in determination of microalgal compositions </span> <span class="source" tagx="source" title="source">Bioresour. Technol.</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">151</span> <span class="fpage" tagx="fpage" title="fpage">347</span> <span class="lpage" tagx="lpage" title="lpage">354</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.biortech.2013.10.064">10.1016/j.biortech.2013.10.064</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24262844">24262844</a> </span> </span> </li> <li tag="ref"> <a name="B19-molecules-24-01593" /> <span class="label" tagx="label" title="label">19.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vongsvivut</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Heraud</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gupta</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Puri</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">McNaughton</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Barrow</span> <span class="given-names" tagx="given-names" title="given-names">C.J.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">FTIR microspectroscopy for rapid screening and monitoring of polyunsaturated fatty acid production in commercially valuable marine yeasts and protists </span> <span class="source" tagx="source" title="source">Analyst</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">138</span> <span class="fpage" tagx="fpage" title="fpage">6016</span> <span class="lpage" tagx="lpage" title="lpage">6031</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1039/c3an00485f">10.1039/c3an00485f</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23957051">23957051</a> </span> </span> </li> <li tag="ref"> <a name="B20-molecules-24-01593" /> <span class="label" tagx="label" title="label">20.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gautam</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Varma</span> <span class="given-names" tagx="given-names" title="given-names">A.K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vinu</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Apparent kinetics of fast pyrolysis of four different microalgae and product analyses using pyrolysis-FTIR and pyrolysis-GC/MS </span> <span class="source" tagx="source" title="source">Energ. Fuel</span> <span class="year" tagx="year" title="year">2017</span> <span class="volume" tagx="volume" title="volume">31</span> <span class="fpage" tagx="fpage" title="fpage">12339</span> <span class="lpage" tagx="lpage" title="lpage">12349</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1021/acs.energyfuels.7b02520">10.1021/acs.energyfuels.7b02520</a> </span> </span> </li> <li tag="ref"> <a name="B21-molecules-24-01593" /> <span class="label" tagx="label" title="label">21.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vo</span> <span class="given-names" tagx="given-names" title="given-names">T.K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lee</span> <span class="given-names" tagx="given-names" title="given-names">O.K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lee</span> <span class="given-names" tagx="given-names" title="given-names">E.Y.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kim</span> <span class="given-names" tagx="given-names" title="given-names">C.H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Seo</span> <span class="given-names" tagx="given-names" title="given-names">J.W.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kim</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kim</span> <span class="given-names" tagx="given-names" title="given-names">S.S.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Kinetics study of the hydrothermal liquefaction of the microalga <i>Aurantiochytrium</i> sp. KRS101 </span> <span class="source" tagx="source" title="source">Chem. Eng. J.</span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">306</span> <span class="fpage" tagx="fpage" title="fpage">763</span> <span class="lpage" tagx="lpage" title="lpage">771</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.cej.2016.07.104">10.1016/j.cej.2016.07.104</a> </span> </span> </li> <li tag="ref"> <a name="B22-molecules-24-01593" /> <span class="label" tagx="label" title="label">22.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sun</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ren</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhuang</span> <span class="given-names" tagx="given-names" title="given-names">X.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ji</span> <span class="given-names" tagx="given-names" title="given-names">X.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yan</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Huang</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Differential effects of nutrient limitations on biochemical constituents and docosahexaenoic acid production of <i>Schizochytrium</i> sp. </span> <span class="source" tagx="source" title="source">Bioresour. Technol.</span> <span class="year" tagx="year" title="year">2014</span> <span class="volume" tagx="volume" title="volume">159</span> <span class="fpage" tagx="fpage" title="fpage">199</span> <span class="lpage" tagx="lpage" title="lpage">206</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.biortech.2014.02.106">10.1016/j.biortech.2014.02.106</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24657750">24657750</a> </span> </span> </li> <li tag="ref"> <a name="B23-molecules-24-01593" /> <span class="label" tagx="label" title="label">23.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Marcotte</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kegelaer</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sandt</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Barbeau</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lafleur</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">An alternative infrared spectroscopy assay for the quantification of polysaccharides in bacterial samples </span> <span class="source" tagx="source" title="source">Anal. Biochem.</span> <span class="year" tagx="year" title="year">2007</span> <span class="volume" tagx="volume" title="volume">361</span> <span class="fpage" tagx="fpage" title="fpage">7</span> <span class="lpage" tagx="lpage" title="lpage">14</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.ab.2006.11.009">10.1016/j.ab.2006.11.009</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17169320">17169320</a> </span> </span> </li> <li tag="ref"> <a name="B24-molecules-24-01593" /> <span class="label" tagx="label" title="label">24.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Palmucci</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ratti</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Giordano</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Ecological and evolutionary implications of carbon allocation in marine phytoplankton as a function of nitrogen availability: A fourier transform infrared spectroscopy approach </span> <span class="source" tagx="source" title="source">J. Phycol.</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">47</span> <span class="fpage" tagx="fpage" title="fpage">313</span> <span class="lpage" tagx="lpage" title="lpage">323</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1529-8817.2011.00963.x">10.1111/j.1529-8817.2011.00963.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27021863">27021863</a> </span> </span> </li> <li tag="ref"> <a name="B25-molecules-24-01593" /> <span class="label" tagx="label" title="label">25.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pistorius</span> <span class="given-names" tagx="given-names" title="given-names">A.M.A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">DeGrip</span> <span class="given-names" tagx="given-names" title="given-names">W.J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Egorova-Zachernyuk</span> <span class="given-names" tagx="given-names" title="given-names">T.A.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Monitoring of biomass composition from microbiological sources by means of FT-IR spectroscopy</span> <span class="source" tagx="source" title="source">Biotechnol. Bioeng.</span> <span class="year" tagx="year" title="year">2009</span> <span class="volume" tagx="volume" title="volume">103</span> <span class="fpage" tagx="fpage" title="fpage">123</span> <span class="lpage" tagx="lpage" title="lpage">129</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1002/bit.22220">10.1002/bit.22220</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19132745">19132745</a> </span> </span> </li> <li tag="ref"> <a name="B26-molecules-24-01593" /> <span class="label" tagx="label" title="label">26.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yu</span> <span class="given-names" tagx="given-names" title="given-names">X.J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yu</span> <span class="given-names" tagx="given-names" title="given-names">Z.Q.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">Y.L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sun</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zheng</span> <span class="given-names" tagx="given-names" title="given-names">J.Y.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">Z.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Utilization of high-fructose corn syrup for biomass production containing high levels of docosahexaenoic acid by a newly isolated <i>Aurantiochytrium</i> sp. YLH70 </span> <span class="source" tagx="source" title="source">Appl. Biochem. Biotechnol.</span> <span class="year" tagx="year" title="year">2015</span> <span class="volume" tagx="volume" title="volume">177</span> <span class="fpage" tagx="fpage" title="fpage">1229</span> <span class="lpage" tagx="lpage" title="lpage">1240</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s12010-015-1809-6">10.1007/s12010-015-1809-6</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26299378">26299378</a> </span> </span> </li> <li tag="ref"> <a name="B27-molecules-24-01593" /> <span class="label" tagx="label" title="label">27.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yokochi</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Honda</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Higashihara</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nakahara</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Optimization of docosahexaenoic acid production by <i>Schizochytrium limacinum</i> SR21 </span> <span class="source" tagx="source" title="source">Appl. Microbiol. Biotechnol.</span> <span class="year" tagx="year" title="year">1998</span> <span class="volume" tagx="volume" title="volume">49</span> <span class="fpage" tagx="fpage" title="fpage">72</span> <span class="lpage" tagx="lpage" title="lpage">76</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s002530051139">10.1007/s002530051139</a> </span> </span> </li> <li tag="ref"> <a name="B28-molecules-24-01593" /> <span class="label" tagx="label" title="label">28.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Huang</span> <span class="given-names" tagx="given-names" title="given-names">T.Y.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lu</span> <span class="given-names" tagx="given-names" title="given-names">W.C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chu</span> <span class="given-names" tagx="given-names" title="given-names">I.M.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">A fermentation strategy for producing docosahexaenoic acid in <i>Aurantiochytrium limacinum</i> SR21 and increasing C22:6 proportions in total fatty acid </span> <span class="source" tagx="source" title="source">Bioresour. Technol.</span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">123</span> <span class="fpage" tagx="fpage" title="fpage">8</span> <span class="lpage" tagx="lpage" title="lpage">14</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.biortech.2012.07.068">10.1016/j.biortech.2012.07.068</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22929740">22929740</a> </span> </span> </li> <li tag="ref"> <a name="B29-molecules-24-01593" /> <span class="label" tagx="label" title="label">29.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ryu</span> <span class="given-names" tagx="given-names" title="given-names">B.G.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kim</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kim</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Han</span> <span class="given-names" tagx="given-names" title="given-names">J.I.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yang</span> <span class="given-names" tagx="given-names" title="given-names">J.W.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Use of organic waste from the brewery industry for high-density cultivation of the docosahexaenoic acid-rich microalga, <i>Aurantiochytrium</i> sp. KRS101 </span> <span class="source" tagx="source" title="source">Bioresour. Technol.</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">129</span> <span class="fpage" tagx="fpage" title="fpage">351</span> <span class="lpage" tagx="lpage" title="lpage">359</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.biortech.2012.11.049">10.1016/j.biortech.2012.11.049</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23262011">23262011</a> </span> </span> </li> <li tag="ref"> <a name="B30-molecules-24-01593" /> <span class="label" tagx="label" title="label">30.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pyle</span> <span class="given-names" tagx="given-names" title="given-names">D.J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Garcia</span> <span class="given-names" tagx="given-names" title="given-names">R.A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wen</span> <span class="given-names" tagx="given-names" title="given-names">Z.Y.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Producing docosahexaenoic acid (DHA)-rich algae from blodiesel-derived crude glycerol: Effects of impurities on DHA production and algal biomass composition </span> <span class="source" tagx="source" title="source">J. Agric. Food Chem.</span> <span class="year" tagx="year" title="year">2008</span> <span class="volume" tagx="volume" title="volume">56</span> <span class="fpage" tagx="fpage" title="fpage">3933</span> <span class="lpage" tagx="lpage" title="lpage">3939</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1021/jf800602s">10.1021/jf800602s</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18465872">18465872</a> </span> </span> </li> <li tag="ref"> <a name="B31-molecules-24-01593" /> <span class="label" tagx="label" title="label">31.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nagano</span> <span class="given-names" tagx="given-names" title="given-names">N.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Taoka</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Honda</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hayashi</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Effect of trace elements on growth of marine eukaryotes, tharaustochytrids</span> <span class="source" tagx="source" title="source">J. Biosci. Bioeng.</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">116</span> <span class="fpage" tagx="fpage" title="fpage">337</span> <span class="lpage" tagx="lpage" title="lpage">339</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.jbiosc.2013.03.017">10.1016/j.jbiosc.2013.03.017</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23639421">23639421</a> </span> </span> </li> <li tag="ref"> <a name="B32-molecules-24-01593" /> <span class="label" tagx="label" title="label">32.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wagner</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">Z.X.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Langner</span> <span class="given-names" tagx="given-names" title="given-names">U.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stehfest</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wilhelm</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The use of FTIR spectroscopy to assess quantitative changes in the biochemical composition of microalgae </span> <span class="source" tagx="source" title="source">J. Biophotonics</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">3</span> <span class="fpage" tagx="fpage" title="fpage">557</span> <span class="lpage" tagx="lpage" title="lpage">566</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1002/jbio.201000019">10.1002/jbio.201000019</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20503222">20503222</a> </span> </span> </li> <li tag="ref"> <a name="B33-molecules-24-01593" /> <span class="label" tagx="label" title="label">33.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Feng</span> <span class="given-names" tagx="given-names" title="given-names">G.D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">F.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cheng</span> <span class="given-names" tagx="given-names" title="given-names">L.H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xu</span> <span class="given-names" tagx="given-names" title="given-names">X.H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">H.L.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Evaluation of FT-IR and Nile Red methods for microalgal lipid characterization and biomass composition determination </span> <span class="source" tagx="source" title="source">Bioresour. Technol.</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">128</span> <span class="fpage" tagx="fpage" title="fpage">107</span> <span class="lpage" tagx="lpage" title="lpage">112</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.biortech.2012.09.123">10.1016/j.biortech.2012.09.123</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23196229">23196229</a> </span> </span> </li> <li tag="ref"> <a name="B34-molecules-24-01593" /> <span class="label" tagx="label" title="label">34.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ethier</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Woisard</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vaughan</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wen</span> <span class="given-names" tagx="given-names" title="given-names">Z.Y.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Continuous culture of the microalgae <i>Schizochytrium limacinum</i> on biodiesel-derived crude glycerol for producing docosahexaenoic acid </span> <span class="source" tagx="source" title="source">Bioresour. Technol.</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">102</span> <span class="fpage" tagx="fpage" title="fpage">88</span> <span class="lpage" tagx="lpage" title="lpage">93</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.biortech.2010.05.021">10.1016/j.biortech.2010.05.021</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20570140">20570140</a> </span> </span> </li> <li tag="ref"> <a name="B35-molecules-24-01593" /> <span class="label" tagx="label" title="label">35.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dean</span> <span class="given-names" tagx="given-names" title="given-names">A.P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nicholson</span> <span class="given-names" tagx="given-names" title="given-names">J.M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sigee</span> <span class="given-names" tagx="given-names" title="given-names">D.C.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Impact of phosphorus quota and growth phase on carbon allocation in <i>Chlamydomonas reinhardtii</i>: An FTIR microspectroscopy study </span> <span class="source" tagx="source" title="source">Eur. J. Phycol.</span> <span class="year" tagx="year" title="year">2008</span> <span class="volume" tagx="volume" title="volume">43</span> <span class="fpage" tagx="fpage" title="fpage">345</span> <span class="lpage" tagx="lpage" title="lpage">354</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1080/09670260801979287">10.1080/09670260801979287</a> </span> </span> </li> <li tag="ref"> <a name="B36-molecules-24-01593" /> <span class="label" tagx="label" title="label">36.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pacheco-Vega</span> <span class="given-names" tagx="given-names" title="given-names">J.M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cadena-Roa</span> <span class="given-names" tagx="given-names" title="given-names">M.A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ascencio</span> <span class="given-names" tagx="given-names" title="given-names">F.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rangel-Davalos</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rojas-Contreras</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Assessment of endemic microalgae as potential food for <i>Artemia franciscana</i> culture </span> <span class="source" tagx="source" title="source">Lat. Am. J. Aquat. Res.</span> <span class="year" tagx="year" title="year">2015</span> <span class="volume" tagx="volume" title="volume">43</span> <span class="fpage" tagx="fpage" title="fpage">23</span> <span class="lpage" tagx="lpage" title="lpage">32</span> <span class="pub-id"> <a href="https://dx.doi.org/10.3856/vol43-issue1-fulltext-3">10.3856/vol43-issue1-fulltext-3</a> </span> </span> </li> <li tag="ref"> <a name="B37-molecules-24-01593" /> <span class="label" tagx="label" title="label">37.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Brown</span> <span class="given-names" tagx="given-names" title="given-names">M.R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">McCausland</span> <span class="given-names" tagx="given-names" title="given-names">M.A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kowalski</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The nutritional value of four Australian microalgal strains fed to Pacific oyster <i>Crassostrea gigas</i> spat </span> <span class="source" tagx="source" title="source">Aquaculture</span> <span class="year" tagx="year" title="year">1998</span> <span class="volume" tagx="volume" title="volume">165</span> <span class="fpage" tagx="fpage" title="fpage">281</span> <span class="lpage" tagx="lpage" title="lpage">293</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/S0044-8486(98)00256-7">10.1016/S0044-8486(98)00256-7</a> </span> </span> </li> </ul> </div> </div> <div class="floats-group" title="floats-group"> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 1</span> </a> </div>   <p>The profile of FT-IR absorption peaks for the H <sub>2</sub>O-treated <i>Aurantiochytrium</i> sp. cell ( <b>A</b>) and the D <sub>2</sub>O-treated <i>Aurantiochytrium</i> sp. cell ( <b>B</b>). </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 2</span> </a> </div>   <p>Three main bio-macromolecules contents of 18 <i>Aurantiochytrium</i> sp. strains deduced from the FT-IR analysis ( <b>A</b>) and the traditional biochemical methods ( <b>B</b>), and bio-macromolecules contents of cell between the H <sub>2</sub>O-treated group (H group) and the D <sub>2</sub>O-treated group (D group) for <i>Aurantiochytrium</i> sp. strain No. 6 ( <b>C</b>). Data are given as means ± standard deviation, <i>n</i> = 3. </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 3</span> </a> </div>   <p>Linear correlations between the results deduced by the traditional and the FT-IR methods for the contents of the main three bio-macromolecules in <i>Aurantiochytrium</i> sp. strains. </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 4</span> </a> </div>   <p>Different concentrations of glucose, peptone and yeast extract on biomass, lipid and DHA productions in <i>Aurantiochytrium</i> sp. strain No. 6 and evaluation of the related FT-IR results. ( <b>A</b>), Effects of concentrations of glucose on biomass, lipid and DHA synthesis; ( <b>B</b>), Effects of concentrations of peptone on biomass, lipid and DHA synthesis; ( <b>C</b>), Effects of concentrations of yeast extract on biomass, lipid and DHA synthesis; ( <b>D</b>), the linear relationship between the lipid contents deduced by the traditional method and the FT-IR method. Data are given as means ± standard deviation, <i>n</i> = 3. Filled blank, biomass; White blank, lipid content; blank with diagonal, DHA yield. * <i>p</i> &amp;lt; 0.05; ** <i>p</i> &amp;lt; 0.01 The yield under the medium composed of 40 g/L glucose, 10 g/L peptone, and 5 g/L yeast extract was set as the control. </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 5</span> </a> </div>   <p>Different concentrations of glucose, peptone and yeast extract on biomass and carbohydrate productions in <i>Aurantiochytrium</i> sp. strain No. 32 and evaluation of the related FT-IR results. ( <b>A</b>), Effects of concentrations of glucose on biomass and carbohydrate production; ( <b>B</b>), Effects of concentrations of peptone on biomass and carbohydrate production; ( <b>C</b>), Effects of concentrations of yeast extract on biomass and carbohydrate production; ( <b>D</b>), the linear relationship between the carbohydrate contents deduced by the traditional method and the FT-IR method. Data are given as means ± standard deviation, <i>n</i> = 3. Filled blank, biomass; White blank, carbohydrate content; blank with diagonal, carbohydrate yield. * <i>p</i> &amp;lt; 0.05; ** <i>p</i> &amp;lt; 0.01 The yield under the medium composed of 40 g/L glucose, 10 g/L peptone, and 5 g/L yeast extract was set as the control. </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 6</span> </a> </div>   <p>The fermentative performances of the <i>Aurantiochytrium</i> sp. strains No. 6 and No. 32 for lipid and carbohydrate productions under FT-IR monitoring. ( <b>A</b>), lipid and DHA production by <i>Aurantiochytrium</i> sp. strains No. 6; ( <b>B</b>), carbohydrate production by <i>Aurantiochytrium</i> sp. strains No. 32. Reducing sugar, filled squares; biomass, filled inverted triangles; lipid content, filled circles; lipid yield, filled triangles; carbohydrate content, filled diamonds; carbohydrate yield, open diamonds. </p>   </div> <div class="table-wrap_UNKNOWN">
molecules-24-01593-t001_Table 1 <span class="label" tagx="label" title="label">Table 1</span>  <p>Evaluation of the FT-IR method compared with the traditional biochemical methods for cellular composition analysis in <i>Aurantiochytrium</i> sp. </p>  <table> <thead> <tr> <th>Evaluation Index</th> <th>Traditional Biochemical Methods</th> <th>FT-IR Method</th> </tr> </thead> <tbody> <tr> <td>Cultivation scale</td> <td>50 mL (in 250 mL flash)</td> <td>200 μL (in tube)</td> </tr> <tr> <td>Sample condition</td> <td>Lyophilized cells</td> <td>Liquid sample</td> </tr> <tr> <td>Time consumed</td> <td>~72 h</td> <td>~4 h</td> </tr> <tr> <td>Chemical reagents</td> <td>Liquid nitrogen, chloroform, and methanol for lipid; sulfuric acid and anthrone for carbohydrate </td> <td>None</td> </tr> <tr> <td>Equipment</td> <td>Centrifuge, freeze dryer, mortar, pestle, evaporimeter, and balance</td> <td>FT-IR spectrometer and spectrophotometer</td> </tr> <tr> <td>Cost</td> <td>Very high </td> <td>Low</td> </tr> <tr> <td>Accuracy</td> <td>High </td> <td>High</td> </tr> </tbody> </table> </div> </div>  </body></html>