<html xmlns="http://www.w3.org/1999/xhtml"><head> <style type="text/css">
         	     		 
         	    a                          {background : #ffffff; }
         	    article                    {border-style : dotted; border-width : 2px; }
         		 	
         		 	div                        {background : #ffffcc;}
         		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
         		 	div.ack                    {border-style : solid ; border-color : red; margin
         : 2em; }
         		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.boxed-text             {margin : 5em; border-style : solid;}
         		 	div.contrib-group          {margin : 1em; }
         		 	div.fig                    {border-style : solid; border-width : 2px; margin :
         2em; }
         		 	div.funding                {font-weight : bold ; font-size : 16pt;}
         		 	div.given-names            {font-style : italic;}
         		 	div.intro                  {border-style : inset; margin : 5px;}
         		 	div.introduction           {border-style : inset; margin : 5px;}
         		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.journal-title-group    {background : #ffeeee;}
         		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
         		 	div.kwd                    {font-style : italic}
         		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
         		 	div.name                   {font-weight : bold;}
         		 	div.alt-title              {font-style : italic; font-size : 12px;}
         		 	
         		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
         		 	div.title                  {font-family : courier; font-weight : bold;
         		 	                            font-size : 16pt; margin : 5px;}
         		 	
         		 	div.materials_methods      {border-style : double; margin : 5px; }
         		 	div.methods                {border-style : double; margin : 5px; }
         		 	
         		 	div.results                {border-style : solid; margin : 5px;}
         		 	div.background             {border-style : dotted; margin : 5px;}
         		 	
         		 	div.discussion             {border-style : groove; margin : 5px;}
         		 	div.conclusion             {border-style : ridge; margin : 5px;}
         		 	div.conclusions             {border-style : ridge; margin : 5px;}
         		 	div.supplementary-material {border-style : inset; margin : 5px;}
         		 	div.abbreviations          {border-style : double; border-color : red; margin
         : 5px;}
         		 	div.competinginterests     {border-style : double; border-color : blue; margin
         : 5px;}
         		 	div.acknowledgements       {border-style : double; border-color : green; margin
         : 5px;}
         		 	div.authors_contributions   {border-style : double; border-color : purple; margin
         : 5px;}
         		 	
         		 	div.publisher              {border-style : outset; margin : 5px;}
         		 	div.fn-type-conflict       {background : #f88; }
         		 	div.fn-type-con            {background : #ddf; }
         		 	div.fn-type-other          {background : #ddd; }
         		 	
         		 	div.unknown                {background : #ffd;
         									 	  border-style : solid;
         									 	  border-width : 1px;
         									 	  padding : 2 px;}
         		 	  
         	    table                      {background : #ffffdd;}
         		 	tr                         {background : #ddddff; padding : 1px;}
         		 	
         		 	span                       {background : #ffcccc;}
         		 	
         		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
         		 	span.collab                {background : #ddffff; }
         		 	span.comment               {font-family : courier; font-size : 6px; background
         : #ffaaff;}
         		 	span.contrib               {background : #ffffff;}
         		 	span.corresp               {background : #ddffdd; }
         		 	span.doi                   {background : #ffffff;}
         		 	span.email                 {font-family : courier; }
         		 	span.etal                  {font-style : italic;}
         		 	span.fpage                 {font-family : courier;}
         		 	span.given-names           {background : #ffffff;}
         		 	span.iso-abbrev            {background : #ffffff;}
         		 	span.issn-epub             {background : #ffffff;}
         		 	span.issn-ppub             {background : #ffffff;}
         		 	span.journal-title         {background : #ffffff;}
         		 	span.lpage                 {font-family : courier;}
         		 	span.mixed-article-title   {font-style : italic ;}
         		 	span.nlm-ta                {background : #ffffff;}
         		 	span.pmc                   {background : #ffffff;}
         		 	span.pmcid                 {background : #ffffff;}
         		 	span.pmid                  {background : #ffffff;}
         		 	span.publisher             {background : #ffffff;}
         		 	span.publisher-id          {background : #ffffff;}
         		 	span.publisher-name        {background : #ffffff;}
         		 	span.source                {background : #ffffff;}
         		 	span.subject               {background : #ffffff;}
         		 	span.surname               {background : #ffffff;}
         		 	span.volume                {font-family : courier; font-weight : bold;}
         		 	span.year                  {font-family : courier ; font-style : italic;}
         			
      </style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"> <span class="nlm-ta" title="nlm-ta">Front Microbiol</span> <span class="iso-abbrev" title="iso-abbrev">Front Microbiol</span> <span class="publisher-id" title="publisher-id">Front. Microbiol.</span> <div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"> <span class="journal-title" tagx="journal-title" title="journal-title">Frontiers in Microbiology</span> </div> <span class="issn-epub" tagx="issn" title="issn-epub">1664-302X</span> <div class="publisher" tagx="publisher" title="publisher"> <span class="publisher-name" tagx="publisher-name" title="publisher-name">Frontiers Media S.A.</span> </div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"> <span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/6218658">6218658</a> </span> <span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.3389/fmicb.2018.02543">10.3389/fmicb.2018.02543</a> </span> <div class="article-categories" title="article-categories"> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Microbiology</span>: <span class="subject" title="subject">Original Research</span> </div> </div> <div class="title-group"> <div class="article-title" title="article-title">
Molecular and Microbiological Insights on the Enrichment Procedures for the Isolation of Petroleum Degrading Bacteria and Fungi </div> </div> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Spini</span> <span class="given-names" tagx="given-names" title="given-names">Giulia</span> </span> <a href="#aff1"> <sup>1</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Spina</span> <span class="given-names" tagx="given-names" title="given-names">Federica</span> </span> <a href="#aff2"> <sup>2</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Poli</span> <span class="given-names" tagx="given-names" title="given-names">Anna</span> </span> <a href="#aff2"> <sup>2</sup> </a> <span class="uri" title="uri"> <a href="http://loop.frontiersin.org/people/493942/overview" /> </span> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Blieux</span> <span class="given-names" tagx="given-names" title="given-names">Anne-Laure</span> </span> <a href="#aff3"> <sup>3</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Regnier</span> <span class="given-names" tagx="given-names" title="given-names">Tiffanie</span> </span> <a href="#aff3"> <sup>3</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gramellini</span> <span class="given-names" tagx="given-names" title="given-names">Carla</span> </span> <a href="#aff4"> <sup>4</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Varese</span> <span class="given-names" tagx="given-names" title="given-names">Giovanna C.</span> </span> <a href="#aff2"> <sup>2</sup> </a> <span class="uri" title="uri"> <a href="http://loop.frontiersin.org/people/139215/overview" /> </span> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Puglisi</span> <span class="given-names" tagx="given-names" title="given-names">Edoardo</span> </span> <a href="#aff1"> <sup>1</sup> </a> <a href="#c001"> <sup>*</sup> </a> <span class="uri" title="uri"> <a href="http://loop.frontiersin.org/people/262755/overview" /> </span> </span> </div> <span class="citation_author_institution" id="aff1">[], <sup>1</sup> <span class="institution" title="institution">Department for Sustainable Food Processes, Università Cattolica del Sacro Cuore</span> <span class="addr-line" title="addr-line">Piacenza</span> <span class="country" tagx="country" title="country">Italy</span> </span> <span class="citation_author_institution" id="aff2">[], <sup>2</sup> <span class="institution" title="institution">Department of Life Sciences and Systems Biology, Mycotheca Universitatis Taurinensis, University of Turin </span> <span class="addr-line" title="addr-line">Turin</span> <span class="country" tagx="country" title="country">Italy</span> </span> <span class="citation_author_institution" id="aff3">[], <sup>3</sup> <span class="institution" title="institution">Satt Grand-EST, Maison Règionale de l’Innovation</span> <span class="addr-line" title="addr-line">Dijon</span> <span class="country" tagx="country" title="country">France</span> </span> <span class="citation_author_institution" id="aff4">[], <sup>4</sup> <span class="institution" title="institution">ARPAE-Emilia Romagna</span> <span class="addr-line" title="addr-line">Bologna</span> <span class="country" tagx="country" title="country">Italy</span> </span> <div class="author-notes" title="author-notes"> <div class="fn-type-edited-by" title="edited-by"> <p>Edited by: Stéphane Pesce, Institut National de Recherche en Sciences et Technologies pour l’Environnement et l’Agriculture (IRSTEA), France </p> </div> <div class="fn-type-edited-by" title="edited-by"> <p>Reviewed by: Mariusz Cycoń, Medical University of Silesia, Poland; Maurizio Petruccioli, Università degli Studi della Tuscia, Italy </p> </div> <div class="corresp" title="corresp">
*Correspondence: Edoardo Puglisi, <span class="email" tagx="email" title="email">edoardo.puglisi@unicatt.it</span> </div> <div class="fn-type-other" title="other"> <p>This article was submitted to Microbiotechnology, Ecotoxicology and Bioremediation, a section of the journal Frontiers in Microbiology </p> </div> </div> <span class="pub-date-epub" title="pub-date-epub">epub: <span>2018-10-10</span> </span> <span class="pub-date-collection" title="pub-date-collection">collection: <span>2018</span> </span> <span class="volume" tagx="volume" title="volume">9</span> <span class="elocation-id" tagx="elocation-id" title="elocation-id">2543</span> <span class="history" title="history"> <span class="received" title="received">received: 2018-7-31</span> <span class="accepted" title="accepted">accepted: 2018-10-05</span> </span> <div class="permissions"> <span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Spini, Spina, Poli, Blieux, Regnier, Gramellini, Varese and Puglisi</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2018</span> </span> <span class="license" title="license"> <span class="license-p" title="license-p">This is an open-access article distributed under the terms of the Creative Commons Attribution License (CC BY). The use, distribution or reproduction in other forums is permitted, provided the original author(s) and the copyright owner(s) are credited and that the original publication in this journal is cited, in accordance with accepted academic practice. No use, distribution or reproduction is permitted which does not comply with these terms. </span> </span> </div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>Autochthonous bioaugmentation, by exploiting the indigenous microorganisms of the contaminated environment to be treated, can represent a successful bioremediation strategy. In this perspective, we have assessed by molecular methods the evolution of bacterial and fungal communities during the selective enrichment on different pollutants of a soil strongly polluted by mixtures of aliphatic and polycyclic hydrocarbons. Three consecutive enrichments were carried out on soil samples from different soil depths (0–1, 1–2, 2–3 m), and analyzed at each step by means of high-throughput sequencing of bacterial and fungal amplicons biomarkers. At the end of the enrichments, bacterial and fungal contaminants degrading strains were isolated and identified in order to (i) compare the composition of enriched communities by culture-dependent and culture-independent molecular methods and to (ii) obtain a collection of hydrocarbon degrading microorganisms potentially exploitable for soil bioremediation. Molecular results highlighted that for both bacteria and fungi the pollutant had a partial shaping effect on the enriched communities, with paraffin creating distinct enriched bacterial community from oil, and polycyclic aromatic hydrocarbons generally overlapping; interestingly neither the soil depth or the enrichment step had significant effects on the composition of the final enriched communities. Molecular analyses well-agreed with culture-dependent analyses in terms of most abundant microbial genera. A total of 95 bacterial and 94 fungal strains were isolated after selective enrichment procedure on different pollutants. On the whole, isolated bacteria where manly ascribed to <i>Pseudomonas</i> genus followed by <i>Sphingobacterium</i>, <i>Bacillus</i>, <i>Stenothrophomonas</i>, <i>Achromobacter</i>, and <i>Serratia</i>. As for fungi, <i>Fusarium</i> was the most abundant genus followed by <i>Trichoderma</i> and <i>Aspergillus</i>. The species comprising more isolates, such as <i>Pseudomonas putida</i>, <i>Achromobacter xylosoxidans</i> and <i>Ochromobactrum anthropi</i> for bacteria, <i>Fusarium oxysporum</i> and <i>Fusarium solani</i> for fungi, were also the dominant OTUs assessed in Illumina. </p> </div> <div class="kwd-group"> <div class="kwd" title="kwd">
bioremediation</div> <div class="kwd" title="kwd">
crude oil</div> <div class="kwd" title="kwd">
soil contamination</div> <div class="kwd" title="kwd">
enrichment culture</div> <div class="kwd" title="kwd">
metagenomics</div> <div class="kwd" title="kwd">
bacteria</div> <div class="kwd" title="kwd">
fungi</div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source">
LIFE programme <span class="named-content" title="named-content">10.13039/100013288</span> </div> </div> </div> <div class="counts" title="counts"> <span class="fig-count" title="fig-count">fig-count: </span> <span class="table-count" title="table-count">table-count: </span> <span class="equation-count" title="equation-count">equation-count: </span> <span class="ref-count" title="ref-count">ref-count: </span> <span class="page-count" title="page-count">page-count: </span> <span class="word-count" title="word-count">word-count: </span> </div> </div> </div> <div class="body" title="body"> <div class="introduction" title="sec"> <div class="title" tagx="title" title="title">
Introduction</div> <p>Oil hydrocarbons are the most widespread environmental pollutants, including n-alkanes, cycloalkanes, and polycyclic aromatic hydrocarbons (PAHs) that have been regarded as serious ecological and public health concerns ( <a href="#B10">Bao et al., 2012</a>). Oil contamination of ecosystems is a serious issue associated with crude oil drilling, transportation, refining, and related activities which demands immediate attention for restoration. Crude petroleum oil is a complex mixture of hydrocarbons mainly composed of saturated and aromatic hydrocarbons, asphaltenes and resins ( <a href="#B68">Moubasher et al., 2015</a>). Aliphatic hydrocarbons consist of readily biodegradable n-alkanes followed by less biodegradable branched and cyclic alkanes ( <a href="#B24">Das and Chandran, 2011</a>). Similarly, PAHs are compounds with two or more aromatic rings that are the most recalcitrant components present at high percentages in crude oil ( <a href="#B45">Haritash and Kaushik, 2009</a>). </p> <p>Due to the public awareness and to the strict legal constraints on the release of pollutants into the environment, it is necessary to find effective and affordable technologies for the treatment of oil industrial wastes. Bioremediation is a biological approach that relies on the metabolic potential of microorganisms to remove contaminants ( <a href="#B61">Maiti et al., 2008</a>; <a href="#B65">Megharaj et al., 2011</a>; <a href="#B44">Hara et al., 2013</a>). The use of bioremediation techniques proved economical, environmentally friendly and flexible ( <a href="#B72">Obi et al., 2016</a>). In this prospect, bioremediation is gaining more and more importance as constructive approach for the remediation of polluted sites. Several studies reported the catabolic abilities of indigenous microorganisms such as fungi, bacteria and algae to degrade hydrocarbons ( <a href="#B26">Dean-Ross et al., 2002</a>; <a href="#B17">Bundy et al., 2004</a>; <a href="#B61">Maiti et al., 2008</a>; <a href="#B105">Wang et al., 2011</a>; <a href="#B9">Badr El-Din et al., 2014</a>). These microorganisms, adapted to the contaminated environments, are equipped with specific enzyme systems that enable them to use hydrocarbons as sole carbon source. Different hydrocarbon degrading microorganisms such as bacteria and archaea have been found in hydrocarbons contaminated environments ( <a href="#B4">Alonso-Gutierrez et al., 2009</a>; <a href="#B6">An et al., 2013</a>; <a href="#B46">Hazen et al., 2013</a>; <a href="#B47">Head et al., 2014</a>; <a href="#B38">Fowler et al., 2016</a>). The structure of the microbial community in a soil, influences deeply the degree of oil hydrocarbons degradation. <a href="#B57">Liu et al. (2011)</a> observed that, at the early stage of remediation, the bacterial community was responsible for the degradation of the saturated and partially aromatic hydrocarbons; the fungal community instead became dominant in decomposing the polar hydrocarbons fraction in post-remediation. Generally, thanks to the variety of extracellular enzymes and fungal hyphae, fungi are the first key players in degrading available contaminants and recalcitrant polymers ( <a href="#B108">Wick et al., 2007</a>; <a href="#B35">Fernandez-Luqueno et al., 2010</a>; <a href="#B28">Deshmukh et al., 2016</a>). Fungal mobilization and degradation of contaminants contribute to release bioavailable intermediates on which, in a later stage, the bacterial community can act more easily ( <a href="#B92">Smits et al., 2005</a>; <a href="#B88">Scullion, 2006</a>; <a href="#B55">Leonardi et al., 2008</a>; <a href="#B117">Yuan et al., 2018</a>). Therefore, in order to characterize and to monitor the native microbial community, the dynamics and the functional potential of both bacteria and fungi in polluted ecosystems is essential for the development of a successful bioremediation strategy ( <a href="#B8">Atlas, 1981</a>; <a href="#B47">Head et al., 2014</a>; <a href="#B22">Covino et al., 2016</a>). </p> <p>The success of on-site bioremediation, employing native microbial populations, could be voided by imbalanced nutrients and/or adverse factors (temperature, moisture content, pH, availability of electron donor and/or acceptor, high pollutant concentration, etc.), which are common in contaminated sites ( <a href="#B91">Smith et al., 2015</a>). Bioremediation can be accomplished by either boosting the growth of the indigenous microbial community through biostimulation or by introducing naturally occurring microorganisms with excellent catabolic abilities (bioagumentation) that are adapted to the ecological conditions of the site ( <a href="#B2">Agnello et al., 2016</a>). </p> <p>Many investigators have reported successful bioremediation, where biostimulation with the addition of appropriate nutrients (N and/or P), to avoid metabolic limitations, resulted in an improved metabolic activity of indigenous microorganisms ( <a href="#B116">Yu et al., 2005</a>; <a href="#B41">Ghaly and Yusran, 2013</a>; <a href="#B94">Suja et al., 2014</a>; <a href="#B91">Smith et al., 2015</a>). On the other hand, autochthonous bioagumentation, based on the re-inoculation in polluted sites of indigenous microorganisms previously enriched under laboratory conditions, enhanced the microbial activities, thus improving the degradation of hydrocarbons ( <a href="#B31">Dueholm et al., 2015</a>). In order to provide an inoculum for bioaugmentation, the isolation of microorganisms in pure culture from these contaminated environments is fundamental. However, due to the complexity of crude oil, a microbial consortium composed of microorganisms endowed with diverse metabolic capacities and syntrophic interactions would work better than a pure culture. Several reports demonstrated the better metabolic versatility of mixed cultures in using hydrocarbon pollutants as sole carbon source in comparison to pure cultures ( <a href="#B18">Cerqueira et al., 2011</a>; <a href="#B24">Das and Chandran, 2011</a>). In laboratory conditions, bacterial, and fungal co-culture(s) showed improved degradation rates of diesel oil and of polyacyclic romatic hydrocarbons (PAHs) ( <a href="#B105">Wang et al., 2011</a>). Hence, catabolic interactions among different microbial groups during biodegradation is extremely important ( <a href="#B8">Atlas, 1981</a>; <a href="#B97">Varjani et al., 2015</a>). Although identifications and characterizations of the microorganisms involved in the degrading processes are available ( <a href="#B27">Desai and Banat, 1997</a>; <a href="#B83">Rojo, 2009</a>), less is known on the biodiversity and dynamics of the native hydrocarbons-degrading microbial community of a contaminated soil, especially during the enrichment process applied to isolate the most effective strains ( <a href="#B74">Omrani et al., 2018</a>). The development of effective bioremediation strategies requires an extensive understanding of the resident microorganisms of these habitats. Recent techniques such as high-throughput sequencing (HTS) have greatly facilitated the advancement of microbial ecological studies in oil-polluted sites. </p> <p>In the present work we have assessed by molecular methods the evolution of bacterial and fungal communities during the enrichment on different pollutants of a soil strongly polluted by mixtures of aliphatic and polycyclic hydrocarbons. Three consecutive enrichments were carried out on soil samples from different soil depths (0–1, 1–2, 2–3 m), and analyzed at each step by means of HTS of bacterial and fungal amplicons biomarkers. At the end of the enrichment, bacterial, and fungal contaminants degrading strains were cultivated and identified. </p> <p>Main aims of the work were: (i) to assess the effect of different pollutants used as sole carbon during a selective enrichment procedure source on the diversity of the microbial community of crude oil contaminated soil (ii) to compare the composition of enriched communities by culture-dependent and culture-independent molecular methods and (iii) to obtain a collection of hydrocarbon degrading fungi and bacteria potentially exploitable for soil bioremediation. </p> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Materials and Methods</div> <div class="samplingandsoilcharacteristic" title="sec"> <div class="title" tagx="title" title="title">
Sampling and Soil Characteristic</div> <p>Crude oil contaminated soil samples from an area of the industrial SIN (Site of National Interest) located in Fidenza (Emilia-Romagna, Italy) were collected at three different depths (0–1, 1–2, and 2–3 m). The polluted site (about 80,000 m <sup>2</sup> wide) has a long history of industrial exploitation and it is contaminated by a variety of pollutants such as Benzene-Toluene-Ethylbenzene-Xylene (BTEX), n-alkanes and PAHs. </p> <p>The chemical composition of the soil samples is described in Supplementary Table <a href="#SM1">S1</a>. </p> </div> <div class="targetorganiccontaminants" title="sec"> <div class="title" tagx="title" title="title">
Target Organic Contaminants</div> <p>Target organic contaminants were chosen based on literature data and on the chemical characterization of the contaminated area of Fidenza SIN. All chemicals were purchased by Sigma-Aldrich (Germany). Benzene was selected as representative of BTEX. Pyrene, phenanthrene, and naphthalene were selected as representatives of 4, 3, and 2 rings PAHs. Stock solutions were prepared in methanol for naphthalene (20 mg/mL) and in ethanol 95% for phenanthrene (15 mg/mL) and for pyrene (5 mg/mL). Paraffin oil, representative of alkanes, and crude oil mixture from the contaminated Fidenza site were used at a final concentration of 1% v/v. </p> </div> <div class="microcosmenrichmentsandmicrobialisolation" title="sec"> <div class="title" tagx="title" title="title">
Microcosm Enrichments and Microbial Isolation</div> <p>Different enrichment cultures were set up using the polluted soil as inoculum and the above-mentioned crude oil hydrocarbons components as the sole carbon source. One hundred g of each soil sample at the three depths (namely, S1, S2, and S3) were added to 900 mL sterile Mineral Medium (MM) supplemented with the target analyte as sole carbon source. To provide the proper microelement amount for bacterial and fungal growth, two mineral media were used: M9 mineral medium (Difco, Sparks, MD, United States) for bacteria and Czapek for fungi. </p> <p>The flasks were incubated on a rotary shaker at 30°C and 180 rpm for bacteria and at 24°C and 120 rpm for fungi. After 7 days, 5 mL of the culture was transferred to another flask with MM and the corresponding target analyte: benzene (50 ppm); naphthalene (200 ppm), phenanthrene (200 ppm), pyrene (200 ppm); paraffin oil (1% v/v), and crude oil (1% v/v). Three consecutive subcultures were performed in the same conditions. </p> <p>Each fungal and bacterial enrichment culture was plate on solid MM containing each pollutant as sole carbon source. To isolate pure microbial cultures, morphologically different colonies were selected and transferred to Malt Extract Agar (MEA) plates for fungi and Tryptone Soy Agar (TSA) plates for bacteria. </p> </div> <div class="molecularcharacterizationoffungalandbacterialisolates" title="sec"> <div class="title" tagx="title" title="title">
Molecular Characterization of Fungal and Bacterial Isolates</div> <div class="bacteria" title="sec"> <div class="title" tagx="title" title="title">
Bacteria</div> <p>DNA from isolate purified colonies was extracted using Microlysis kit (Labogen, London, United Kingdom) according to the manufacturer’s protocol. The isolates were screened by randomly amplified polymorphic DNA-polymerase chain reaction (RAPD-PCR) using the single stranded oligonucleotides primer RAPD2 (5′-AGCAGCGTGG-3′) ( <a href="#B81">Rebecchi et al., 2015</a>) and GTG-5 (5′-GTG GTG GTG GTG GTG-3′) ( <a href="#B101">Versalovic et al., 1994</a>). The PCR fragment profiles were digitally captured using the BioImaging System Gene Genius and pattern analysis was performed with the Fingerprinting II software (Bio-Rad Laboratories, Hercules, CA, United States). The similarity in the profiles of bands was based on the Pearson correlation coefficient and the cluster analyses were performed by unweighted pair group method with arithmetic mean (UPGMA). A correlation coefficient of 70% was arbitrarily selected to distinguish the clusters, and one representative for each cluster was amplified using the primers P0 (5′-GAG AGT TTG ATC CTG GCT-3′) and P6 (5′-CTA CGG CTA CCT TGT TAC-3′) ( <a href="#B29">Di Cello and Fani, 1996</a>). PCR products were visualized by electrophoresis on 2.5% agarose gel in Tris-Acetate-EDTA (TAE). The PCR amplicons of approximately 1.5 kb, corresponding to the size of the full 16S rRNA gene were purified using the NucleoSpin gel and PCR clean-up according to the package insert (Macherey-Nagel, DE) and sequenced at the GATC Biotech (Germany). The taxonomical identification of sequences was performed using BLAST (Basic Local Alignment Search Tool) ( <a href="#B5">Altschul et al., 1997</a>) and by alignment against the Ribosomal Database Project (RDP) database using the Naïve Bayesian Classifier ( <a href="#B104">Wang et al., 2007</a>). </p> </div> <div class="fungi" title="sec"> <div class="title" tagx="title" title="title">
Fungi</div> <p>Genomic DNA of each strains was extracted from about 100 mg of mycelium scraped from the MEA petri dishes using the NucleoSpin <sup>®</sup> Plant II kit (Macherey-Nagel), according to the manufacturer’s instruction. The quality and quantity of extracted DNA was measured spectrophotometrically by Infinite M200 (TECAN Trading, Austria). De-replication of fungal isolates was performed by using the minisatellite core sequence derived from the wildtype phage M13 (5′-GAG GGT GGC GGT TCT-3′) as specific primer to amplify variable number tandem repeat (VNTR) ( <a href="#B78">Poli et al., 2016</a>). Molecular identification of each fungal isolate was carried out by amplification of specific markers ( <a href="#B106">White et al., 1990</a>; <a href="#B43">Glass and Donaldson, 1995</a>; <a href="#B12">Bensch et al., 2012</a>). PCR products were visualized by electrophoresis on a 1.5% agarose gel stained with ethidium bromide in Tris-Borate-EDTA (TBE). PCR products were purified and sequenced at Macrogen Europe (Amsterdam, Netherlands). Consensus sequences were obtained by using Sequencer 5.0 (Gene Code Corporation). Taxonomic assignments were inferred by querying with the Blastn algorithm (default setting), hosted at NCBI (National Center for Biotechnology Information), the newly generated sequences against the nucleotide database of NCBI (GenBank). Pairwise alignments were also performed against the CBS-Knaw Fungal Biodiversity Centre (Centraalbureau voor Schimmel cultures) database. Similarity values equal or higher than 98% (e-value &amp;gt; e-100) were considered reliable; results were confirmed morphologically. </p> </div> </div> <div class="dnaextractionfromoriginalsoilsandenrichments" title="sec"> <div class="title" tagx="title" title="title">
DNA Extraction From Original Soils and Enrichments</div> <p>Total microbial DNA was extracted from original soil samples and from each enrichment step of fungal and bacterial culture/microcosms with the PowerLyzer PowerSoil kit (MoBIO Laboratories, Inc., Carlsbad, CA, United States) according to the manufacturer’s instructions. DNA purity was checked with electrophoresis on a 0.8% agarose gel, while quantification was performed with the Quant-iTTM HS ds-DNA assay kit (Invitrogen, Paisley, United Kingdom) method in combination with the QuBit <sup>TM</sup> fluorometer. </p> </div> <div class="molecularanalysesofmicrobialdiversity" title="sec"> <div class="title" tagx="title" title="title">
Molecular Analyses of Microbial Diversity</div> <div class="illuminasequencingof16spcramplicons" title="sec"> <div class="title" tagx="title" title="title">
Illumina Sequencing of 16S PCR Amplicons</div> <p>For bacteria, PCR amplicons covering the V3–V4 regions of the 16S rRNA were analyzed in Illumina MiSeq with V3 chemistry in 300 bp paired-reads mode. PCR reactions were performed using indexed primer pairs 343F (5′-TACGGRAGGCAGCAG-3′) and 802R (5′-TACNVGGGTWTCTAATCC-3′), as described in <a href="#B99">Vasileiadis et al. (2015)</a>. A multiplexing strategy was employed to analyze several amplicon samples simultaneously in the same sequencing run. A nine nucleic acids extension was added to the 5′ end of the forward primer, where the first seven bases served as a tag to identify to each sample, and the following two bases were a linker designed not to match bacterial sequences in the same position according to RDP entries. In order to reduce possible biases related to the primer extension, the two step-PCR approach described in <a href="#B14">Berry et al. (2011)</a> was adopted. The PCR conditions were set as described by <a href="#B99">Vasileiadis et al. (2015)</a>. The final PCR products were checked on 0.8% agarose gel and pooled in equimolar amounts according to QuBit measurements. The final pool was cleaned with the SPRI (Solid Phase Reverse Immobilization Method) using the Agencourt <sup>®</sup> AMPure <sup>®</sup> XP kit (Beckman Coulter, Milan, Italy). Finally, the pool was sequenced by BioFab Company (Rome, Italy) with a MiSeq Illumina instrument (Illumina, Inc., San Diego, CA, United States) operating with V3 chemistry and producing 300 bp paired-reads. </p> </div> <div class="illuminasequencingofitspcramplicons" title="sec"> <div class="title" tagx="title" title="title">
Illumina Sequencing of ITS PCR Amplicons</div> <p>The nr ITS2 region was amplified from all DNA by means of a semi-nested PCR approach. In the first PCR, the nr ITS (ITS1-5.8S-ITS2) was amplified with universal primers ITS1F-ITS4 ( <a href="#B106">White et al., 1990</a>). For the second PCR, ITS3 and ITS4 ( <a href="#B106">White et al., 1990</a>) tagged primers were used to amplify the ITS2 region of each DNA sample ( <a href="#B103">Voyron et al., 2017</a>). </p> <p>PCR products were pooled and purified using Wizard SV Gel and PCR Clean-Up System (Promega) following the manufacturer’s instructions. After quantification with Qubit 2.0 (Thermo Fisher Scientific, Waltham, MA, United States), the purified PCR products were mixed in equimolar amounts to prepare sequencing libraries. The libraries were paired-end sequenced using the Illumina MiSeq technology (2 bp × 250 bp) by IGA Technology Services S.r.l. Unipersonale (Udine, Italy). </p> </div> </div> <div class="sequencesdatapreparation,bioinformatics,andstatisticalanalyses" title="sec"> <div class="title" tagx="title" title="title">
Sequences Data Preparation, Bioinformatics, and Statistical Analyses</div> <p>The first steps for sequences processing and filtering were the same for both 16S and ITS amplicons. Raw paired Illumina sequences were merged with the "pandaseq" script ( <a href="#B63">Masella et al., 2012</a>) with a minimum overlap of 30 bp between read pairs and 2 maximum allowed mismatches. Sequences were multiplexed according to sample indexes and primers with the fastx-toolkit <sup> <a href="#fn01">1</a> </sup>. Both bacterial and fungal amplicons were analyzed with taxonomy-based and OTU-based analyses: in the first case, all sequences were individually classified at taxonomical level against relevant database (GreenGenes for bacteria, UNITE for fungi), while in the OTU-based analyses, sequences were grouped at 97% similarities. For 16S amplicons, both operational taxonomic units (OTUs) and taxonomy-based matrixes were produced with a pipeline in Mothur ( <a href="#B87">Schloss et al., 2009</a>). For ITS amplicons, taxonomy-based analyses were also performed in Mothur, whereas OTUs were determined in UPARSE ( <a href="#B32">Edgar, 2013</a>). The reason for this discrepancy was that no aligned databases are available for ITS, and the OTU-clustering method implemented in Mothur does not allow analysis of sequences that have dissimilar lengths, which was the case with the ITS amplicons. </p> <p>For bacterial sequences, Mothur v.1.39.5 ( <a href="#B87">Schloss et al., 2009</a>) was applied in order to remove sequences with large homopolymers (≥10), sequences that did not align within the targeted V3–V4 region, chimeric sequences ( <a href="#B33">Edgar et al., 2011</a>) and sequences that were not classified as bacterial after alignment against the Mothur version of the RDP training data set. The resulting high- quality sequences were analyzed with Mothur and R <sup> <a href="#fn02">2</a> </sup> following the OTU and the taxonomy-based approach. Sequences were first aligned against the SILVA reference aligned database for bacteria ( <a href="#B79">Pruesse et al., 2007</a>) using the NAST algorithm and a kmer approach ( <a href="#B86">Schloss, 2010</a>) and then clustered at the 3% distance using the average linkage algorithm. OTUs were classified into taxa by alignment against the Greengenes database ( <a href="#B64">McDonald et al., 2012</a>). </p> <p>ITS taxonomy-based analyses were conducted in Mothur: sequences shorter than 120 bp were discarded. We discarded homopolymers &amp;gt; 10 bp and chimeras, which were identified with the UCHIME algorithm implemented in Mothur, with the UNITE database version 6 as reference. The same database was used to classify the retained sequences and to eliminate non-fungal sequences. OTUs were produced in USEARCH with the –fastx_uniques and - cluster_otus commands. Sequences that did not belong to fungi were identified with the sintax command against the Utax reference database and discarded. </p> <p>The OTU- and taxonomy-based matrixes obtained were analyzed in R to estimate the associated α and β diversity of the samples. The Good’s coverage estimate was calculated to assess the "percentage diversity" captured by sequencing. The most abundant OTUs identified were confirmed with BLAST (Basic Local Alignment Search) searches against the GenBank and the RDP database. </p> </div> </div> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <div class="hydrocarbondegradingbacterialandfungalstrains" title="sec"> <div class="title" tagx="title" title="title">
Hydrocarbon Degrading Bacterial and Fungal Strains</div> <p>A collection of hydrocarbon-degrading bacterial and fungal strains was isolated from microcosms enriched on each pollutant (benzene, paraffin, pyrene, naphthalene, phenanthrene, and crude oil) and from three depths of the contaminated soil of the SIN in Fidenza. Molecular de-replication of isolated colonies resulted in 95 and 94 unique colonies for bacteria and fungi, respectively. Molecular identification of the strains was inferred by 16S rDNA and ITS sequencing and is reported in Table <a href="#T1">1</a>. </p> <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 1</span>  <p>Identification of bacteria and fungi isolated from benzene, paraffin, crude oil, naphthalene, pyrene, and phenanthrene enrichment microcosms. </p>  <table> <thead> <tr> <td /> <th>Bacterial isolates</th> <th>Fungal isolates</th> </tr> </thead> <tbody> <tr> <td>Benzene</td> <td> <i>Acinetobacter calcoaceticus (1)</i> </td> <td> <i>Acremonium sclerotigenum (1)</i> </td> </tr> <tr> <td /> <td> <i>Bacillus subtilis (1)</i> </td> <td> <i>Aspergillus creber (1)</i> </td> </tr> <tr> <td /> <td> <i>Benzo[a]pyrene-degrading bacterium (1)</i> </td> <td> <i>Bjerkabndera adusta (1)</i> </td> </tr> <tr> <td /> <td> <i>Cellulosimicrobium</i> sp. <i>(1)</i> </td> <td> <i>Eutypella scoparia (1)</i> </td> </tr> <tr> <td /> <td> <i>Pseudomonas mosselii (2)</i> </td> <td> <i>Fusarium oxysporum (1)</i> </td> </tr> <tr> <td /> <td> <i>Pseudomonas putida (1)</i> </td> <td> <i>Fusarium solani (5)</i> </td> </tr> <tr> <td /> <td> <i>Pseudoxanthomonas indica (1)</i> </td> <td> <i>Irpex lacteus (1)</i> </td> </tr> <tr> <td /> <td> <i>Rhizobium petrolearium (1)</i> </td> <td> <i>Pseudoallescheria boydii (2)</i> </td> </tr> <tr> <td /> <td> <i>Sphingobacterium</i> sp. ( <i>3)</i> </td> <td> <i>Scedosporium apiospermum (1)</i> </td> </tr> <tr> <td /> <td> <i>Stenotrophomonas acidaminiphila (2)</i> </td> <td> <i>Scedosporium dehoogii (1)</i> </td> </tr> <tr> <td /> <td> <i>Stenotrophomonas maltophilia (1)</i> </td> <td /> </tr> <tr> <td>Paraffin</td> <td> <i>Acholeplasma vituli (1)</i> </td> <td> <i>Clonostachys rosea (1)</i> </td> </tr> <tr> <td /> <td> <i>Achromobacter xylosoxidans (3)</i> </td> <td> <i>Fusarium oxysporum (5)</i> </td> </tr> <tr> <td /> <td> <i>Bacillus subtilis (3)</i> </td> <td> <i>Fusarium solani (2)</i> </td> </tr> <tr> <td /> <td> <i>Bacillus xiamenentis (1)</i> </td> <td /> </tr> <tr> <td /> <td> <i>Cupriavidus campinensis (1)</i> </td> <td /> </tr> <tr> <td /> <td> <i>Gordonia rubripertincus (1)</i> </td> <td /> </tr> <tr> <td /> <td> <i>Helicobacter</i> sp. <i>(1)</i> </td> <td /> </tr> <tr> <td /> <td> <i>Paenibacillus</i> spp. ( <i>1)</i> </td> <td /> </tr> <tr> <td /> <td> <i>Pseudomonas putida (7)</i> </td> <td /> </tr> <tr> <td /> <td> <i>Pseudoxanthomonas mexicana (3)</i> </td> <td /> </tr> <tr> <td /> <td> <i>Sphingobacterium multivorum (6)</i> </td> <td /> </tr> <tr> <td /> <td> <i>Stenotrophomonas acidaminiphila (1)</i> </td> <td /> </tr> <tr> <td>Crude oil</td> <td> <i>Pseudomonas aeruginosa (1)</i> </td> <td> <i>Aspergillus jensenii (1)</i> </td> </tr> <tr> <td /> <td> <i>Pseudomonas fluorescens (2)</i> </td> <td> <i>Aspergillus protuberus (1)</i> </td> </tr> <tr> <td /> <td> <i>Serratia marcescens (3)</i> </td> <td> <i>Aspergillus versicolor (6)</i> </td> </tr> <tr> <td /> <td /> <td> <i>Cladosporium cladosporioides (1)</i> </td> </tr> <tr> <td /> <td /> <td> <i>Cladosporium perangustum (2)</i> </td> </tr> <tr> <td /> <td /> <td> <i>Epicoccum nigrum (1)</i> </td> </tr> <tr> <td /> <td /> <td> <i>Penicillium crustosum (1)</i> </td> </tr> <tr> <td /> <td /> <td> <i>Trichoderma harzianum (1)</i> </td> </tr> <tr> <td /> <td /> <td> <i>Wallemia mellicola (1)</i> </td> </tr> <tr> <td>Naphtalene</td> <td> <i>Achromobacter</i> sp. <i>(1)</i> </td> <td> <i>Aspergillus sclerotiorum (1)</i> </td> </tr> <tr> <td /> <td> <i>Pseudomonas fluorescens (4)</i> </td> <td> <i>Aspergillus sydowii (1)</i> </td> </tr> <tr> <td /> <td> <i>Pseudomonas putida (2)</i> </td> <td> <i>Aureobasidium pullulans (1)</i> </td> </tr> <tr> <td /> <td> <i>Pseudomonas</i> sp. <i>(7)</i> </td> <td> <i>Cladosporium cladosporioides (1)</i> </td> </tr> <tr> <td /> <td> <i>Pseudomonas veronii (3)</i> </td> <td> <i>Eutypella scoparia (1)</i> </td> </tr> <tr> <td /> <td> <i>Sphingobacterium multivorum (1)</i> </td> <td> <i>Fusarium oxysporum (1)</i> </td> </tr> <tr> <td /> <td> <i>Stenotrophomonas maltophilia (1)</i> </td> <td> <i>Penicillium brevicompactum (1)</i> </td> </tr> <tr> <td /> <td /> <td> <i>Penicillium catenatum (1)</i> </td> </tr> <tr> <td /> <td /> <td> <i>Scedosporium apiospermum (1)</i> </td> </tr> <tr> <td /> <td /> <td> <i>Sulcatispora acerina (1)</i> </td> </tr> <tr> <td /> <td /> <td> <i>Trametes gibbosa (1)</i> </td> </tr> <tr> <td>Pyrene</td> <td> <i>Ochrobactrum anthropi (1)</i> </td> <td> <i>Aspergillus waksmanii (1)</i> </td> </tr> <tr> <td /> <td> <i>Pseudomonas fluorescens (4)</i> </td> <td> <i>Cladosporium cladosporioides (2)</i> </td> </tr> <tr> <td /> <td> <i>Pseudomonas putida (1)</i> </td> <td> <i>Fusarium oxysporum (2)</i> </td> </tr> <tr> <td /> <td> <i>Pseudomonas</i> sp. <i>(4)</i> </td> <td> <i>Fusarium solani (9)</i> </td> </tr> <tr> <td /> <td /> <td> <i>Fusarium solani/keratoplasticum (3)</i> </td> </tr> <tr> <td /> <td /> <td> <i>Hypocrea lixii (2)</i> </td> </tr> <tr> <td /> <td /> <td> <i>Polyporus gayanus (1)</i> </td> </tr> <tr> <td /> <td /> <td> <i>Trichoderma harzianum (3)</i> </td> </tr> <tr> <td>Phenanthrene</td> <td> <i>Achromobacter</i> sp. <i>(1)</i> </td> <td> <i>Epicoccum nigrum (1)</i> </td> </tr> <tr> <td /> <td> <i>Pseudomonas fluorescens (3)</i> </td> <td> <i>Fusarium oxysporum (6)</i> </td> </tr> <tr> <td /> <td> <i>Pseudomonas putida (6)</i> </td> <td> <i>Fusarium solani (6)</i> </td> </tr> <tr> <td /> <td> <i>Pseudomonas</i> sp. <i>(5)</i> </td> <td> <i>Fusarium solani/keratoplasticum (2)</i> </td> </tr> <tr> <td /> <td /> <td> <i>Hypocrea lixii (2)</i> </td> </tr> <tr> <td /> <td /> <td> <i>Trichoderma harzianum (5)</i> </td> </tr> <tr> <td /> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <span class="attrib" tagx="attrib" title="attrib"> <i>The number of isolates per species is indicated in parenthesis.</i> </span> </div> </div> <p>For bacteria, sequence similarity searches in NCBI GenBank and RDP database revealed the presence of cultivable members of different genera under the phyla Proteobacteria, Firmicutes, and Sphingobacteria. On the whole, the bacteria isolated where ascribed mainly to 12 genera belonging to both Gram-negative and Gram-positive. <i>Pseudomonas</i> was the most abundant genera (56.8%) followed by <i>Sphingobacterium</i> (12%), <i>Bacillus</i> (6%), <i>Stenothrophomonas</i> (6%), <i>Achromobacter</i> (6%), and <i>Serratia</i> (3%). Most of the bacterial strains (46) were isolated from enrichments inoculated with soil S2 (Supplementary Table <a href="#SM1">S2</a>): the most represented bacterial genera in soil S2 were <i>Pseudomonas</i> (23 isolates), <i>Serratia</i> (3 isolates), and <i>Sphingobacterium</i> (9 isolates). A lower number of bacterial strains was isolated from enrichments of soil S1 and S3 (Supplementary Table <a href="#SM1">S2</a>). A total of 26 isolates were obtained from soil S1 and were mainly affiliated to the genera <i>Bacillus</i>, <i>Pseudomonas</i> and <i>Pseudoxanthomonas</i>, while the 23 bacterial strains isolated from the deepest soil S3 were mainly affiliated to the genus <i>Pseudomonas</i> (Supplementary Table <a href="#SM1">S2</a>). The influence of pollutants used in the enrichments on the distribution of bacterial species is reported in Table <a href="#T1">1</a>. The highest number of bacterial isolates was obtained from paraffin (29 isolates), followed by naphthalene (19 isolates), benzene (15 isolates), and phenanthrene (15 isolates) microcosms; 10 strains were isolated in the presence of pyrene and only six strains were isolated in the presence of crude oil (Table <a href="#T1">1</a>). The genus <i>Pseudomonas</i> dominated all the microcosms enriched on pollutants. Benzene and paraffin enrichments showed the highest diversity of bacterial genera. In particular, benzene enrichment selected also bacterial species of the genera <i>Sphingobacterium</i> and <i>Stenotrophomonas</i>, while paraffin enrichment resulted also in the isolation of species of <i>Bacillus</i>, <i>Achromobacter</i>, and <i>Pseudoxanthomonas</i>. Beside <i>Pseudomonas</i>, <i>Serratia</i>, and <i>Ochromobactrum</i> were the most abundant genera in crude oil and pyrene microcosms (Table <a href="#T1">1</a>). At species level, <i>Pseudomonas putida</i> was dominant and was isolated in the presence of all the tested pollutants. Other frequently isolated species were <i>Pseudomonas fluorescence</i> (four pollutants), <i>Stenotrophomonas maltophilia (</i>two pollutants), <i>Stenotrophomonas acidaminiphila</i> (two pollutants), <i>Bacillus subtilis</i> (two pollutants), and <i>Sphingobacterium multivorum</i> (two pollutants). Three strains of <i>Serratia marcescens</i> were isolated only from the enriched microcosms with the crude oil of the contaminated site (Table <a href="#T1">1</a>). </p> <p>Fungal species with their relative abundance are reported in Supplementary Table <a href="#SM1">S3</a>, according to the soil depth and in Table <a href="#T1">1</a> according to the pollutant used in the enrichments. Isolated fungi belonged mostly to the phylum Ascomycota, and only 5% to Basidiomycota; Mucoromycota were not isolated. Most of the strains were isolated from enrichments inoculated with contaminated soils S1 (35 strains) and S3 (36 strains); a lower number of isolates (23) were retrieved from the contaminated soil S2 (Supplementary Table <a href="#SM1">S3</a>). In general, <i>Fusarium</i> was the most abundant genus (43.8%) followed by <i>Trichoderma</i> (and its teleomorph <i>Hypocrea lixii</i>) (13.8%) and <i>Aspergillus</i> (11.0%). The majority of the isolates detected in soils S1 and S3 were indeed ascribable to these three genera, while in S2 only 44.4% of the strains were affiliated to the genus <i>Fusarium</i> and 16.7% belonged to the genus <i>Cladosporium</i> (Supplementary Table <a href="#SM1">S3</a>). </p> <p>The retrieval of fungal species was influenced by the pollutant used as sole carbon source in the enrichments, as reported in Table <a href="#T1">1</a>. The highest number of isolates was obtained from pyrene and phenanthrene (23-22 isolates) followed by benzene and crude oil (15 isolates); 11 strains were isolated in the presence of naphthalene and only eight strains were isolated in the presence of paraffin oil (Table <a href="#T1">1</a>). With the only exception of crude oil, the genus <i>Fusarium</i> was the most abundant in each pollutant. Benzene and paraffin enrichments revealed the highest variety of genus and species. </p> </div> <div class="molecularanalysesofbacterialandfungalcommunities" title="sec"> <div class="title" tagx="title" title="title">
Molecular Analyses of Bacterial and Fungal Communities</div> <p>Illumina HTS of PCR amplicons of the 16S V3–V4 region resulted in 1,991,866 sequences, which were reduced to 1,742,447 after the exclusion of homopolymers, of sequences &amp;lt;380 bp, sequences not aligning to the targeted V3–V4 regions, chimera and sequences not classified as bacterial after alignment against the RDP 16S training set. In order to reduce biases in diversity estimates related to the analyses of samples with different number of sequences, rarefaction to a common minimum number of 10,065 sequences per sample was carried out ( <a href="#B42">Gihring et al., 2012</a>). The rarefaction step resulted in the loss of only 5 out of 82 samples, while retaining a sequencing depth able to depict most of the bacterial diversity in the samples, as indicated by an average Good’s coverage of 90.4%. </p> <p>As for fungi, Illumina sequencing of ITS2 amplicons resulted in 795,742 high quality sequences. Taxonomy-based analyses were performed in Mothur and consisted in a screening to remove homopolymers, sequences shorter than 120 bp, chimera, and sequences that were not correctly classified as fungal after alignment against the UNITE v6 database. Rarefaction was performed with a common number of 5,001 sequences per sample. The coverage of fungal diversity was not affected by this rarefaction step: average Good’s coverage was 99.7%, thus showing that the sequencing effort covered the totality of fungal biodiversity in the samples. OTU-based analyses for fungi were performed in USEARCH and resulted in a very similar number of sequences per sample (4,942) and a total of 188 fungal OTUs. </p> <p>Multivariate canonical correspondence analysis (CCA) was performed on bacterial and fungal OTUs abundance tables in order to assess how the microbial communities responded to contaminants, depth of soil collection and enrichment time (Figure <a href="#F1">1</a>). Among the tested factors, in the case of bacteria, pollutant was the most significant: the percentage of variance explained (36.1%, Figure <a href="#F1">1A</a>) was much higher than the variance explained by the soil (6.1%, Figure <a href="#F1">1B</a>) and by the time (8.3%, Figure <a href="#F1">1C</a>) factors. The bacterial communities differed among the six tested contaminants: pollutants that shared the chemical structure (e.g., the three PAHs) grouped together, while oil, benzene, and paraffin formed separate groups (Figure <a href="#F1">1A</a>). A similar picture was obtained by the CCA analyses performed on fungal communities: only the pollutant factor was significant (25.6% of variance explained, Figure <a href="#F1">1D</a>) contrary to soil and time factors. The pollutant used in the enrichment significantly affected the evolution of fungal communities: phenanthrene and pyrene enriched fungal communities formed groups that separated from those enriched from oil (Figure <a href="#F1">1D</a>). For both fungi and bacteria, the groups formed by soil-depth (Figures <a href="#F1">1B–E</a>) and time (Figures <a href="#F1">1C–F</a>) were mainly overlapping: this indicates that the depth, as well as the time of enrichment, did not affect the evolution of enriched microbial communities. Grouping of samples was also evaluated by UPGMA clustering of sequencing classified at genus level. Results are reported in Figure <a href="#F2">2</a> for bacteria and in Figure <a href="#F3">3</a> for fungi: samples are labeled according to the soil depth, followed by the pollutant and the enrichment step. Results confirmed that the relative composition in bacterial genera was partly dependent on the pollutant: several pyrene and phenanthrene samples formed two sub-clusters in a common cluster, with <i>Azospirillum</i>, <i>Achromobacter</i>, and <i>Pseudomonas</i> being the dominant genera; naphthalene enrichments were dominated by <i>Achromobacter</i> and <i>Pseudomonas</i> in similar proportions, while oil enrichments showed a dominance of <i>Pseudomonas</i> followed by <i>Achromobacter</i>. Finally, benzene and paraffin enrichments revealed a more diverse community, with high relative presences of <i>Acinetobacter</i>, <i>Pseudoxanthomonas</i>, and <i>Pseudomonas</i>. Neither the soil depth nor the time of enrichment influenced the samples clustering, confirming the CCA results presented above. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">FIGURE 1</span> </a> </div>   <p>Canonical correspondence analyses (CCAs) to test the significance of the effects of pollutant <b>(A,D)</b>, soil depth <b>(B,E),</b> and time <b>(C,F)</b> on the total structure of bacterial and fungal communities as determined by the relative abundances of all the OTUs resulting by Illumina sequencing of bacterial 16S and fungal ITS amplicons. <b>(A–C)</b> Graphs in the first line are for bacteria, <b>(D–F)</b> are for fungi. </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">FIGURE 2</span> </a> </div>   <p>Hierarchical clustering of bacterial 16S sequences classified at the genus level. Only taxa participating with &amp;gt;20% in at least one sample are shown, while taxa with lower participation are grouped in the "other" sequence group. </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">FIGURE 3</span> </a> </div>   <p>Hierarchical clustering of bacterial ITS sequences classified at the genus level. Only taxa participating with &amp;gt;5% in at least one sample are shown, while taxa with lower participation are grouped in the "other" sequence group. </p>   </div> <p>Concerning fungi, no clear clustering of genera was shown according to pollutant, soil depth, or enrichment time (Figure <a href="#F3">3</a>). All samples were dominated by <i>Fusarium</i> (more than 80% in average), followed by <i>Aspergillus</i>, <i>Penicillium</i>, <i>Trichoderma</i>, and <i>Arthrinium</i>. </p> </div> <div class="comparisonbetweenculture-dependentandculture-independentanalyses" title="sec"> <div class="title" tagx="title" title="title">
Comparison Between Culture-Dependent and Culture-Independent Analyses</div> <p>A number of analyses were performed in order to compare the results on the diversity of bacterial and fungal strains isolated with culture-dependent and culture-independent methods. In Table <a href="#T2">2</a> the relative percentages of bacterial genera retrieved from each of the three soil depths are compared. Results generally agree: <i>Pseudomonas</i> was the dominant genus in both cases, with 24.2, 25.2, and 63.5% of relative abundances in Illumina and 57.7, 48.9, and 69.6% as isolates for S1, S2, and S3, respectively. The other genera, in terms of relative percentages, were <i>Bacillus</i>, <i>Achromobacter</i>, <i>Sphingobacterium</i>, and <i>Pseudotrophomonas</i> for the culture-dependent data; <i>Achromobacer</i>, <i>Agrobacterium</i>, <i>Azospirilllum</i>, <i>Shinella</i>, <i>Sphingobacterium</i>, and <i>Pseudotrophomonas</i> for the Illumina data. It is worth noting that for some genera, the soil depth with the highest relative percentage was the same with both approaches. For instance, <i>Pseudomonas</i> reached 60% of abundance in soil three from both the approaches. There are a number of discrepancies: <i>Agrobacterium</i> sequences were above 9% in all soils while no strains belonging to this genus was isolated; and 2.2% of isolates (i.e., two strains) from S2 belong to <i>Helicobacter</i> while no sequences affiliated to this genus were detected with Illumina. </p> <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 2</span>  <p>Relative abundances of bacterial genera determined by Illumina sequencing of 16S amplicns or by isolation on selective media; the comparison was performed at the end of the enrichment (step III). </p>  <table> <thead> <tr> <th>Bacterial genus</th> <th>16S Illumina <hr /> </th> <th>Bacterial isolates <hr /> </th> </tr> <tr> <td /> <th>S1</th> <th>S2</th> <th>S3</th> <th>S1</th> <th>S2</th> <th>S3</th> </tr> </thead> <tbody> <tr> <td> <i>Acholeplasma</i> </td> <td>nd</td> <td>nd</td> <td>nd</td> <td>nd</td> <td>nd</td> <td>4,3</td> </tr> <tr> <td> <i>Achromobacter</i> </td> <td>26.68</td> <td>29.37</td> <td>8.39</td> <td>7.7</td> <td>4.4</td> <td>4.3</td> </tr> <tr> <td> <i>Acinetobacter</i> </td> <td>0.38</td> <td>0.18</td> <td>1.60</td> <td>nd</td> <td>nd</td> <td>4.3</td> </tr> <tr> <td> <i>Agrobacterium</i> </td> <td>9.08</td> <td>11.79</td> <td>11.62</td> <td>nd</td> <td>nd</td> <td>nd</td> </tr> <tr> <td> <i>Ancylobacter</i> </td> <td>0.30</td> <td>2.53</td> <td>0.40</td> <td>nd</td> <td>nd</td> <td>nd</td> </tr> <tr> <td> <i>Azospirillum</i> </td> <td>4.17</td> <td>2.50</td> <td>0.24</td> <td>nd</td> <td>nd</td> <td>nd</td> </tr> <tr> <td> <i>Bacillus</i> </td> <td>0.11</td> <td>0.09</td> <td>0.09</td> <td>7.7</td> <td>8.9</td> <td>nd</td> </tr> <tr> <td> <i>Cellulosimicrobium</i> </td> <td>0.03</td> <td>0.02</td> <td>0.08</td> <td>nd</td> <td>nd</td> <td>4.3</td> </tr> <tr> <td> <i>Cupriavidus</i> </td> <td>4.64</td> <td>0.46</td> <td>0.08</td> <td>nd</td> <td>2.2</td> <td>nd</td> </tr> <tr> <td> <i>Gordonia</i> </td> <td>3.00</td> <td>0.04</td> <td>0.06</td> <td>3.8</td> <td>nd</td> <td>nd</td> </tr> <tr> <td> <i>Helicobacter</i> </td> <td>nd</td> <td>nd</td> <td>nd</td> <td>nd</td> <td>2.2</td> <td>nd</td> </tr> <tr> <td> <i>Klebsiella</i> </td> <td>0.08</td> <td>2.63</td> <td>0.08</td> <td>nd</td> <td>nd</td> <td>nd</td> </tr> <tr> <td> <i>Ochrobactrum</i> </td> <td>2.89</td> <td>0.67</td> <td>0.32</td> <td>3.8</td> <td>nd</td> <td>nd</td> </tr> <tr> <td> <i>Olivibacter</i> </td> <td>1.13</td> <td>6.57</td> <td>1.28</td> <td>nd</td> <td>nd</td> <td>nd</td> </tr> <tr> <td> <i>Paenibacillus</i> </td> <td>0.02</td> <td>0.24</td> <td>0.05</td> <td>nd</td> <td>2.2</td> <td>nd</td> </tr> <tr> <td> <i>Pseudomonas</i> </td> <td>24.21</td> <td>25.25</td> <td>63.49</td> <td>57.7</td> <td>48.9</td> <td>69.6</td> </tr> <tr> <td> <i>Pseudoxanthomonas</i> </td> <td>2.95</td> <td>2.62</td> <td>0.66</td> <td>8.7</td> <td>4.4</td> <td>nd</td> </tr> <tr> <td> <i>Rhizobium</i> </td> <td>nd</td> <td>nd</td> <td>nd</td> <td>nd</td> <td>nd</td> <td>4.3</td> </tr> <tr> <td> <i>Serratia</i> </td> <td>0.00</td> <td>0.24</td> <td>0.01</td> <td>nd</td> <td>6.7</td> <td>nd</td> </tr> <tr> <td> <i>Shinella</i> </td> <td>2.79</td> <td>0.25</td> <td>0.03</td> <td>nd</td> <td>nd</td> <td>nd</td> </tr> <tr> <td> <i>Sphingobacterium</i> </td> <td>0.95</td> <td>0.94</td> <td>1.60</td> <td>3.8</td> <td>17.8</td> <td>nd</td> </tr> <tr> <td> <i>Stenotrophomonas</i> </td> <td>1.23</td> <td>0.77</td> <td>1.42</td> <td>7.7</td> <td>2.2</td> <td>8.7</td> </tr> <tr> <td /> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <span class="attrib" tagx="attrib" title="attrib"> <i>Data are expressed as percentages (among 10,065 sequences for Illumina data, among 96 strains for isolates). </i> </span> </div> </div> <p>The same analyses were conducted on fungi (Table <a href="#T3">3</a>). Sordariomycetes was the most abundant class, with the order Hypocreales (i.e., <i>Acremonium</i>, <i>Clonostachys, Fusarium, Hypocrea/Trichoderma</i>) and Microascales (i.e., <i>Ceriosporopsis, Pseudoallescheria/Scedosporium</i>). Xylariales were instead a minor component of the detected fungi (i.e., <i>Arthrinium, Eutypella</i>). Results from culture-dependent and culture-indepentent methods confirmed each other in the case of the genus <i>Fusarium</i>, whose relative percentages were highly similar (ranging 40 and 69%). <i>Clonotachys</i>, <i>Pseudallescheria</i>, and <i>Trichoderma</i> were mostly isolated from S1 soils where also Illumina data showed the highest relative percentage. In the case of fungi too, there are some contrasting results, as fungal genera isolated in pure culture that were not detected with Illumina sequencing of ITS2 amplicons (e.g., <i>Bjerkandera</i>, <i>Epicoccum</i>, <i>Eutypella, Irpex, Polyporus</i>, and <i>Sulcatispora</i>). As for the genera <i>Ceriosporopsis</i> and <i>Phanerochaete</i>, they were detected with Illumina in significant amounts (above 5% of total sequences) but no isolates were obtained. </p> <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 3</span>  <p>Relative abundances of fungal genera as determined by Illumina sequencing of 16S PCR of amplicons or by isolation on selective media; the comparison was performed at the end of the enrichment (step III). </p>  <table> <thead> <tr> <th>Fungal genus</th> <th>ITS Illumina <hr /> </th> <th>Fungal isolates <hr /> </th> </tr> <tr> <td /> <th>S1</th> <th>S2</th> <th>S3</th> <th>S1</th> <th>S2</th> <th>S3</th> </tr> </thead> <tbody> <tr> <td> <i>Acremonium</i> </td> <td>0.001</td> <td>0.003</td> <td>nd</td> <td>2.9</td> <td>nd</td> <td>nd</td> </tr> <tr> <td> <i>Arthrinium</i> </td> <td>2.0</td> <td>0.4</td> <td>0.3</td> <td>nd</td> <td>nd</td> <td>nd</td> </tr> <tr> <td> <i>Aspergillus</i> </td> <td>1.8</td> <td>2.0</td> <td>0.8</td> <td>8.6</td> <td>8.7</td> <td>19.4</td> </tr> <tr> <td> <i>Aureobasidium</i> </td> <td>0.9</td> <td>0.3</td> <td>0.3</td> <td>2.9</td> <td>nd</td> <td>nd</td> </tr> <tr> <td> <i>Bjerkandera</i> </td> <td>nd</td> <td>nd</td> <td>nd</td> <td>nd</td> <td>4.3</td> <td>nd</td> </tr> <tr> <td> <i>Capnodiales_unclassified</i> </td> <td>12.0</td> <td>11.3</td> <td>3.4</td> <td>nd</td> <td>nd</td> <td>nd</td> </tr> <tr> <td> <i>Ceriosporopsis</i> </td> <td>0.2</td> <td>0.2</td> <td>14.8</td> <td>nd</td> <td>nd</td> <td>nd</td> </tr> <tr> <td> <i>Cladosporium</i> </td> <td>0.2</td> <td>0.6</td> <td>0.1</td> <td>8.6</td> <td>13.0</td> <td>nd</td> </tr> <tr> <td> <i>Clonostachys</i> </td> <td>11.0</td> <td>0.6</td> <td>0.5</td> <td>2.9</td> <td>nd</td> <td>nd</td> </tr> <tr> <td> <i>Epicoccum</i> </td> <td>nd</td> <td>nd</td> <td>nd</td> <td>2.9</td> <td>4.3</td> <td /> </tr> <tr> <td> <i>Eutypella</i> </td> <td>nd</td> <td>nd</td> <td>nd</td> <td>nd</td> <td>4.3</td> <td>2.8</td> </tr> <tr> <td> <i>Fusarium</i> </td> <td>43.8</td> <td>68.8</td> <td>63.4</td> <td>40.0</td> <td>56.5</td> <td>41.7</td> </tr> <tr> <td> <i>Hypocrea</i> </td> <td>nd</td> <td>nd</td> <td>nd</td> <td>nd</td> <td>nd</td> <td>11.1</td> </tr> <tr> <td> <i>Irpex</i> </td> <td>nd</td> <td>nd</td> <td>nd</td> <td>nd</td> <td>4.3</td> <td>nd</td> </tr> <tr> <td> <i>Penicillium</i> </td> <td>0.8</td> <td>2.1</td> <td>1.2</td> <td>5.7</td> <td>nd</td> <td>2.8</td> </tr> <tr> <td> <i>Phanerochaete</i> </td> <td>0.5</td> <td>1.7</td> <td>0.3</td> <td>nd</td> <td>nd</td> <td>nd</td> </tr> <tr> <td> <i>Polyporus</i> </td> <td>nd</td> <td>nd</td> <td>nd</td> <td>nd</td> <td>nd</td> <td>2.8</td> </tr> <tr> <td> <i>Pseudallescheria</i> </td> <td>15.5</td> <td>0.8</td> <td>1.5</td> <td>5.7</td> <td>nd</td> <td>nd</td> </tr> <tr> <td> <i>Scedosporium</i> </td> <td>0.01</td> <td>0.01</td> <td>0.12</td> <td>nd</td> <td>4.3</td> <td>5.6</td> </tr> <tr> <td> <i>Sulcatispora</i> </td> <td>nd</td> <td>nd</td> <td>nd</td> <td>nd</td> <td>nd</td> <td>2.8</td> </tr> <tr> <td> <i>Trametes</i> </td> <td>nd</td> <td>0.1</td> <td>0.04</td> <td>nd</td> <td>nd</td> <td>2.8</td> </tr> <tr> <td> <i>Saccharomyces</i> </td> <td>0.9</td> <td>0.7</td> <td>0.1</td> <td>nd</td> <td>nd</td> <td>nd</td> </tr> <tr> <td> <i>Trichoderma</i> </td> <td>3.6</td> <td>0.4</td> <td>0.5</td> <td>20.0</td> <td>nd</td> <td>5.6</td> </tr> <tr> <td> <i>Wallemia</i> </td> <td>0.02</td> <td>nd</td> <td>0.02</td> <td>nd</td> <td>nd</td> <td>2.8</td> </tr> <tr> <td> <i>Trichosporon</i> </td> <td>1.2</td> <td>3.2</td> <td>4.4</td> <td>nd</td> <td>nd</td> <td>nd</td> </tr> <tr> <td /> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <span class="attrib" tagx="attrib" title="attrib"> <i>Data are expressed as percentages (among 10065 sequences for Illumina data, among 94 strains for isolates). </i> </span> </div> </div> <p>The most abundant OTUs (i.e., those who had a relative abundance higher than 1% in at least one sample) are reported in Tables <a href="#T4">4</a>, <a href="#T5">5</a> for bacteria and fungi, respectively. This screening criterion retained 19 OTUs for bacteria, and 21 OTUs for fungi: in both tables, the highest taxonomic affiliation is reported, together with the relative abundances in each enrichment microcosm. In the case of bacteria, most OTUs (16 out of 19) were classified at species level, two at genus and only one at the family level. Nine out of 19 OTUs were retrieved in the cultivated isolates. By comparing the data on the isolates in Table <a href="#T1">1</a> with the molecular data in Table <a href="#T4">4</a>, it is possible to notice that the species comprising more isolates ( <i>Pseudomonas putida</i>, <i>Achromobacter xylosoxidans</i>, and <i>Pseudomonas veronii</i>) were also the most abundant in terms of Illumina OTUs. As above reported for genera, also for OTUs classified at genus or species level there was a good agreement between molecular and microbiological data. </p> <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 4</span>  <p>Relative percentages of most abundant bacterial OTUs after the last enrichment step for each pollutant and at each of the three soil depths tested. </p>  <table> <thead> <tr> <th>OTU nr</th> <th>Taxonomy</th> <th>Benzene <hr /> </th> <th>Naphtalene <hr /> </th> <th>Oil <hr /> </th> <th>Paraffin <hr /> </th> <th>Phenanthrene <hr /> </th> <th>Pyrene <hr /> </th> </tr> <tr> <td /> <td /> <th>S1</th> <th>S2</th> <th>S3</th> <th>S1</th> <th>S2</th> <th>S3</th> <th>S1</th> <th>S2</th> <th>S3</th> <th>S1</th> <th>S2</th> <th>S1</th> <th>S2</th> <th>S1</th> <th>S2</th> <th>S3</th> </tr> </thead> <tbody> <tr> <td>OTU001</td> <td> <i>Pseudomonas putida <sup>∗</sup> </i> </td> <td>5.2</td> <td>0.9</td> <td>2.2</td> <td>26.1</td> <td>37.1</td> <td>73.6</td> <td>6.0</td> <td>1.6</td> <td>58.3</td> <td>0.3</td> <td>0.2</td> <td>10.6</td> <td>20.3</td> <td>0.6</td> <td>0.4</td> <td>65.8</td> </tr> <tr> <td>OTU002</td> <td> <i>Achromobacter xylosoxidans <sup>∗</sup> </i> </td> <td>7.0</td> <td>2.9</td> <td>4.8</td> <td>46.4</td> <td>20.7</td> <td>4.1</td> <td>10.2</td> <td>20.3</td> <td>6.0</td> <td>8.5</td> <td>11.1</td> <td>23.5</td> <td>23.4</td> <td>26.4</td> <td>51.5</td> <td>6.5</td> </tr> <tr> <td>OTU003</td> <td> <i>Pseudomonas stutzeri</i> </td> <td>0.1</td> <td>0.1</td> <td>0.1</td> <td>0.4</td> <td>0.9</td> <td>0.8</td> <td>1.0</td> <td>0.9</td> <td>5.4</td> <td>0.2</td> <td>0.1</td> <td>0.6</td> <td>1.5</td> <td>0.9</td> <td>0.9</td> <td>1.3</td> </tr> <tr> <td>OTU004</td> <td> <i>Pseudomonas fluorescens <sup>∗</sup> </i> </td> <td>0.1</td> <td>0.1</td> <td>0.0</td> <td>1.9</td> <td>4.4</td> <td>3.7</td> <td>16.4</td> <td>20.7</td> <td>9.7</td> <td>0.2</td> <td>0.3</td> <td>1.1</td> <td>3.5</td> <td>0.3</td> <td>0.6</td> <td>2.1</td> </tr> <tr> <td>OTU005</td> <td> <i>Rhizobium petrolearium <sup>∗</sup> </i> </td> <td>50.1</td> <td>31.5</td> <td>44.9</td> <td>0.1</td> <td>0.3</td> <td>0.2</td> <td>0.1</td> <td>0.1</td> <td>0.2</td> <td>1.8</td> <td>4.3</td> <td>0.1</td> <td>0.3</td> <td>0.3</td> <td>0.2</td> <td>0.6</td> </tr> <tr> <td>OTU006</td> <td> <i>Acinetobacter calcoaceticus <sup>∗</sup> </i> </td> <td>1.1</td> <td>0.0</td> <td>4.8</td> <td>0.1</td> <td>0.1</td> <td>0.0</td> <td>0.1</td> <td>0.1</td> <td>0.1</td> <td>0.0</td> <td>0.1</td> <td>0.1</td> <td>0.1</td> <td>0.1</td> <td>0.0</td> <td>0.1</td> </tr> <tr> <td>OTU008</td> <td> <i>Azospirillum lipoferum</i> </td> <td>0.0</td> <td>0.0</td> <td>0.0</td> <td>0.0</td> <td>0.0</td> <td>0.1</td> <td>0.0</td> <td>0.0</td> <td>0.0</td> <td>0.0</td> <td>0.1</td> <td>0.1</td> <td>10.7</td> <td>4.6</td> <td>0.4</td> <td>0.1</td> </tr> <tr> <td>OTU009</td> <td> <i>Unclassified Comanomonadacea</i> </td> <td>0.0</td> <td>0.0</td> <td>0.0</td> <td>0.0</td> <td>0.0</td> <td>0.1</td> <td>6.0</td> <td>12.2</td> <td>0.0</td> <td>0.2</td> <td>0.3</td> <td>0.0</td> <td>0.0</td> <td>0.0</td> <td>0.0</td> <td>0.0</td> </tr> <tr> <td>OTU010</td> <td> <i>Ochrobactrum anthropi <sup>∗</sup> </i> </td> <td>0.8</td> <td>0.2</td> <td>0.9</td> <td>0.0</td> <td>0.0</td> <td>0.0</td> <td>0.0</td> <td>0.0</td> <td>0.0</td> <td>11.9</td> <td>15.4</td> <td>0.2</td> <td>2.3</td> <td>1.7</td> <td>0.0</td> <td>0.0</td> </tr> <tr> <td>OTU011</td> <td> <i>Rhizobium radiobacter</i> </td> <td>0.5</td> <td>4.8</td> <td>1.8</td> <td>0.2</td> <td>0.3</td> <td>0.0</td> <td>0.0</td> <td>0.0</td> <td>0.0</td> <td>0.9</td> <td>3.8</td> <td>1.8</td> <td>1.7</td> <td>1.4</td> <td>19.1</td> <td>0.0</td> </tr> <tr> <td>OTU013</td> <td> <i>Cupriavidus necator</i> </td> <td>0.6</td> <td>1.2</td> <td>0.2</td> <td>0.1</td> <td>0.0</td> <td>0.0</td> <td>0.0</td> <td>0.0</td> <td>0.0</td> <td>1.2</td> <td>0.3</td> <td>0.1</td> <td>0.0</td> <td>20.3</td> <td>0.0</td> <td>0.0</td> </tr> <tr> <td>OTU014</td> <td> <i>Gordonia alkanivorans</i> </td> <td>0.0</td> <td>0.0</td> <td>0.0</td> <td>0.0</td> <td>0.0</td> <td>0.1</td> <td>0.1</td> <td>0.0</td> <td>0.0</td> <td>16.8</td> <td>2.2</td> <td>0.1</td> <td>0.1</td> <td>0.0</td> <td>0.0</td> <td>0.1</td> </tr> <tr> <td>OTU015</td> <td> <i>Pseudomonas stutzeri <sup>∗</sup> </i> </td> <td>0.0</td> <td>0.0</td> <td>0.0</td> <td>0.0</td> <td>0.0</td> <td>0.0</td> <td>24.2</td> <td>0.0</td> <td>0.0</td> <td>0.0</td> <td>0.0</td> <td>0.0</td> <td>0.1</td> <td>0.0</td> <td>0.1</td> <td>0.0</td> </tr> <tr> <td>OTU016</td> <td> <i>Sphingobacterium</i> spp. <i> <sup>∗</sup> </i> </td> <td>1.3</td> <td>20.2</td> <td>4.6</td> <td>0.0</td> <td>0.0</td> <td>0.0</td> <td>0.0</td> <td>0.0</td> <td>0.0</td> <td>0.4</td> <td>0.0</td> <td>0.0</td> <td>7.3</td> <td>0.0</td> <td>2.7</td> <td>0.0</td> </tr> <tr> <td>OTU017</td> <td> <i>Bordetella</i> spp. </td> <td>5.1</td> <td>0.0</td> <td>1.8</td> <td>0.1</td> <td>0.6</td> <td>0.0</td> <td>0.0</td> <td>0.0</td> <td>0.0</td> <td>0.5</td> <td>2.9</td> <td>0.1</td> <td>0.5</td> <td>0.2</td> <td>0.2</td> <td>0.0</td> </tr> <tr> <td>OTU018</td> <td> <i>Pseudoxanthomonas mexicana <sup>∗</sup> </i> </td> <td>2.7</td> <td>1.1</td> <td>1.6</td> <td>0.0</td> <td>1.1</td> <td>0.1</td> <td>0.0</td> <td>0.0</td> <td>0.0</td> <td>0.8</td> <td>0.3</td> <td>2.9</td> <td>2.9</td> <td>5.4</td> <td>2.8</td> <td>0.1</td> </tr> <tr> <td>OTU021</td> <td> <i>Sinorhizobium</i> spp. </td> <td>2.0</td> <td>2.2</td> <td>1.2</td> <td>0.0</td> <td>0.2</td> <td>0.0</td> <td>0.0</td> <td>0.0</td> <td>0.0</td> <td>1.5</td> <td>7.0</td> <td>0.0</td> <td>0.1</td> <td>0.2</td> <td>0.1</td> <td>0.0</td> </tr> <tr> <td>OTU023</td> <td> <i>Xanthobacter flavus</i> </td> <td>0.0</td> <td>0.1</td> <td>0.1</td> <td>0.0</td> <td>0.0</td> <td>0.0</td> <td>0.0</td> <td>0.0</td> <td>0.0</td> <td>5.6</td> <td>21.1</td> <td>0.0</td> <td>0.1</td> <td>0.0</td> <td>0.1</td> <td>0.0</td> </tr> <tr> <td>OTU024</td> <td> <i>Pseudomonas resinovorans</i> </td> <td>0.1</td> <td>0.3</td> <td>0.0</td> <td>0.5</td> <td>0.0</td> <td>0.1</td> <td>0.4</td> <td>0.5</td> <td>2.1</td> <td>0.0</td> <td>0.0</td> <td>12.0</td> <td>0.2</td> <td>1.0</td> <td>0.6</td> <td>0.1</td> </tr> <tr> <td>OTU025</td> <td> <i>Azospirillum thiophilum</i> </td> <td>0.0</td> <td>0.0</td> <td>0.0</td> <td>0.0</td> <td>0.0</td> <td>0.0</td> <td>0.0</td> <td>0.0</td> <td>0.0</td> <td>0.0</td> <td>0.0</td> <td>17.7</td> <td>0.0</td> <td>0.0</td> <td>0.0</td> <td>0.5</td> </tr> <tr> <td /> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <span class="attrib" tagx="attrib" title="attrib"> <i>OTUs that represent more than 1% of total bacterial community in at least one sample are presented. OTUs that were also detected among isolates are highlighted with an asterisk. </i> </span> </div> </div> <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 5</span>  <p>Relative percentages of most abundant fungal OTUs after the last enrichment step for each pollutant and at each of the three soil depths tested. OTUs that represent more than 1% of total fungal community in at least one sample are presented. </p>  <table> <thead> <tr> <th>OTU nr</th> <th>Taxonomy</th> <th>Benzene <hr /> </th> <th>Naphtalene <hr /> </th> <th>Oil <hr /> </th> <th>Paraffin <hr /> </th> <th>Pheanthrene <hr /> </th> <th>Pyrene <hr /> </th> </tr> <tr> <td /> <td /> <th>S1</th> <th>S2</th> <th>S3</th> <th>S1</th> <th>S2</th> <th>S3</th> <th>S1</th> <th>S2</th> <th>S3</th> <th>S1</th> <th>S2</th> <th>S3</th> <th>S1</th> <th>S2</th> <th>S3</th> <th>S1</th> <th>S2</th> </tr> </thead> <tbody> <tr> <td>OTU002</td> <td> <i>Fusarium oxysporum <sup>∗</sup> </i> </td> <td>3.5</td> <td>39.1</td> <td>16.6</td> <td>29.4</td> <td>23.4</td> <td>36.3</td> <td>4.7</td> <td>4.6</td> <td>3.3</td> <td>9.8</td> <td>15.3</td> <td>13.4</td> <td>39.3</td> <td>56.4</td> <td>63.5</td> <td>56.4</td> <td>78.3</td> </tr> <tr> <td>OTU003</td> <td> <i>Cladosporium</i> spp. <i> <sup>∗</sup> </i> </td> <td>1.3</td> <td>7.5</td> <td>2.4</td> <td>5.6</td> <td>2.5</td> <td>6.4</td> <td>71.2</td> <td>61.8</td> <td>8.4</td> <td>1.1</td> <td>1.4</td> <td>1.8</td> <td>1.5</td> <td>1.7</td> <td>1.4</td> <td>2.0</td> <td>3.4</td> </tr> <tr> <td>OTU004</td> <td> <i>Fusarium</i> spp. <sup>∗</sup> </td> <td>0.9</td> <td>5.1</td> <td>2.1</td> <td>5.5</td> <td>5.8</td> <td>6.5</td> <td>1.3</td> <td>3.5</td> <td>0.9</td> <td>9.8</td> <td>64.9</td> <td>30.2</td> <td>35.9</td> <td>26.1</td> <td>2.0</td> <td>2.9</td> <td>3.4</td> </tr> <tr> <td>OTU001</td> <td> <i>Fusarium solani <sup>∗</sup> </i> </td> <td>2.6</td> <td>21.8</td> <td>64.3</td> <td>13.2</td> <td>22.6</td> <td>15.1</td> <td>2.4</td> <td>10.1</td> <td>1.8</td> <td>12.7</td> <td>4.4</td> <td>51.2</td> <td>18.5</td> <td>11.7</td> <td>7.6</td> <td>7.9</td> <td>7.6</td> </tr> <tr> <td>OTU007</td> <td> <i>Unidentified Halosphaeriacea</i> </td> <td>0.2</td> <td>0.0</td> <td>0.0</td> <td>0.2</td> <td>0.2</td> <td>0.2</td> <td>0.2</td> <td>0.2</td> <td>80.4</td> <td>0.1</td> <td>0.2</td> <td>0.1</td> <td>0.2</td> <td>0.3</td> <td>0.2</td> <td>0.2</td> <td>0.2</td> </tr> <tr> <td>OTU014</td> <td> <i>Fusarium merismoides</i> </td> <td>0.0</td> <td>0.0</td> <td>0.0</td> <td>0.2</td> <td>0.1</td> <td>0.4</td> <td>0.1</td> <td>0.1</td> <td>0.0</td> <td>0.1</td> <td>0.1</td> <td>0.2</td> <td>0.2</td> <td>0.2</td> <td>20.2</td> <td>0.4</td> <td>0.1</td> </tr> <tr> <td>OTU005</td> <td> <i>Pseudallescheria</i> spp. </td> <td>89.0</td> <td>0.7</td> <td>6.6</td> <td>1.1</td> <td>1.5</td> <td>0.6</td> <td>0.6</td> <td>0.5</td> <td>0.3</td> <td>0.7</td> <td>1.0</td> <td>0.9</td> <td>0.7</td> <td>0.6</td> <td>0.8</td> <td>1.0</td> <td>0.9</td> </tr> <tr> <td>OTU008</td> <td> <i>Apiospora montagnei</i> </td> <td>0.2</td> <td>0.1</td> <td>0.1</td> <td>5.9</td> <td>0.7</td> <td>0.6</td> <td>0.3</td> <td>0.4</td> <td>0.1</td> <td>0.2</td> <td>0.3</td> <td>0.4</td> <td>0.5</td> <td>0.3</td> <td>0.4</td> <td>5.4</td> <td>0.8</td> </tr> <tr> <td>OTU012</td> <td> <i>Aspergillus flavus</i> </td> <td>0.1</td> <td>2.8</td> <td>0.2</td> <td>5.4</td> <td>3.2</td> <td>1.6</td> <td>0.1</td> <td>0.2</td> <td>0.0</td> <td>0.0</td> <td>0.1</td> <td>0.1</td> <td>0.1</td> <td>0.1</td> <td>0.1</td> <td>0.6</td> <td>0.2</td> </tr> <tr> <td>OTU030</td> <td> <i>Saccharomyces cerevisiae</i> </td> <td>0.1</td> <td>0.2</td> <td>0.1</td> <td>0.7</td> <td>1.8</td> <td>0.2</td> <td>3.6</td> <td>1.8</td> <td>0.2</td> <td>0.1</td> <td>0.1</td> <td>0.0</td> <td>0.1</td> <td>0.1</td> <td>0.1</td> <td>0.6</td> <td>0.1</td> </tr> <tr> <td>OTU009</td> <td> <i>Penicillium</i> spp. <i> <sup>∗</sup> </i> </td> <td>0.1</td> <td>4.6</td> <td>0.8</td> <td>1.3</td> <td>1.5</td> <td>2.7</td> <td>0.2</td> <td>1.9</td> <td>0.2</td> <td>0.1</td> <td>0.1</td> <td>0.3</td> <td>0.1</td> <td>0.1</td> <td>0.1</td> <td>0.3</td> <td>0.2</td> </tr> <tr> <td>OTU010</td> <td> <i>Trichoderma harzianum <sup>∗</sup> </i> </td> <td>0.2</td> <td>0.2</td> <td>0.1</td> <td>5.1</td> <td>0.7</td> <td>0.6</td> <td>0.2</td> <td>0.3</td> <td>0.1</td> <td>0.3</td> <td>0.3</td> <td>0.3</td> <td>0.5</td> <td>0.3</td> <td>0.8</td> <td>16.0</td> <td>0.7</td> </tr> <tr> <td>OTU006</td> <td> <i>Fusarium cuneirostrum</i> </td> <td>0.1</td> <td>0.1</td> <td>0.1</td> <td>0.5</td> <td>0.3</td> <td>0.3</td> <td>0.3</td> <td>1.9</td> <td>1.4</td> <td>64.0</td> <td>0.3</td> <td>0.2</td> <td>0.2</td> <td>0.5</td> <td>0.4</td> <td>0.4</td> <td>0.4</td> </tr> <tr> <td>OTU028</td> <td> <i>Aureobasidium pullulans <sup>∗</sup> </i> </td> <td>0.0</td> <td>0.5</td> <td>0.6</td> <td>4.0</td> <td>1.2</td> <td>0.4</td> <td>0.0</td> <td>0.1</td> <td>0.1</td> <td>0.0</td> <td>0.0</td> <td>0.0</td> <td>0.0</td> <td>0.1</td> <td>0.0</td> <td>0.5</td> <td>0.3</td> </tr> <tr> <td>OTU024</td> <td> <i>Trichosporon</i> spp. </td> <td>0.1</td> <td>0.7</td> <td>0.0</td> <td>3.1</td> <td>3.4</td> <td>4.5</td> <td>0.0</td> <td>0.0</td> <td>0.0</td> <td>0.0</td> <td>0.1</td> <td>0.0</td> <td>0.1</td> <td>0.1</td> <td>0.0</td> <td>0.1</td> <td>0.1</td> </tr> <tr> <td>OTU019</td> <td> <i>Galactomyces geotrichum</i> </td> <td>0.0</td> <td>0.1</td> <td>0.2</td> <td>0.5</td> <td>3.3</td> <td>0.1</td> <td>0.0</td> <td>0.0</td> <td>0.0</td> <td>0.0</td> <td>0.0</td> <td>0.0</td> <td>0.0</td> <td>0.0</td> <td>0.0</td> <td>0.0</td> <td>0.0</td> </tr> <tr> <td>OTU021</td> <td> <i>Cadophora malorum</i> </td> <td>0.0</td> <td>0.1</td> <td>0.3</td> <td>1.3</td> <td>0.1</td> <td>0.3</td> <td>3.5</td> <td>1.0</td> <td>0.1</td> <td>0.1</td> <td>1.8</td> <td>0.0</td> <td>0.1</td> <td>0.1</td> <td>0.1</td> <td>0.1</td> <td>0.0</td> </tr> <tr> <td>OTU011</td> <td> <i>Trichosporon loubieri</i> </td> <td>0.0</td> <td>1.8</td> <td>0.1</td> <td>3.0</td> <td>10.3</td> <td>16.2</td> <td>0.1</td> <td>0.1</td> <td>0.1</td> <td>0.1</td> <td>0.2</td> <td>0.0</td> <td>0.1</td> <td>0.1</td> <td>0.1</td> <td>0.3</td> <td>0.2</td> </tr> <tr> <td>OTU018</td> <td> <i>Phanerochaete chrysosporium</i> </td> <td>1.0</td> <td>7.3</td> <td>0.5</td> <td>1.5</td> <td>2.7</td> <td>0.8</td> <td>0.7</td> <td>0.0</td> <td>0.5</td> <td>0.0</td> <td>0.0</td> <td>0.0</td> <td>0.1</td> <td>0.1</td> <td>0.1</td> <td>0.2</td> <td>0.2</td> </tr> <tr> <td>OTU114</td> <td> <i>Fusarium solani <sup>∗</sup> </i> </td> <td>0.0</td> <td>0.1</td> <td>0.1</td> <td>0.2</td> <td>3.9</td> <td>0.0</td> <td>0.0</td> <td>0.0</td> <td>0.0</td> <td>0.0</td> <td>0.0</td> <td>0.0</td> <td>0.0</td> <td>0.0</td> <td>0.0</td> <td>0.0</td> <td>0.0</td> </tr> <tr> <td>OTU132</td> <td> <i>Fusarium</i> spp. <i> <sup>∗</sup> </i> </td> <td>0.1</td> <td>0.1</td> <td>0.0</td> <td>0.2</td> <td>0.1</td> <td>0.1</td> <td>0.0</td> <td>0.0</td> <td>0.1</td> <td>0.1</td> <td>1.0</td> <td>0.0</td> <td>0.2</td> <td>0.2</td> <td>1.2</td> <td>3.1</td> <td>0.1</td> </tr> <tr> <td>OTU033</td> <td> <i>Hypocrea gamsii</i> </td> <td>0.0</td> <td>0.0</td> <td>0.0</td> <td>0.1</td> <td>0.1</td> <td>0.0</td> <td>2.5</td> <td>0.0</td> <td>0.1</td> <td>0.0</td> <td>0.1</td> <td>0.0</td> <td>0.1</td> <td>0.0</td> <td>0.0</td> <td>0.0</td> <td>0.0</td> </tr> <tr> <td /> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <span class="attrib" tagx="attrib" title="attrib"> <i>OTUs that were also detected among isolates are highlighted with an asterisk.</i> </span> </div> </div> <p>Regarding fungi, the rate of OTUs that could be classified at species level is similar to the one for bacteria: 14 out of 21. The number of OTUs also detected among isolates was instead slightly lower, 9 isolates (Table <a href="#T5">5</a>). It is worth noting that, as for bacteria, the taxa comprising more isolates were also the most abundant OTUs: that was the case for <i>Fusarium oxysporum</i>, <i>Cladosporium</i> spp., and <i>Fusarium solani</i>. These data highlighted the different adaptation abilities developed by fungi: while species of the genus <i>Fusarium</i> were abundant and widely distributed among pollutants and depth, other fungi seemed to be particularly adapted to individual contaminant. For instance, the relative percentage of <i>Cladosporium</i> spp. and <i>Pseudoallescheria</i> spp. was significantly high only in the presence of oil and benzene, respectively. </p> </div> </div> <div class="discussion" title="sec"> <div class="title" tagx="title" title="title">
Discussion</div> <p>Since the discovery of the great plate count anomaly in the middle ‘80s of the last century ( <a href="#B93">Staley and Konopka, 1985</a>), microbiologists are facing the issue of non-cultivable microorganisms and the underestimation of the actual microbial community by means of direct isolation techniques. This has become more and more relevant with the advent of HTS techniques, and it is of utmost importance in the environmental microbiology sector of bioremediation, where studies must rely on the isolation and identification of bacterial and fungal strains to be inoculated in soil for efficient degradation of target pollutants ( <a href="#B65">Megharaj et al., 2011</a>). These strains are usually obtained through enrichment procedures, aimed at isolating autochthonous microorganisms capable of using the pollutants as sole carbon sources ( <a href="#B95">Thompson et al., 2005</a>). Although many data are available in literature about the assessment of microbial community dynamics following the addition of enriched microbial strains ( <a href="#B59">MacNaughton et al., 1999</a>; <a href="#B1">Aburto-Medina et al., 2012</a>; <a href="#B39">Fuentes et al., 2014</a>), little is known on the developments that occur within the bacterial and fungal communities during the enrichment steps. Furthermore, to the best of our knowledge, no attempt was made to assess how soil samples with a high petroleum hydrocarbons content and collected at different depths affect the organization of the bacterial and fungal communities. </p> <p>In this work, we assessed whether the pollutant and the collection depth had significant effects on the microbial community arrangements. Enrichments were performed using five pollutants (three PAHs, benzene, and paraffin) and a complex mixture (i.e., crude oil collected from the polluted site). The results obtained (Figure <a href="#F1">1</a>) highlighted that for both bacteria and fungi, the pollutant partially shaped the enriched communities. Although it was difficult to highlight separate clusters according to the pollutants, some differences were observed as bacteria grown in the presence of paraffin and benzene or with PAHs, and fungi grown with paraffin or PAHs (Figure <a href="#F1">1</a>). This is not surprising, if we consider that different metabolic pathways are involved in the degradation of aliphatic or aromatic hydrocarbons, but that single strains can be also equipped with both pathways ( <a href="#B107">Whyte et al., 1997</a>). The microbial community responded uniquely to the presence of crude oil: only those strains, highly adapted to this extreme and toxic environment, were capable of colonizing this ecological niche. </p> <p>According to literature, a relation between pollutants load and microbial communities can be drawn. Several studies assessed a different distribution of PAHs and other oil pollutants along soil depths ( <a href="#B21">Cousins et al., 1999</a>; <a href="#B76">Ping et al., 2007</a>). It is well-known that microbial community organization is affected by soil depths, commonly causing a decline in biomass and diversity from the surface downward ( <a href="#B16">Blume et al., 2002</a>; <a href="#B36">Fierer et al., 2003</a>; <a href="#B73">Oehl et al., 2005</a>). As far as we know, no experiments were previously conducted to assess the impact of the soil samples depths on the community structure and enrichment outcomes. Contrary to this general background, in the present study, neither the soil depth nor the enrichment step had significant effects on the community structure (Figures <a href="#F1">1</a>– <a href="#F3">3</a>). Regarding fungi, this could be explained by their physiological growing capabilities. Thanks to their hyphae growth, they can widely colonize the soil, creating a homogenous mycoflora in the space ( <a href="#B49">Hesham et al., 2017</a>). </p> <p>The total load of bacteria and fungi was lower than unpolluted soils (Supplementary Table <a href="#SM1">S1</a>), although no differences were observed among the collection depths: a strong and long-lasting pollution of this site may shape a microbial community adapted to the pollutants pressure mostly composed by oil-utilizing strains. As illustrated in Figures <a href="#F2">2</a>, <a href="#F3">3</a>, differences among samples are more evident for bacteria. However, it is not possible to define clear clusters based on soil depths; the structure of the microbial community was not dependent by the collection point. Since the microbial community is selected by the total content of the pollutants, we can speculate that, in this soil, a pollutants threshold has been reached in the surface topsoil: beyond this level, only the selected and adapted microflora can survive. Indeed, in a less polluted site, the presence of both aerobic and anaerobic microorganisms decreased with the vertical soil profile ( <a href="#B15">Biró et al., 2014</a>). Regarding the enrichment steps, it was surprising to find that already after 1 week of enrichment, the communities were similar to those found after 4 weeks. This indicates that when dealing with strong polluted soils, as in this study, a single enrichment step may be enough. </p> <p>A total of 95 bacterial and 94 fungal strains were identified. These strains were well-distributed along soil depths and pollutant (Supplementary Tables <a href="#SM1">S2</a>, <a href="#SM1">S3</a>). The taxonomical identification of bacteria confirmed the selection of specific bacterial communities following the addition of different hydrocarbons, indicating a high specialization of bacterial taxa involved in their degradation. <i>Proteobacteria, Firmicutes</i>, and <i>Sphingobacteria</i> were the prevailing bacterial phyla: all these phyla comprised several phylogenetic groups involved in the aerobic degradation of hydrocarbons ( <a href="#B102">Vinas et al., 2005</a>; <a href="#B48">Head et al., 2006</a>; <a href="#B114">Yang et al., 2012</a>; <a href="#B115">Yergeau et al., 2012</a>). <i>Proteobacteria</i> was also found as the most abundant phylum distributed in an environment contaminated with petroleum muck and in activated biomass from a petrochemical industry wastewater sample ( <a href="#B51">Joshi et al., 2014</a>; <a href="#B112">Yadav et al., 2015</a>). This phylum is divided into five major classes, all with oil-degrading genera ( <a href="#B40">Gao et al., 2014</a>). On the whole, isolated bacteria where ascribed mainly to genera belonging to Gamma-proteobacteria class, whose dominance within the enriched microcosms can be expected since it comprises many of the hydrocarbonoclastic bacteria responsible for the first steps of hydrocarbons degradation ( <a href="#B48">Head et al., 2006</a>; <a href="#B113">Yakimov et al., 2007</a>; <a href="#B11">Barbato et al., 2016</a>). </p> <p>Bacterial species of the genera <i>Sphingobacterium, Achromobacter, Pseudomonas, Stenotrophomonas</i>, and <i>Bacillus</i>, isolated from this crude-oil contaminated soil, showed close lineage with previously reported hydrocarbons degrading members of same genera and most of them have been reported earlier as hydrocarbon degraders ( <a href="#B109">Widdel and Rabus, 2001</a>; <a href="#B84">Roy et al., 2002</a>; <a href="#B37">Foght, 2008</a>; <a href="#B20">Chandra et al., 2013</a>; <a href="#B97">Varjani et al., 2015</a>; <a href="#B98">Varjani and Upasani, 2016</a>). Members of the genus <i>Pseudomonas</i> are widespread environmental microorganisms isolated from a variety of natural sources ( <a href="#B100">Velmurugan et al., 2011</a>) and well-known biodegraders, capable of metabolizing a range of compounds ( <a href="#B54">Kostka et al., 2011</a>; <a href="#B69">Mulet et al., 2011</a>; <a href="#B60">Mahjoubi et al., 2013</a>). </p> <p>According to our results, the PAHs-degrading isolates in soil belong to the <i>Sphingomonas</i> and <i>Pseudomonas</i> bacteria. <i>Pseudomonas</i> usually contain <i>nah</i>-like genes that encode for the dioxygenase subfamily; the products of these genes are capable of degrading low-molecular- weight PAHs ( <a href="#B50">Johnsen et al., 2007</a>) and producing a rhamnolipid biosurfactant in soils contaminated with petroleum hydrocarbons ( <a href="#B85">Sadouk et al., 2008</a>; <a href="#B111">Xia et al., 2014</a>). Unlike other gram-negative bacteria strains, members of the genus <i>Sphingomonas</i> are able to degrade a wide range of natural and xenobiotic compounds ( <a href="#B34">Eguchi et al., 1996</a>; <a href="#B77">Pinhassi and Hagstrom, 2000</a>). Three strains of <i>Serratia marcescens</i> were also isolated from the enriched microcosms with the crude oil of the contaminated site, supporting previous evidences of degradation of crude oil and petroleum products by species of <i>Serratia</i> ( <a href="#B25">De La Fuente et al., 1991</a>; <a href="#B82">Rojas-Avelizapa et al., 2002</a>; <a href="#B110">Wongsa et al., 2004</a>; <a href="#B80">Rajasekar et al., 2007</a>). </p> <p>Generally, fungi involved in the degradation of PAHs include ligninolytic and non-ligninolytic fungi. Most of the strains isolated after the enrichment steps belonged to the phylum Ascomycota, although few Basidiomycota were isolated. The dominance of Ascomycota in polluted soil has been extensively acknowledged: <a href="#B119">Zhang et al. (2018)</a> reported that Ascomycota represented up to 73–96% of the total 18S sequences of a coking area soil; eight of the 10 strains isolated from a PAHs contaminated pond were Ascomycetes and were also able to remove anthracene ( <a href="#B7">Aranda et al., 2017</a>). Finally, an oilfield with a history of 50 years pollution influenced mainly the bacterial community, whereas fungi, mostly related to this phylum, were abundant, demonstrating to be less sensitive to soil PAHs ( <a href="#B120">Zhou et al., 2017</a>). </p> <p>Basidiomycota represented only the 5.3% of the total isolates. Similarly, in a toxic coking area, only a small fraction of the isolates belonged to Basidiomycota (0.34–7.0%) ( <a href="#B119">Zhang et al., 2018</a>). Besides <i>Bjerkandera adusta, Irpex lacteus, Wallemia mellicola, Polyporus gayanus, Trametes gibbosa </i>, the presence of species of <i>Lopharia, Phanerochaete</i>, and <i>Trichosporon</i> was assessed by HTS. Among them, <i>T. gibbosa, P. gayanus</i>, and <i>W. mellicola</i> were isolated for the first time in hydrocarbons polluted soil. </p> <p>On the other hand, Mucoromycota were absent in this contaminated soil. Unlike <a href="#B67">Morais et al. (2016)</a> who found more than 50% of Mucoromycota in a petroleum contaminated costal site, we did not identified any organism belonging to this phylum. Mucoromycota represented a small fraction of the isolates also in an estuarine sediment contaminated mainly by PAHs ( <a href="#B23">Da Silva et al., 2003</a>) and in heavy crude oil-contaminated soil ( <a href="#B118">Zafra et al., 2014</a>). In the present study, Mucoromycota could be included in the "unclassified" group (Figure <a href="#F3">3</a>), even though they would be a negligible fraction of the mycoflora. </p> <p>All samples were dominated by <i>Fusarium</i>, followed by <i>Aspergillus</i>, <i>Penicillium</i>, and <i>Trichoderma</i>. The identification of these genera as polluted soil inhabitants has been already discussed elsewhere. Sordariomycetes is the most represented class with <i>Acremonium, Arthrinium, Clonostachys, Eutypella, Fusarium, Hypocrea/Trichoderma</i>, and <i>Pseudoallescheria/Scedosporium</i>. Fungal taxonomy analysis of an oil-contaminated soil detected almost 25% of Sordariomyecetes ( <a href="#B67">Morais et al., 2016</a>). <i>Trichoderma harzianum</i> ( <a href="#B23">Da Silva et al., 2003</a>), <i>Scedosporium apiospermum</i> and <i>Acremonium</i> sp. ( <a href="#B19">Chaıneau et al., 1999</a>; <a href="#B118">Zafra et al., 2014</a>), <i>Fusarium oxysporum</i> ( <a href="#B62">Marchand et al., 2017</a>), and <i>Fusarium solani</i> ( <a href="#B19">Chaıneau et al., 1999</a>) were also isolated from contaminated soil or sediments. <i>Fusarium</i> (three strains), <i>Trichoderma</i> (three strains), and <i>Pseudallescheria</i> (two strains) represented the 40% of the isolates from heavy and extra-heavy crude oil polluted soils ( <a href="#B70">Naranjo et al., 2007</a>). <a href="#B120">Zhou et al. (2017)</a> detected more than 20% of <i>Gibberella</i> species in an oilfield site. Since <i>Gibberella</i> is the teleomorph of the majority of <i>Fusarium</i> species ( <a href="#B56">Leslie and Summerell, 2008</a>) we can state that this finding is in agreement with our observations. The isolation of 42 strains of <i>Fusarium</i>, among which 64% were ascribable to <i>F. solani</i>, opens to intriguing solutions for their application in bioremediation processes. Indeed <i>F. solani</i> isolated from oil polluted soil by enrichment method using phenanthrene as the sole source of carbon was capable of degrading a mixture of low and high molecular weight PAHs (up to 84.8%) ( <a href="#B49">Hesham et al., 2017</a>). To the best of our knowledge, this is the first time that a strain of <i>Eutypella scoparia</i> has been isolated from a polluted soil. Previous researches reported the isolation of this species only in a non-polluted Arctic soil ( <a href="#B13">Bergero et al., 1999</a>; <a href="#B58">Liu et al., 2014</a>). </p> <p>Surprisingly, <a href="#B67">Morais et al. (2016)</a> did not found Eurotiomycetes in a contaminated soil. On the contrary, in the present study, the fungal community was rich of <i>Aspergillus</i> and <i>Penicillium. Aspergillus</i> and <i>Penicillium</i> strains have been already isolated in several polluted samples ( <a href="#B19">Chaıneau et al., 1999</a>; <a href="#B23">Da Silva et al., 2003</a>; <a href="#B118">Zafra et al., 2014</a>; <a href="#B3">Al-Hawash et al., 2018</a>). Data on soil isolation of <i>Aspergillus niger</i> ( <a href="#B19">Chaıneau et al., 1999</a>) <i>, Aspergillus fumigatus</i> ( <a href="#B19">Chaıneau et al., 1999</a>; <a href="#B118">Zafra et al., 2014</a>), <i>Aspergillus terreus</i> ( <a href="#B23">Da Silva et al., 2003</a>), <i>Aspergillus flavus, Aspergillus niger, Aspergillus nomius</i> ( <a href="#B118">Zafra et al., 2014</a>), <i>Aspergillus sydowii</i> ( <a href="#B96">Tian et al., 2016</a>), <i>Aspergillus versicolor</i> ( <a href="#B89">Šimonovièová et al., 2017</a>) have been reported although few information deals with the here-detected species of <i>Aspergillus</i> (i.e., <i>A. creber, A. jensenii, A. protuberus</i>). For instance, <i>A. creber</i> and <i>A. jensenii</i>, described as new species in <a href="#B52">Jurjevic et al. (2012)</a>, are mostly known as air-born contaminants or of clinical relevance ( <a href="#B66">Micheluz et al., 2016</a>; <a href="#B90">Siqueira et al., 2016</a>); <i>A. protuberus</i> has been isolated from air and the mussel <i>Mytilus edulis galloprovincialis</i> ( <a href="#B90">Siqueira et al., 2016</a>; <a href="#B53">Klas et al., 2018</a>). There is no evidence in literature of these species as a terrestrial inhabitant. Although already detected in soil, <i>Aspergillus sclerotiorum</i> and <i>Aspergillus waskmanii</i> have been only associated to not-contaminated lands ( <a href="#B71">Nesci et al., 2006</a>; <a href="#B75">Phainuphong et al., 2017</a>) or as part of the rhizosphere of tussock ( <a href="#B30">Di Menna et al., 2007</a>). Although this is the first time a <i>Penicillium catenatum</i> is found in a polluted soil, it must be considered that the identification of soil fungi is not always accurate at species level ( <a href="#B3">Al-Hawash et al., 2018</a>). </p> <p>A thorough comparison was performed between the relative abundances of bacterial and fungal strains. The comparisons carried out at genus level (Table <a href="#T2">2</a> for bacteria, Table <a href="#T3">3</a> for fungi) and at OTU level (Table <a href="#T4">4</a> for bacteria, Table <a href="#T5">5</a> for fungi) showed an agreement between the culture-dependent and the culture-independent approaches. In most cases, species level was reached, thus confirming that the Illumina sequencing of the two hypervariable V3–V4 regions and of the partial ITS2 are sufficient to reach the species level for bacteria and fungi respectively. The species with more isolates (such as <i>P. putida</i>, <i>A. xylosoxidans</i> and <i>Ochromobactrum anthropi</i> for bacteria, <i>F. oxysporum</i> and <i>F. solani</i> for fungi) were also the dominant OTUs, thus showing that the culturing methods reflect the relative composition of total communities. Furthermore, the two approaches also agree for the less abundant genera: this is expected since enrichment procedures indeed select for the cultivable species among the total bacterial and fungal communities that is originally present in the polluted soil. A novel aspect shown in our work is that the two approaches had detection levels that were generally in good agreement. Composition of the actual microbial community and are not much subjected to biases. </p> </div> <div class="conclusion" title="sec"> <div class="title" tagx="title" title="title">
Conclusion</div> <p>In the present study, we evaluated the evolution of bacterial and fungal communities enriched from polluted soil by culture-independent and dependent methods. The results showed that already after 1 week of enrichment, both bacterial and fungal communities resembled those found after 4 weeks, and that the soil depth did not influence the evolution of microbial communities, contrary to the pollutant used. Isolation results were in agreement with HTS molecular data, indicating that the strains isolated reflected the microbial composition of the enriched consortia. Future studies will focus on the biodegradation abilities of the isolates microorganisms. </p> </div> <div class="authorcontributions" title="sec"> <div class="title" tagx="title" title="title">
Author Contributions</div> <p>EP and GV ideated and supervised the study. GS, FS, and AP performed the experiments and wrote the paper. A-LB and TR performed the experiments. CG carried out chemical analyses. </p> </div> <div class="conflictofintereststatement" title="sec"> <div class="title" tagx="title" title="title">
Conflict of Interest Statement</div> <p>The authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest. </p> </div> </div> <div class="back" title="back"> <div class="fn-group" title="fn-group"> <div class="fn-type-financial-disclosure" title="financial-disclosure"> <p> <b>Funding.</b> This research was conducted within the LIFE15 ENV/IT/000396 project "Bioremediation and revegetation to restore the public use of contaminated land." </p> </div> </div> <div class="ack" title="ack"> <p>The authors are grateful to Sotirios Vasileiadis for a number of scripts used for HTS data elaboration, and to Prof Mariangela Girlanda and Dr. Samuele Voyron for helping in Fungal metabarcoding analysis. </p> </div> <div class="fn-group" title="fn-group"> <div class="fn-type-" title=""> <span class="label" tagx="label" title="label">1</span> <p> <a href="http://hannonlab.cshl.edu/fastx_toolkit/">http://hannonlab.cshl.edu/fastx_toolkit/</a> </p> </div> <div class="fn-type-" title=""> <span class="label" tagx="label" title="label">2</span> <p> <a href="http://www.R-project.org">http://www.R-project.org</a> </p> </div> </div> <div class="supplementary material" title="supplementary material"> <div class="title" tagx="title" title="title">
Supplementary Material</div> <p>The Supplementary Material for this article can be found online at: <a href="https://www.frontiersin.org/articles/10.3389/fmicb.2018.02543/full#supplementary-material">https://www.frontiersin.org/articles/10.3389/fmicb.2018.02543/full#supplementary-material</a> </p> <div class="local-data" title="local-data"> <div class="media" title="media"> <a href="Table_1.DOCX">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"> <a name="B1" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Aburto-Medina</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Adetutu</span> <span class="given-names" tagx="given-names" title="given-names">E. M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Aleer</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Weber</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Patil</span> <span class="given-names" tagx="given-names" title="given-names">S. S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sheppard</span> <span class="given-names" tagx="given-names" title="given-names">P. J.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> ( <span class="year" tagx="year" title="year">2012</span>). <span class="mixed-article-title" title="mixed-article-title">Comparison of indigenous and exogenous microbial populations during slurry phase biodegradation of long-term hydrocarbon-contaminated soil. </span> <span class="source" tagx="source" title="source"> <i>Biodegradation</i> </span> <span class="volume" tagx="volume" title="volume">23</span> <span class="fpage" tagx="fpage" title="fpage">813</span>– <span class="lpage" tagx="lpage" title="lpage">822</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s10532-012-9563-8">10.1007/s10532-012-9563-8</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22684213">22684213</a> </span> </span> </li> <li tag="ref"> <a name="B2" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Agnello</span> <span class="given-names" tagx="given-names" title="given-names">A. C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bagard</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Van Hullebusch</span> <span class="given-names" tagx="given-names" title="given-names">E. D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Esposito</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Huguenot</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> </span> ( <span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">Comparative bioremediation of heavy metals and petroleum hydrocarbons co-contaminated soil by natural attenuation, phytoremediation, bioaugmentation and bioaugmentation-assisted phytoremediation. </span> <span class="source" tagx="source" title="source"> <i>Sci. Total. Environ.</i> </span> <span class="volume" tagx="volume" title="volume">563</span>- <span class="issue" tagx="issue" title="issue">564</span> <span class="fpage" tagx="fpage" title="fpage">693</span>– <span class="lpage" tagx="lpage" title="lpage">703</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.scitotenv.2015.10.061">10.1016/j.scitotenv.2015.10.061</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26524994">26524994</a> </span> </span> </li> <li tag="ref"> <a name="B3" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Al-Hawash</span> <span class="given-names" tagx="given-names" title="given-names">A. B.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Alkooranee</span> <span class="given-names" tagx="given-names" title="given-names">J. T.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Abbood</span> <span class="given-names" tagx="given-names" title="given-names">H. A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sun</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">X.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> ( <span class="year" tagx="year" title="year">2018</span>). <span class="mixed-article-title" title="mixed-article-title">Isolation and characterization of two crude oil-degrading fungi strains from Rumaila oil field. </span> <span class="source" tagx="source" title="source"> <i>Iraq. Biotechnol. Rep.</i> </span> <span class="volume" tagx="volume" title="volume">17</span> <span class="fpage" tagx="fpage" title="fpage">104</span>– <span class="lpage" tagx="lpage" title="lpage">109</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.btre.2017.12.006">10.1016/j.btre.2017.12.006</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/29541603">29541603</a> </span> </span> </li> <li tag="ref"> <a name="B4" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Alonso-Gutierrez</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Figueras</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Albaiges</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jimenez</span> <span class="given-names" tagx="given-names" title="given-names">N.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vinas</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Solanas</span> <span class="given-names" tagx="given-names" title="given-names">A. M.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> ( <span class="year" tagx="year" title="year">2009</span>). <span class="mixed-article-title" title="mixed-article-title">Bacterial communities from shoreline environments (Costa da morte, northwestern Spain) affected by the prestige oil spill. </span> <span class="source" tagx="source" title="source"> <i>Appl. Environ. Microbiol.</i> </span> <span class="volume" tagx="volume" title="volume">75</span> <span class="fpage" tagx="fpage" title="fpage">3407</span>– <span class="lpage" tagx="lpage" title="lpage">3418</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1128/AEM.01776-08">10.1128/AEM.01776-08</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19376924">19376924</a> </span> </span> </li> <li tag="ref"> <a name="B5" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Altschul</span> <span class="given-names" tagx="given-names" title="given-names">S. F.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Madden</span> <span class="given-names" tagx="given-names" title="given-names">T. L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schaffer</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">Z.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Miller</span> <span class="given-names" tagx="given-names" title="given-names">W.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> ( <span class="year" tagx="year" title="year">1997</span>). <span class="mixed-article-title" title="mixed-article-title">Gapped BLAST and PSI-BLAST: a new generation of protein database search programs.</span> <span class="source" tagx="source" title="source"> <i>Nucleic. Acid. Res.</i> </span> <span class="volume" tagx="volume" title="volume">25</span> <span class="fpage" tagx="fpage" title="fpage">3389</span>– <span class="lpage" tagx="lpage" title="lpage">3404</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/25.17.3389">10.1093/nar/25.17.3389</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9254694">9254694</a> </span> </span> </li> <li tag="ref"> <a name="B6" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">An</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Caffrey</span> <span class="given-names" tagx="given-names" title="given-names">S. M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Soh</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Agrawal</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Brown</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Budwill</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> ( <span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">Metagenomics of hydrocarbon resource environments indicates aerobic taxa and genes to be unexpectedly common. </span> <span class="source" tagx="source" title="source"> <i>Environ. Sci. Technol.</i> </span> <span class="volume" tagx="volume" title="volume">47</span> <span class="fpage" tagx="fpage" title="fpage">10708</span>– <span class="lpage" tagx="lpage" title="lpage">10717</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1021/es4020184">10.1021/es4020184</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23889694">23889694</a> </span> </span> </li> <li tag="ref"> <a name="B7" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Aranda</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Godoy</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Reina</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Badia-Fabregat</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rosell</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Marco-Urrea</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> ( <span class="year" tagx="year" title="year">2017</span>). <span class="mixed-article-title" title="mixed-article-title">Isolation of Ascomycota fungi with capability to transform PAHs: insights into the biodegradation mechanisms of <i>Penicillium oxalicum</i>. </span> <span class="source" tagx="source" title="source"> <i>Int. Biodeter. Biodegr.</i> </span> <span class="volume" tagx="volume" title="volume">122</span> <span class="fpage" tagx="fpage" title="fpage">141</span>– <span class="lpage" tagx="lpage" title="lpage">150</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.ibiod.2017.05.015">10.1016/j.ibiod.2017.05.015</a> </span> </span> </li> <li tag="ref"> <a name="B8" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Atlas</span> <span class="given-names" tagx="given-names" title="given-names">R. M.</span> </span> </span> ( <span class="year" tagx="year" title="year">1981</span>). <span class="mixed-article-title" title="mixed-article-title">Microbial degradation of petroleum hydrocarbons: an environmental perspective.</span> <span class="source" tagx="source" title="source"> <i>Microbiol. Rev.</i> </span> <span class="volume" tagx="volume" title="volume">45</span> <span class="fpage" tagx="fpage" title="fpage">180</span>– <span class="lpage" tagx="lpage" title="lpage">209</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/7012571">7012571</a> </span> </span> </li> <li tag="ref"> <a name="B9" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Badr El-Din</span> <span class="given-names" tagx="given-names" title="given-names">S. M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Moussa</span> <span class="given-names" tagx="given-names" title="given-names">T. A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Moawad</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sharaf</span> <span class="given-names" tagx="given-names" title="given-names">O. A.</span> </span> </span> ( <span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">Isolation and characterization of polyaromatic hydrocarbons degrading bacteria from compost leachate. </span> <span class="source" tagx="source" title="source"> <i>J. Adv. Biol.</i> </span> <span class="volume" tagx="volume" title="volume">5</span> <span class="fpage" tagx="fpage" title="fpage">651</span>– <span class="lpage" tagx="lpage" title="lpage">660</span>. </span> </li> <li tag="ref"> <a name="B10" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bao</span> <span class="given-names" tagx="given-names" title="given-names">M. T.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">L. N.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sun</span> <span class="given-names" tagx="given-names" title="given-names">P. Y.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cao</span> <span class="given-names" tagx="given-names" title="given-names">L. X.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zou</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">Y. M.</span> </span> </span> ( <span class="year" tagx="year" title="year">2012</span>). <span class="mixed-article-title" title="mixed-article-title">Biodegradation of crude oil using an efficient microbial consortium in a simulated marine environment. </span> <span class="source" tagx="source" title="source"> <i>Mar. Pollut. Bull.</i> </span> <span class="volume" tagx="volume" title="volume">64</span> <span class="fpage" tagx="fpage" title="fpage">1177</span>– <span class="lpage" tagx="lpage" title="lpage">1185</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.marpolbul.2012.03.020">10.1016/j.marpolbul.2012.03.020</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22498316">22498316</a> </span> </span> </li> <li tag="ref"> <a name="B11" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Barbato</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mapelli</span> <span class="given-names" tagx="given-names" title="given-names">F.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Magagnini</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chouaia</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Armeni</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Marasco</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> ( <span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">Hydrocarbon pollutants shape bacterial community assembly of harbor sediments.</span> <span class="source" tagx="source" title="source"> <i>Mar. Pollut. Bull.</i> </span> <span class="volume" tagx="volume" title="volume">104</span> <span class="fpage" tagx="fpage" title="fpage">211</span>– <span class="lpage" tagx="lpage" title="lpage">220</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.marpolbul.2016.01.029">10.1016/j.marpolbul.2016.01.029</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26849913">26849913</a> </span> </span> </li> <li tag="ref"> <a name="B12" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bensch</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Braun</span> <span class="given-names" tagx="given-names" title="given-names">U.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Groenewald</span> <span class="given-names" tagx="given-names" title="given-names">J. Z.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Crous</span> <span class="given-names" tagx="given-names" title="given-names">P. W.</span> </span> </span> ( <span class="year" tagx="year" title="year">2012</span>). <span class="mixed-article-title" title="mixed-article-title">The genus <i>Cladosporium</i>. </span> <span class="source" tagx="source" title="source"> <i>Stud. Mycol.</i> </span> <span class="volume" tagx="volume" title="volume">72</span> <span class="fpage" tagx="fpage" title="fpage">1</span>– <span class="lpage" tagx="lpage" title="lpage">401</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.3114/sim0003">10.3114/sim0003</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22815589">22815589</a> </span> </span> </li> <li tag="ref"> <a name="B13" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bergero</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Girlanda</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Varese</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Intili</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Luppi</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> </span> ( <span class="year" tagx="year" title="year">1999</span>). <span class="mixed-article-title" title="mixed-article-title">Psychrooligotrophic fungi from arctic soils of Franz Joseph Land.</span> <span class="source" tagx="source" title="source"> <i>Polar Biol.</i> </span> <span class="volume" tagx="volume" title="volume">21</span> <span class="fpage" tagx="fpage" title="fpage">361</span>– <span class="lpage" tagx="lpage" title="lpage">368</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s003000050374">10.1007/s003000050374</a> </span> </span> </li> <li tag="ref"> <a name="B14" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Berry</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mahfoudh</span> <span class="given-names" tagx="given-names" title="given-names">K. B.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wagner</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Loy</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> </span> ( <span class="year" tagx="year" title="year">2011</span>). <span class="mixed-article-title" title="mixed-article-title">Barcoded primers used in multiplex amplicon pyrosequencing bias amplification.</span> <span class="source" tagx="source" title="source"> <i>Appl. Environ. Microb.</i> </span> <span class="volume" tagx="volume" title="volume">77</span> <span class="fpage" tagx="fpage" title="fpage">7846</span>– <span class="lpage" tagx="lpage" title="lpage">7849</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1128/AEM.05220-11">10.1128/AEM.05220-11</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21890669">21890669</a> </span> </span> </li> <li tag="ref"> <a name="B15" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Biró</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Toscano</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Horváth</span> <span class="given-names" tagx="given-names" title="given-names">N.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Matics</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Domonkos</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Scotti</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> ( <span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">Vertical and horizontal distributions of microbial abundances and enzymatic activities in propylene-glycol-affected soils. </span> <span class="source" tagx="source" title="source"> <i>Environ. Sci. Pollut. R.</i> </span> <span class="volume" tagx="volume" title="volume">21</span> <span class="fpage" tagx="fpage" title="fpage">9095</span>– <span class="lpage" tagx="lpage" title="lpage">9108</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s11356-014-2686-1">10.1007/s11356-014-2686-1</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24627198">24627198</a> </span> </span> </li> <li tag="ref"> <a name="B16" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Blume</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bischoff</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Reichert</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Moorman</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Konopka</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Turco</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> </span> ( <span class="year" tagx="year" title="year">2002</span>). <span class="mixed-article-title" title="mixed-article-title">Surface and subsurface microbial biomass, community structure and metabolic activity as a function of soil depth and season. </span> <span class="source" tagx="source" title="source"> <i>Appl. Soil Ecol.</i> </span> <span class="volume" tagx="volume" title="volume">20</span> <span class="fpage" tagx="fpage" title="fpage">171</span>– <span class="lpage" tagx="lpage" title="lpage">181</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/S0929-1393(02)00025-2">10.1016/S0929-1393(02)00025-2</a> </span> </span> </li> <li tag="ref"> <a name="B17" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bundy</span> <span class="given-names" tagx="given-names" title="given-names">J. G.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Paton</span> <span class="given-names" tagx="given-names" title="given-names">G. I.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Campbell</span> <span class="given-names" tagx="given-names" title="given-names">C. D.</span> </span> </span> ( <span class="year" tagx="year" title="year">2004</span>). <span class="mixed-article-title" title="mixed-article-title">Combined microbial community level and single species biosensor responses to monitor recovery of oil polluted soil. </span> <span class="source" tagx="source" title="source"> <i>Soil Biol. Biochem.</i> </span> <span class="volume" tagx="volume" title="volume">36</span> <span class="fpage" tagx="fpage" title="fpage">1149</span>– <span class="lpage" tagx="lpage" title="lpage">1159</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.soilbio.2004.02.025">10.1016/j.soilbio.2004.02.025</a> </span> </span> </li> <li tag="ref"> <a name="B18" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cerqueira</span> <span class="given-names" tagx="given-names" title="given-names">V. S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hollenbach</span> <span class="given-names" tagx="given-names" title="given-names">E. B.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Maboni</span> <span class="given-names" tagx="given-names" title="given-names">F.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vainstein</span> <span class="given-names" tagx="given-names" title="given-names">M. H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Camargo</span> <span class="given-names" tagx="given-names" title="given-names">F. A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Do Carmo</span> <span class="given-names" tagx="given-names" title="given-names">R. P. M.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> ( <span class="year" tagx="year" title="year">2011</span>). <span class="mixed-article-title" title="mixed-article-title">Biodegradation potential of oily sludge by pure and mixed bacterial cultures.</span> <span class="source" tagx="source" title="source"> <i>Bioresour. Technol.</i> </span> <span class="volume" tagx="volume" title="volume">102</span> <span class="fpage" tagx="fpage" title="fpage">11003</span>– <span class="lpage" tagx="lpage" title="lpage">11010</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.biortech.2011.09.074">10.1016/j.biortech.2011.09.074</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21993328">21993328</a> </span> </span> </li> <li tag="ref"> <a name="B19" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chaıneau</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Morel</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dupont</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bury</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Oudot</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> </span> ( <span class="year" tagx="year" title="year">1999</span>). <span class="mixed-article-title" title="mixed-article-title">Comparison of the fuel oil biodegradation potential of hydrocarbon-assimilating microorganisms isolated from a temperate agricultural soil. </span> <span class="source" tagx="source" title="source"> <i>Sci. Tot. Environ.</i> </span> <span class="volume" tagx="volume" title="volume">227</span> <span class="fpage" tagx="fpage" title="fpage">237</span>– <span class="lpage" tagx="lpage" title="lpage">247</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/S0048-9697(99)00033-9">10.1016/S0048-9697(99)00033-9</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10231986">10231986</a> </span> </span> </li> <li tag="ref"> <a name="B20" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chandra</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sharma</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Singh</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> </span> ( <span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">Application of bioremediation technology in the environment contaminated with petroleum hydrocarbon. </span> <span class="source" tagx="source" title="source"> <i>Ann. Microbiol.</i> </span> <span class="volume" tagx="volume" title="volume">63</span> <span class="fpage" tagx="fpage" title="fpage">417</span>– <span class="lpage" tagx="lpage" title="lpage">431</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s13213-012-0543-3">10.1007/s13213-012-0543-3</a> </span> </span> </li> <li tag="ref"> <a name="B21" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cousins</span> <span class="given-names" tagx="given-names" title="given-names">I. T.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gevao</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jones</span> <span class="given-names" tagx="given-names" title="given-names">K. C.</span> </span> </span> ( <span class="year" tagx="year" title="year">1999</span>). <span class="mixed-article-title" title="mixed-article-title">Measuring and modelling the vertical distribution of semi-volatile organic compounds in soils. I: PCB and PAH soil core data. </span> <span class="source" tagx="source" title="source"> <i>Chemosphere</i> </span> <span class="volume" tagx="volume" title="volume">39</span> <span class="fpage" tagx="fpage" title="fpage">2507</span>– <span class="lpage" tagx="lpage" title="lpage">2518</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/S0045-6535(99)00164-2">10.1016/S0045-6535(99)00164-2</a> </span> </span> </li> <li tag="ref"> <a name="B22" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Covino</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stella</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cajthaml</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> </span> ( <span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">"Mycoremediation of organic pollutants: principles, opportunities, and pitfalls," in </span> <span class="source" tagx="source" title="source"> <i>Fungal Applications in Sustainable Environmental Biotechnology</i> </span>, <span class="role" tagx="role" title="role">ed.</span> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Purchase</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> </span> ( <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Berlin</span>: <span class="publisher-name" tagx="publisher-name" title="publisher-name">Springer</span>), <span class="fpage" tagx="fpage" title="fpage">185</span>– <span class="lpage" tagx="lpage" title="lpage">231</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1007/978-3-319-42852-9_8">10.1007/978-3-319-42852-9_8</a> </span> </span> </li> <li tag="ref"> <a name="B23" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Da Silva</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Umbuzeiro</span> <span class="given-names" tagx="given-names" title="given-names">G. A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pfenning</span> <span class="given-names" tagx="given-names" title="given-names">L. H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Canhos</span> <span class="given-names" tagx="given-names" title="given-names">V. P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Esposito</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span> </span> ( <span class="year" tagx="year" title="year">2003</span>). <span class="mixed-article-title" title="mixed-article-title">Filamentous fungi isolated from estuarine sediments contaminated with industrial discharges.</span> <span class="source" tagx="source" title="source"> <i>Soil Sediment Contam.</i> </span> <span class="volume" tagx="volume" title="volume">12</span> <span class="fpage" tagx="fpage" title="fpage">345</span>– <span class="lpage" tagx="lpage" title="lpage">356</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1080/713610976">10.1080/713610976</a> </span> </span> </li> <li tag="ref"> <a name="B24" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Das</span> <span class="given-names" tagx="given-names" title="given-names">N.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chandran</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> </span> ( <span class="year" tagx="year" title="year">2011</span>). <span class="mixed-article-title" title="mixed-article-title">Microbial degradation of petroleum hydrocarbon contaminants: an overview.</span> <span class="source" tagx="source" title="source"> <i>Biotechnol. Res. Int.</i> </span> <span class="volume" tagx="volume" title="volume">2011</span>: <span class="issue" tagx="issue" title="issue">941810</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.4061/2011/941810">10.4061/2011/941810</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21350672">21350672</a> </span> </span> </li> <li tag="ref"> <a name="B25" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">De La Fuente</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Perestelo</span> <span class="given-names" tagx="given-names" title="given-names">F.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rodriguez-Perez</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Falcon</span> <span class="given-names" tagx="given-names" title="given-names">M. A.</span> </span> </span> ( <span class="year" tagx="year" title="year">1991</span>). <span class="mixed-article-title" title="mixed-article-title">Oxidation of aromatic aldehydes by <i>Serratia marcescens</i>. </span> <span class="source" tagx="source" title="source"> <i>Appl. Environ. Microb.</i> </span> <span class="volume" tagx="volume" title="volume">57</span> <span class="fpage" tagx="fpage" title="fpage">1275</span>– <span class="lpage" tagx="lpage" title="lpage">1276</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/2059046">2059046</a> </span> </span> </li> <li tag="ref"> <a name="B26" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dean-Ross</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Moody</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cerniglia</span> <span class="given-names" tagx="given-names" title="given-names">C. E.</span> </span> </span> ( <span class="year" tagx="year" title="year">2002</span>). <span class="mixed-article-title" title="mixed-article-title">Utilization of mixtures of polycyclic aromatic hydrocarbons by bacteria isolated from contaminated sediment. </span> <span class="source" tagx="source" title="source"> <i>FEMS Microbiol. Ecol.</i> </span> <span class="volume" tagx="volume" title="volume">41</span> <span class="fpage" tagx="fpage" title="fpage">1</span>– <span class="lpage" tagx="lpage" title="lpage">7</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1574-6941.2002.tb00960.x">10.1111/j.1574-6941.2002.tb00960.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19709233">19709233</a> </span> </span> </li> <li tag="ref"> <a name="B27" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Desai</span> <span class="given-names" tagx="given-names" title="given-names">J. D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Banat</span> <span class="given-names" tagx="given-names" title="given-names">I. M.</span> </span> </span> ( <span class="year" tagx="year" title="year">1997</span>). <span class="mixed-article-title" title="mixed-article-title">Microbial production of surfactants and their commercial potential.</span> <span class="source" tagx="source" title="source"> <i>Microbiol. Mol. Biol. R.</i> </span> <span class="volume" tagx="volume" title="volume">61</span> <span class="fpage" tagx="fpage" title="fpage">47</span>– <span class="lpage" tagx="lpage" title="lpage">64</span>. </span> </li> <li tag="ref"> <a name="B28" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Deshmukh</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Khardenavis</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Purohit</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> </span> ( <span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">Diverse metabolic capacities of fungi for bioremediation.</span> <span class="source" tagx="source" title="source"> <i>Indian J. Microbiol.</i> </span> <span class="volume" tagx="volume" title="volume">56</span> <span class="fpage" tagx="fpage" title="fpage">247</span>– <span class="lpage" tagx="lpage" title="lpage">264</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s12088-016-0584-6">10.1007/s12088-016-0584-6</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27407289">27407289</a> </span> </span> </li> <li tag="ref"> <a name="B29" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Di Cello</span> <span class="given-names" tagx="given-names" title="given-names">F.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fani</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> </span> ( <span class="year" tagx="year" title="year">1996</span>). <span class="mixed-article-title" title="mixed-article-title">A molecular strategy for the study of natural bacterial communities by PCR-based techniques.</span> <span class="source" tagx="source" title="source"> <i>Minerva Biotecnol.</i> </span> <span class="volume" tagx="volume" title="volume">8</span> <span class="fpage" tagx="fpage" title="fpage">126</span>– <span class="lpage" tagx="lpage" title="lpage">134</span>. </span> </li> <li tag="ref"> <a name="B30" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Di Menna</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sayer</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Barratt</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bell</span> <span class="given-names" tagx="given-names" title="given-names">N.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ferguson</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Townsend</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> </span> ( <span class="year" tagx="year" title="year">2007</span>). <span class="mixed-article-title" title="mixed-article-title">Biodiversity of indigenous tussock grassland sites in Otago, Canterbury and the central North Island. V. <i>Penicillia</i> and <i>Aspergilli</i>. </span> <span class="source" tagx="source" title="source"> <i>J. Roy. Soc. New Zeal.</i> </span> <span class="volume" tagx="volume" title="volume">37</span> <span class="fpage" tagx="fpage" title="fpage">131</span>– <span class="lpage" tagx="lpage" title="lpage">137</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1080/03014220709510541">10.1080/03014220709510541</a> </span> </span> </li> <li tag="ref"> <a name="B31" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dueholm</span> <span class="given-names" tagx="given-names" title="given-names">M. S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Marques</span> <span class="given-names" tagx="given-names" title="given-names">I. G.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Karst</span> <span class="given-names" tagx="given-names" title="given-names">S. M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">D’imperio</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tale</span> <span class="given-names" tagx="given-names" title="given-names">V. P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lewis</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> ( <span class="year" tagx="year" title="year">2015</span>). <span class="mixed-article-title" title="mixed-article-title">Survival and activity of individual bioaugmentation strains.</span> <span class="source" tagx="source" title="source"> <i>Bioresour. Technol.</i> </span> <span class="volume" tagx="volume" title="volume">186</span> <span class="fpage" tagx="fpage" title="fpage">192</span>– <span class="lpage" tagx="lpage" title="lpage">199</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.biortech.2015.02.111">10.1016/j.biortech.2015.02.111</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25817029">25817029</a> </span> </span> </li> <li tag="ref"> <a name="B32" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Edgar</span> <span class="given-names" tagx="given-names" title="given-names">R. C.</span> </span> </span> ( <span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">UPARSE: highly accurate OTU sequences from microbial amplicon reads.</span> <span class="source" tagx="source" title="source"> <i>Nat. Methods</i> </span> <span class="volume" tagx="volume" title="volume">10</span> <span class="fpage" tagx="fpage" title="fpage">996</span>– <span class="lpage" tagx="lpage" title="lpage">998</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nmeth.2604">10.1038/nmeth.2604</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23955772">23955772</a> </span> </span> </li> <li tag="ref"> <a name="B33" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Edgar</span> <span class="given-names" tagx="given-names" title="given-names">R. C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Haas</span> <span class="given-names" tagx="given-names" title="given-names">B. J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Clemente</span> <span class="given-names" tagx="given-names" title="given-names">J. C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Quince</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Knight</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> </span> ( <span class="year" tagx="year" title="year">2011</span>). <span class="mixed-article-title" title="mixed-article-title">UCHIME improves sensitivity and speed of chimera detection.</span> <span class="source" tagx="source" title="source"> <i>Bioinformatics</i> </span> <span class="volume" tagx="volume" title="volume">27</span> <span class="fpage" tagx="fpage" title="fpage">2194</span>– <span class="lpage" tagx="lpage" title="lpage">2200</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1093/bioinformatics/btr381">10.1093/bioinformatics/btr381</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21700674">21700674</a> </span> </span> </li> <li tag="ref"> <a name="B34" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Eguchi</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nishikawa</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Macdonald</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cavicchioli</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gottschal</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kjelleberg</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> </span> ( <span class="year" tagx="year" title="year">1996</span>). <span class="mixed-article-title" title="mixed-article-title">Responses to stress and nutrient availability by the marine ultramicrobacterium <i>Sphingomonas</i> sp. strain RB2256. </span> <span class="source" tagx="source" title="source"> <i>App. Environ. Microb.</i> </span> <span class="volume" tagx="volume" title="volume">62</span> <span class="fpage" tagx="fpage" title="fpage">1287</span>– <span class="lpage" tagx="lpage" title="lpage">1294</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16535292">16535292</a> </span> </span> </li> <li tag="ref"> <a name="B35" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fernandez-Luqueno</span> <span class="given-names" tagx="given-names" title="given-names">F.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Valenzuela-Encinas</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Marsch</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Martinez-Suarez</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vazquez-Nunez</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dendooven</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span> </span> ( <span class="year" tagx="year" title="year">2010</span>). <span class="mixed-article-title" title="mixed-article-title">Microbial communities to mitigate contamination of PAHs in soil-possibilities and challenge: a review. </span> <span class="source" tagx="source" title="source"> <i>Environ. Sci. Pollut. R.</i> </span> <span class="volume" tagx="volume" title="volume">18</span> <span class="fpage" tagx="fpage" title="fpage">12</span>– <span class="lpage" tagx="lpage" title="lpage">30</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s11356-010-0371-6">10.1007/s11356-010-0371-6</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20623198">20623198</a> </span> </span> </li> <li tag="ref"> <a name="B36" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fierer</span> <span class="given-names" tagx="given-names" title="given-names">N.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schimel</span> <span class="given-names" tagx="given-names" title="given-names">J. P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Holden</span> <span class="given-names" tagx="given-names" title="given-names">P. A.</span> </span> </span> ( <span class="year" tagx="year" title="year">2003</span>). <span class="mixed-article-title" title="mixed-article-title">Variations in microbial community composition through two soil depth profiles.</span> <span class="source" tagx="source" title="source"> <i>Soil Biol. Biochem.</i> </span> <span class="volume" tagx="volume" title="volume">35</span> <span class="fpage" tagx="fpage" title="fpage">167</span>– <span class="lpage" tagx="lpage" title="lpage">176</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/S0038-0717(02)00251-1">10.1016/S0038-0717(02)00251-1</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17822097">17822097</a> </span> </span> </li> <li tag="ref"> <a name="B37" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Foght</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> </span> ( <span class="year" tagx="year" title="year">2008</span>). <span class="mixed-article-title" title="mixed-article-title">Anaerobic biodegradation of aromatic hydrocarbons: pathways and prospects.</span> <span class="source" tagx="source" title="source"> <i>J Mol. Microbiol. Biotechnol.</i> </span> <span class="volume" tagx="volume" title="volume">15</span> <span class="fpage" tagx="fpage" title="fpage">93</span>– <span class="lpage" tagx="lpage" title="lpage">120</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1159/000121324">10.1159/000121324</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18685265">18685265</a> </span> </span> </li> <li tag="ref"> <a name="B38" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fowler</span> <span class="given-names" tagx="given-names" title="given-names">S. J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Toth</span> <span class="given-names" tagx="given-names" title="given-names">C. R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gieg</span> <span class="given-names" tagx="given-names" title="given-names">L. M.</span> </span> </span> ( <span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">Community structure in methanogenic enrichments provides insight into syntrophic interactions in hydrocarbon-impacted environments. </span> <span class="source" tagx="source" title="source"> <i>Front. Microbiol.</i> </span> <span class="volume" tagx="volume" title="volume">7</span>: <span class="issue" tagx="issue" title="issue">562</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.3389/fmicb.2016.00562">10.3389/fmicb.2016.00562</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27148240">27148240</a> </span> </span> </li> <li tag="ref"> <a name="B39" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fuentes</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Méndez</span> <span class="given-names" tagx="given-names" title="given-names">V.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Aguila</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Seeger</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> </span> ( <span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">Bioremediation of petroleum hydrocarbons: catabolic genes, microbial communities, and applications. </span> <span class="source" tagx="source" title="source"> <i>Appl. Microbiol. Biotechnol.</i> </span> <span class="volume" tagx="volume" title="volume">98</span> <span class="fpage" tagx="fpage" title="fpage">4781</span>– <span class="lpage" tagx="lpage" title="lpage">4794</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s00253-014-5684-9">10.1007/s00253-014-5684-9</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24691868">24691868</a> </span> </span> </li> <li tag="ref"> <a name="B40" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gao</span> <span class="given-names" tagx="given-names" title="given-names">Y. C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Guo</span> <span class="given-names" tagx="given-names" title="given-names">S. H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">J. N.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zeng</span> <span class="given-names" tagx="given-names" title="given-names">D. H.</span> </span> </span> ( <span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">Effects of different remediation treatments on crude oil contaminated saline soil.</span> <span class="source" tagx="source" title="source"> <i>Chemosphere</i> </span> <span class="volume" tagx="volume" title="volume">117</span> <span class="fpage" tagx="fpage" title="fpage">486</span>– <span class="lpage" tagx="lpage" title="lpage">493</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.chemosphere.2014.08.070">10.1016/j.chemosphere.2014.08.070</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25240723">25240723</a> </span> </span> </li> <li tag="ref"> <a name="B41" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ghaly</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yusran</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> </span> ( <span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">Effects of biostimulation and bioaugmentation on the degradation of pyrene in soil.</span> <span class="source" tagx="source" title="source"> <i>J. Bioremed. Biod.</i> </span> <span class="volume" tagx="volume" title="volume">5</span> <span class="fpage" tagx="fpage" title="fpage">1</span>– <span class="lpage" tagx="lpage" title="lpage">13</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.chemosphere.2011.11.032">10.1016/j.chemosphere.2011.11.032</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22197016">22197016</a> </span> </span> </li> <li tag="ref"> <a name="B42" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gihring</span> <span class="given-names" tagx="given-names" title="given-names">T. M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Green</span> <span class="given-names" tagx="given-names" title="given-names">S. J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schadt</span> <span class="given-names" tagx="given-names" title="given-names">C. W.</span> </span> </span> ( <span class="year" tagx="year" title="year">2012</span>). <span class="mixed-article-title" title="mixed-article-title">Massively parallel rRNA gene sequencing exacerbates the potential for biased community diversity comparisons due to variable library sizes. </span> <span class="source" tagx="source" title="source"> <i>Environ. Microbiol.</i> </span> <span class="volume" tagx="volume" title="volume">14</span> <span class="fpage" tagx="fpage" title="fpage">285</span>– <span class="lpage" tagx="lpage" title="lpage">290</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1462-2920.2011.02550.x">10.1111/j.1462-2920.2011.02550.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21923700">21923700</a> </span> </span> </li> <li tag="ref"> <a name="B43" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Glass</span> <span class="given-names" tagx="given-names" title="given-names">N. L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Donaldson</span> <span class="given-names" tagx="given-names" title="given-names">G. C.</span> </span> </span> ( <span class="year" tagx="year" title="year">1995</span>). <span class="mixed-article-title" title="mixed-article-title">Development of primer sets designed for use with the PCR to amplify conserved genes from filamentous ascomycetes. </span> <span class="source" tagx="source" title="source"> <i>Appl. Environ. Microbiol.</i> </span> <span class="volume" tagx="volume" title="volume">61</span> <span class="fpage" tagx="fpage" title="fpage">1323</span>– <span class="lpage" tagx="lpage" title="lpage">1330</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/7747954">7747954</a> </span> </span> </li> <li tag="ref"> <a name="B44" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hara</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kurihara</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nomura</span> <span class="given-names" tagx="given-names" title="given-names">N.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nakajima</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Uchiyama</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> </span> ( <span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">Bioremediation field trial of oil-contaminated soil with food-waste compost.</span> <span class="source" tagx="source" title="source"> <i>J. JSCE</i> </span> <span class="volume" tagx="volume" title="volume">1</span> <span class="fpage" tagx="fpage" title="fpage">125</span>– <span class="lpage" tagx="lpage" title="lpage">132</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.2208/journalofjsce.1.1_125">10.2208/journalofjsce.1.1_125</a> </span> </span> </li> <li tag="ref"> <a name="B45" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Haritash</span> <span class="given-names" tagx="given-names" title="given-names">A. K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kaushik</span> <span class="given-names" tagx="given-names" title="given-names">C. P.</span> </span> </span> ( <span class="year" tagx="year" title="year">2009</span>). <span class="mixed-article-title" title="mixed-article-title">Biodegradation aspects of polycyclic aromatic hydrocarbons (PAHs): a review.</span> <span class="source" tagx="source" title="source"> <i>J. Hazard Mater.</i> </span> <span class="volume" tagx="volume" title="volume">169</span> <span class="fpage" tagx="fpage" title="fpage">1</span>– <span class="lpage" tagx="lpage" title="lpage">15</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.jhazmat.2009.03.137">10.1016/j.jhazmat.2009.03.137</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19442441">19442441</a> </span> </span> </li> <li tag="ref"> <a name="B46" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hazen</span> <span class="given-names" tagx="given-names" title="given-names">T. C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rocha</span> <span class="given-names" tagx="given-names" title="given-names">A. M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Techtmann</span> <span class="given-names" tagx="given-names" title="given-names">S. M.</span> </span> </span> ( <span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">Advances in monitoring environmental microbes.</span> <span class="source" tagx="source" title="source"> <i>Curr. Opin. Biotechnol.</i> </span> <span class="volume" tagx="volume" title="volume">24</span> <span class="fpage" tagx="fpage" title="fpage">526</span>– <span class="lpage" tagx="lpage" title="lpage">533</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.copbio.2012.10.020">10.1016/j.copbio.2012.10.020</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23183250">23183250</a> </span> </span> </li> <li tag="ref"> <a name="B47" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Head</span> <span class="given-names" tagx="given-names" title="given-names">I. M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gray</span> <span class="given-names" tagx="given-names" title="given-names">N. D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Larter</span> <span class="given-names" tagx="given-names" title="given-names">S. R.</span> </span> </span> ( <span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">Life in the slow lane; biogeochemistry of biodegraded petroleum containing reservoirs and implications for energy recovery and carbon management. </span> <span class="source" tagx="source" title="source"> <i>Front. Microbiol.</i> </span> <span class="volume" tagx="volume" title="volume">5</span>: <span class="issue" tagx="issue" title="issue">566</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.3389/fmicb.2014.00566">10.3389/fmicb.2014.00566</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25426105">25426105</a> </span> </span> </li> <li tag="ref"> <a name="B48" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Head</span> <span class="given-names" tagx="given-names" title="given-names">I. M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jones</span> <span class="given-names" tagx="given-names" title="given-names">D. M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Roling</span> <span class="given-names" tagx="given-names" title="given-names">W. F.</span> </span> </span> ( <span class="year" tagx="year" title="year">2006</span>). <span class="mixed-article-title" title="mixed-article-title">Marine microorganisms make a meal of oil.</span> <span class="source" tagx="source" title="source"> <i>Nat. Rev. Microbiol.</i> </span> <span class="volume" tagx="volume" title="volume">4</span> <span class="fpage" tagx="fpage" title="fpage">173</span>– <span class="lpage" tagx="lpage" title="lpage">182</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1038/nrmicro1348">10.1038/nrmicro1348</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16489346">16489346</a> </span> </span> </li> <li tag="ref"> <a name="B49" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hesham</span> <span class="given-names" tagx="given-names" title="given-names">A. E.-L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mohamed</span> <span class="given-names" tagx="given-names" title="given-names">E. A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mawad</span> <span class="given-names" tagx="given-names" title="given-names">A. M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Elfarash</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Abd El-Fattah</span> <span class="given-names" tagx="given-names" title="given-names">B. S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">El-Rawy</span> <span class="given-names" tagx="given-names" title="given-names">M. A.</span> </span> </span> ( <span class="year" tagx="year" title="year">2017</span>). <span class="mixed-article-title" title="mixed-article-title">Molecular characterization of <i>Fusarium solani</i> degrades a mixture of low and high molecular weight polycyclic aromatic hydrocarbons. </span> <span class="source" tagx="source" title="source"> <i>Biotechnol. J.</i> </span> <span class="volume" tagx="volume" title="volume">11</span> <span class="fpage" tagx="fpage" title="fpage">27</span>– <span class="lpage" tagx="lpage" title="lpage">35</span>. </span> </li> <li tag="ref"> <a name="B50" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Johnsen</span> <span class="given-names" tagx="given-names" title="given-names">A. R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schmidt</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hybholt</span> <span class="given-names" tagx="given-names" title="given-names">T. K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Henriksen</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jacobsen</span> <span class="given-names" tagx="given-names" title="given-names">C. S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Andersen</span> <span class="given-names" tagx="given-names" title="given-names">O.</span> </span> </span> ( <span class="year" tagx="year" title="year">2007</span>). <span class="mixed-article-title" title="mixed-article-title">Strong impact on the polycyclic aromatic hydrocarbon (PAH)-degrading community of a PAH-polluted soil but marginal effect on PAH degradation when priming with bioremediated soil dominated by mycobacteria. </span> <span class="source" tagx="source" title="source"> <i>Appl. Environ. Microbiol.</i> </span> <span class="volume" tagx="volume" title="volume">73</span> <span class="fpage" tagx="fpage" title="fpage">1474</span>– <span class="lpage" tagx="lpage" title="lpage">1480</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1128/AEM.02236-06">10.1128/AEM.02236-06</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17209064">17209064</a> </span> </span> </li> <li tag="ref"> <a name="B51" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Joshi</span> <span class="given-names" tagx="given-names" title="given-names">M. N.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dhebar</span> <span class="given-names" tagx="given-names" title="given-names">S. V.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dhebar</span> <span class="given-names" tagx="given-names" title="given-names">S. V.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bhargava</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pandit</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Patel</span> <span class="given-names" tagx="given-names" title="given-names">R. P.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> ( <span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">Metagenomics of petroleum muck: revealing microbial diversity and depicting microbial syntrophy. </span> <span class="source" tagx="source" title="source"> <i>Arch. Microbiol.</i> </span> <span class="volume" tagx="volume" title="volume">196</span> <span class="fpage" tagx="fpage" title="fpage">531</span>– <span class="lpage" tagx="lpage" title="lpage">544</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s00203-014-0992-0">10.1007/s00203-014-0992-0</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24838250">24838250</a> </span> </span> </li> <li tag="ref"> <a name="B52" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jurjevic</span> <span class="given-names" tagx="given-names" title="given-names">Z.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Peterson</span> <span class="given-names" tagx="given-names" title="given-names">S. W.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Horn</span> <span class="given-names" tagx="given-names" title="given-names">B. W.</span> </span> </span> ( <span class="year" tagx="year" title="year">2012</span>). <span class="mixed-article-title" title="mixed-article-title"> <i>Aspergillus</i> section versicolores: nine new species and multilocus DNA sequence based phylogeny. </span> <span class="source" tagx="source" title="source"> <i>IMA Fungus</i> </span> <span class="volume" tagx="volume" title="volume">3</span> <span class="fpage" tagx="fpage" title="fpage">59</span>– <span class="lpage" tagx="lpage" title="lpage">59</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.5598/imafungus.2012.03.01.07">10.5598/imafungus.2012.03.01.07</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23155501">23155501</a> </span> </span> </li> <li tag="ref"> <a name="B53" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Klas</span> <span class="given-names" tagx="given-names" title="given-names">K. R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kato</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Frisvad</span> <span class="given-names" tagx="given-names" title="given-names">J. C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yu</span> <span class="given-names" tagx="given-names" title="given-names">F.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Newmister</span> <span class="given-names" tagx="given-names" title="given-names">S. A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fraley</span> <span class="given-names" tagx="given-names" title="given-names">A. E.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> ( <span class="year" tagx="year" title="year">2018</span>). <span class="mixed-article-title" title="mixed-article-title">Structural and stereochemical diversity in prenylated indole alkaloids containing the bicyclo-[2.2.2]-diazaoctane ring system from marine and terrestrial fungi. </span> <span class="source" tagx="source" title="source"> <i>Nat. Prod. Rep.</i> </span> <span class="volume" tagx="volume" title="volume">35</span> <span class="fpage" tagx="fpage" title="fpage">532</span>– <span class="lpage" tagx="lpage" title="lpage">558</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1039/c7np00042a">10.1039/c7np00042a</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/29632911">29632911</a> </span> </span> </li> <li tag="ref"> <a name="B54" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kostka</span> <span class="given-names" tagx="given-names" title="given-names">J. E.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Prakash</span> <span class="given-names" tagx="given-names" title="given-names">O.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Overholt</span> <span class="given-names" tagx="given-names" title="given-names">W. A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Green</span> <span class="given-names" tagx="given-names" title="given-names">S. J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Freyer</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Canion</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> ( <span class="year" tagx="year" title="year">2011</span>). <span class="mixed-article-title" title="mixed-article-title">Hydrocarbon-degrading bacteria and the bacterial community response in gulf of Mexico beach sands impacted by the deepwater horizon oil spill. </span> <span class="source" tagx="source" title="source"> <i>Appl. Environ. Microbiol.</i> </span> <span class="volume" tagx="volume" title="volume">77</span> <span class="fpage" tagx="fpage" title="fpage">7962</span>– <span class="lpage" tagx="lpage" title="lpage">7974</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1128/AEM.05402-11">10.1128/AEM.05402-11</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21948834">21948834</a> </span> </span> </li> <li tag="ref"> <a name="B55" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Leonardi</span> <span class="given-names" tagx="given-names" title="given-names">V.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Giubilei</span> <span class="given-names" tagx="given-names" title="given-names">M. A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Federici</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Spaccapelo</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sasek</span> <span class="given-names" tagx="given-names" title="given-names">V.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Novotny</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> ( <span class="year" tagx="year" title="year">2008</span>). <span class="mixed-article-title" title="mixed-article-title">Mobilizing agents enhance fungal degradation of polycyclic aromatic hydrocarbons and affect diversity of indigenous bacteria in soil. </span> <span class="source" tagx="source" title="source"> <i>Biotechnol. Bioeng.</i> </span> <span class="volume" tagx="volume" title="volume">101</span> <span class="fpage" tagx="fpage" title="fpage">273</span>– <span class="lpage" tagx="lpage" title="lpage">285</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1002/bit.21909">10.1002/bit.21909</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18727031">18727031</a> </span> </span> </li> <li tag="ref"> <a name="B56" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Leslie</span> <span class="given-names" tagx="given-names" title="given-names">J. F.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Summerell</span> <span class="given-names" tagx="given-names" title="given-names">B. A.</span> </span> </span> ( <span class="year" tagx="year" title="year">2008</span>). <span class="source" tagx="source" title="source"> <i>The Fusarium Laboratory Manual.</i> </span> <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Hoboken, NJ</span>: <span class="publisher-name" tagx="publisher-name" title="publisher-name">John Wiley &amp;amp; Sons</span>. </span> </li> <li tag="ref"> <a name="B57" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">G. P.-W.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chang</span> <span class="given-names" tagx="given-names" title="given-names">T. C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Whang</span> <span class="given-names" tagx="given-names" title="given-names">L.-M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kao</span> <span class="given-names" tagx="given-names" title="given-names">C.-H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pan</span> <span class="given-names" tagx="given-names" title="given-names">P.-T.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cheng</span> <span class="given-names" tagx="given-names" title="given-names">S.-S.</span> </span> </span> ( <span class="year" tagx="year" title="year">2011</span>). <span class="mixed-article-title" title="mixed-article-title">Bioremediation of petroleum hydrocarbon contaminated soil: effects of strategies and microbial community shift. </span> <span class="source" tagx="source" title="source"> <i>Int. Biodeter. Biodegr.</i> </span> <span class="volume" tagx="volume" title="volume">65</span> <span class="fpage" tagx="fpage" title="fpage">1119</span>– <span class="lpage" tagx="lpage" title="lpage">1127</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.ibiod.2011.09.002">10.1016/j.ibiod.2011.09.002</a> </span> </span> </li> <li tag="ref"> <a name="B58" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">J. T.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hu</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gao</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">J. P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jiao</span> <span class="given-names" tagx="given-names" title="given-names">B. H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lu</span> <span class="given-names" tagx="given-names" title="given-names">X. L.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> ( <span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">Bioactive tyrosine-derived cytochalasins from fungus <i>Eutypella</i> sp. D-1. </span> <span class="source" tagx="source" title="source"> <i>Chem. Biodivers.</i> </span> <span class="volume" tagx="volume" title="volume">11</span> <span class="fpage" tagx="fpage" title="fpage">800</span>– <span class="lpage" tagx="lpage" title="lpage">806</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1002/cbdv.201300218">10.1002/cbdv.201300218</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24827690">24827690</a> </span> </span> </li> <li tag="ref"> <a name="B59" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">MacNaughton</span> <span class="given-names" tagx="given-names" title="given-names">S. J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stephen</span> <span class="given-names" tagx="given-names" title="given-names">J. R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Venosa</span> <span class="given-names" tagx="given-names" title="given-names">A. D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Davis</span> <span class="given-names" tagx="given-names" title="given-names">G. A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chang</span> <span class="given-names" tagx="given-names" title="given-names">Y.-J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">White</span> <span class="given-names" tagx="given-names" title="given-names">D. C.</span> </span> </span> ( <span class="year" tagx="year" title="year">1999</span>). <span class="mixed-article-title" title="mixed-article-title">Microbial population changes during bioremediation of an experimental oil spill.</span> <span class="source" tagx="source" title="source"> <i>Appl. Environ. Microbiol.</i> </span> <span class="volume" tagx="volume" title="volume">65</span> <span class="fpage" tagx="fpage" title="fpage">3566</span>– <span class="lpage" tagx="lpage" title="lpage">3574</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10427050">10427050</a> </span> </span> </li> <li tag="ref"> <a name="B60" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mahjoubi</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jaouani</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Guesmi</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ben Amor</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jouini</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cherif</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> ( <span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">Hydrocarbonoclastic bacteria isolated from petroleum contaminated sites in Tunisia: isolation, identification and characterization of the biotechnological potential. </span> <span class="source" tagx="source" title="source"> <i>N. Biotechnol.</i> </span> <span class="volume" tagx="volume" title="volume">30</span> <span class="fpage" tagx="fpage" title="fpage">723</span>– <span class="lpage" tagx="lpage" title="lpage">733</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.nbt.2013.03.004">10.1016/j.nbt.2013.03.004</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23541698">23541698</a> </span> </span> </li> <li tag="ref"> <a name="B61" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Maiti</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chandra</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mondal</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ojha</span> <span class="given-names" tagx="given-names" title="given-names">A. K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Das</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Roy</span> <span class="given-names" tagx="given-names" title="given-names">S. K.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> ( <span class="year" tagx="year" title="year">2008</span>). <span class="mixed-article-title" title="mixed-article-title">Isolation and characterization of a heteroglycan from the fruits of <i>Astraeus hygrometricus</i>. </span> <span class="source" tagx="source" title="source"> <i>Carbohyd. Res.</i> </span> <span class="volume" tagx="volume" title="volume">343</span> <span class="fpage" tagx="fpage" title="fpage">817</span>– <span class="lpage" tagx="lpage" title="lpage">824</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.carres.2007.12.003">10.1016/j.carres.2007.12.003</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18206864">18206864</a> </span> </span> </li> <li tag="ref"> <a name="B62" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Marchand</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">St-Arnaud</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hogland</span> <span class="given-names" tagx="given-names" title="given-names">W.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bell</span> <span class="given-names" tagx="given-names" title="given-names">T. H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hijri</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> </span> ( <span class="year" tagx="year" title="year">2017</span>). <span class="mixed-article-title" title="mixed-article-title">Petroleum biodegradation capacity of bacteria and fungi isolated from petroleum-contaminated soil. </span> <span class="source" tagx="source" title="source"> <i>Int. Biodeter. Biodegr.</i> </span> <span class="volume" tagx="volume" title="volume">116</span> <span class="fpage" tagx="fpage" title="fpage">48</span>– <span class="lpage" tagx="lpage" title="lpage">57</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.ibiod.2016.09.030">10.1016/j.ibiod.2016.09.030</a> </span> </span> </li> <li tag="ref"> <a name="B63" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Masella</span> <span class="given-names" tagx="given-names" title="given-names">A. P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bartram</span> <span class="given-names" tagx="given-names" title="given-names">A. K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Truszkowski</span> <span class="given-names" tagx="given-names" title="given-names">J. M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Brown</span> <span class="given-names" tagx="given-names" title="given-names">D. G.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Neufeld</span> <span class="given-names" tagx="given-names" title="given-names">J. D.</span> </span> </span> ( <span class="year" tagx="year" title="year">2012</span>). <span class="mixed-article-title" title="mixed-article-title">PANDAseq: paired-end assembler for illumina sequences.</span> <span class="source" tagx="source" title="source"> <i>BMC Bioinformatics</i> </span> <span class="volume" tagx="volume" title="volume">13</span>: <span class="issue" tagx="issue" title="issue">31</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1186/1471-2105-13-31">10.1186/1471-2105-13-31</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22333067">22333067</a> </span> </span> </li> <li tag="ref"> <a name="B64" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">McDonald</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Price</span> <span class="given-names" tagx="given-names" title="given-names">M. N.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Goodrich</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nawrocki</span> <span class="given-names" tagx="given-names" title="given-names">E. P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Desantis</span> <span class="given-names" tagx="given-names" title="given-names">T. Z.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Probst</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> ( <span class="year" tagx="year" title="year">2012</span>). <span class="mixed-article-title" title="mixed-article-title">An improved Greengenes taxonomy with explicit ranks for ecological and evolutionary analyses of bacteria and archaea. </span> <span class="source" tagx="source" title="source"> <i>ISME J.</i> </span> <span class="volume" tagx="volume" title="volume">6</span> <span class="fpage" tagx="fpage" title="fpage">610</span>– <span class="lpage" tagx="lpage" title="lpage">618</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1038/ismej.2011.139">10.1038/ismej.2011.139</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22134646">22134646</a> </span> </span> </li> <li tag="ref"> <a name="B65" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Megharaj</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ramakrishnan</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Venkateswarlu</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sethunathan</span> <span class="given-names" tagx="given-names" title="given-names">N.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Naidu</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> </span> ( <span class="year" tagx="year" title="year">2011</span>). <span class="mixed-article-title" title="mixed-article-title">Bioremediation approaches for organic pollutants: a critical perspective.</span> <span class="source" tagx="source" title="source"> <i>Environ. Int.</i> </span> <span class="volume" tagx="volume" title="volume">37</span> <span class="fpage" tagx="fpage" title="fpage">1362</span>– <span class="lpage" tagx="lpage" title="lpage">1375</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.envint.2011.06.003">10.1016/j.envint.2011.06.003</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21722961">21722961</a> </span> </span> </li> <li tag="ref"> <a name="B66" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Micheluz</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sulyok</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Manente</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Krska</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Varese</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ravagnan</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span> </span> ( <span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">Fungal secondary metabolite analysis applied to cultural heritage: the case of a contaminated library in Venice. </span> <span class="source" tagx="source" title="source"> <i>World Mycotoxin J.</i> </span> <span class="volume" tagx="volume" title="volume">9</span> <span class="fpage" tagx="fpage" title="fpage">397</span>– <span class="lpage" tagx="lpage" title="lpage">407</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.3920/WMJ2015.1958">10.3920/WMJ2015.1958</a> </span> </span> </li> <li tag="ref"> <a name="B67" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Morais</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pylro</span> <span class="given-names" tagx="given-names" title="given-names">V.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Clark</span> <span class="given-names" tagx="given-names" title="given-names">I. M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hirsch</span> <span class="given-names" tagx="given-names" title="given-names">P. R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tótola</span> <span class="given-names" tagx="given-names" title="given-names">M. R.</span> </span> </span> ( <span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">Responses of microbial community from tropical pristine coastal soil to crude oil contamination. </span> <span class="source" tagx="source" title="source"> <i>PeerJ</i> </span> <span class="volume" tagx="volume" title="volume">4</span>: <span class="issue" tagx="issue" title="issue">e1733</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.7717/peerj.1733">10.7717/peerj.1733</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26925341">26925341</a> </span> </span> </li> <li tag="ref"> <a name="B68" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Moubasher</span> <span class="given-names" tagx="given-names" title="given-names">H. A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hegazy</span> <span class="given-names" tagx="given-names" title="given-names">A. K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mohamed</span> <span class="given-names" tagx="given-names" title="given-names">N. H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Moustafa</span> <span class="given-names" tagx="given-names" title="given-names">Y. M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kabiel</span> <span class="given-names" tagx="given-names" title="given-names">H. F.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hamad</span> <span class="given-names" tagx="given-names" title="given-names">A. A.</span> </span> </span> ( <span class="year" tagx="year" title="year">2015</span>). <span class="mixed-article-title" title="mixed-article-title">Phytoremediation of soils polluted with crude petroleum oil using <i>Bassia scoparia</i> and its associated rhizosphere microorganisms. </span> <span class="source" tagx="source" title="source"> <i>Int. Biodeter. Biodegr.</i> </span> <span class="volume" tagx="volume" title="volume">98</span> <span class="fpage" tagx="fpage" title="fpage">113</span>– <span class="lpage" tagx="lpage" title="lpage">120</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.ibiod.2014.11.019">10.1016/j.ibiod.2014.11.019</a> </span> </span> </li> <li tag="ref"> <a name="B69" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mulet</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">David</span> <span class="given-names" tagx="given-names" title="given-names">Z.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nogales</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bosch</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lalucat</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Garcia-Valdes</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span> </span> ( <span class="year" tagx="year" title="year">2011</span>). <span class="mixed-article-title" title="mixed-article-title"> <i>Pseudomonas</i> diversity in crude-oil-contaminated intertidal sand samples obtained after the Prestige oil spill. </span> <span class="source" tagx="source" title="source"> <i>Appl. Environ. Microbiol.</i> </span> <span class="volume" tagx="volume" title="volume">77</span> <span class="fpage" tagx="fpage" title="fpage">1076</span>– <span class="lpage" tagx="lpage" title="lpage">1085</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1128/AEM.01741-10">10.1128/AEM.01741-10</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21131512">21131512</a> </span> </span> </li> <li tag="ref"> <a name="B70" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Naranjo</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Urbina</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sisto</span> <span class="given-names" tagx="given-names" title="given-names">A. D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Leon</span> <span class="given-names" tagx="given-names" title="given-names">V.</span> </span> </span> ( <span class="year" tagx="year" title="year">2007</span>). <span class="mixed-article-title" title="mixed-article-title">Isolation of autochthonous non-white rot fungi with potential for enzymatic upgrading of Venezuelan extra-heavy crude oil. </span> <span class="source" tagx="source" title="source"> <i>Biocatal. Biotransfor.</i> </span> <span class="volume" tagx="volume" title="volume">25</span> <span class="fpage" tagx="fpage" title="fpage">341</span>– <span class="lpage" tagx="lpage" title="lpage">349</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1080/10242420701379908">10.1080/10242420701379908</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18833334">18833334</a> </span> </span> </li> <li tag="ref"> <a name="B71" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nesci</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Barros</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Castillo</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Etcheverry</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> </span> ( <span class="year" tagx="year" title="year">2006</span>). <span class="mixed-article-title" title="mixed-article-title">Soil fungal population in preharvest maize ecosystem in different tillage practices in Argentina. </span> <span class="source" tagx="source" title="source"> <i>Soil Till. Res.</i> </span> <span class="volume" tagx="volume" title="volume">91</span> <span class="fpage" tagx="fpage" title="fpage">143</span>– <span class="lpage" tagx="lpage" title="lpage">149</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.still.2005.11.014">10.1016/j.still.2005.11.014</a> </span> </span> </li> <li tag="ref"> <a name="B72" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Obi</span> <span class="given-names" tagx="given-names" title="given-names">L. U.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Atagana</span> <span class="given-names" tagx="given-names" title="given-names">H. I.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Adeleke</span> <span class="given-names" tagx="given-names" title="given-names">R. A.</span> </span> </span> ( <span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">Isolation and characterisation of crude oil sludge degrading bacteria.</span> <span class="source" tagx="source" title="source"> <i>SpringerPlus</i> </span> <span class="volume" tagx="volume" title="volume">5</span>: <span class="issue" tagx="issue" title="issue">1946</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1186/s40064-016-3617-z">10.1186/s40064-016-3617-z</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27933233">27933233</a> </span> </span> </li> <li tag="ref"> <a name="B73" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Oehl</span> <span class="given-names" tagx="given-names" title="given-names">F.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sieverding</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ineichen</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ris</span> <span class="given-names" tagx="given-names" title="given-names">E. A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Boller</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wiemken</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> </span> ( <span class="year" tagx="year" title="year">2005</span>). <span class="mixed-article-title" title="mixed-article-title">Community structure of arbuscular mycorrhizal fungi at different soil depths in extensively and intensively managed agroecosystems. </span> <span class="source" tagx="source" title="source"> <i>New Phytol.</i> </span> <span class="volume" tagx="volume" title="volume">165</span> <span class="fpage" tagx="fpage" title="fpage">273</span>– <span class="lpage" tagx="lpage" title="lpage">283</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1469-8137.2004.01235.x">10.1111/j.1469-8137.2004.01235.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15720639">15720639</a> </span> </span> </li> <li tag="ref"> <a name="B74" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Omrani</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Spini</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Puglisi</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Saidane</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> </span> ( <span class="year" tagx="year" title="year">2018</span>). <span class="mixed-article-title" title="mixed-article-title">Modulation of microbial consortia enriched from different polluted environments during petroleum biodegradation. </span> <span class="source" tagx="source" title="source"> <i>Biodegradation</i> </span> <span class="volume" tagx="volume" title="volume">29</span> <span class="fpage" tagx="fpage" title="fpage">187</span>– <span class="lpage" tagx="lpage" title="lpage">209</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s10532-018-9823-3">10.1007/s10532-018-9823-3</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/29492776">29492776</a> </span> </span> </li> <li tag="ref"> <a name="B75" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Phainuphong</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rukachaisirikul</span> <span class="given-names" tagx="given-names" title="given-names">V.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tadpetch</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sukpondma</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Saithong</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Phongpaichit</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> ( <span class="year" tagx="year" title="year">2017</span>). <span class="mixed-article-title" title="mixed-article-title">γ-butenolide and furanone derivatives from the soil-derived fungus <i>Aspergillus sclerotiorum</i> PSU-RSPG178. </span> <span class="source" tagx="source" title="source"> <i>Phytochemistry</i> </span> <span class="volume" tagx="volume" title="volume">137</span> <span class="fpage" tagx="fpage" title="fpage">165</span>– <span class="lpage" tagx="lpage" title="lpage">173</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.phytochem.2017.02.008">10.1016/j.phytochem.2017.02.008</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/28228227">28228227</a> </span> </span> </li> <li tag="ref"> <a name="B76" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ping</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Luo</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">Q.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wu</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span> </span> ( <span class="year" tagx="year" title="year">2007</span>). <span class="mixed-article-title" title="mixed-article-title">Distribution of polycyclic aromatic hydrocarbons in thirty typical soil profiles in the Yangtze river delta region, east China. </span> <span class="source" tagx="source" title="source"> <i>Environ. Poll.</i> </span> <span class="volume" tagx="volume" title="volume">147</span> <span class="fpage" tagx="fpage" title="fpage">358</span>– <span class="lpage" tagx="lpage" title="lpage">365</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.envpol.2006.05.027">10.1016/j.envpol.2006.05.027</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16815614">16815614</a> </span> </span> </li> <li tag="ref"> <a name="B77" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pinhassi</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hagstrom</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> </span> ( <span class="year" tagx="year" title="year">2000</span>). <span class="mixed-article-title" title="mixed-article-title">Seasonal succession in marine bacterioplanktoton.</span> <span class="source" tagx="source" title="source"> <i>Aquat. Microb. Ecol.</i> </span> <span class="volume" tagx="volume" title="volume">21</span> <span class="fpage" tagx="fpage" title="fpage">245</span>– <span class="lpage" tagx="lpage" title="lpage">256</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.3354/ame021245">10.3354/ame021245</a> </span> </span> </li> <li tag="ref"> <a name="B78" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Poli</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lazzari</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Prigione</span> <span class="given-names" tagx="given-names" title="given-names">V.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Voyron</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Spadaro</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Varese</span> <span class="given-names" tagx="given-names" title="given-names">G. C.</span> </span> </span> ( <span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">Influence of plant genotype on the cultivable fungi associated to tomato rhizosphere and roots in differnet soils. </span> <span class="source" tagx="source" title="source"> <i>Fungal Biol. U.K.</i> </span> <span class="volume" tagx="volume" title="volume">120</span> <span class="fpage" tagx="fpage" title="fpage">862</span>– <span class="lpage" tagx="lpage" title="lpage">872</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.funbio.2016.03.008">10.1016/j.funbio.2016.03.008</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27268246">27268246</a> </span> </span> </li> <li tag="ref"> <a name="B79" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pruesse</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Quast</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Knittel</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fuchs</span> <span class="given-names" tagx="given-names" title="given-names">B. M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ludwig</span> <span class="given-names" tagx="given-names" title="given-names">W.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Peplies</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> ( <span class="year" tagx="year" title="year">2007</span>). <span class="mixed-article-title" title="mixed-article-title">SILVA: a comprehensive online resource for quality checked and aligned ribosomal RNA sequence data compatible with ARB. </span> <span class="source" tagx="source" title="source"> <i>Nucleic Acids Res.</i> </span> <span class="volume" tagx="volume" title="volume">35</span> <span class="fpage" tagx="fpage" title="fpage">7188</span>– <span class="lpage" tagx="lpage" title="lpage">7196</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gkm864">10.1093/nar/gkm864</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17947321">17947321</a> </span> </span> </li> <li tag="ref"> <a name="B80" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rajasekar</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Babu</span> <span class="given-names" tagx="given-names" title="given-names">T. G.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pandian</span> <span class="given-names" tagx="given-names" title="given-names">S. T.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Maruthamuthu</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Palaniswamy</span> <span class="given-names" tagx="given-names" title="given-names">N.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rajendran</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> </span> ( <span class="year" tagx="year" title="year">2007</span>). <span class="mixed-article-title" title="mixed-article-title">Role of <i>Serratia marcescens</i> ACE2 on diesel degradation and its influence on corrosion. </span> <span class="source" tagx="source" title="source"> <i>J. Ind. Microbiol. Biotechnol.</i> </span> <span class="volume" tagx="volume" title="volume">34</span> <span class="fpage" tagx="fpage" title="fpage">589</span>– <span class="lpage" tagx="lpage" title="lpage">598</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s10295-007-0225-5">10.1007/s10295-007-0225-5</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17605058">17605058</a> </span> </span> </li> <li tag="ref"> <a name="B81" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rebecchi</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pisacane</span> <span class="given-names" tagx="given-names" title="given-names">V.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Miragoli</span> <span class="given-names" tagx="given-names" title="given-names">F.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Polka</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Falasconi</span> <span class="given-names" tagx="given-names" title="given-names">I.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Morelli</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> ( <span class="year" tagx="year" title="year">2015</span>). <span class="mixed-article-title" title="mixed-article-title">High-throughput assessment of bacterial ecology in hog, cow and ovine casings used in sausages production. </span> <span class="source" tagx="source" title="source"> <i>Int. J. Food. Microbiol.</i> </span> <span class="volume" tagx="volume" title="volume">212</span> <span class="fpage" tagx="fpage" title="fpage">49</span>– <span class="lpage" tagx="lpage" title="lpage">59</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.ijfoodmicro.2015.04.047">10.1016/j.ijfoodmicro.2015.04.047</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26003605">26003605</a> </span> </span> </li> <li tag="ref"> <a name="B82" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rojas-Avelizapa</span> <span class="given-names" tagx="given-names" title="given-names">N. G.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cervantes-Gonzalez</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cruz-Camarillo</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rojas-Avelizapa</span> <span class="given-names" tagx="given-names" title="given-names">L. I.</span> </span> </span> ( <span class="year" tagx="year" title="year">2002</span>). <span class="mixed-article-title" title="mixed-article-title">Degradation of aromatic and asphaltenic fractions by <i>Serratia liquefasciens</i> and <i>Bacillus</i> sp. </span> <span class="source" tagx="source" title="source"> <i>Bull. Environ. Contam. Toxicol.</i> </span> <span class="volume" tagx="volume" title="volume">69</span> <span class="fpage" tagx="fpage" title="fpage">835</span>– <span class="lpage" tagx="lpage" title="lpage">842</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s00128-002-0135-1">10.1007/s00128-002-0135-1</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12428160">12428160</a> </span> </span> </li> <li tag="ref"> <a name="B83" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rojo</span> <span class="given-names" tagx="given-names" title="given-names">F.</span> </span> </span> ( <span class="year" tagx="year" title="year">2009</span>). <span class="mixed-article-title" title="mixed-article-title">Degradation of alkanes by bacteria.</span> <span class="source" tagx="source" title="source"> <i>Environ. Microbiol.</i> </span> <span class="volume" tagx="volume" title="volume">11</span> <span class="fpage" tagx="fpage" title="fpage">2477</span>– <span class="lpage" tagx="lpage" title="lpage">2490</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1462-2920.2009.01948.x">10.1111/j.1462-2920.2009.01948.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19807712">19807712</a> </span> </span> </li> <li tag="ref"> <a name="B84" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Roy</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hens</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Biswas</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Biswas</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kumar</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> </span> ( <span class="year" tagx="year" title="year">2002</span>). <span class="mixed-article-title" title="mixed-article-title">Survey of petroleum-degrading bacteria in coastal waters of Sunderban biosphere reserve.</span> <span class="source" tagx="source" title="source"> <i>World J. Microb. Biotech.</i> </span> <span class="volume" tagx="volume" title="volume">18</span> <span class="fpage" tagx="fpage" title="fpage">575</span>– <span class="lpage" tagx="lpage" title="lpage">581</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1023/A:1016362819746">10.1023/A:1016362819746</a> </span> </span> </li> <li tag="ref"> <a name="B85" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sadouk</span> <span class="given-names" tagx="given-names" title="given-names">Z.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tazerouti</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hacene</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> </span> ( <span class="year" tagx="year" title="year">2008</span>). <span class="mixed-article-title" title="mixed-article-title">Biodegradation of diesel oil and production of fatty acid esters by a newly isolated <i>Pseudomonas</i> citronellolis KHA. </span> <span class="source" tagx="source" title="source"> <i>World J. Microb. Biotech.</i> </span> <span class="volume" tagx="volume" title="volume">25</span> <span class="fpage" tagx="fpage" title="fpage">65</span>– <span class="lpage" tagx="lpage" title="lpage">70</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s11274-008-9863-7">10.1007/s11274-008-9863-7</a> </span> </span> </li> <li tag="ref"> <a name="B86" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schloss</span> <span class="given-names" tagx="given-names" title="given-names">P. D.</span> </span> </span> ( <span class="year" tagx="year" title="year">2010</span>). <span class="mixed-article-title" title="mixed-article-title">The effects of alignment quality, distance calculation method, sequence filtering, and region on the analysis of 16S rRNA gene-based studies. </span> <span class="source" tagx="source" title="source"> <i>PLoS Comput. Biolog.</i> </span> <span class="volume" tagx="volume" title="volume">6</span>: <span class="issue" tagx="issue" title="issue">e1000844</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1371/journal.pcbi.1000844">10.1371/journal.pcbi.1000844</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20628621">20628621</a> </span> </span> </li> <li tag="ref"> <a name="B87" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schloss</span> <span class="given-names" tagx="given-names" title="given-names">P. D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Westcott</span> <span class="given-names" tagx="given-names" title="given-names">S. L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ryabin</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hall</span> <span class="given-names" tagx="given-names" title="given-names">J. R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hartmann</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hollister</span> <span class="given-names" tagx="given-names" title="given-names">E. B.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> ( <span class="year" tagx="year" title="year">2009</span>). <span class="mixed-article-title" title="mixed-article-title">Introducing mothur: open-source, platform-independent, community-supported software for describing and comparing microbial communities. </span> <span class="source" tagx="source" title="source"> <i>Appl. Environ. Microbiol.</i> </span> <span class="volume" tagx="volume" title="volume">75</span> <span class="fpage" tagx="fpage" title="fpage">7537</span>– <span class="lpage" tagx="lpage" title="lpage">7541</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1128/AEM.01541-09">10.1128/AEM.01541-09</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19801464">19801464</a> </span> </span> </li> <li tag="ref"> <a name="B88" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Scullion</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> </span> ( <span class="year" tagx="year" title="year">2006</span>). <span class="mixed-article-title" title="mixed-article-title">Remediating polluted soils.</span> <span class="source" tagx="source" title="source"> <i>Naturwissenschaften</i> </span> <span class="volume" tagx="volume" title="volume">93</span> <span class="fpage" tagx="fpage" title="fpage">51</span>– <span class="lpage" tagx="lpage" title="lpage">65</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s00114-005-0079-5">10.1007/s00114-005-0079-5</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16468025">16468025</a> </span> </span> </li> <li tag="ref"> <a name="B89" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Šimonovièová</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ferianc</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vojtková</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pangallo</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hanajík</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kraková</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> ( <span class="year" tagx="year" title="year">2017</span>). <span class="mixed-article-title" title="mixed-article-title">Alkaline technosol contaminated by former mining activity and its culturable autochthonous microbiota. </span> <span class="source" tagx="source" title="source"> <i>Chemosphere</i> </span> <span class="volume" tagx="volume" title="volume">171</span> <span class="fpage" tagx="fpage" title="fpage">89</span>– <span class="lpage" tagx="lpage" title="lpage">96</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.chemosphere.2016.11.131">10.1016/j.chemosphere.2016.11.131</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/28006667">28006667</a> </span> </span> </li> <li tag="ref"> <a name="B90" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Siqueira</span> <span class="given-names" tagx="given-names" title="given-names">J. P. Z.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sutton</span> <span class="given-names" tagx="given-names" title="given-names">D. A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">García</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gené</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Thomson</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wiederhold</span> <span class="given-names" tagx="given-names" title="given-names">N.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> ( <span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">Species diversity of <i>Aspergillus</i> section versicolores in clinical samples and antifungal susceptibility. </span> <span class="source" tagx="source" title="source"> <i>Fungal Biol. U.K.</i> </span> <span class="volume" tagx="volume" title="volume">120</span> <span class="fpage" tagx="fpage" title="fpage">1458</span>– <span class="lpage" tagx="lpage" title="lpage">1467</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.funbio.2016.02.006">10.1016/j.funbio.2016.02.006</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27742099">27742099</a> </span> </span> </li> <li tag="ref"> <a name="B91" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Smith</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Thavamani</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ramadass</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Naidu</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Srivastava</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Megharaj</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> </span> ( <span class="year" tagx="year" title="year">2015</span>). <span class="mixed-article-title" title="mixed-article-title">Remediation trials for hydrocarbon-contaminated soils in arid environments: evaluation of bioslurry and biopiling techniques. </span> <span class="source" tagx="source" title="source"> <i>Int. Biodeter.Biodegr.</i> </span> <span class="volume" tagx="volume" title="volume">101</span> <span class="fpage" tagx="fpage" title="fpage">56</span>– <span class="lpage" tagx="lpage" title="lpage">65</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.ibiod.2015.03.029">10.1016/j.ibiod.2015.03.029</a> </span> </span> </li> <li tag="ref"> <a name="B92" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Smits</span> <span class="given-names" tagx="given-names" title="given-names">T. H. M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ford</span> <span class="given-names" tagx="given-names" title="given-names">R. M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Keel</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Harms</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wick</span> <span class="given-names" tagx="given-names" title="given-names">L. Y.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kohlmeier</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> </span> ( <span class="year" tagx="year" title="year">2005</span>). <span class="mixed-article-title" title="mixed-article-title">Taking the fungal highway: mobilization of pollutant-degrading bacteria by fungi.</span> <span class="source" tagx="source" title="source"> <i>Environ. Sci. Tech.</i> </span> <span class="volume" tagx="volume" title="volume">39</span> <span class="fpage" tagx="fpage" title="fpage">4640</span>– <span class="lpage" tagx="lpage" title="lpage">4646</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1021/es047979z">10.1021/es047979z</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16047804">16047804</a> </span> </span> </li> <li tag="ref"> <a name="B93" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Staley</span> <span class="given-names" tagx="given-names" title="given-names">J. T.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Konopka</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> </span> ( <span class="year" tagx="year" title="year">1985</span>). <span class="mixed-article-title" title="mixed-article-title">Measurement of in situ activities of nonphotosynthetic microorganisms in aquatic and terrestrial habitats. </span> <span class="source" tagx="source" title="source"> <i>Ann. Rev. Microbiol.</i> </span> <span class="volume" tagx="volume" title="volume">39</span> <span class="fpage" tagx="fpage" title="fpage">321</span>– <span class="lpage" tagx="lpage" title="lpage">346</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1146/annurev.mi.39.100185.001541">10.1146/annurev.mi.39.100185.001541</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/3904603">3904603</a> </span> </span> </li> <li tag="ref"> <a name="B94" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Suja</span> <span class="given-names" tagx="given-names" title="given-names">F.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rahim</span> <span class="given-names" tagx="given-names" title="given-names">F.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Taha</span> <span class="given-names" tagx="given-names" title="given-names">M. R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hambali</span> <span class="given-names" tagx="given-names" title="given-names">N.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rizal Razali</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Khalid</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> ( <span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">Effects of local microbial bioaugmentation and biostimulation on the bioremediation of total petroleum hydrocarbons (TPH) in crude oil contaminated soil based on laboratory and field observations. </span> <span class="source" tagx="source" title="source"> <i>Int. Biodeter.Biodegr.</i> </span> <span class="volume" tagx="volume" title="volume">90</span> <span class="fpage" tagx="fpage" title="fpage">115</span>– <span class="lpage" tagx="lpage" title="lpage">122</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.ibiod.2014.03.006">10.1016/j.ibiod.2014.03.006</a> </span> </span> </li> <li tag="ref"> <a name="B95" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Thompson</span> <span class="given-names" tagx="given-names" title="given-names">I. P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Van Der Gast</span> <span class="given-names" tagx="given-names" title="given-names">C. J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ciric</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Singer</span> <span class="given-names" tagx="given-names" title="given-names">A. C.</span> </span> </span> ( <span class="year" tagx="year" title="year">2005</span>). <span class="mixed-article-title" title="mixed-article-title">Bioaugmentation for bioremediation: the challenge of strain selection.</span> <span class="source" tagx="source" title="source"> <i>Environ. Microbiol.</i> </span> <span class="volume" tagx="volume" title="volume">7</span> <span class="fpage" tagx="fpage" title="fpage">909</span>– <span class="lpage" tagx="lpage" title="lpage">915</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1462-2920.2005.00804.x">10.1111/j.1462-2920.2005.00804.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15946288">15946288</a> </span> </span> </li> <li tag="ref"> <a name="B96" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tian</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dong</span> <span class="given-names" tagx="given-names" title="given-names">Q.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yu</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhao</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span> </span> ( <span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">Biodegradation of the organophosphate trichlorfon and its major degradation products by a novel <i>Aspergillus sydowii</i> PA F-2. </span> <span class="source" tagx="source" title="source"> <i>J. Agr. Food Chem.</i> </span> <span class="volume" tagx="volume" title="volume">64</span> <span class="fpage" tagx="fpage" title="fpage">4280</span>– <span class="lpage" tagx="lpage" title="lpage">4287</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1021/acs.jafc.6b00909">10.1021/acs.jafc.6b00909</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27161040">27161040</a> </span> </span> </li> <li tag="ref"> <a name="B97" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Varjani</span> <span class="given-names" tagx="given-names" title="given-names">S. J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rana</span> <span class="given-names" tagx="given-names" title="given-names">D. P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jain</span> <span class="given-names" tagx="given-names" title="given-names">A. K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bateja</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Upasani</span> <span class="given-names" tagx="given-names" title="given-names">V. N.</span> </span> </span> ( <span class="year" tagx="year" title="year">2015</span>). <span class="mixed-article-title" title="mixed-article-title">Synergistic ex-situ biodegradation of crude oil by halotolerant bacterial consortium of indigenous strains isolated from on shore sites of Gujarat, India. </span> <span class="source" tagx="source" title="source"> <i>Int. Biodeter. Biodegr.</i> </span> <span class="volume" tagx="volume" title="volume">103</span> <span class="fpage" tagx="fpage" title="fpage">116</span>– <span class="lpage" tagx="lpage" title="lpage">124</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.ibiod.2015.03.030">10.1016/j.ibiod.2015.03.030</a> </span> </span> </li> <li tag="ref"> <a name="B98" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Varjani</span> <span class="given-names" tagx="given-names" title="given-names">S. J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Upasani</span> <span class="given-names" tagx="given-names" title="given-names">V. N.</span> </span> </span> ( <span class="year" tagx="year" title="year">2016</span>). <span class="mixed-article-title" title="mixed-article-title">Biodegradation of petroleum hydrocarbons by oleophilic strain of <i>Pseudomonas aeruginosa</i> NCIM 5514. </span> <span class="source" tagx="source" title="source"> <i>Bioresour. Technol.</i> </span> <span class="volume" tagx="volume" title="volume">222</span> <span class="fpage" tagx="fpage" title="fpage">195</span>– <span class="lpage" tagx="lpage" title="lpage">201</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.biortech.2016.10.006">10.1016/j.biortech.2016.10.006</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27718402">27718402</a> </span> </span> </li> <li tag="ref"> <a name="B99" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vasileiadis</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Puglisi</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Trevisa</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Scheckel</span> <span class="given-names" tagx="given-names" title="given-names">K. G.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Langdon</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">McLaughlin</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> ( <span class="year" tagx="year" title="year">2015</span>). <span class="mixed-article-title" title="mixed-article-title">Changes in soil bacterial communities and diversity in response to long-term silver exposure. </span> <span class="source" tagx="source" title="source"> <i>FEMS Microbiol. Ecol.</i> </span> <span class="volume" tagx="volume" title="volume">91</span>: <span class="issue" tagx="issue" title="issue">fiv114</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1093/femsec/fiv114">10.1093/femsec/fiv114</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26391377">26391377</a> </span> </span> </li> <li tag="ref"> <a name="B100" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Velmurugan</span> <span class="given-names" tagx="given-names" title="given-names">N.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kalpana</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cho</span> <span class="given-names" tagx="given-names" title="given-names">J.-Y.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lee</span> <span class="given-names" tagx="given-names" title="given-names">G.-H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Park</span> <span class="given-names" tagx="given-names" title="given-names">S.-H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lee</span> <span class="given-names" tagx="given-names" title="given-names">Y.-S.</span> </span> </span> ( <span class="year" tagx="year" title="year">2011</span>). <span class="mixed-article-title" title="mixed-article-title">Phylogenetic analysis of culturable marine bacteria in sediments from South Korean Yellow Sea. </span> <span class="source" tagx="source" title="source"> <i>Microbiology</i> </span> <span class="volume" tagx="volume" title="volume">80</span> <span class="fpage" tagx="fpage" title="fpage">261</span>– <span class="lpage" tagx="lpage" title="lpage">272</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1134/S0026261711010188">10.1134/S0026261711010188</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21675219">21675219</a> </span> </span> </li> <li tag="ref"> <a name="B101" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Versalovic</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schneider</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">De Bruijn</span> <span class="given-names" tagx="given-names" title="given-names">F. J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lupski</span> <span class="given-names" tagx="given-names" title="given-names">J. R.</span> </span> </span> ( <span class="year" tagx="year" title="year">1994</span>). <span class="mixed-article-title" title="mixed-article-title">Genomic fingerprinting of bacteria using repetitive sequence-based polymerase chain reaction. </span> <span class="source" tagx="source" title="source"> <i>Method Mol. Cell. Bio.</i> </span> <span class="volume" tagx="volume" title="volume">5</span> <span class="fpage" tagx="fpage" title="fpage">25</span>– <span class="lpage" tagx="lpage" title="lpage">40</span>. </span> </li> <li tag="ref"> <a name="B102" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vinas</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sabate</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Espuny</span> <span class="given-names" tagx="given-names" title="given-names">M. J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Solanas</span> <span class="given-names" tagx="given-names" title="given-names">A. M.</span> </span> </span> ( <span class="year" tagx="year" title="year">2005</span>). <span class="mixed-article-title" title="mixed-article-title">Bacterial community dynamics and polycyclic aromatic hydrocarbon degradation during bioremediation of heavily creosote-contaminated soil. </span> <span class="source" tagx="source" title="source"> <i>Appl. Environ. Microbiol.</i> </span> <span class="volume" tagx="volume" title="volume">71</span> <span class="fpage" tagx="fpage" title="fpage">7008</span>– <span class="lpage" tagx="lpage" title="lpage">7018</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1128/AEM.71.11.7008-7018.2005">10.1128/AEM.71.11.7008-7018.2005</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16269736">16269736</a> </span> </span> </li> <li tag="ref"> <a name="B103" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Voyron</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ercole</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ghignone</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Perotto</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Girlanda</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> </span> ( <span class="year" tagx="year" title="year">2017</span>). <span class="mixed-article-title" title="mixed-article-title">Fine-scale spatial distribution of orchid mycorrhizal fungi in the soil of host-rich grasslands. </span> <span class="source" tagx="source" title="source"> <i>New Phytol.</i> </span> <span class="volume" tagx="volume" title="volume">213</span> <span class="fpage" tagx="fpage" title="fpage">1428</span>– <span class="lpage" tagx="lpage" title="lpage">1439</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1111/nph.14286">10.1111/nph.14286</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27861936">27861936</a> </span> </span> </li> <li tag="ref"> <a name="B104" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">Q.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Garrity</span> <span class="given-names" tagx="given-names" title="given-names">G. M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tiedje</span> <span class="given-names" tagx="given-names" title="given-names">J. M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cole</span> <span class="given-names" tagx="given-names" title="given-names">J. R.</span> </span> </span> ( <span class="year" tagx="year" title="year">2007</span>). <span class="mixed-article-title" title="mixed-article-title">Naive Bayesian classifier for rapid assignment of rRNA sequences into the new bacterial taxonomy. </span> <span class="source" tagx="source" title="source"> <i>Appl. Environ. Microbio.</i> </span> <span class="volume" tagx="volume" title="volume">73</span> <span class="fpage" tagx="fpage" title="fpage">5261</span>– <span class="lpage" tagx="lpage" title="lpage">5267</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1128/AEM.00062-07">10.1128/AEM.00062-07</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17586664">17586664</a> </span> </span> </li> <li tag="ref"> <a name="B105" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">X. B.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chi</span> <span class="given-names" tagx="given-names" title="given-names">C. Q.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nie</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tang</span> <span class="given-names" tagx="given-names" title="given-names">Y. Q.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tan</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wu</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> ( <span class="year" tagx="year" title="year">2011</span>). <span class="mixed-article-title" title="mixed-article-title">Degradation of petroleum hydrocarbons (C6-C40) and crude oil by a novel <i>Dietzia</i> strain. </span> <span class="source" tagx="source" title="source"> <i>Bioresour. Technol.</i> </span> <span class="volume" tagx="volume" title="volume">102</span> <span class="fpage" tagx="fpage" title="fpage">7755</span>– <span class="lpage" tagx="lpage" title="lpage">7761</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.biortech.2011.06.009">10.1016/j.biortech.2011.06.009</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21715162">21715162</a> </span> </span> </li> <li tag="ref"> <a name="B106" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">White</span> <span class="given-names" tagx="given-names" title="given-names">T. J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bruns</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lee</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Taylor</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> </span> ( <span class="year" tagx="year" title="year">1990</span>). <span class="mixed-article-title" title="mixed-article-title">Amplification and direct sequencing of fungal ribosomal RNA genes for phylogenetics.</span> <span class="source" tagx="source" title="source"> <i>PCR Protocols</i> </span> <span class="volume" tagx="volume" title="volume">18</span> <span class="fpage" tagx="fpage" title="fpage">315</span>– <span class="lpage" tagx="lpage" title="lpage">322</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1186/s12866-017-1046-y">10.1186/s12866-017-1046-y</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/28651582">28651582</a> </span> </span> </li> <li tag="ref"> <a name="B107" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Whyte</span> <span class="given-names" tagx="given-names" title="given-names">L. G.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bourbonniere</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Greer</span> <span class="given-names" tagx="given-names" title="given-names">C. W.</span> </span> </span> ( <span class="year" tagx="year" title="year">1997</span>). <span class="mixed-article-title" title="mixed-article-title">Biodegradation of petroleum hydrocarbons by psychrotrophic <i>Pseudomonas</i> strains possessing both alkane (alk) and naphthalene (nah) catabolic pathways. </span> <span class="source" tagx="source" title="source"> <i>Appl. Environ. Microbiol.</i> </span> <span class="volume" tagx="volume" title="volume">63</span> <span class="fpage" tagx="fpage" title="fpage">3719</span>– <span class="lpage" tagx="lpage" title="lpage">3723</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9293024">9293024</a> </span> </span> </li> <li tag="ref"> <a name="B108" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wick</span> <span class="given-names" tagx="given-names" title="given-names">L. Y.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Remer</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Würz</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Reichenbach</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Braun</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schäfer</span> <span class="given-names" tagx="given-names" title="given-names">F.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Harms</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> </span> ( <span class="year" tagx="year" title="year">2007</span>). <span class="mixed-article-title" title="mixed-article-title">Effect of fungal hyphae on the access of bacteria to phenanthrene in soil.</span> <span class="source" tagx="source" title="source"> <i>Environ. Sci. Technol.</i> </span> <span class="volume" tagx="volume" title="volume">41</span> <span class="fpage" tagx="fpage" title="fpage">500</span>– <span class="lpage" tagx="lpage" title="lpage">505</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1021/es061407s">10.1021/es061407s</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17310713">17310713</a> </span> </span> </li> <li tag="ref"> <a name="B109" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Widdel</span> <span class="given-names" tagx="given-names" title="given-names">F.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rabus</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> </span> ( <span class="year" tagx="year" title="year">2001</span>). <span class="mixed-article-title" title="mixed-article-title">Anaerobic biodegradation of saturated and aromatic hydrocarbons.</span> <span class="source" tagx="source" title="source"> <i>Curr. Opin. Biotech.</i> </span> <span class="volume" tagx="volume" title="volume">12</span> <span class="fpage" tagx="fpage" title="fpage">259</span>– <span class="lpage" tagx="lpage" title="lpage">276</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/S0958-1669(00)00209-3">10.1016/S0958-1669(00)00209-3</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11404104">11404104</a> </span> </span> </li> <li tag="ref"> <a name="B110" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wongsa</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tanaka</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ueno</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hasanuzzaman</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yumoto</span> <span class="given-names" tagx="given-names" title="given-names">I.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Okuyama</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> </span> ( <span class="year" tagx="year" title="year">2004</span>). <span class="mixed-article-title" title="mixed-article-title">Isolation and characterization of novel strains of <i>Pseudomonas aeruginosa</i> and <i>Serratia marcescens</i> possessing high efficiency to degrade gasoline, kerosene, diesel oil, and lubricating oil. </span> <span class="source" tagx="source" title="source"> <i>Curr. Microbiol.</i> </span> <span class="volume" tagx="volume" title="volume">49</span> <span class="fpage" tagx="fpage" title="fpage">415</span>– <span class="lpage" tagx="lpage" title="lpage">422</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s00284-004-4347-y">10.1007/s00284-004-4347-y</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15696617">15696617</a> </span> </span> </li> <li tag="ref"> <a name="B111" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xia</span> <span class="given-names" tagx="given-names" title="given-names">W.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Du</span> <span class="given-names" tagx="given-names" title="given-names">Z.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cui</span> <span class="given-names" tagx="given-names" title="given-names">Q.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dong</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">F.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">He</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> ( <span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">Biosurfactant produced by novel <i>Pseudomonas</i> sp. WJ6 with biodegradation of n-alkanes and polycyclic aromatic hydrocarbons. </span> <span class="source" tagx="source" title="source"> <i>J. Hazard. Mater.</i> </span> <span class="volume" tagx="volume" title="volume">276</span> <span class="fpage" tagx="fpage" title="fpage">489</span>– <span class="lpage" tagx="lpage" title="lpage">498</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.jhazmat.2014.05.062">10.1016/j.jhazmat.2014.05.062</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24929788">24929788</a> </span> </span> </li> <li tag="ref"> <a name="B112" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yadav</span> <span class="given-names" tagx="given-names" title="given-names">T. C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pal</span> <span class="given-names" tagx="given-names" title="given-names">R. R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shastri</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jadeja</span> <span class="given-names" tagx="given-names" title="given-names">N. B.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kapley</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> </span> ( <span class="year" tagx="year" title="year">2015</span>). <span class="mixed-article-title" title="mixed-article-title">Comparative metagenomics demonstrating different degradative capacity of activated biomass treating hydrocarbon contaminated wastewater. </span> <span class="source" tagx="source" title="source"> <i>Bioresour. Technol.</i> </span> <span class="volume" tagx="volume" title="volume">188</span> <span class="fpage" tagx="fpage" title="fpage">24</span>– <span class="lpage" tagx="lpage" title="lpage">32</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.biortech.2015.01.141">10.1016/j.biortech.2015.01.141</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25727998">25727998</a> </span> </span> </li> <li tag="ref"> <a name="B113" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yakimov</span> <span class="given-names" tagx="given-names" title="given-names">M. M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Timmis</span> <span class="given-names" tagx="given-names" title="given-names">K. N.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Golyshin</span> <span class="given-names" tagx="given-names" title="given-names">P. N.</span> </span> </span> ( <span class="year" tagx="year" title="year">2007</span>). <span class="mixed-article-title" title="mixed-article-title">Obligate oil-degrading marine bacteria.</span> <span class="source" tagx="source" title="source"> <i>Curr. Opin. Biotech.</i> </span> <span class="volume" tagx="volume" title="volume">18</span> <span class="fpage" tagx="fpage" title="fpage">257</span>– <span class="lpage" tagx="lpage" title="lpage">266</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.copbio.2007.04.006">10.1016/j.copbio.2007.04.006</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17493798">17493798</a> </span> </span> </li> <li tag="ref"> <a name="B114" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yang</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wen</span> <span class="given-names" tagx="given-names" title="given-names">X.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jin</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wu</span> <span class="given-names" tagx="given-names" title="given-names">Q.</span> </span> </span> ( <span class="year" tagx="year" title="year">2012</span>). <span class="mixed-article-title" title="mixed-article-title">Pyrosequencing investigation into the bacterial community in permafrost soils along the China-Russia crude oil pipeline (CRCOP). </span> <span class="source" tagx="source" title="source"> <i>PLoS One</i> </span> <span class="volume" tagx="volume" title="volume">7</span>: <span class="issue" tagx="issue" title="issue">e52730</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1371/journal.pone.0052730">10.1371/journal.pone.0052730</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23300754">23300754</a> </span> </span> </li> <li tag="ref"> <a name="B115" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yergeau</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sanschagrin</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Beaumier</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Greer</span> <span class="given-names" tagx="given-names" title="given-names">C. W.</span> </span> </span> ( <span class="year" tagx="year" title="year">2012</span>). <span class="mixed-article-title" title="mixed-article-title">Metagenomic analysis of the bioremediation of diesel-contaminated Canadian high arctic soils. </span> <span class="source" tagx="source" title="source"> <i>PLoS One</i> </span> <span class="volume" tagx="volume" title="volume">7</span>: <span class="issue" tagx="issue" title="issue">e30058</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1371/journal.pone.0030058">10.1371/journal.pone.0030058</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22253877">22253877</a> </span> </span> </li> <li tag="ref"> <a name="B116" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yu</span> <span class="given-names" tagx="given-names" title="given-names">K. S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wong</span> <span class="given-names" tagx="given-names" title="given-names">A. H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yau</span> <span class="given-names" tagx="given-names" title="given-names">K. W.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wong</span> <span class="given-names" tagx="given-names" title="given-names">Y. S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tam</span> <span class="given-names" tagx="given-names" title="given-names">N. F.</span> </span> </span> ( <span class="year" tagx="year" title="year">2005</span>). <span class="mixed-article-title" title="mixed-article-title">Natural attenuation, biostimulation and bioaugmentation on biodegradation of polycyclic aromatic hydrocarbons (PAHs) in mangrove sediments. </span> <span class="source" tagx="source" title="source"> <i>Mar. Pollut. Bull.</i> </span> <span class="volume" tagx="volume" title="volume">51</span> <span class="fpage" tagx="fpage" title="fpage">1071</span>– <span class="lpage" tagx="lpage" title="lpage">1077</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.marpolbul.2005.06.006">10.1016/j.marpolbul.2005.06.006</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16023146">16023146</a> </span> </span> </li> <li tag="ref"> <a name="B117" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yuan</span> <span class="given-names" tagx="given-names" title="given-names">X.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">X.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">X.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kong</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">X.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shen</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> </span> ( <span class="year" tagx="year" title="year">2018</span>). <span class="mixed-article-title" title="mixed-article-title">Synergistic degradation of crude oil by indigenous bacterial consortium and exogenous fungus <i>Scedosporium boydii</i>. </span> <span class="source" tagx="source" title="source"> <i>Bioresour. Technol.</i> </span> <span class="volume" tagx="volume" title="volume">264</span> <span class="fpage" tagx="fpage" title="fpage">190</span>– <span class="lpage" tagx="lpage" title="lpage">197</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.biortech.2018.05.072">10.1016/j.biortech.2018.05.072</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/29803810">29803810</a> </span> </span> </li> <li tag="ref"> <a name="B118" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zafra</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Absalón</span> <span class="given-names" tagx="given-names" title="given-names">ÁE.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cuevas</span> <span class="given-names" tagx="given-names" title="given-names">M. D. C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cortés-Espinosa</span> <span class="given-names" tagx="given-names" title="given-names">D. V.</span> </span> </span> ( <span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">Isolation and selection of a highly tolerant microbial consortium with potential for PAH biodegradation from heavy crude oil-contaminated soils. </span> <span class="source" tagx="source" title="source"> <i>Water Air Soil Poll.</i> </span> <span class="volume" tagx="volume" title="volume">225</span> <span class="issue" tagx="issue" title="issue">1826</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s11270-013-1826-4">10.1007/s11270-013-1826-4</a> </span> </span> </li> <li tag="ref"> <a name="B119" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Guo</span> <span class="given-names" tagx="given-names" title="given-names">X.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhu</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">X.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Han</span> <span class="given-names" tagx="given-names" title="given-names">Z.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sun</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> ( <span class="year" tagx="year" title="year">2018</span>). <span class="mixed-article-title" title="mixed-article-title">The effects of different biochars on microbial quantity, microbial community shift, enzyme activity, and biodegradation of polycyclic aromatic hydrocarbons in soil. </span> <span class="source" tagx="source" title="source"> <i>Geoderma</i> </span> <span class="volume" tagx="volume" title="volume">328</span> <span class="fpage" tagx="fpage" title="fpage">100</span>– <span class="lpage" tagx="lpage" title="lpage">108</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.geoderma.2018.05.009">10.1016/j.geoderma.2018.05.009</a> </span> </span> </li> <li tag="ref"> <a name="B120" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhou</span> <span class="given-names" tagx="given-names" title="given-names">Z.-F.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">M.-X.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zuo</span> <span class="given-names" tagx="given-names" title="given-names">X.-H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yao</span> <span class="given-names" tagx="given-names" title="given-names">Y.-H.</span> </span> </span> ( <span class="year" tagx="year" title="year">2017</span>). <span class="mixed-article-title" title="mixed-article-title">Comparative investigation of bacterial, fungal, and archaeal community structures in soils in a typical oilfield in Jianghan, China. </span> <span class="source" tagx="source" title="source"> <i>Arch. Environ. Con. Tox.</i> </span> <span class="volume" tagx="volume" title="volume">72</span> <span class="fpage" tagx="fpage" title="fpage">65</span>– <span class="lpage" tagx="lpage" title="lpage">77</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s00244-016-0333-1">10.1007/s00244-016-0333-1</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27900422">27900422</a> </span> </span> </li> </ul> </div> </div>  </body></html>