<html xmlns="http://www.w3.org/1999/xhtml"><head> <style type="text/css">
         	     		 
         	    a                          {background : #ffffff; }
         	    article                    {border-style : dotted; border-width : 2px; }
         		 	
         		 	div                        {background : #ffffcc;}
         		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
         		 	div.ack                    {border-style : solid ; border-color : red; margin
         : 2em; }
         		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.boxed-text             {margin : 5em; border-style : solid;}
         		 	div.contrib-group          {margin : 1em; }
         		 	div.fig                    {border-style : solid; border-width : 2px; margin :
         2em; }
         		 	div.funding                {font-weight : bold ; font-size : 16pt;}
         		 	div.given-names            {font-style : italic;}
         		 	div.intro                  {border-style : inset; margin : 5px;}
         		 	div.introduction           {border-style : inset; margin : 5px;}
         		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.journal-title-group    {background : #ffeeee;}
         		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
         		 	div.kwd                    {font-style : italic}
         		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
         		 	div.name                   {font-weight : bold;}
         		 	div.alt-title              {font-style : italic; font-size : 12px;}
         		 	
         		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
         		 	div.title                  {font-family : courier; font-weight : bold;
         		 	                            font-size : 16pt; margin : 5px;}
         		 	
         		 	div.materials_methods      {border-style : double; margin : 5px; }
         		 	div.methods                {border-style : double; margin : 5px; }
         		 	
         		 	div.results                {border-style : solid; margin : 5px;}
         		 	div.background             {border-style : dotted; margin : 5px;}
         		 	
         		 	div.discussion             {border-style : groove; margin : 5px;}
         		 	div.conclusion             {border-style : ridge; margin : 5px;}
         		 	div.conclusions             {border-style : ridge; margin : 5px;}
         		 	div.supplementary-material {border-style : inset; margin : 5px;}
         		 	div.abbreviations          {border-style : double; border-color : red; margin
         : 5px;}
         		 	div.competinginterests     {border-style : double; border-color : blue; margin
         : 5px;}
         		 	div.acknowledgements       {border-style : double; border-color : green; margin
         : 5px;}
         		 	div.authors_contributions   {border-style : double; border-color : purple; margin
         : 5px;}
         		 	
         		 	div.publisher              {border-style : outset; margin : 5px;}
         		 	div.fn-type-conflict       {background : #f88; }
         		 	div.fn-type-con            {background : #ddf; }
         		 	div.fn-type-other          {background : #ddd; }
         		 	
         		 	div.unknown                {background : #ffd;
         									 	  border-style : solid;
         									 	  border-width : 1px;
         									 	  padding : 2 px;}
         		 	  
         	    table                      {background : #ffffdd;}
         		 	tr                         {background : #ddddff; padding : 1px;}
         		 	
         		 	span                       {background : #ffcccc;}
         		 	
         		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
         		 	span.collab                {background : #ddffff; }
         		 	span.comment               {font-family : courier; font-size : 6px; background
         : #ffaaff;}
         		 	span.contrib               {background : #ffffff;}
         		 	span.corresp               {background : #ddffdd; }
         		 	span.doi                   {background : #ffffff;}
         		 	span.email                 {font-family : courier; }
         		 	span.etal                  {font-style : italic;}
         		 	span.fpage                 {font-family : courier;}
         		 	span.given-names           {background : #ffffff;}
         		 	span.iso-abbrev            {background : #ffffff;}
         		 	span.issn-epub             {background : #ffffff;}
         		 	span.issn-ppub             {background : #ffffff;}
         		 	span.journal-title         {background : #ffffff;}
         		 	span.lpage                 {font-family : courier;}
         		 	span.mixed-article-title   {font-style : italic ;}
         		 	span.nlm-ta                {background : #ffffff;}
         		 	span.pmc                   {background : #ffffff;}
         		 	span.pmcid                 {background : #ffffff;}
         		 	span.pmid                  {background : #ffffff;}
         		 	span.publisher             {background : #ffffff;}
         		 	span.publisher-id          {background : #ffffff;}
         		 	span.publisher-name        {background : #ffffff;}
         		 	span.source                {background : #ffffff;}
         		 	span.subject               {background : #ffffff;}
         		 	span.surname               {background : #ffffff;}
         		 	span.volume                {font-family : courier; font-weight : bold;}
         		 	span.year                  {font-family : courier ; font-style : italic;}
         			
      </style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"> <span class="nlm-ta" title="nlm-ta">DNA Res</span> <span class="iso-abbrev" title="iso-abbrev">DNA Res</span> <span class="publisher-id" title="publisher-id">dnares</span> <div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"> <span class="journal-title" tagx="journal-title" title="journal-title">DNA Research: An International Journal for Rapid Publication of Reports on Genes and Genomes </span> </div> <span class="issn-ppub" tagx="issn" title="issn-ppub">1340-2838</span> <span class="issn-epub" tagx="issn" title="issn-epub">1756-1663</span> <div class="publisher" tagx="publisher" title="publisher"> <span class="publisher-name" tagx="publisher-name" title="publisher-name">Oxford University Press</span> </div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"> <span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/6289780">6289780</a> </span> <span class="pmid" title="pmid"> pmid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/30188980">30188980</a> </span> <span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.1093/dnares/dsy028">10.1093/dnares/dsy028</a> </span> <span class="publisher-id" title="publisher-id">dsy028</span> <div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Full Papers</span> </div> <div class="title-group"> <div class="article-title" title="article-title"> <i>De novo</i> transcriptome sequencing and metabolite profiling analyses reveal the complex metabolic genes involved in the terpenoid biosynthesis in Blue Anise Sage ( <i>Salvia guaranitica</i> L.) </div> </div> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ali</span> <span class="given-names" tagx="given-names" title="given-names">Mohammed</span> </span> <a href="#dsy028-aff1">1</a> <a href="#dsy028-aff2">2</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hussain</span> <span class="given-names" tagx="given-names" title="given-names">Reem M</span> </span> <a href="#dsy028-aff1">1</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rehman</span> <span class="given-names" tagx="given-names" title="given-names">Naveed Ur</span> </span> <a href="#dsy028-aff1">1</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">She</span> <span class="given-names" tagx="given-names" title="given-names">Guangbiao</span> </span> <a href="#dsy028-aff3">3</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">Penghui</span> </span> <a href="#dsy028-aff3">3</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wan</span> <span class="given-names" tagx="given-names" title="given-names">Xiaochun</span> </span> <a href="#dsy028-aff3">3</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Guo</span> <span class="given-names" tagx="given-names" title="given-names">Liang</span> </span> <a href="#dsy028-aff1">1</a> <a href="#dsy028-cor1" /> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhao</span> <span class="given-names" tagx="given-names" title="given-names">Jian</span> </span> <a href="#dsy028-aff1">1</a> <a href="#dsy028-aff3">3</a> <a href="#dsy028-cor1" /> </span> </div> <span class="citation_author_institution" id="dsy028-aff1">[1], </span> <span class="citation_author_institution" id="dsy028-aff2">[2], </span> <span class="citation_author_institution" id="dsy028-aff3">[3], </span> <div class="author-notes" title="author-notes"> <div class="corresp" title="corresp">
To whom correspondence should be addressed. Tel. +86 27 87385199. Email: <span class="email" tagx="email" title="email">guoliang@mail.hzau.edu.cn</span> (L.G.) ; Tel. +86 15527429202. Fax. +86 27 87385199. Email: <span class="email" tagx="email" title="email">jzhao2@qq.com</span> (J.Z.) </div> </div> <span class="pub-date-ppub" title="pub-date-ppub">ppub: <span>2018-12</span> </span> <span class="pub-date-epub" title="pub-date-epub">epub: <span>2018-9-9</span> </span> <span class="pub-date-pmc-release" title="pub-date-pmc-release">pmc-release: <span>2018-9-9</span> </span> <span class="volume" tagx="volume" title="volume">25</span> <span class="issue" tagx="issue" title="issue">6</span> <span class="fpage" tagx="fpage" title="fpage">597</span> <span class="lpage" tagx="lpage" title="lpage">617</span> <span class="history" title="history"> <span class="received" title="received">received: 2018-1-17</span> <span class="accepted" title="accepted">accepted: 2018-7-25</span> </span> <div class="permissions"> <span class="copyright" title="copyright">(C) , <span class="copyright-year" tagx="copyright-year" title="copyright-year">2018</span> </span> <span class="license" title="license"> <span class="license-p" title="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License ( <a href="http://creativecommons.org/licenses/by-nc/4.0/">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com </span> </span> </div> <a class="self-uri" href="dsy028.pdf" title="self-uri">dsy028.pdf</a> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <div class="title" tagx="title" title="title">
Abstract</div> <p>Many terpenoid compounds have been extracted from different tissues of <i>Salvia guaranitica.</i> However, the molecular genetic basis of terpene biosynthesis pathways is virtually unknown. In this study, approximately 4 Gb of raw data were generated from the transcriptome of <i>S. guaranitica</i> leaves using Illumina HiSeq 2000 sequencing. After filtering and removing the adapter sequences from the raw data, the number of reads reached 32 million, comprising 186 million of high-quality nucleotide bases. A total of 61,400 unigenes were assembled <i>de novo</i> and annotated for establishing a valid database for studying terpenoid biosynthesis. We identified 267 unigenes that are putatively involved in terpenoid metabolism (including, 198 mevalonate and methyl-erythritol phosphate (MEP) pathways, terpenoid backbone biosynthesis genes and 69 terpene synthases genes). Moreover, three terpene synthase genes were studied for their functions in terpenoid biosynthesis by using transgenic Arabidopsis; most transgenic Arabidopsis plants expressing these terpene synthetic genes produced increased amounts of terpenoids compared with wild-type control. The combined data analyses from the transcriptome and metabolome provide new insights into our understanding of the complex metabolic genes in terpenoid-rich blue anise sage, and our study paves the way for the future metabolic engineering of the biosynthesis of useful terpene compounds in <i>S. guaranitica</i>. </p> </div> <div class="kwd-group"> <div class="kwd" title="kwd"> <i>Salvia guaranitica</i> </div> <div class="kwd" title="kwd">
transcriptome</div> <div class="kwd" title="kwd">
terpene synthase genes</div> <div class="kwd" title="kwd">
transgenic Arabidopsis</div> <div class="kwd" title="kwd">
functional characterization</div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <span class="named-content" title="named-content">China Scholarship Council</span> <span class="named-content" title="named-content">10.13039/501100004543</span> </div> </div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <span class="named-content" title="named-content">CSC</span> <span class="named-content" title="named-content">10.13039/501100000867</span> </div> </div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <span class="named-content" title="named-content">National Basic Research Development Program of China</span> </div> <span class="award-id" tagx="award-id" title="award-id">2013CB127001</span> </div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <span class="named-content" title="named-content">Huazhong Agricultural University</span> <span class="named-content" title="named-content">10.13039/501100007925</span> </div> </div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source"> <span class="named-content" title="named-content">Anhui Agricultural University</span> </div> </div> </div> <div class="counts" title="counts"> <span class="page-count" title="page-count">page-count: </span> </div> </div> </div> <div class="body" title="body"> <div class="1.introduction" title="sec"> <div class="title" tagx="title" title="title">
1. Introduction</div> <p>Blue Anise Sage ( <i>Salvia guaranitica</i> L.), belongs to the genus <i>Salvia</i>, which is one of the economically best-known genera due to its vast medicinal properties and rich aromatic oils. The genus <i>Salvia</i> (tribe Mentheae) is the largest of the Lamiaceae family, which comprises nearly 1,000 species. <i>Salvia</i> plants are widely distributed in three regions around the world but mainly exist in Central and South America (~500 species), West Asia (~200 species) and East Asia (~100 species), while the other <i>Salvia</i> species are spread throughout the world. <a href="#dsy028-B1"> <sup>1</sup> </a> Most of these plants contain various medicinally active components used throughout history in folk medicine, e.g. <i>S. officinalis</i>, <i>S. japonica</i>, <i>S. santolinifolia</i>, <i>S. hydrangea</i>, <i>S. tomentosa</i>, <i>S. tuxtlensis</i>, <i>S. miltiorrhiza</i>, <i>S. chloroleuca</i>, <i>S. nipponica</i>, <i>S. fruticosa</i>, <i>S. aureus</i>, <i>S. przewalskii</i>, <i>S. epidermindis</i>, <i>S. isensis</i>, <i>S. lavandulifolia</i>, <i>S. glabrescens</i>, <i>S. allagospadonopsis</i>, <i>S. macrochlamys and S. recognita.</i> Recently, <i>Salvia</i> species have become a valuable source for pharmaceutical research for identifying and discovering biologically active compounds. <a href="#dsy028-B2"> <sup>2</sup> </a> Essential oils of <i>Salvia</i> species exhibit significant bioactivities, including antimicrobial, antimutagenic, anticancer, antioxidant, anti-inflammatory, choleretic and antimicrobial activities. <i>Salvia</i> essential oils contain more than 100 active compounds with pharmacological effects, and they can be categorized into monoterpenes, sesquiterpenes, diterpenes and triterpenes. <a href="#dsy028-B2"> <sup>2</sup> </a> During their biosynthesis, these terpenoids are sequentially built up from the isoprene units (C5) building block, isopentyl diphosphate (IPP) and dimethylallyl diphosphate (DMAPP). These components are condensed in a sequential manner by prenyltransferases, resulting in the formation of prenyl diphosphates, such as geranyl diphosphate (GPP), farnesyl pyrophosphate (FPP) and geranylgeranyl pyrophosphate (GGPP). <a href="#dsy028-B3"> <sup>3</sup> </a> These prenyl diphosphates are the immediate precursors for the biosynthesis of mono-, sesqui-, di- and tetraterpenes. Despite the scientific and medicinal interests in these terpenoids of <i>S. guaranitica</i>, the genes that are related to the biosynthesis of these compounds have not yet been fully identified or understood. Plant secondary metabolites have significant use in the food and pharmaceutical industries, such as in fine chemicals, and cosmetics. The biosynthesis, regulation and metabolic engineering of useful secondary metabolites have been extensively studied. <a href="#dsy028-B4"> <sup>4</sup> </a> In recent years, next-generation sequencing (NGS)-based RNA sequencing (RNA-Seq) has become a powerful tool for discovering genes that are involved in the biosynthesis of various secondary metabolite pathways in medicinal plants. <a href="#dsy028-B5"> <sup>5</sup> </a> For example, the volatile terpenoid biosynthesis in <i>Salvia officinalis</i>, <a href="#dsy028-B6"> <sup>6</sup> </a> the phenylpropanoid and terpenoid biosynthesis pathways in <i>Ocimum sanctum</i> and <i>Ocimum basilicum</i>, <a href="#dsy028-B7"> <sup>7</sup> </a> the biosynthesis of active ingredients in <i>Salvia miltiorrhiza</i>, <a href="#dsy028-B8"> <sup>8</sup> </a> the essential oil biosynthesis in aromatic <i>Cymbopogon flexuosus</i>, <a href="#dsy028-B9"> <sup>9</sup> </a> the biosynthesis of carotenoids in <i>Momordica cochinchinensis</i>, <a href="#dsy028-B10"> <sup>10</sup> </a> the biosynthesis of cellulose and lignin in <i>Cunninghamia lanceolata</i> <a href="#dsy028-B11"> <sup>11</sup> </a> and the biosynthesis of tea-specific compounds, i.e. catechins, caffeine and theanine pathways in <i>Camellia sinens</i>, <a href="#dsy028-B12"> <sup>12</sup> </a> have been explored by using NGS. Characterization of plant terpene synthases is typically carried out by the production of the recombinant enzymes in <i>Escherichia coli</i>. This is often difficult due to enzyme solubility and codon usage issues. Furthermore, plant terpene synthases that are localized to the plastids, such as diterpene synthases, must be abridged in a more or less experimental approach to ameliorate expression. <a href="#dsy028-B13"> <sup>13</sup> </a> <sup>,</sup> <a href="#dsy028-B14"> <sup>14</sup> </a> Transgenic Arabidopsis ( <i>A. thaliana</i>) is very efficient and has been successfully used for the characterization of one sesquiterpene synthase ( <i>PmSTS</i>) genes from <i>Polygonum minus</i>: β-sesquiphellandrene, and also has been successfully used for the characterization the strawberry linalool/nerolidol synthase (monoterpene) and taxadiene synthase. <a href="#dsy028-B15"> <sup>15</sup> </a> <sup>,</sup> <a href="#dsy028-B16"> <sup>16</sup> </a> Here, we characterized genes that are involved in terpenoid biosynthesis in <i>S. guaranitica</i> and determined their biological significance in <i>S. guaranitica</i> for terpenoid production in various tissues. In this study, a transcriptome database was established for <i>S. guaranitica</i> leaves using NGS technology to identify and to characterize genes that are related to the terpenoid biosynthesis pathway. The criteria used to achieve these objectives and to elucidate the complex metabolic pathways and genes for the understanding of terpenoid production in <i>S. guaranitica</i> included the following: (i) transcriptome analysis of leaves using Illumina HiSeq 2000 sequencing; (ii) GC-MS analysis for six fresh plant parts (old leaves, young leaves, stems, flowers, bud flowers and roots); (iii) characterization of three terpene genes in transgenic <i>A. thaliana</i>; (iv) qRT-PCR of highly expressed genes that are involved in the biosynthesis of terpenoids; (v) and the combination of data from the transcriptome, qRT-PCR and metabolome with GC-MS for revealing the functions of metabolic genes that are involved in the biosynthesis of valuable terpenoids. </p> </div> <div class="2.materialsandmethods" title="sec"> <div class="title" tagx="title" title="title">
2. Materials and methods</div> <div class="2.1.plantmaterialsandtissuecollection" title="sec"> <div class="title" tagx="title" title="title">
2.1. Plant materials and tissue collection</div> <p>Seedlings of <i>Salvia guaranitica</i> L. were collected from the Wuhan Botanical Garden, China, and grown at National Key Laboratory of Crop Genetic Improvement farm of Huazhong Agricultural University, Wuhan, China. Different tissues were sampled from one-year-old <i>S. guaranitica</i> plants. For RNA-Seq, three biological replicates from leaves were sampled and handled. Each replicate consisted of two young and two old leaves from the same plant. For qRT-PCR, three biological replicates were collected from the following six parts (old leaves, young leaves, stems, flowers, bud flowers and roots). All samples were directly frozen in liquid nitrogen and then stored at −80 °C until RNA extraction. Furthermore, another three biological replicates from the individual six fresh parts were collected for isolation of the essential oil. </p> </div> <div class="2.2.isolationofchemicalcompounds" title="sec"> <div class="title" tagx="title" title="title">
2.2. Isolation of chemical compounds</div> <p>The correct method to reduce technical variability throughout a sampling procedure is essential to stop cell metabolism and to avoid leaking of metabolites during the various preparation steps before the actual metabolite extraction. Therefore, three biological replicates from each of the six fresh parts were immediately frozen on dry ice. In the laboratory, the frozen three biological replicates from each of the three fresh part samples were homogenized with a mortar and pestle in liquid nitrogen, after which the plant material (ca. 10 g) was directly soaked in <i>n</i>-hexane as a solvent in Amber storage bottles, 60 ml screw-top vials with silicone/PTFE septum lids ( <a href="http://www.sigmaaldrich.com">http://www.sigmaaldrich.com</a>) were used to reduce loss of volatiles to the headspace then incubated with shaking at 37 °C and 200 rpm for 72 h. Afterward, the solvent was transferred using a glass pipette to a 10-ml glass centrifuge tube with screw-top vials with silicone/PTFE septum lids and centrifuged at 5,000 rpm for 10 min at 4 °C to remove plant debris. The supernatant was pipetted into glass vials with a screw cap, and oil was concentrated until remaining 1.5 ml of concentrated oils under a stream of nitrogen gas in a nitrogen evaporator (Organomation) with a water bath at room temperature (Toption-China-WD-12). The concentrated oils transferred to a fresh crimp vial amber glass, 1.5 ml screw-top vials with silicone/PTFE septum lids were used to reduce a loss of volatiles to the headspace. For absolute oil recovery, the remaining film crude oil in the internal surface of concentrated glass vials was dissolved in the minimum volume of <i>n</i>-hexane, thoroughly mixed and transferred to the same fresh crimp vial amber glass, 1.5 ml. And the crimp vial was placed on the auto-sampler of the gas chromatography-mass spectrometer (GC-MS) system for GC-MS analysis, or each tube was covered with parafilm after closed with screw-top vials with silicone/PTFE septum lids and stored at −20 °C until GC-MS analysis. <a href="#dsy028-B6"> <sup>6</sup> </a> </p> </div> <div class="2.3.gc-msanalysisofessentialoilcomponents" title="sec"> <div class="title" tagx="title" title="title">
2.3. GC-MS analysis of essential oil components</div> <p>GC analysis was performed using a Shimadzu model GCMS-QP2010 Ultra (Tokyo, Japan) system. An approximately 1 µl aliquot of each sample was injected (split ratios of 15: 1) into a GC-MS equipped with an HP-5 fused silica capillary column (30 m × 0.25 mm ID, 0.25 µm film thicknesses). And we used Helium as a carrier gas at a constant flow of 1.0 ml min <sup>−1</sup>. The mass spectra were monitored between 50 and 450  <i>m</i>/ <i>z</i>. Temperature was initially under isothermal conditions at 60 °C for 10 min. Temperature was then increased at a rate of 4 °C min <sup>−1</sup> to 220 °C, held isothermal at 220 °C for 10 min, increased by 1 °C min <sup>−1</sup> to 240 °C, held isothermal at 240 °C for 2 min and finally held isothermal for 10 min at 350 °C. The identification of the volatile constituents were determined by parallel comparison of their recorded mass spectra with the data stored in the Wiley GC/MS Library (10th edition) (Wiley, New York, NY, USA), the volatile organic compounds (VOC) analysis S/W software and the NIST Library (2014 edition). The relative % amount of each component was calculated by comparing its average peak area to the total areas, as well as retention time index. All of the experiments were performed simultaneously three times under the same conditions for each isolation technique with total GC running time was 80 min. <a href="#dsy028-B6"> <sup>6</sup> </a> </p> </div> <div class="2.4.rnaextraction" title="sec"> <div class="title" tagx="title" title="title">
2.4. RNA extraction</div> <p>Total RNAs from the three biological leaf replicates were extracted for RNA-Seq. Moreover, total RNAs from three biological replicates from each of the plant parts (old leaves, young leaves, stems, flowers, bud flowers and roots) were extracted for qRT-PCR. Additionally, total RNAs from three biological replicates of <i>A. thaliana</i> were extracted for semiquantitative RT-PCR using the TRIzol Reagent (Invitrogen, USA) and treated with DNase I (Takara). RNA quality was examined on 1% agarose gels, and the purity was analysed using a Nano-Photometer <sup>®</sup> spectrophotometer (IMPLEN, CA, USA). RNA concentration was determined using a Qubit <sup>®</sup> RNA Assay Kit in a Qubit <sup>®</sup> 2.0 Fluorometer (Life Technologies, CA, USA). RNA pools were prepared for cDNA libraries by mixing equal volumes from the three RNAs replications in one tube. </p> </div> <div class="2.5.cdnalibrarypreparationandsequencing" title="sec"> <div class="title" tagx="title" title="title">
2.5. cDNA library preparation and sequencing</div> <p>Three micrograms of RNA per sample were used for generating a sequencing library. cDNA was synthesized using an RNA Library Prep Kit for Illumina <sup>®</sup> (NEB, USA) for generated sequencing libraries according to the manufacturer’s instructions. The first strand of cDNA was synthesized in the presence of random hexamer primers and M-MuLV Reverse Transcriptase (RNase H), and the second strand of cDNA was synthesized in the presence of DNA Polymerase I and RNase H. The remaining cDNA was converted into blunt ends in the presence of exonuclease/polymerase activities. After the adenylation of three ends of DNA fragments, NEB Next, an adaptor with a hairpin loop structure, was ligated to prepare for hybridization. To select cDNA fragments of preferentially 150∼200 bp in length, the library fragments were purified using an AMPure XP system (Beckman Coulter, Beverly, USA). Then, 3 μl of USER Enzyme (NEB, USA) was used with size-selected, adaptor-ligated cDNA at 37 °C for 15 min followed by 95 °C for 5 min. Afterward, PCR was performed with Phusion High-Fidelity DNA polymerase, universal PCR primers and Index (X) Primer. Finally, PCR products were purified (AMPure XP system), and the library quality was assessed using an Agilent Bioanalyzer 2100 system (Agilent Technologies, CA, USA). Clustering of the index-coded samples was performed on a cBot Cluster Generation System using a TruSeq PE Cluster Kit v3-cBot-HS (Illumina) according to the manufacturer’s instructions (Novogene Experimental Department). After cluster generation, the library preparations were sequenced on an Illumina HiSeq 2000 platform, and paired-end (PE) reads were generated. </p> </div> <div class="2.6.qualitycontrol" title="sec"> <div class="title" tagx="title" title="title">
2.6. Quality control</div> <p>Raw data (raw reads) in the fastq format were first processed through in-house Perl scripts. During this step, clean data (clean reads) were obtained by removing reads containing adapters, reads containing ploy-N and low-quality reads from the raw data. At the same time, Q20, Q30, GC content and sequence duplication level of the clean data were calculated. All of the downstream analyses were based on high-quality clean data. </p> </div> <div class="2.7.denovotranscriptomeassembly" title="sec"> <div class="title" tagx="title" title="title">
2.7. <i>De novo</i> transcriptome assembly </div> <p> <i>De novo</i> assembly of the processed reads was carried out using Trinity program (Version: trinityaseq_r 2012-10-05) <sup>17,</sup> <a href="#dsy028-B18"> <sup>18</sup> </a> with the min_kmer_cov set to 2 by default and all other parameters set to default. The Trinity method consists of three software modules, (1) Inchworm, (2) Chrysalis and (3) Butterfly, applied sequentially to process large volumes of RNA-Seq reads. In the first step, read datasets were assembled into linear contigs by the first module (Inchworm program). The minimally overlapping contigs were then clustered into sets of connected components (build graph components) by the second module (Chrysalis program), and the transcripts were then constructed from each de Bruijn graph by the third software module (Butterfly program). Finally, the transcripts were clustered by a similarity of correct match length beyond 80% for longer transcripts or 90% for shorter transcripts using the multiple sequence alignment tool. The transcriptome data from <i>S. guaranitica</i> was submitted to the NCBI under submission ID (1955911). And the accession number from BankIt1954130 (KX869088) to BankIt1954278 (KX869125), and from BankIt1955703 (KX893913) to BankIt1955935 (KX894017) see <a href="#dsy028-T1">Tables 1</a> and <a href="#dsy028-T2">2</a>. And any inquiries about my submission should be sent to <span class="email" tagx="email" title="email">gb-admin@ncbi.nlm.nih.gov</span> or sent to <span class="email" tagx="email" title="email">info@ncbi.nlm.nih.gov</span>. <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 1.</span>  </div></p><p>Transcript abundance of MEP, MVA and other terpenoid backbone biosynthesis pathway genes as per the <i>S. guaranitica</i> transcriptome data annotation </p>  <table> <div class="colgroup_UNKNOWN"> <div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div>  <thead> <tr> <th>Pathway</th> <th>Gene name</th> <th>Kegg entry</th> <th>Gene bank accession ID</th> <th>EC.No.</th> <th>Read in leaf</th> <th>FPKM</th> </tr> </thead> <tbody> <tr> <td>MEP</td> <td> <i>SgDXS 1</i> </td> <td>K01662</td> <td>KX869088</td> <td>2.2.1.7</td> <td>8968.47</td> <td>223.94</td> </tr> <tr> <td> <i>SgDXS2</i> </td> <td>K01662</td> <td>KX869089</td> <td>2.2.1.7</td> <td>40</td> <td>41.89</td> </tr> <tr> <td> <i>SgDXS3</i> </td> <td>K01662</td> <td>KX869090</td> <td>2.2.1.7</td> <td>169</td> <td>3.68</td> </tr> <tr> <td> <i>SgDXS4</i> </td> <td>K01662</td> <td>KX869091</td> <td>2.2.1.7</td> <td>3634.82</td> <td>85.89</td> </tr> <tr> <td> <i>SgDXS5</i> </td> <td>K01662</td> <td>KX869092</td> <td>2.2.1.7</td> <td>697.93</td> <td>14.94</td> </tr> <tr> <td> <i>SgDXR 1</i> </td> <td>K00099</td> <td>KX869093</td> <td>1.1.1.267</td> <td>4080.05</td> <td>175.32</td> </tr> <tr> <td> <i>SgDXR 2</i> </td> <td>K00099</td> <td>KX869094</td> <td>1.1.1.267</td> <td>158.7</td> <td>37.97</td> </tr> <tr> <td> <i>SgMCT</i> </td> <td>K00991</td> <td>KX869095</td> <td>2.7.7.60</td> <td>563.54</td> <td>25.17</td> </tr> <tr> <td> <i>SgCMK</i> </td> <td>K00919</td> <td>KX869096</td> <td>2.7.1.148</td> <td>1588.67</td> <td>53.65</td> </tr> <tr> <td> <i>SgHDS 1</i> </td> <td>K03526</td> <td>KX869097</td> <td>1.17.7.1</td> <td>350</td> <td>13.57</td> </tr> <tr> <td> <i>SgHDS2</i> </td> <td>K03526</td> <td>KX869098</td> <td>1.17.7.1</td> <td>123.02</td> <td>8.35</td> </tr> <tr> <td> <i>SgHDS3</i> </td> <td>K03526</td> <td>KX869099</td> <td>1.17.7.1</td> <td>11</td> <td>4.08</td> </tr> <tr> <td> <i>SgHDS4</i> </td> <td>K03526</td> <td>KX869100</td> <td>1.17.7.1</td> <td>17316.85</td> <td>304.95</td> </tr> <tr> <td> <i>SgHDR 1</i> </td> <td>K03527</td> <td>KX869101</td> <td>1.17.1.2</td> <td>4</td> <td>2.72</td> </tr> <tr> <td> <i>SgHDR2</i> </td> <td>K03527</td> <td>KX869102</td> <td>1.17.1.2</td> <td>850</td> <td>175.65</td> </tr> <tr> <td> <i>SgHDR3</i> </td> <td>K03527</td> <td>KX869103</td> <td>1.17.1.2</td> <td>5</td> <td>2.7</td> </tr> <tr> <td> <i>SgHDR4</i> </td> <td>K03527</td> <td>KX869104</td> <td>1.17.1.2</td> <td>1034</td> <td>209.85</td> </tr> <tr> <td> <i>SgHDR5</i> </td> <td>K03527</td> <td>KX869105</td> <td>1.17.1.2</td> <td>296</td> <td>16.78</td> </tr> <tr> <td> <i>SgHDR6</i> </td> <td>K03527</td> <td>KX869106</td> <td>1.17.1.2</td> <td>858.92</td> <td>106.05</td> </tr> <tr> <td> <i>SgHDR7</i> </td> <td>K03527</td> <td>KX869107</td> <td>1.17.1.2</td> <td>67</td> <td>17.91</td> </tr> <tr> <td> <i>SgHDR8</i> </td> <td>K03527</td> <td>KX869108</td> <td>1.17.1.2</td> <td>9</td> <td>4.12</td> </tr> <tr> <td> <i>SgHDR9</i> </td> <td>K03527</td> <td>KX869109</td> <td>1.17.1.2</td> <td>42686.56</td> <td>1228.23</td> </tr> <tr> <td> <i>SgHDR10</i> </td> <td>K03527</td> <td>KX869110</td> <td>1.17.1.2</td> <td>43</td> <td>2.94</td> </tr> <tr> <td> <i>SgIDI1</i> </td> <td>K01823</td> <td>KX869111</td> <td>5.3.3.2</td> <td>2</td> <td>1.27</td> </tr> <tr> <td> <i>SgIDI2</i> </td> <td>K01823</td> <td>KX869112</td> <td>5.3.3.2</td> <td>2344.77</td> <td>98.18</td> </tr> <tr> <td> <i>SgIDI3</i> </td> <td>K01823</td> <td>KX869113</td> <td>5.3.3.2</td> <td>1</td> <td>1.44</td> </tr> <tr> <td>MVA</td> <td> <i>SgAACT 1</i> </td> <td>K00626</td> <td>KX869114</td> <td>2.3.1.9</td> <td>624.59</td> <td>22.69</td> </tr> <tr> <td> <i>SgAACT 2</i> </td> <td>K00626</td> <td>KX869115</td> <td>2.3.1.9</td> <td>2001.02</td> <td>67.38</td> </tr> <tr> <td> <i>SgHMGS</i> </td> <td>K01641</td> <td>KX869116</td> <td>2.3.3.10</td> <td>1897.92</td> <td>61.34</td> </tr> <tr> <td> <i>SgHMGR1</i> </td> <td>K00021</td> <td>KX869117</td> <td>1.1.1.34</td> <td>40</td> <td>20.67</td> </tr> <tr> <td> <i>SgHMGR2</i> </td> <td>K00021</td> <td>KX869118</td> <td>1.1.1.34</td> <td>2300.82</td> <td>56.02</td> </tr> <tr> <td> <i>SgHMGR3</i> </td> <td>K00021</td> <td>KX869119</td> <td>1.1.1.34</td> <td>23</td> <td>4.09</td> </tr> <tr> <td> <i>SgHMGR4</i> </td> <td>K00021</td> <td>KX869120</td> <td>1.1.1.34</td> <td>70</td> <td>11.16</td> </tr> <tr> <td> <i>SgHMGR5</i> </td> <td>K00021</td> <td>KX869121</td> <td>1.1.1.34</td> <td>144</td> <td>25.27</td> </tr> <tr> <td> <i>SgHMGR6</i> </td> <td>K00021</td> <td>KX869122</td> <td>1.1.1.34</td> <td>1691.49</td> <td>68.13</td> </tr> <tr> <td> <i>SgHMGR7</i> </td> <td>K00021</td> <td>KX869123</td> <td>1.1.1.34</td> <td>14</td> <td>2.17</td> </tr> <tr> <td> <i>SgHMGR8</i> </td> <td>K00021</td> <td>KX869124</td> <td>1.1.1.34</td> <td>39</td> <td>20.82</td> </tr> <tr> <td> <i>SgHMGR9</i> </td> <td>K00021</td> <td>KX869125</td> <td>1.1.1.34</td> <td>14</td> <td>1.25</td> </tr> <tr> <td> <i>SgMVK1</i> </td> <td>K00869</td> <td>KX893913</td> <td>2.7.1.36</td> <td>441.39</td> <td>15.27</td> </tr> <tr> <td> <i>SgMVK2</i> </td> <td>K00869</td> <td>KX893914</td> <td>2.7.1.36</td> <td>4</td> <td>1.83</td> </tr> <tr> <td> <i>SgPMK</i> </td> <td>K00938</td> <td>KX893915</td> <td>2.7.4.2</td> <td>754</td> <td>19.93</td> </tr> <tr> <td> <i>SgMDC</i> </td> <td>K01597</td> <td>KX893916</td> <td>4.1.1.33</td> <td>786.17</td> <td>56.1</td> </tr> <tr> <td>Monoterpene</td> <td> <i>SgGPPS</i> </td> <td>K14066</td> <td>KX893917</td> <td>2.5.1.1</td> <td>1292.8</td> <td>47.81</td> </tr> <tr> <td>Sesqui and triterpene</td> <td> <i>SgFPPS</i> </td> <td>K00787</td> <td>KX893918</td> <td>2.5.1.10</td> <td>1909.92</td> <td>80.21</td> </tr> <tr> <td>Diterpene</td> <td> <i>SgGGPSΙΙ1</i> </td> <td>K13789</td> <td>KX893925</td> <td>2.5.1.29</td> <td>1393</td> <td>60.95</td> </tr> <tr> <td> <i>SgGGPSΙΙ2</i> </td> <td>K13789</td> <td>KX893926</td> <td>2.5.1.29</td> <td>2153.95</td> <td>74.07</td> </tr> <tr> <td> <i>SgGGPSΙΙ3</i> </td> <td>K13789</td> <td>KX893919</td> <td>2.5.1.29</td> <td>2</td> <td>0.67</td> </tr> <tr> <td> <i>SgGGPSΙΙ4</i> </td> <td>K13789</td> <td>KX893920</td> <td>2.5.1.29</td> <td>125</td> <td>9.99</td> </tr> <tr> <td> <i>SgGGPSΙΙ5</i> </td> <td>K13789</td> <td>KX893921</td> <td>2.5.1.29</td> <td>54.01</td> <td>25.45</td> </tr> <tr> <td> <i>SgGGPSΙΙ6</i> </td> <td>K13789</td> <td>KX893927</td> <td>2.5.1.29</td> <td>180.01</td> <td>12.7</td> </tr> <tr> <td> <i>SgGGPSΙΙ7</i> </td> <td>K13789</td> <td>KX893922</td> <td>2.5.1.29</td> <td>25</td> <td>5.21</td> </tr> <tr> <td> <i>SgGGPSΙΙ8</i> </td> <td>K13789</td> <td>KX893923</td> <td>2.5.1.29</td> <td>44</td> <td>6.97</td> </tr> <tr> <td> <i>SgGGPSΙΙ9</i> </td> <td>K13789</td> <td>KX893924</td> <td>2.5.1.29</td> <td>16</td> <td>7.69</td> </tr> <tr> <td> <i>SgGGPSΙΙ10</i> </td> <td>K13789</td> <td>KX893928</td> <td>2.5.1.29</td> <td>673.01</td> <td>28.66</td> </tr> <tr> <td>Other terpenoid backbone biosynthesis</td> <td> <i>SgFOHSDR</i> </td> <td>K15890</td> <td>KX893929</td> <td>1.1.1.216</td> <td>241.13</td> <td>9.68</td> </tr> <tr> <td> <i>SgFOLK1</i> </td> <td>K15892</td> <td>KX893930</td> <td>2.7.1.</td> <td>998.85</td> <td>85.9</td> </tr> <tr> <td> <i>SgFOLK2</i> </td> <td>K15892</td> <td>KX893931</td> <td>2.7.1.</td> <td>892.1</td> <td>43.74</td> </tr> <tr> <td> <i>SgPCYOX1</i> </td> <td>K05906</td> <td>KX893932</td> <td>1.8.3.5 1.8.3.6</td> <td>523.75</td> <td>14.1</td> </tr> <tr> <td> <i>SgSTE24</i> </td> <td>K06013</td> <td>KX893933</td> <td>3.4.24.84</td> <td>1467.53</td> <td>48.98</td> </tr> <tr> <td> <i>SgCHLP1</i> </td> <td>K10960</td> <td>KX893934</td> <td>1.3.1.83</td> <td>14506.79</td> <td>419.11</td> </tr> <tr> <td> <i>SgCHLP2</i> </td> <td>K10960</td> <td>KX893935</td> <td>1.3.1.83</td> <td>83</td> <td>10.71</td> </tr> <tr> <td> <i>SgCHLP3</i> </td> <td>K10960</td> <td>KX893936</td> <td>1.3.1.83</td> <td>675.74</td> <td>26.22</td> </tr> <tr> <td> <i>SgFACE2</i> </td> <td>K08658</td> <td>KX893937</td> <td>3.4.22.-</td> <td>297</td> <td>36.52</td> </tr> <tr> <td> <i>SgPCME1</i> </td> <td>K15889</td> <td>KX893938</td> <td>3.1.1.-</td> <td>232.82</td> <td>11.54</td> </tr> <tr> <td> <i>SgPCME2</i> </td> <td>K15889</td> <td>KX893939</td> <td>3.1.1.-</td> <td>36</td> <td>1.81</td> </tr> <tr> <td> <i>SgFNTB</i> </td> <td>K05954</td> <td>KX893940</td> <td>2.5.1.58</td> <td>626.09</td> <td>21.79</td> </tr> <tr> <td> <i>SgSPS</i> </td> <td>K05356</td> <td>KX893941</td> <td>2.5.1.84 2.5.1.85</td> <td>6500.62</td> <td>224.55</td> </tr> <tr> <td> <i>SgDHDDS1</i> </td> <td>K11778</td> <td>KX893942</td> <td>2.5.1.87</td> <td>129.01</td> <td>7.82</td> </tr> <tr> <td> <i>SgDHDDS2</i> </td> <td>K11778</td> <td>KX893943</td> <td>2.5.1.87</td> <td>4818.35</td> <td>227.35</td> </tr> <tr> <td> <i>SgDHDDS3</i> </td> <td>K11778</td> <td>KX893944</td> <td>2.5.1.87</td> <td>778.03</td> <td>31.45</td> </tr> <tr> <td> <i>SgDHDDS4</i> </td> <td>K11778</td> <td>KX893945</td> <td>2.5.1.87</td> <td>3175</td> <td>179.52</td> </tr> <tr> <td> <i>SgDHDDS5</i> </td> <td>K11778</td> <td>KX893946</td> <td>2.5.1.87</td> <td>219</td> <td>9.45</td> </tr> <tr> <td> <i>SgICMT1</i> </td> <td>K00587</td> <td>KX893947</td> <td>2.1.1.100</td> <td>187</td> <td>15.99</td> </tr> <tr> <td> <i>SgICMT2</i> </td> <td>K00587</td> <td>KX893948</td> <td>2.1.1.100</td> <td>119</td> <td>12.97</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p>FPKM, fragments per kilobase of transcripts per million mapped fragments; DXS, 1-deoxy- <div class="sc_UNKNOWN">
d</div>-xylulose-5-phosphate synthase; DXR, 1-deoxy- <div class="sc_UNKNOWN">
d</div>-xylulose-5-phosphate reductoisomerase; MCT, 2-C-methyl- <div class="sc_UNKNOWN">
d</div>-erythritol 4-phosphate cytidylyltransferase; CMK, 4-diphosphocytidyl-2-C-methyl-d-erythritol kinase; HDS, (E)-4-hydroxy-3-methylbut-2-enyl-diphosphate synthase; HDR, 4-hydroxy-3-methylbut-2-enyl diphosphate reductase; IDI, isopentenyl-diphosphate delta-isomerase, AACT, acetyl-CoA C-acetyltransferase; HMGS, hydroxylmethylglutaryl-CoA synthase; HMGR, hydroxymethylglutaryl-CoA reductase (NADPH); MVK, mevalonate kinase; PMK, 5-phosphomevalonate kinase, MDC, mevalonate diphosphate decarboxylase; GPPS, geranyl diphosphate synthase; FPPS, farnesyl pyrophosphate synthase; GGPS, geranylgeranyldiphosphate synthase, type II; FOHSDR, farnesol dehydrogenase; FOLK, farnesol kinase; PCYOX1, prenylcysteine oxidases/farnesylcysteine lyase; STE24, STE24 endopeptidases; CHLP, geranylgeranyl diphosphate reductase; FACE2, farnesylated protein-converting enzyme 2; PCME, prenylcysteine alpha-carboxyl methylesterase; FNTB, protein farnesyltransferase subunit beta; SPS, all-trans-nonaprenyl-diphosphate synthase; DHDDS, ditrans, polycis-polyprenyl diphosphate synthase; ICMT, protein-S-isoprenylcysteine O-methyltransferase. </p> </div> </div>  <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 2.</span>  <p>Transcript abundance of TPS genes as per the <i>S. guaranitica</i> transcriptome </p>  <table> <div class="colgroup_UNKNOWN"> <div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div>  <thead> <tr> <th>Terpene synthase</th> <th>Kegg entry</th> <th>Gene bank accession ID</th> <th>Annotation</th> <th>Length (bp)</th> <th>E.C. no.</th> <th>Read in leaf</th> <th>FPKM</th> </tr> </thead> <tbody> <tr> <td>Monoterpene</td> <td>K12467</td> <td>KX893949</td> <td>Myrcene/ocimene synthase</td> <td>371</td> <td>4.2.3.15</td> <td>7</td> <td>1.92</td> </tr> <tr> <td>K12467</td> <td>KX893950</td> <td>Myrcene/ocimene synthase</td> <td>223</td> <td>4.2.3.15</td> <td>3</td> <td>4.41</td> </tr> <tr> <td>K12467</td> <td>KX893951</td> <td>Myrcene/ocimene synthase</td> <td>217</td> <td>4.2.3.15</td> <td>0</td> <td>0</td> </tr> <tr> <td>K12467</td> <td>KX893952</td> <td>Myrcene/ocimene synthase</td> <td>208</td> <td>4.2.3.15</td> <td>3</td> <td>6.39</td> </tr> <tr> <td>K12467</td> <td>KX893953</td> <td>Myrcene/ocimene synthase</td> <td>449</td> <td>4.2.3.15</td> <td>11</td> <td>2.05</td> </tr> <tr> <td>K12467</td> <td>KX893954</td> <td>Myrcene/ocimene synthase</td> <td>366</td> <td>4.2.3.15</td> <td>5</td> <td>1.41</td> </tr> <tr> <td>K15095</td> <td>KX893955</td> <td>(+)-neomenthol dehydrogenase1</td> <td>2831</td> <td>1.1.1.208</td> <td>2112.05</td> <td>97.37</td> </tr> <tr> <td>K15095</td> <td>KX893956</td> <td>(+)-neomenthol dehydrogenase2</td> <td>1065</td> <td>1.1.1.208</td> <td>122.03</td> <td>9.02</td> </tr> <tr> <td>K15095</td> <td>KX893957</td> <td>(+)-neomenthol dehydrogenase3</td> <td>1660</td> <td>1.1.1.208</td> <td>3417.92</td> <td>106.62</td> </tr> <tr> <td>K15095</td> <td>KX893958</td> <td>(+)-neomenthol dehydrogenase</td> <td>352</td> <td>1.1.1.208</td> <td>4</td> <td>1.24</td> </tr> <tr> <td>K15095</td> <td>KX893959</td> <td>(+)-neomenthol dehydrogenase</td> <td>216</td> <td>1.1.1.208</td> <td>21.01</td> <td>36.39</td> </tr> <tr> <td>K15095</td> <td>KX893960</td> <td>(+)-neomenthol dehydrogenase</td> <td>232</td> <td>1.1.1.208</td> <td>7</td> <td>8.53</td> </tr> <tr> <td>K07385</td> <td>KX893961</td> <td>1, 8-cineole synthase</td> <td>303</td> <td>4.2.3.108</td> <td>3</td> <td>1.37</td> </tr> <tr> <td>K07385</td> <td>KX893962</td> <td>1, 8-cineole synthase</td> <td>230</td> <td>4.2.3.108</td> <td>3</td> <td>3.8</td> </tr> <tr> <td>K07385</td> <td>KX893963</td> <td>1, 8-cineole synthase</td> <td>268</td> <td>4.2.3.108</td> <td>7</td> <td>4.76</td> </tr> <tr> <td>K07385</td> <td>KX893964</td> <td>1, 8-cineole synthase</td> <td>2338</td> <td>4.2.3.108</td> <td>896.71</td> <td>29.15</td> </tr> <tr> <td>K15086</td> <td>KX893965</td> <td>(3S)-linalool synthase1</td> <td>2099</td> <td>4.2.3.25</td> <td>2191.73</td> <td>64.89</td> </tr> <tr> <td>K15086</td> <td>KX893966</td> <td>(3S)-linalool synthase2</td> <td>1251</td> <td>4.2.3.25</td> <td>521</td> <td>13.14</td> </tr> <tr> <td>K17982</td> <td>KX893967</td> <td>(E, E)-geranyl linalool synthase</td> <td>2541</td> <td>4.2.3.144</td> <td>1277.71</td> <td>22.13</td> </tr> <tr> <td>K15099</td> <td>KX893968</td> <td>Geraniol isomerase synthase</td> <td>823</td> <td>1.14.13.152</td> <td>33</td> <td>2.36</td> </tr> <tr> <td>Sesquiterpene</td> <td>K15891</td> <td>KX893969</td> <td>Farnesol dehydrogenase</td> <td>1385</td> <td>1.1.1.216</td> <td>241.13</td> <td>9.68</td> </tr> <tr> <td>K14184</td> <td>KX893970</td> <td>α-humulene/β-caryophyllene synthase</td> <td>497</td> <td>4.2.3.57</td> <td>11</td> <td>1.71</td> </tr> <tr> <td>K14184</td> <td>KX893971</td> <td>α-humulene/β-caryophyllene synthase</td> <td>212</td> <td>4.2.3.57</td> <td>2</td> <td>3.83</td> </tr> <tr> <td>K14184</td> <td>KX893972</td> <td>α-humulene/β-caryophyllene synthase</td> <td>299</td> <td>4.2.3.57</td> <td>2</td> <td>0.95</td> </tr> <tr> <td>K14184</td> <td>KX893973</td> <td>α-humulene/β-caryophyllene synthase</td> <td>1425</td> <td>4.2.3.57</td> <td>374</td> <td>14.89</td> </tr> <tr> <td>K14181</td> <td>KX893974</td> <td>Valencene synthase (TPS-V)</td> <td>1653</td> <td>4.2.3.73 4.2.3.86</td> <td>1901</td> <td>51.85</td> </tr> <tr> <td>K14179</td> <td>KX893975</td> <td>Germacrene- A synthase (TPS-1)</td> <td>858</td> <td>4.2.3.23</td> <td>192</td> <td>13.53</td> </tr> <tr> <td>K15809 </td> <td>KX893976</td> <td> <i>Cis</i>-muuroladiene synthase </td> <td>1695</td> <td>4.2.3.67</td> <td>619.04</td> <td>19.21</td> </tr> <tr> <td>K15803</td> <td>KX893977</td> <td>Germacrene- <div class="sc_UNKNOWN">
d</div> synthase (TPS-6) </td> <td>1764</td> <td>4.2.3.22 4.2.3.75</td> <td>116</td> <td>3.35</td> </tr> <tr> <td>K15806</td> <td>KX893978</td> <td>Selinene synthase (TPS-3)</td> <td>1680</td> <td>4.2.3.76</td> <td>2141.91</td> <td>59.35</td> </tr> <tr> <td>K14183</td> <td>KX893979</td> <td>Gamma-cadinene synthase</td> <td>1533</td> <td>4.2.3.13</td> <td>67</td> <td>2.07</td> </tr> <tr> <td>k01000</td> <td>KX893980</td> <td>Bicyclogermacrene synthase (TPS-4)</td> <td>1326</td> <td>4.2.3.100</td> <td>2020.06</td> <td>64.65</td> </tr> <tr> <td>Diterpene</td> <td>K13070</td> <td>KX893981</td> <td>Momilactone-A synthase</td> <td>374</td> <td>1.1.1.295</td> <td>8</td> <td>2.15</td> </tr> <tr> <td>K13070</td> <td>KX893982</td> <td>Momilactone-A synthase</td> <td>347</td> <td>1.1.1.295</td> <td>5</td> <td>1.6</td> </tr> <tr> <td>K04124</td> <td>KX893983</td> <td>Gibberellin 3-beta-dioxygenase</td> <td>815</td> <td>1.14.11.15</td> <td>19</td> <td>1.41</td> </tr> <tr> <td>K04124</td> <td>KX893984</td> <td>Gibberellin 3-beta-dioxygenase</td> <td>319</td> <td>1.14.11.15</td> <td>8</td> <td>3.17</td> </tr> <tr> <td>K04125</td> <td>KX893985</td> <td>Gibberellin 2-oxidase</td> <td>1307</td> <td>1.14.11.13</td> <td>480.99</td> <td>31.01</td> </tr> <tr> <td>K04125</td> <td>KX893986</td> <td>Gibberellin 2-oxidase</td> <td>214</td> <td>1.14.11.13</td> <td>1</td> <td>1.82</td> </tr> <tr> <td>K04120</td> <td>KX893987</td> <td>Ent-copalyl diphosphate synthase</td> <td>1298</td> <td>5.5.1.13</td> <td>69</td> <td>2.87</td> </tr> <tr> <td>K04120</td> <td>KX893988</td> <td>Ent-copalyl diphosphate synthase</td> <td>2497</td> <td>5.5.1.13</td> <td>2616.28</td> <td>51.8</td> </tr> <tr> <td>K04120</td> <td>KX893989</td> <td>Ent-copalyl diphosphate synthase</td> <td>642</td> <td>5.5.1.13</td> <td>16</td> <td>1.66</td> </tr> <tr> <td>K04120</td> <td>KX893990</td> <td>Ent-copalyl diphosphate synthase</td> <td>213</td> <td>5.5.1.13</td> <td>1</td> <td>1.87</td> </tr> <tr> <td>K04120</td> <td>KX893991</td> <td>Ent-copalyl diphosphate synthase</td> <td>255</td> <td>5.5.1.13</td> <td>2</td> <td>1.63</td> </tr> <tr> <td>K04120</td> <td>KX893992</td> <td>Ent-copalyl diphosphate synthase</td> <td>345</td> <td>5.5.1.13</td> <td>6</td> <td>1.94</td> </tr> <tr> <td>K04120</td> <td>KX893993</td> <td>Ent-copalyl diphosphate synthase</td> <td>691</td> <td>5.5.1.13</td> <td>21</td> <td>3.06</td> </tr> <tr> <td>K04120</td> <td>KX893994</td> <td>Ent-copalyl diphosphate synthase</td> <td>300</td> <td>5.5.1.13</td> <td>4</td> <td>1.88</td> </tr> <tr> <td>K04121</td> <td>KX893995</td> <td>Ent-kaurene synthase-1</td> <td>326</td> <td>4.2.3.19</td> <td>11</td> <td>4.11</td> </tr> <tr> <td>K04121</td> <td>KX893996</td> <td>Ent-kaurene synthase-5</td> <td>252</td> <td>4.2.3.19</td> <td>6</td> <td>5.14</td> </tr> <tr> <td>K04121</td> <td>KX893997</td> <td>Ent-kaurene synthase-3</td> <td>227</td> <td>4.2.3.19</td> <td>2</td> <td>2.7</td> </tr> <tr> <td>K04121</td> <td>KX893998</td> <td>Ent-kaurene synthase-4</td> <td>1541</td> <td>4.2.3.19</td> <td>126</td> <td>4.46</td> </tr> <tr> <td>K04121</td> <td>KX893999</td> <td>Ent-kaurene synthase-2</td> <td>1151</td> <td>4.2.3.19</td> <td>65</td> <td>3.21</td> </tr> <tr> <td>K04121</td> <td>KX894000</td> <td>Ent-kaurene synthase-6</td> <td>1646</td> <td>4.2.3.19</td> <td>164</td> <td>7.12</td> </tr> <tr> <td>K04121</td> <td>KX894001</td> <td>Ent-kaurene synthase-7</td> <td>1743</td> <td>4.2.3.19</td> <td>372</td> <td>15.7</td> </tr> <tr> <td>K04123</td> <td>KX894002</td> <td>Ent-kaurenoic acid hydroxylase</td> <td>2463</td> <td>1.14.13.79</td> <td>361.97</td> <td>8.26</td> </tr> <tr> <td>K04123</td> <td>KX894003</td> <td>Ent-kaurenoic acid hydroxylase</td> <td>1766</td> <td>1.14.13.79</td> <td>253</td> <td>7.36</td> </tr> <tr> <td>K16085</td> <td>KX894004</td> <td>9beta-pimara-7, 15-diene oxidase</td> <td>434</td> <td>1.14.13.144</td> <td>133.16</td> <td>26.44</td> </tr> <tr> <td>K16083</td> <td>KX894005</td> <td>Ent-isokaurene C2-hydroxylase</td> <td>415</td> <td>1.14.13.143</td> <td>19</td> <td>4.11</td> </tr> <tr> <td>K05282</td> <td>KX894006</td> <td>Gibberellin 20-oxidase-1</td> <td>354</td> <td>1.14.11.12</td> <td>2</td> <td>0.61</td> </tr> <tr> <td>K05282</td> <td>KX894007</td> <td>Gibberellin 20-oxidase-5</td> <td>213</td> <td>1.14.11.12</td> <td>1</td> <td>1.87</td> </tr> <tr> <td>K05282</td> <td>KX894008</td> <td>Gibberellin 20-oxidase-3</td> <td>256</td> <td>1.14.11.12</td> <td>2</td> <td>1.61</td> </tr> <tr> <td>K05282</td> <td>KX894009</td> <td>Gibberellin 20-oxidase-4</td> <td>433</td> <td>1.14.11.12</td> <td>7</td> <td>1.4</td> </tr> <tr> <td>K05282</td> <td>KX894010</td> <td>Gibberellin 20-oxidase-2</td> <td>1367</td> <td>1.14.11.12</td> <td>566.34</td> <td>22.38</td> </tr> <tr> <td>Triterpene</td> <td>K15813</td> <td>KX894011</td> <td>Beta-amyrin synthase</td> <td>2745</td> <td>5.4.99.39</td> <td>2784.23</td> <td>92.71</td> </tr> <tr> <td>K15813</td> <td>KX894012</td> <td>Beta-amyrin synthase</td> <td>2739</td> <td>5.4.99.39</td> <td>2606.94</td> <td>46.69</td> </tr> <tr> <td>K00511</td> <td>KX894013</td> <td>Squalene monooxygenase</td> <td>1972</td> <td>1.14.13.132</td> <td>1324.67</td> <td>35.44</td> </tr> <tr> <td>K00511</td> <td>KX894014</td> <td>Squalene monooxygenase</td> <td>1909</td> <td>1.14.13.132</td> <td>1101.83</td> <td>43.23</td> </tr> <tr> <td>K00801</td> <td>KX894015</td> <td>Farnesyl-diphosphate farnesyltransferase</td> <td>1664</td> <td>2.5.1.21</td> <td>2268.89</td> <td>108.66</td> </tr> <tr> <td>K00801</td> <td>KX894016</td> <td>Farnesyl-diphosphate farnesyltransferase</td> <td>728</td> <td>2.5.1.21</td> <td>327.94</td> <td>29.06</td> </tr> <tr> <td>K15822</td> <td>KX894017</td> <td>Camelliol C synthase</td> <td>312</td> <td>5.4.99.38</td> <td>12</td> <td>5.66</td> </tr> </tbody> </table> </div> <p /> </div> <div class="2.8.annotationofunigenes" title="sec"> <div class="title" tagx="title" title="title">
2.8. Annotation of unigenes</div> <p>Unigenes were used as query sequences to search the annotation databases, including the NCBI non-redundant protein sequences database (NR) ( <a href="http://www.ncbi.nlm.nih.gov/">http://www.ncbi.nlm.nih.gov/</a>) and Swiss-Prot (a manually annotated and reviewed protein sequence database) ( <a href="http://www.ebi.ac.uk/uniprot/">http://www.ebi.ac.uk/uniprot/</a>), based on sequence homology to entries in the Gene Ontology (GO) database ( <a href="http://www.geneontology.org/">http://www.geneontology.org/</a>). Unigene sequences from <i>S. guaranitica</i> were categorized into three general sections: biological process (BP), cellular component (CC) and molecular function (MF). Additionally, the unigenes were used as query sequences for searching the Kyoto Encyclopedia of Genes and Genome (KEGG) pathways database ( <a href="http://www.genome.jp/kegg/">http://www.genome.jp/kegg/</a>) and the Pfam (Protein family) database ( <a href="http://pfam.sanger.ac.uk/">http://pfam.sanger.ac.uk/</a>). </p> </div> <div class="2.9.differentialexpressionanalysis" title="sec"> <div class="title" tagx="title" title="title">
2.9. Differential expression analysis</div> <p>Expression levels of unigenes were normalized and calculated as the values of fragments per kilobase of transcripts per million mapped fragments (FPKM) during the assembly and clustering process. Differential expression analysis of unigenes was performed using the DESeq R package (1.10.1). DESeq provides statistical routines for assessing the differential genes expression in leaf tissues and assigns genes as differential expressed when the <i>P</i>-value &amp;lt; 0.05. <i>P</i>-value results were corrected using the Benjamini and Hochberg approach for controlling the false discovery rate (FDR). <a href="#dsy028-B19"> <sup>19</sup> </a> </p> </div> <div class="2.10.quantitativereal-timepcr(qrt-pcr)analysis" title="sec"> <div class="title" tagx="title" title="title">
2.10. Quantitative real-time PCR (qRT-PCR) analysis</div> <p>Quantitative real-time PCR was performed using an IQ <sup>TM</sup>5 Multicolor Real-Time PCR Detection System (Bio-Rad, USA) as described previously <a href="#dsy028-B60"> <sup>60</sup> </a> with SYBR Green Master (ROX) (Newbio Industry, China) following the manufacturer’s instructions at a total reaction volume of 20 µl. Gene-specific primers for <i>SgActin</i> as a reference gene and for the other 15 gene ( <i>SgGPPS</i>, <i>SgFPPS</i>, <i>SgHUMS</i>, <i>SgNEOD-1</i>, <i>SgNEOD-2</i>, <i>SgNEOD-3</i>, <i>SgTPS-1</i>, <i>SgTPS-3</i>, <i>SgTPS-6</i>, <i>SgLINS-1</i>, <i>SgLINS -2</i>, <i>SgGLNS</i>, <i>SgGERIS</i>, <i>SgTPS-V</i> and <i>SgFARD</i>) involved in the biosynthesis of terpenes were designed using the primer designing tools of IDTdna ( <a href="http://www.idtdna.com">http://www.idtdna.com</a>), as listed in <a href="#sup2">Supplementary Table S1</a>. The quantitative RT-PCR conditions were set as standard conditions: 95 °C for 3 min, 40 cycles of amplification (95 °C for 10 s, 60 or 58 °C for 30 s and 72 °C for 20 s), and a final extension at 65 °C for 1 min. The values are means ± SE of three replicates was normalized using <i>SgActin</i> as a reference gene. The relative expression levels were calculated by comparing the cycle thresholds (CTs) of the target genes with that of the reference gene <i>SgActin</i> using the 2 <sup>-ΔΔCt</sup> method. <a href="#dsy028-B6"> <sup>6</sup> </a> <sup>,</sup> <a href="#dsy028-B20"> <sup>20</sup> </a> <sup>,</sup> <a href="#dsy028-B21"> <sup>21</sup> </a> The sizes of amplification products were 140–160 bp. The quantified data were analysed using the Bio-Rad IQ <sup>TM</sup> 5 Multicolor Real-Time Manager software. Finally, the relative expression levels of <i>SgGPPS</i>, <i>SgFPPS</i>, <i>SgHUMS</i>, <i>SgNEOD-1</i>, <i>SgNEOD-2</i>, <i>SgNEOD-3</i>, <i>SgTPS-1</i>, <i>SgTPS-3</i>, <i>SgTPS-6</i>, <i>SgLINS-1</i>, <i>SgLINS-2</i>, <i>SgGLNS</i>, <i>SgGERIS</i>, <i>SgTPS-V</i> and <i>SgFARD</i> were detected. </p> </div> <div class="2.11.identificationofsimplesequencerepeats(ssrs)" title="sec"> <div class="title" tagx="title" title="title">
2.11. Identification of simple sequence repeats (SSRs)</div> <p>All of the transcripts of <i>S. guaranitica</i> were analysed with the MISA program version 1.0 ( <a href="http://pgrc.ipk-gatersleben.de/misa/misa.html">http://pgrc.ipk-gatersleben.de/misa/misa.html</a>) for the detection of simple sequence repeat (SSR) motifs that have mono- to hexanucleotide repeats. In addition, primers for each SSR were designed using Primer3 version 2.3.5 ( <a href="http://primer3.sourceforge.net/releases.php">http://primer3.sourceforge.net/releases.php</a>). The minimum number of SSR repeat units during analysis was ≥24 for mono- and dinucleotides and was 8, 7, 7 and 9 for tri-, tetra-, penta- and hexanucleotide repeats, respectively. The default parameters corresponding to each unit size of the minimum number of repetitions were 1-10, 2-6, 3, 5, 4, 5, 5, 5 and 6-5 for Unigene SSR detection. </p> </div> <div class="2.12.full-lengthterpenesynthasecdnaclonesandvectors" title="sec"> <div class="title" tagx="title" title="title">
2.12. Full-length terpene synthase cDNA clones and vectors</div> <p>Full-length cDNAs for <i>SgFPPS</i>, <i>SgGPPS</i> and <i>SgLINS</i> were obtained by PCR amplification using short and long gene-specific primers ( <a href="#sup2">Supplementary Table S2</a>, <a href="#sup1">Fig. S1</a>) based on RNA-Seq sequence information from the transcriptome sequencing of <i>S. guaranitica</i> leaves. Leaf cDNA was used as a template for the initial PCR amplification and performed using short primers with the KOD-Plus DNA polymerase (Novagen) under the following PCR conditions: 3 min at 94 °C followed by 10 s at 98 °C; 30 s at 60, 60 and 59 °C (different annealing temperatures), 1.30 min at 68 °C, and then 10 min at 68 °C. This process was repeated for 35 cycles. The first PCR products was used as a template for PCR cloning using long primers with the KOD-Plus DNA polymerase for the Gateway pDONR221 vector. The amplified PCR products were purified and cloned into the Gateway entry vector pDONR221 using BP Clonase (Invitrogen, USA). The resulting pDONR221 constructs harbouring target genes were sequenced, and Gateway LR Clonase (Invitrogen, USA) was used for recombination into the destination vector pB2GW7 for <i>A. thaliana</i> transformation. All final constructs containing <i>SgFPPS</i>, <i>SgGPPS</i> and <i>SgLINS</i> were confirmed by sequencing. </p> </div> <div class="2.13.arabidopsisplantgrowthconditionsandpreparationofagrobacteriumculturesforfloral-diptransformation" title="sec"> <div class="title" tagx="title" title="title">
2.13. <i>Arabidopsis</i> plant growth conditions and preparation of <i>Agrobacterium</i> cultures for floral-dip transformation </div> <p>Ecotype of <i>A. thaliana</i> plant seeds Columbia-0 (Col-0) were pre-germinated by adding 1 ml sterilized water on some seed at 1.5 ml Eppendorf tube, then incubated at 4 °C for three days at the refrigerator. After that <i>A. thaliana</i> seeds had been growing in our Key Lab growth chamber at a temperature of 22 °C day/20 °C night with humidity of 50–70%, and photoperiod at 16 h day/8 h night, with a light density of 100–150 μmoles m  <sup>−</sup> <sup>2</sup> s  <sup>−</sup> <sup>1</sup> using fluorescent bulbs. After 2 months plants were ready for floral-dip transformation, and one week after the primary inflorescences were clipped. Plant watering was stopped 3 days prior to transformation for improved and increase the transformation efficiency. In addition, the constructs of pB2GW7 vectors with all inserted genes were introduced into <i>Agrobacterium tumefaciens</i> strain EHA105 by direct electroporation. Recombinant <i>A. tumefaciens</i> was grown for 2 days at 28 °C in solid LB media supplemented with 50 μg/ml each of rifampicin and spectinomycin. An individual colony of each sample was inoculated into 1.0 ml of liquid medium and grown at 28 °C under 200 rpm agitation overnight with the same media composition. After 24 h, 1.0 ml of each sample of liquid medium was transferred to a 250-ml conical flask containing 50 ml of LB media supplemented with the same compositions; the samples were grown at 28 °C in a shaker overnight until an optical density of 0.6–8.0 (OD 600) was reached. Overnight cell cultures were harvested by centrifugation at 5,000 rpm for 10 min at 4 °C, and the pellet was resuspended in the floral-dip inoculation medium contained 5% sucrose and 0.05% Silwet. <i>A. thaliana</i> was transformed by soaked the secondary inflorescences in the inoculation medium and stirred gently to allow the intake of <i>Agrobacterium</i> harbouring the pB2GW7 vector into the flower gynoecium. The transformed plants were kept in the dark and wrapped with plastic cover overnight to maintain humidity. The next day, the plants were returned back to their normal growth conditions. The transformation was repeated after 1 week to increase the transformation efficiency. Plants were grown for additional 4–5 weeks until all of the siliques became brown and dry. The seeds were harvested and stored at 4 °C under desiccation. <a href="#dsy028-B15"> <sup>15</sup> </a> <sup>,</sup> <a href="#dsy028-B16"> <sup>16</sup> </a> BASTA was used for selection of transformant seedlings which were also confirmed with PCR for positive transgenic lines, more than 10 positive plant lines from each gene were analysed for terpenoid profiling and target gene expression. </p> </div> <div class="2.14.semiquantitativert-pcranalysis" title="sec"> <div class="title" tagx="title" title="title">
2.14. Semiquantitative RT-PCR analysis</div> <p>Semiquantitative real-time PCR was performed on an Eppendorf PCR (Eppendorf Mastercycler-Nexus GSX1, POCD Scientific, Australia) system with a total reaction volume of 25 µl. A gene-specific primer for <i>AT-B-actin</i> was used as a reference gene, and the other three gene-specific primers for <i>SgFPPS</i>, <i>SgGPPS</i> and <i>SoLINS</i> which are involved in the biosynthesis of terpenes, were designed using the primer designing tools of IDTdna ( <a href="http://www.idtdna.com/scitools/">http://www.idtdna.com/scitools/</a> Applications/RealTimePCR/); the primer sequences are listed in ( <a href="#sup2">Supplementary Table S1</a>). The semiquantitative RT-PCR conditions were as follows: predenaturation step at 95 °C for 4 min, 35 cycles of amplification (95 °C for 30 s, 60or 59 °C for 30 s and 72 °C for 1 min), and a final extension step at 72 °C for 10 min. The PCR products were resolved on 1% agarose gel, and the expression levels of <i>AT-BActin</i>, <i>SgFPPS</i>, <i>SgGPPS</i> and <i>SgLINS</i> genes were detected. </p> </div> <div class="2.15.metaboliteextractionfromtransgenica.thalianaleaves" title="sec"> <div class="title" tagx="title" title="title">
2.15. Metabolite extraction from transgenic <i>A. thaliana</i> leaves </div> <p>Terpenoid compounds from non-transgenic <i>A. thaliana</i> leaves (control) and transgenic <i>A. thaliana</i> leaves containing either <i>SgFPPS</i>, <i>SgGPPS</i> and <i>SgLINS</i> expression constructs were extracted and isolated. For this, 15 leaves from each transgenic <i>A. thaliana</i> line (three leaf from each plant) were homogenized in liquid nitrogen with a mortar and pestle, after which the plant material powder was directly soaked in <i>n</i>-hexane as a solvent in Amber storage bottles, 30 ml screw-top vials with silicone/PTFE septum lids ( <a href="http://www.sigmaaldrich.com">http://www.sigmaaldrich.com</a>) were used to reduce loss of volatiles to the headspace then incubated with shaking at 37 °C and 200 rpm for 72 h. Afterward, the solvent was transferred using a glass pipette to a 10-ml glass centrifuge tube with screw-top vials with silicone/PTFE septum lids and centrifuged at 5,000 rpm for 10 min at 4 °C to remove plant debris. The supernatant was pipetted into glass vials with a screw cap and oil was concentrated until remaining 1.5 ml of concentrated oils under a stream of nitrogen gas with a nitrogen evaporator (Organomation) and water bath at room temperature (Toption-China-WD-12). The concentrated oils transferred to a fresh crimp vial amber glass, 1.5 ml screw-top vials with silicone/PTFE septum lids were used to reduce a loss of volatiles to the headspace. For absolute oil recovery, the remaining film crude oil in the internal surface of concentrated glass vials was dissolved in the minimum volume of <i>n</i>-hexane, thoroughly mixed and transferred to the same fresh crimp vial amber glass, 1.5 ml. And the crimp vial was placed on the auto-sampler of the GC-MS system for GC-MS analysis, or each tube was covered with parafilm after closed with screw-top vials with silicone/PTFE septum lids and stored at −20 °C until GC-MS analysis. The same programme and standard conditions that were used for GC-MS analysis with <i>S. officinalis</i> essential oil components were applied. <a href="#dsy028-B6"> <sup>6</sup> </a> </p> </div> </div> <div class="3.resultsanddiscussion" title="sec"> <div class="title" tagx="title" title="title">
3. Results and discussion</div> <div class="3.1.identificationofessentialoilcomponents" title="sec"> <div class="title" tagx="title" title="title">
3.1. Identification of essential oil components</div> <p>For GC-MS analysis, 204 compounds were identified using <i>n</i>-hexane extracts from six fresh parts of <i>S. guaranitica</i>. The numbers of obtained compounds from old leaves, young leaves, stems, flowers, bud flowers and roots were 71 (98.73%), 29 (74.58%), 21 (83.87%), 45 (93.51%), 32 (80.06%) and 45 (96.79%), respectively. The results of the qualitative and quantitative analyses of all compounds from the essential oils are reported in ( <a href="#dsy028-T3">Table 3</a> and <a href="#sup2">Supplementary Table S3</a>). The identified compounds are listed based on the retention time, compounds mass and percentage of peak area ( <a href="#dsy028-F1">Fig. 1A and B</a>). In old leaves, one triterpene was shown as the main compound (32.31%), followed by the group of diterpene (21.16%) and sesquiterpenes group (16.7%). In young leaves, the sesquiterpenes compounds were observed to be the main group (25.78%), followed by one diterpene and one monoterpene compounds represented (11.45 and 0.19%), respectively. The main compound in the stem was one triterpene (0.15%). Furthermore, in flowers the sesquiterpenes compounds were observed to be the main group (0.42%), followed by one diterpene compound (0.01%). <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 3.</span>  </div></p><p>The major chemical compositions in the essential oils of <i>S. guaranitica</i> </p>  <table> <div class="colgroup_UNKNOWN"> <div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div>  <thead> <tr> <th> <i>N</i> </th> <th>Compound name</th> <th>Retention time (min.)</th> <th>Retention time index</th> <th>Formula</th> <th>Molecular Mass (g mol <sup>-1</sup>) </th> <th>Terpene type</th> <th>Old leaf <hr /> </th> <th>Young leaf <hr /> </th> <th>Stem <hr /> </th> <th>Flower <hr /> </th> <th>Bud Flower <hr /> </th> <th>Root <hr /> </th> <th /> </tr> <tr> <th> <b>% Peak area</b> </th> <th> <b>% Peak area</b> </th> <th> <b>% Peak area</b> </th> <th> <b>% Peak area</b> </th> <th> <b>% Peak area</b> </th> <th> <b>% Peak area</b> </th> <th> <b>O.S.S</b> </th> </tr> </thead> <tbody> <tr> <td>1</td> <td>α-Pinene</td> <td>7.323</td> <td>939</td> <td>C10H16</td> <td>136.234</td> <td>Mon</td> <td>–</td> <td>–</td> <td>–</td> <td>–</td> <td>–</td> <td>0.96</td> <td>SO, SL, SA, SF, SC</td> </tr> <tr> <td>2</td> <td>Camphene</td> <td>8.367</td> <td>951</td> <td>C10H16</td> <td>136.234</td> <td>Mon</td> <td>–</td> <td>–</td> <td>–</td> <td>–</td> <td>–</td> <td>0.55</td> <td>SO, SL, SA, SF, SC</td> </tr> <tr> <td>3</td> <td>laevo-β-Pinene</td> <td>10.108</td> <td>974</td> <td>C10H16</td> <td>136.234</td> <td>Mon</td> <td>–</td> <td>–</td> <td>–</td> <td>–</td> <td>–</td> <td>0.97</td> <td>SO, SL, SA, SF, SC</td> </tr> <tr> <td>4</td> <td>beta.-Pinene</td> <td>11.27</td> <td>980</td> <td>C10H16</td> <td>136.234</td> <td>Mon</td> <td>–</td> <td>–</td> <td>–</td> <td>–</td> <td>–</td> <td>0.36</td> <td>SO, SL, SA, SF, SC</td> </tr> <tr> <td>5</td> <td>1,8-cineol</td> <td>13.99</td> <td>1030</td> <td>C10H18O</td> <td>154.2493</td> <td>Mon</td> <td>–</td> <td>–</td> <td>–</td> <td>–</td> <td>–</td> <td>2.61</td> <td>SO, SL, SA, SF</td> </tr> <tr> <td>6</td> <td>Thujone</td> <td>18.35</td> <td>1112</td> <td>C10H16O</td> <td>152.2334</td> <td>Mon</td> <td>–</td> <td>–</td> <td>–</td> <td>–</td> <td>–</td> <td>0.35</td> <td>SO, SF</td> </tr> <tr> <td>7</td> <td>(-)-Camphor</td> <td>20.325</td> <td>1141</td> <td>C10H16O</td> <td>152.2334</td> <td>Mono</td> <td>–</td> <td>–</td> <td>–</td> <td>–</td> <td>–</td> <td>1.14</td> <td>SO, SL, SA, SF</td> </tr> <tr> <td>8</td> <td>(+)-borneol</td> <td>21.485</td> <td>1152</td> <td>C10H18O</td> <td>154.2493</td> <td>Mono</td> <td>–</td> <td>–</td> <td>–</td> <td>–</td> <td>–</td> <td>0.53</td> <td>SO, SL, SA, SF, SC</td> </tr> <tr> <td>9</td> <td>Cis-α-terpineol</td> <td>22.648</td> <td>1209</td> <td>C10H18O</td> <td>154.2493</td> <td>Mono</td> <td>–</td> <td>–</td> <td>–</td> <td>–</td> <td>–</td> <td>0.13</td> <td>SO</td> </tr> <tr> <td>10</td> <td>Farnesan</td> <td>26.943</td> <td>1376</td> <td>C15H32</td> <td>212.4146</td> <td>Sesqui</td> <td>–</td> <td>–</td> <td>–</td> <td>–</td> <td>–</td> <td>0.22</td> <td /> </tr> <tr> <td>11</td> <td>(-)-.beta.-Bourbonene</td> <td>27.023</td> <td>1386</td> <td>C15H24</td> <td>204.351</td> <td>Sesqui</td> <td>0.72</td> <td>–</td> <td>–</td> <td>–</td> <td>–</td> <td>–</td> <td>SO</td> </tr> <tr> <td>12</td> <td>(E)-β-Elemene</td> <td>27.29</td> <td>1389</td> <td>C15H24</td> <td>204.3511</td> <td>Sesqui</td> <td>0.95</td> <td>–</td> <td>–</td> <td>–</td> <td>–</td> <td>–</td> <td /> </tr> <tr> <td>13</td> <td>α-Terpineol acetate</td> <td>27.638</td> <td>1351</td> <td>C12H20O2</td> <td>196.286</td> <td>Mono</td> <td>–</td> <td>–</td> <td>–</td> <td>–</td> <td>–</td> <td>0.21</td> <td /> </tr> <tr> <td>14</td> <td>β-Caryophyllene</td> <td>28.225</td> <td>1420</td> <td>C15H24</td> <td>204.3511</td> <td>Sesqui</td> <td>1.33</td> <td>1.98</td> <td>–</td> <td>0.2</td> <td>–</td> <td>0.31</td> <td>SO, SF</td> </tr> <tr> <td>15</td> <td>Humulene</td> <td>29.46</td> <td>1454</td> <td>C15H24</td> <td>204.357</td> <td>Sesqui</td> <td>0.07</td> <td>–</td> <td>–</td> <td>–</td> <td>–</td> <td>–</td> <td>SO, SL SA</td> </tr> <tr> <td>16</td> <td>(-)-Germacrene D</td> <td>30.293</td> <td>1481</td> <td>C15H24</td> <td>204.3511</td> <td>Sesqui</td> <td>5.43</td> <td>10.7</td> <td>–</td> <td>–</td> <td>–</td> <td>–</td> <td>SO</td> </tr> <tr> <td>17</td> <td>pi- α-Muurolene</td> <td>30.908</td> <td>1496</td> <td>C15H24</td> <td>204.3511</td> <td>Sesqui</td> <td>0.41</td> <td>–</td> <td>–</td> <td>–</td> <td>–</td> <td>–</td> <td>SO, SL SA</td> </tr> <tr> <td>18</td> <td>1-Ethenyl-1-methyl-2, 4-bis-(1methylethenyl) cyclohexane</td> <td>31.195</td> <td>1449</td> <td>C15H24</td> <td>204.351105</td> <td>Sesqui</td> <td>–</td> <td>–</td> <td>–</td> <td>0.22</td> <td>–</td> <td>–</td> <td>SO</td> </tr> <tr> <td>19</td> <td>δ-Cadinene</td> <td>31.518</td> <td>1507</td> <td>C15H24</td> <td>204.3511</td> <td>Sesqui</td> <td>0.67</td> <td>–</td> <td>–</td> <td>–</td> <td>–</td> <td>–</td> <td /> </tr> <tr> <td>20</td> <td>Germacrene-A</td> <td>33.407</td> <td>1510</td> <td>C15H26O</td> <td>222.3663</td> <td>Sesqui</td> <td>0.49</td> <td>–</td> <td>–</td> <td>–</td> <td>–</td> <td>–</td> <td /> </tr> <tr> <td>21</td> <td>Caryophyllene oxide</td> <td>33.462</td> <td>1546</td> <td>C15H24</td> <td>204.3511</td> <td>Sesqui</td> <td>0.43</td> <td>4.16</td> <td>–</td> <td>–</td> <td>–</td> <td>–</td> <td>SO, SA, SL, SN</td> </tr> <tr> <td>22</td> <td>Ledol</td> <td>35.185</td> <td>1600</td> <td>C15H26O</td> <td>222.3663</td> <td>Sesqui</td> <td>–</td> <td>–</td> <td>–</td> <td>–</td> <td>–</td> <td>1.56</td> <td>SO, SL, SA</td> </tr> <tr> <td>23</td> <td>α-Cadinol</td> <td>35.727</td> <td>1653</td> <td>C15H26O</td> <td>222.3663</td> <td>Sesqui</td> <td>0.54</td> <td>–</td> <td>–</td> <td>–</td> <td>–</td> <td>–</td> <td /> </tr> <tr> <td>24</td> <td>Indene, 6-methyl-3a, 4, 7, 7a-tetrahydro</td> <td>35.85</td> <td>1113</td> <td>C10H14</td> <td>134.2182</td> <td>Mono</td> <td>–</td> <td>0.19</td> <td>–</td> <td>–</td> <td>–</td> <td>–</td> <td /> </tr> <tr> <td>25</td> <td>Longipinocarveol, trans-</td> <td>37.823</td> <td>1618</td> <td>C15H24O</td> <td>220.350494</td> <td>Sesqui</td> <td>0.78</td> <td>–</td> <td>–</td> <td>–</td> <td>–</td> <td>–</td> <td /> </tr> <tr> <td>26</td> <td> <i>Trans</i>-bisabolene epoxide </td> <td>37.93</td> <td>1529</td> <td>C15H24O</td> <td>220.350494</td> <td>Sesqui</td> <td /> <td>0.14</td> <td>–</td> <td>–</td> <td>–</td> <td>–</td> <td>SO</td> </tr> <tr> <td>27</td> <td> <i>trans</i>-phytol, (E) Phytol </td> <td>47.172</td> <td>2110</td> <td>C20H40O</td> <td>296.531</td> <td>Diter</td> <td>21.11</td> <td>11.45</td> <td>–</td> <td>–</td> <td>0.48</td> <td>–</td> <td /> </tr> <tr> <td>28</td> <td>Phytan</td> <td>49.165</td> <td>1811</td> <td>C20H42</td> <td>282.5475</td> <td>Diter</td> <td>0.05</td> <td>–</td> <td>–</td> <td>–</td> <td>–</td> <td>–</td> <td /> </tr> <tr> <td>29</td> <td>Caryophyllene oxide</td> <td>52.368</td> <td>1582</td> <td>C15H24O</td> <td>220.3505</td> <td>Sesqui</td> <td>–</td> <td>–</td> <td>–</td> <td>–</td> <td>–</td> <td>1.91</td> <td>SO, SA, SL, SN</td> </tr> <tr> <td>30</td> <td>δ-Decalactone</td> <td>54.81</td> <td>1504</td> <td>C10H18O2</td> <td>170.248703</td> <td>Mono</td> <td>–</td> <td>–</td> <td>–</td> <td>–</td> <td>0.28</td> <td>–</td> <td /> </tr> <tr> <td>31</td> <td>Kauran-18-al, 17-(acetyloxy)-, (4.beta.)-</td> <td>68.623</td> <td>2040</td> <td>C22H34O3</td> <td>346.503601</td> <td>Diter</td> <td>–</td> <td>–</td> <td>–</td> <td>0.01</td> <td>–</td> <td>–</td> <td /> </tr> <tr> <td>32</td> <td>Peppermint camphor</td> <td>74.07</td> <td>1179</td> <td>C10H20O</td> <td>156.265198</td> <td>Mono</td> <td>–</td> <td>–</td> <td>–</td> <td>–</td> <td>0.83</td> <td>–</td> <td /> </tr> <tr> <td>33</td> <td>Squalene</td> <td>74.235</td> <td>2831</td> <td>C30H50</td> <td>410.718</td> <td>Tri</td> <td>32.31</td> <td>–</td> <td>0.15</td> <td>–</td> <td>–</td> <td>–</td> <td>SO</td> </tr> <tr> <td /> <td>Total percentage ( % ) of Monoterpenes</td> <td /> <td /> <td /> <td /> <td /> <td>0.19</td> <td /> <td /> <td>1.11</td> <td>7.81</td> <td /> </tr> <tr> <td /> <td>Total percentage ( % ) of sesquiterpenes</td> <td /> <td /> <td /> <td /> <td>16.7</td> <td>25.78</td> <td /> <td>0.42</td> <td /> <td>4.0</td> <td /> </tr> <tr> <td /> <td>Total percentage ( % ) of diterpenes</td> <td /> <td /> <td /> <td /> <td>21.16</td> <td>11.45</td> <td /> <td>0.01</td> <td>0.48</td> <td /> <td /> </tr> <tr> <td /> <td>Total percentage ( % ) of triterpenes</td> <td /> <td /> <td /> <td /> <td>32.31</td> <td /> <td>0.15</td> <td /> <td /> <td /> <td /> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p>RT, retention time; OSS, other salvia species; SA, <i>Salvia acetabulosa</i>; SL, <i>Salvia leriifolia</i>; SF, <i>Salvia fruticosa</i> <b>;</b> SN, <i>Salvia nemorosa</i>; SC, <i>Salvia compressa</i>; SO, <i>Salvia officinalis</i>; Mono, monoterpene; Sesqui, sesquiterpene; Dit, diterpene; Tri, triterpene; –, terpene compounds not detected. </p> </div> </div>  <p /> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 1.</span> </a> </div>   <p>Typical GC-MS mass spectragraphs for terpenoids from old leaf, young leaf, stem, bud flower, flower and root of <i>S. guaranitica</i>. (A) GC-MS Peak of the essential oil, (B) mass spectrum of GC peak with retention time for the major compound, (C) Six-way Venn diagram to show the number of unique and common compounds in the essential oil extracts from old leaf (A), young leaf (B), stem (C), flower (D), bud flower (E) and root (F) of <i>S. guaranitica</i>. </p>   </div> <p>On the other hand, in bud flowers, the monoterpenes compounds were shown as the main group (1.11%), followed by one diterpene and one sesquiterpene compound represented (0.48 and 0.03%), respectively. Finally, Monoterpenes from the main group of compounds (7.81%) found in the roots, followed by sesquiterpene group represented 4.0%. Moreover, the six hexane extracts from the different tissue essential oils have unique, common and major compounds. For example, the extracts of old leaf essential oils (A) had 57 unique compounds, three common compounds shared with extracts from young leaf essential oils, one common compound shared with extracts from stem essential oils, three common compounds shared with extracts from flower essential oils and two common compounds shared with extracts from bud flower essential oils. Furthermore, the young leaf essential oils (B) contained 19 unique compounds. While the stem essential oils (C) contained 11 unique compounds and two common compounds shared with extracts from flower essential oils and three common compounds shared with extracts from root essential oils. Also, the extracts from flower essential oils (D) had 30 unique compounds, four common compounds shared with extracts from bud flower essential oils. Moreover, the extracts from bud flower essential oils (E) and the root essential oils (F) had 24 and 37 unique compounds, respectively ( <a href="#dsy028-F1">Fig. 1C</a>). On the other hand, we found one common compound named (Cyclooctasiloxane, hexadecamethyl) shared with all six plant parts. Additionally, we found some other common compounds shared among all six plant parts, such as (trans-phytol, 2-methyloctacosane, β-caryophyllene, cyclohexasiloxane, dodecanethiol-, cycloheptasiloxane, tetradecane- methyl- and cyclononasiloxane, octa deca methyl-) ( <a href="#sup2">Supplementary Table S3</a> and <a href="#dsy028-F1">Fig. 1C</a>). Regarding the major compounds, squalene (32.31%) was the major compound in the essential oils extracts of old leaves, followed by trans-phytol (21.11%), (-)-germacrene D (5.43%), <i>n</i>-octadecanal (5.15%), 8-isopropenyl-1, 5-dimethyl-1, 5-cyclodecadiene (4.88%) and β-caryophyllene (1.33%), whereas the essential oil of young leaves was also characterized by trans-phytol (11.45%), followed by (-)-germacrene D (10.7%), 8-isopropenyl-1, 5-dimethyl-1, 5-cyclodecadiene (8.8%), caryophyllene oxide (4.16%), 3-methyl-cis-3a, 4, 7, 7a-tetrahydroindan (3.49%) and β-caryophyllene (1.98%). N-octadecanal was characterized as the major compound in stem extracts (38.78%), followed by undecane, 2-methyl (1.27%), and then squalene (0.15%). Furthermore, the essential oils of flowers was also characterized by 2-methyloctacosane (5.34%), followed by 10-methyleicosane (1.9%), and tetracosane, 2-methyl (1.7%). Moreover, the essential oil of bud flowers was also characterized by peppermint camphor (0.83%) as a major compound, followed by trans-phytol (0.48%), angelicoidenol (0.28%) and longi borneol (0.03%). Finally, the root essential oil was characterized by 1,8-cineol (2.61) as a major compound, followed by caryophyllene oxide (1.91), ledol (1.56), (-)-camphor (1.14), laevo-β-pinene (0.97) and α-pinene (0.96) ( <a href="#dsy028-T3">Table 3</a>). When comparing the composition of the six essential oil extracts of <i>S. guaranitica</i>, we deduced that some common compounds exist at different levels within the parts of <i>S. guaranitica</i> ( <a href="#dsy028-F1">Fig. 1A</a>). Additionally, some of the compounds that have been found in <i>S. guaranitica</i> were detected in the other <i>Salvia</i> plant species ( <a href="#dsy028-T3">Table 3</a> and <a href="#sup2">Supplementary Table S3</a>). <a href="#dsy028-B6"> <sup>6</sup> </a> <sup>,</sup> <a href="#dsy028-B13"> <sup>13</sup> </a> <sup>,</sup> <a href="#dsy028-B14"> <sup>14</sup> </a> <sup>,</sup> <a href="#dsy028-B22"> <sup>22</sup> </a> Therefore, we suggest that plant parts can have a major effect on the metabolic composition of their essential oils. <a href="#dsy028-B23"> <sup>23</sup> </a> <sup>,</sup> <a href="#dsy028-B24"> <sup>24</sup> </a> From the previous GC-MS data, an important question has been raised, why do the triterpenes, sesquiterpenes and monoterpene compounds of <i>S. guaranitica</i> mostly accumulated in old leaves, young leaves and roots, respectively? This question was difficult to answer before conducting the present work because there was a lack of information at the genetic level regarding the terpenoid biosynthetic pathway and how these compounds are synthesized in <i>S. guaranitica</i>. </p> </div> <div class="3.2.illuminasequencingandthedenovoassemblyofthes.guaraniticaleaftranscriptome" title="sec"> <div class="title" tagx="title" title="title">
3.2. Illumina sequencing and the <i>de novo</i> assembly of the <i>S. guaranitica</i> leaf transcriptome </div> <p>In the past few years, the Illumina sequencing platform has become a powerful method for analysing and discovering the genomes of non-model plants. <a href="#dsy028-B17"> <sup>17</sup> </a> <sup>,</sup> <a href="#dsy028-B25"> <sup>25</sup> </a>In this context, to generate transcriptome sequences, complementary DNA (cDNA) libraries were prepared from leaf tissues of <i>S. guaranitica</i>, and cDNA was then sequenced using PE reads sequencing using an Illumina HiSeq 2000 platform. Previous reports involving Illumina sequencing reported that the use of PE sequencing showed significant improvement in the efficiency of <i>de novo</i> assembly and increased the depth of sequencing. <a href="#dsy028-B10"> <sup>10</sup> </a> <sup>,</sup> <a href="#dsy028-B26"> <sup>26</sup> </a> The cDNA sequencing generated 4 Gb of raw data from <i>S. guaranitica</i> leaves. After filtering and removing the adapter sequences from the raw data, the number of reads was 32,862,861 (32.86 million), comprising of 186,299,510 high-quality nucleotide bases, with 96.32% Q20, 92.42% Q30 and 47.55% GC content. For further analysis, high-quality reads were selected, and the transcriptome was assembled using the Trinity program, <a href="#dsy028-B18"> <sup>18</sup> </a> which produced 179,369 transcripts with an N50 length of 1,603 bp, an N90 length of 462 bp and a mean length of 1,039 bp. Moreover, 61,400 unigenes could be detected with an N50 length of 1,334 bp, an N90 length of 277 bp and a mean length of 731 bp. The distribution of the assembled transcript length ranged from 200 to &amp;gt; 2,000 bases; the maximum number of transcripts (66,664 transcripts, 37.165%) ranged from 200 to 500 bp, followed by 48,716 transcripts (27.159%) ranging from 1,000 to 2,000 bp and then 40,323 transcripts (22.480%) ranging from 500 to 1,000 bp. On the contrary, the lowest number of transcripts (23,666 transcripts, 13.194%) was obtained for a size of more than 2,000 bp. In contrast, the assembled unigene lengths were distributed between 200 and &amp;gt; 2,000 bp. The maximum number of unigenes (37,659 unigenes, 37.165%) ranged from 200 to 500 bp, followed by 10,132 unigenes (16.501%) ranged from 500 to 1,000 bp, and then 8,777 unigenes (14.294%) ranging from 1,000 to 2,000 bp. Finally, the lowest number of unigenes (4,832 unigenes, 7.869%) was obtained for a size of &amp;gt;2,000 bp. The length distribution of the transcripts and unigenes are shown in ( <a href="#sup2">Supplementary Table S4</a> and <a href="#sup1">Fig. S2</a>). Our results are in agreement with those for <i>Salvia officinalis</i>, <i>Boehmeria nivea</i>, <i>Curcuma longa</i>, <i>Medicago sativa</i>, <i>Centella as</i>iatica and <i>Apium graveolens</i> in which the largest number of both transcript and unigene lengths were found to range between 75 and 500 bp. <a href="#dsy028-B6"> <sup>6</sup> </a> <sup>,</sup> <a href="#dsy028-B27"> <sup>27</sup> </a> <sup>,</sup> <a href="#dsy028-B28"> <sup>28</sup> </a> </p> </div> <div class="3.3.functionalannotationandclassificationofassembleds.guaraniticaunigenes" title="sec"> <div class="title" tagx="title" title="title">
3.3. Functional annotation and classification of assembled <i>S. guaranitica</i> unigenes </div> <p>The total number of unigenes (61,400, 100% of all unigenes) was compared against the NR, NT, KO, Swiss-Prot, PFAM, GO and KEGG annotation database ( <a href="#sup2">Supplementary Table S5</a> and <a href="#sup1">Fig. S3</a>). The annotation percentage results in this research were higher than the annotation percentages in other non-model plant studies (58% in <i>Carthamus tinctorius</i> and 58. 01% in <i>C. lanceolata</i>). <a href="#dsy028-B11"> <sup>11</sup> </a> <sup>,</sup> <a href="#dsy028-B29"> <sup>29</sup> </a> <sup>,</sup> <a href="#dsy028-B30"> <sup>30</sup> </a>The international standardized gene functional annotation system (GO Annotation) provides a powerful way to recognize the functions and properties of sequences that have not been characterized for an organism. <a href="#dsy028-B31"> <sup>31</sup> </a> The BLAST2 GO program was used to categorize the functions of these annotated unigenes, and a total of 23,198 unigenes (37.78% of all of the assembled unigenes) were mapped to at least one GO term. Based on sequence homology, the unigene sequences from <i>S. guaranitica</i> were categorized into 47 functional groups under three general sections: 60,139 were assigned to the BP, 42,494 were assigned to the CC and 29,574 were assigned to the MF sections. As a result, cellular process (13,830) and metabolic process (13,253) were the most enriched GO terms in the BP section. Regarding the CC section, the cell (8,590) and cell part (8,553) were the most enriched. Within the MF section, binding (13,723) and catalytic activities (11,368) were highly enriched ( <a href="#dsy028-F2">Fig. 2</a>). These results revealed that the main GO classifications in the annotated unigenes were responsible for metabolism and fundamental biological regulation. These results were similar to previous results with the <i>S. miltiorrhiza</i>, <i>S. officinalis</i> transcriptome, and with the transcriptome of <i>O. sanctum</i> and <i>O. basilicum</i> (members of the same family), which have the highest percentages of metabolic process, cellular process, cell, cell part, binding and catalytic activity. <a href="#dsy028-B6"> <sup>6</sup> </a> <sup>,</sup> <a href="#dsy028-B32"> <sup>32</sup> </a> <sup>,</sup> <a href="#dsy028-B33"> <sup>33</sup> </a> Moreover, these results are in agreement way with previous studies on <i>de novo</i> transcriptome assembly in the tuberous root of sweet potato, transcriptome sequencing from <i>S. officinalis</i>, <i>de novo</i> transcriptome sequencing from <i>Raphanus sativus</i> and <i>de novo</i> characterization of roots from the Chinese medicinal plant <i>Polygonum cuspidatum.</i> <a href="#dsy028-B6"> <sup>6</sup> </a> <sup>,</sup> <a href="#dsy028-B30"> <sup>30</sup> </a> <sup>,</sup> <a href="#dsy028-B34"> <sup>34</sup> </a> The lowest percentage of unigenes categories included channel regulator activity (66), extracellular matrix parts (54) and cell junction (25). Therefore, the present work suggests that the enormous potential data that exist in the GO classifications can be used to identify the new genes. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 2.</span> </a> </div>   <p>Functional annotation and classification of assembled unigenes in <i>S. guaranitica</i>. GO terms are summarized in three general sections of the BP, CC and MF. </p>   </div> </div> <div class="3.4.kegganalysisofs.guaraniticatranscriptomes" title="sec"> <div class="title" tagx="title" title="title">
3.4. KEGG analysis of S <i>. guaranitica</i> transcriptomes </div> <p>The KEGG pathway database can facilitate the understanding of the functional annotations of enzymes and the biological functions of genes regarding their networks. <a href="#dsy028-B8"> <sup>8</sup> </a> <sup>,</sup> <a href="#dsy028-B35"> <sup>35</sup> </a> To identify active biological functional pathways in the leaf tissues of <i>S. guaranitica</i>, all 61,400 unigenes sequences were mapped in reference to the canonical pathways of KEGG, but 9,163 (14.92%) unigene sequence were assigned to 260 KEGG pathways. Furthermore, all transcripts were classified into five larger pathways categories, including cellular processes, environmental information processing, genetic information processing, metabolism and organismal systems ( <a href="#dsy028-F3">Fig. 3</a>). The highest number of transcripts from <i>S. guaranitica</i> could be assigned to the metabolism category, followed by genetic information processing, organismal systems and cellular processes, whereas the lowest number of transcripts was related to the category environmental information processing. Interestingly, 570 transcripts of <i>S. guaranitica</i> were related to the biosynthesis of various secondary metabolite pathways, which were sorted into 26 subcategories, with phenylpropanoid biosynthesis (ko00940), terpenoid backbone biosynthesis (ko00900) and carotenoid biosynthesis (ko00906) representing the largest subcategories ( <a href="#sup2">Supplementary Table S6</a>). These results were in agreement with previous results from the transcriptome of <i>S. officinalis</i>, <i>O. sanctum</i> and <i>O. basilicum</i>, which are members of the same family, and from <i>de novo</i> transcriptome sequencing from <i>R. sativus</i>, the transcriptome of which had the highest percentages of phenylpropanoid biosynthesis and terpenoid backbone biosynthesis. <a href="#dsy028-B6"> <sup>6</sup> </a> <sup>,</sup> <a href="#dsy028-B7"> <sup>7</sup> </a> <sup>,</sup> <a href="#dsy028-B30"> <sup>30</sup> </a> </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 3.</span> </a> </div>   <p>KEGG classified into five largest categories pathways includes cellular processes (A), environmental information processing (B), genetic information processing (C), metabolism (D) and organismal systems (E). </p>   </div> </div> <div class="3.5.genesrelatedtothebiosynthesisofisoprenoids" title="sec"> <div class="title" tagx="title" title="title">
3.5. Genes related to the biosynthesis of isoprenoids</div> <p>Various types of terpenoids were found in the essential oil extracts of <i>S. guaranitica.</i> The mixture contained mainly α-pinene, camphene, laevo-β-pinene, beta-pinene, 1,8-cineol, thujone, (-)-camphor, (+)-borneol, <i>cis</i>-α-terpineol, farnesan, (-)-beta-bourbonene, (E)-β-elemene, β-caryophyllene, humulene, (-)-germacrene D, pi-α-muurolene, δ-cadinene, germacrene-A, ledol, α-cadinol, trans-longipinocarveol, trans-phytol, phytan, kauran-18-al, 17-(acetyloxy)-, (4.beta.)- and squalene. Precursor molecules for terpenoid biosynthesis are derived from the cytosolic mevalonate (Ac-MVA) and plastidial MEP pathways. Therefore, queries against the Lamiaceae family transcriptome libraries were applied to identify and to determine the genes that encode the enzymes involved in the different steps of the terpenoid biosynthesis pathway, such as, IPPS (isopentyl diphosphate isomerase), DMAPPS (dimethylallyl diphosphate isomerase), GPPS (geranyl diphosphate synthases), FPPS (farnesyl pyrophosphate synthases) and GGPS (geranylgeranyl diphosphate synthases). <a href="#dsy028-B36"> <sup>36</sup> </a> <sup>,</sup> <a href="#dsy028-B37"> <sup>37</sup> </a> Furthermore, we identified and estimated the expression levels of isoprenoid genes by using uniprot annotations against the transcriptome libraries ( <a href="#dsy028-T1">Table 1</a>). From the annotation data analyses, we found many transcript genes related to isoprenoid biosynthesis from the MEP pathway with higher expression levels, including gene transcripts such as <i>SgDXS1</i>, 4 and 5 (1-deoxy- <div class="sc_UNKNOWN">
d</div>-xylulose-5-phosphate synthase 1, 4 and 5), <i>SgDXR1</i> (1-deoxy- <div class="sc_UNKNOWN">
d</div>-xylulose-5-phosphate reductoisomerase 1), <i>SgMCT</i> (2-C-methyl- <div class="sc_UNKNOWN">
d</div>-erythritol 4-phosphate cytidylyltransferase), <i>SgCMK</i> (4-diphosphocytidyl-2-C-methyl- <div class="sc_UNKNOWN">
d</div>-erythritol kinase), <i>SgHDS 2</i> and <i>4</i> ((E)-4-hydroxy-3-methylbut-2-enyl-diphosphate synthase 2 and 4) <i>SgHDR 4</i>, <i>6</i> and <i>9</i> (4-hydroxy-3-methylbut-2-enyl diphosphate reductase 4, 6 and 9), <i>SgIDI 2</i> (isopentenyl-diphosphate delta-isomerase 2). Additionally, we obtained some gene transcripts that were related to isoprenoid biosynthesis from the MVA pathway with higher expression levels, such as, <i>SgAACT 1</i> and <i>4</i> (acetyl-CoA C-acetyltransferase 1 and 4), <i>SgHMGS</i> (hydroxymethyl glutaryl-CoA synthase), <i>SgHMGR 3</i> and <i>4</i> (hydroxymethyl glutaryl-CoA reductase (NADPH) 3 and 4), <i>SgMVK</i> (mevalonate kinase), <i>SgPMK</i> (phosphomevalonate kinase). Moreover, the transcriptome dataset of <i>S. guaranitica</i> presented other genes, such as <i>SgGPPS</i>, <i>SgFPPS</i>, and <i>SgGGPSΙΙ2</i> that are the immediate precursor of the mono-, sesqui- and di-terpene biosynthesis pathway. The <i>SgGPPS</i>, <i>SgFPPS</i> and <i>SgGGPSΙΙ2</i> genes were highly abundant in leaves and had higher values of fragments per kilobase of transcripts per million mapped fragments (FPKM), which were 47.81, 80.21 and 74.07, respectively ( <a href="#dsy028-F4">Fig. 4</a> and <a href="#dsy028-T1">Table 1</a>). Our results were similar to previously obtained results from the transcriptomes of <i>S. officinalis</i>, <i>O. sanctum</i>, <i>O. basilicum</i> and <i>S. miltiorrhiza</i>, which are members of the same family and have a higher number of transcripts for the isoprenoid biosynthesis genes related to the terpenoid biosynthesis pathway. <a href="#dsy028-B6"> <sup>6–8</sup> </a> </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 4.</span> </a> </div>   <p>Representative terpenoid biosynthesis pathway with cognate heat maps for transcript levels of genes from <i>S. guaranitica</i> transcriptome data with substrates and products, coloured arrows connect substrates to their corresponding products. Green/red colour-coded heat maps represent relative transcript levels of different terpenoid genes determined by Illumina HiSeq 2000 sequencing; red, up-regulated; green, down-regulated. Transcript levels data represent by FPKM: fragments per kilobase of transcripts per million mapped fragments. MeV, multi-experiment Viewer software was used to depict transcript levels. <i>DXS</i>, 1-deoxy- <div class="sc_UNKNOWN">
d</div>-xylulose-5-phosphate synthase; <i>DXR</i>, 1-deoxy- <div class="sc_UNKNOWN">
d</div>-xylulose-5-phosphate reductoisomerase; <i>MCT</i>, 2-C-methyl- <div class="sc_UNKNOWN">
d</div>-erythritol 4-phosphate cytidylyltransferase; <i>ISPF</i>, 2-C-methyl- <div class="sc_UNKNOWN">
d</div>-erythritol 2, 4-cyclodiphos-phate synthase; <i>HDS</i>, (E)-4-hydroxy-3-methylbut-2-enyl-diphosphate synthase; <i>HDR</i>, 4-hydroxy-3-methylbut-2-enyl diphosphate reductases; <i>IDI</i>, isopentenyl-diphosphate delta isomerase; <i>AACT</i>, acetyl-CoA C-acetyl transferase; <i>HMGS</i>, hydroxyl methyl glutaryl-CoA synthase; <i>HMGR</i>, hydroxymethyl glutaryl-CoA reductase (NADPH); <i>MVK</i>, mevalonate kinase; <i>PMK</i>, phospho-mevalonate kinase; <i>GPPS</i>, geranyl diphosphate synthase; <i>FPPS</i>, farnesyl pyrophosphate synthase; <i>GGPS</i>, geranylgeranyl diphosphate synthase, type II; <i>CINS</i>, 1,8-cineole synthase; <i>MYS</i>, myrcene/ocimene synthase; <i>LINS</i>, (3S)-linalool synthase; <i>NEOM</i>, (+)-neomenthol dehydrogenase; <i>SABI</i>, (+)-sabinene synthase; <i>TPS6</i>, (-)-germacrene <div class="sc_UNKNOWN">
d</div> synthase; <i>AMS</i>, beta-amyrin synthase; <i>FARNESOL</i>, farnesol dehydrogenase; <i>SEQ</i>, squalene monooxygenase; <i>HUMS</i>, α-humulene/β-caryophyllene synthase; <i>FAR</i>, farnesyl-diphosphate farnesyltransferase; <i>GA2</i>, gibberellin 2-oxidase; <i>GA20</i>, gibberellin 20-oxidase; <i>E-KS</i>, ent-kaurene synthase; <i>MAS</i>, momilactone-A synthase; <i>GA3</i>, gibberellin 3-beta-dioxygenase; <i>E-KIA</i>, ent-iso-kaurene C2-hydroxylase; <i>E-KIH</i>, ent-kaurenoic acid hydroxylase; <i>E-CDS</i>, ent-copalyl diphosphate synthase. </p>   </div> </div> <div class="3.6.genesrelatedtoterpenesynthesis" title="sec"> <div class="title" tagx="title" title="title">
3.6. Genes related to terpene synthesis</div> <p>Plants produce various terpenoid compounds with highly diverse structures. These compounds play an important role and functions in the interactions with environmental factors and in fundamental BPs. <a href="#dsy028-B37"> <sup>37</sup> </a> <sup>,</sup> <a href="#dsy028-B38"> <sup>38</sup> </a> Multiple terpenoids are synthesized in plants by the expression of many terpene syntheses (TPSs) genes. Moreover, some TPS genes have the ability to catalyse the production of multiple products. Thus, the TPS genes family was classified according to phylogenetic relationships into eight subfamilies (TPS a, b, c, d, e/f, g and h), which comprises mono-, sesqui-, di- and triterpene synthases. <a href="#dsy028-B39"> <sup>39</sup> </a> Therefore, the annotation of transcriptome data from <i>S. guaranitica</i> against the Lamiaceae family and Arabidopsis revealed many terpene synthases involved in the terpenoid biosynthesis pathway, e.g. myrcene, (+)-neomenthol, 1, 8-cineole, (3S)-linalool, (E, E)-geranyl linalool, geraniol isomerase, farnesol, α-humulene, valencene, germacrene-A, <i>cis</i>-muuroladiene, selinene, gamma-cadinene, bicyclogermacrene, momilactone-A, gibberellin 3-beta-dioxygenase, gibberellin 2-oxidase, ent-copalyl diphosphate, ent-kaurene, ent-kaurenoic acid, 9 beta-pimara-7, 15-diene, ent-isokaurene C2-, gibberellin 20-, beta-amyrin, squalene, farnesyl- pyrophosphate and camelliol C. From the dataset, 69 TPS unigenes were identified and determined based on sequence similarities with a TPS sequence in the canonical annotation reference database. Twenty unigenes were annotated as being involved in monoterpene biosynthesis, including myrcene/ocimene synthase, (+)-neomenthol dehydrogenase, 1,8-cineole synthase, (3S)-linalool synthase, (E, E)-geranyl linalool synthase and geraniol isomerase synthase, and 12 other unigenes were annotated as being involved in sesquiterpene biosyntheses, including farnesol dehydrogenase, α-humulene/β-caryophyllene synthase, valencene synthase, germacrene-A synthase, <i>cis</i>-muuroladiene synthase, germacrene-D synthase, selinene synthase, gamma-cadinene synthase and bicyclo-germacrene synthase. Additionally, 30 unigenes were annotated as being involved in diterpene biosynthesis, including momilactone-A synthase, gibberellin 3-beta-dioxygenase, gibberellin 2-oxidase, ent-copalyl diphosphate synthase, ent-kaurene synthase, ent-kaurenoic acid hydroxylase, 9beta-pimara-7, 15-diene oxidase, ent-iso kaurene C2-hydroxylase and gibberellin 20-oxidase. Finally, seven unigenes were annotated as being involved in triterpene biosyntheses, including beta-amyrin synthase, camelliol C synthase, squalene monooxygenase and farnesyl-diphosphate farnesyl transferase, but some of these previous genes showed high abundance in leaves and higher FPKM values ( <a href="#dsy028-F4">Fig. 4</a> and <a href="#dsy028-T2">Table 2</a>). The previous compounds have significant pharmacological activities, such as anticancer, anti-HIV, antiviral, anti-inflammatory and antibacterial activities. <a href="#dsy028-B40"> <sup>40</sup> </a> Sesquiterpenoids are similar to triterpenoids as both share the same origin and originate from FPP. Triterpenoid compounds originate from the conversion of FDP into squalene by squalene synthase (SQS) and then to (S)-2, 3-epoxysqualene by squalene monooxygenase (SQE). Subsequently, (S)-2,3-epoxysqualene is converted to beta-amyrin and camelliol C in the presence of multifunctional (S)-2,3-epoxysqualene cyclase via beta-amyrin synthase and camelliol C synthase, respectively. Similar reports about triterpenoid biosynthesis from (S)-2,3-epoxysqualene cyclases are available for <i>O. basilicum</i> and <i>Catharanthus roseus.</i> <a href="#dsy028-B41"> <sup>41</sup> </a> <sup>,</sup> <a href="#dsy028-B42"> <sup>42</sup> </a> </p> </div> <div class="3.7.ssrdiscoveryandanalysis" title="sec"> <div class="title" tagx="title" title="title">
3.7. SSR discovery and analysis</div> <p>The Illumina HiSeq 2000 system offers the opportunity to analyse molecular markers such as SSRs that are related to terpenoid pathway genes. SSR molecular markers have proven to be a powerful method for understanding genetic variation. Moreover, polymorphic SSR markers are very important for the investigation of related comparative genomics, genetic diversity, evolution, linkage mapping, gene-based association studies and relatedness. Even though SNP markers have become promising, especially for studying complex genetic traits and high-throughput mapping, SSRs provide many advantages compared with other marker systems. Hence, SSRs have become the preferable codominant molecular marker for a construction of linkage maps. <a href="#dsy028-B43"> <sup>43</sup> </a> Therefore, the development of novel SSR molecular markers for <i>S. guaranitica</i> plants could be a valuable tool for breeding studies and genetic applications. Therefore, SSR markers were identified from transcriptome sequencing data using MISA (MIcroSAtellite) ( <a href="http://pgrc.ipkgatersle-ben.de/misa/misa.html">http://pgrc.ipkgatersle-ben.de/misa/misa.html</a>). Of the 61,400 transcripts of <i>S. guaranitica</i>, 5,262 transcripts were observed to have SSRs ( <a href="#sup2">Supplementary Table S7</a>). The total number of SSR-containing sequences in <i>S. guaranitica</i> was 5,931, following stringent selection criteria used to identify these SSRs. The analysis data showed that dinucleotide repeats were the most abundant motif type in <i>S. guaranitica</i> (2,787; 45.25%), followed by trinucleotide (1,555; 23.58%), mononucleotide (1,452; 23.58%), tetranucleotide (92; 1.493%), and hexanucleotide (28; 0.454%) types, while the pentanucleotide type was the least abundant motif (17; 0.276%) ( <a href="#sup2">Supplementary Table S8</a> and <a href="#sup1">Fig. S4</a>). Except for the absence of mononucleotide, these results were similar to the previous results obtained from the transcriptome of <i>O. sanctum</i> and <i>O. basilicum</i> (members of the same family), which have dinucleotide repeats as the most abundant motif type, followed by tri-, tetra-, hexa- and pentanucleotide types as the least abundant motif. <a href="#dsy028-B7"> <sup>7</sup> </a> After analysing the data from mono- to hexanucleotide motifs to obtain the number of repeat units, we found that the highest repeat unit of potential SSRs was 10, which accounted for 1,376 SSRs (27.08%), followed by 5 SSRs (1,049; 20.65%), 7 (728; 14.33%), and 6 (573; 11.28%), and the smallest repeat unit of potential SSRs was ≥24 (3; 0.0 5%) ( <a href="#sup2">Supplementary Table S9</a>). The AG/CT dinucleotide repeat was the most prevalent motif detected in all SSRs (1,893; 30.73%), followed by A/T as a mononucleotide repeat (1,408; 22.86%). In contrast, the least abundant motif in all SSRs (3; 0.048%) was detected in (AAAAC/GTTTT/AAAAG/CTTTT/AAACC/GGTTT) as pentanucleotide repeat and in (AAACAC/GTGTTT/AAACGG/CCGTTT/AAAGAC/CTTTGT) as hexanucleotide repeat. Finally, several SSR motifs were associated with many unique sequences that encode enzymes (e.g. <i>Sg</i>DXS1, <i>Sg</i>DXR1, <i>Sg</i>MCT, <i>Sg</i>HDR9, <i>Sg</i>IDI3, <i>Sg</i>AACT1, <i>Sg</i>HMGS, <i>Sg</i>HMGR2, <i>Sg</i>HMGR6, <i>Sg</i>MVK2, <i>Sg</i>GGPSΙΙ2, <i>Sg</i>Gibberellin 20-oxidase, <i>Sg</i>Beta-amyrin synthase, <i>Sg</i>Squalene monooxygenase and <i>Sg</i>farnesyl-diphosphate farnesyltransferase) involved in terpenoid biosynthesis ( <a href="#sup2">Supplementary Table S10</a>). </p> </div> <div class="3.8.validationofthegeneexpressionpatternsbyquantitativert-pcr" title="sec"> <div class="title" tagx="title" title="title">
3.8. Validation of the gene expression patterns by quantitative RT-PCR</div> <p>To determinate the reliability of the Illumina HiSeq 2000 read analysis, 15 candidate genes with a higher differential expression were selected, and their expression profiles were compared within young leaf, old leaf, stem, flower, bud flower and root samples. Quantitative real-time (qRT) PCR was used to determine the ‘transcriptional control’ which indicates the number of mRNA copies of the enzyme that complements the end-product quantity. Therefore, the correlation between the TPS mRNAs with their products and the end-products showed a relationship between the chosen differentially expressed genes (DEGs), monoterpene synthase ( <i>SgGPPS; KX893917</i>), sesquiterpene synthase ( <i>SgFPPS; KX893918</i>), β-caryophyllene ( <i>SgHUMS; KX893973</i>), neomenthol synthase-1 ( <i>SgNEOD-1; KX893955</i>), neomenthol synthase-2 ( <i>SgNEOD-2; KX893956</i>), neomenthol synthase-3 ( <i>SgNEOD-3; KX893957</i>), germacrene-A synthase ( <i>SgTPS-1; KX893975</i>), selinene synthase ( <i>SgTPS-3; KX893978</i>), germacrene- <div class="sc_UNKNOWN">
d</div> synthase ( <i>SgTPS-6; KX893977</i>), linalool synthase-1 ( <i>SgLINS-1; KX893965</i>), linalool synthase -2 ( <i>SgLINS-2; KX893966</i>), (E, E)-geranyl linalool synthase ( <i>SgGLNS; KX893967</i>), geraniol isomerase synthase ( <i>SgGERIS; KX893968</i>), valencene synthase ( <i>SgTPS-V; KX893974</i>), farnesol dehydrogenase ( <i>SgFARD; KX893969</i>) and the terpenoid biosynthesis pathway of <i>S. guaranitica</i>. <i>SgACTIN</i> was used as an internal reference gene ( <a href="#sup2">Supplementary Table S1</a>). The expression patterns of the 15 selected DEGs in the young leaf, old leaf, stem, flower, bud flower and root samples were examined ( <a href="#dsy028-F5">Fig. 5</a>) by qRT-PCR, and the results were consistent with the results from the Illumina HiSeq 2000 read analysis. At the current stage, we may be able to answer the question which terpenoid compounds of <i>S. guaranitica</i> accumulated mostly in which tissue. From our results, we found that the next gene, geranyl diphosphate synthase ( <i>SgGPPS</i>) gene showed the highest expression levels in the young leaves, followed by roots, stems, old leaves, bud flowers and flowers. These results were nearly compatible with our GC-MS analysis data indicating that the main group of terpenes in roots, bud flowers and young leaves consisted of monoterpene. According to the findings of the GC-MS analysis, we found eight monoterpene compound are accumulated in the root, two monoterpene compounds in bud flowers and one monoterpene compound are accumulated in young leaves ( <a href="#dsy028-T3">Table 3</a>).Therefore, we suggest that the roots are the primer site for monoterpene biosynthesis and accumulation, followed by, bud flower, and young leave. These results are not in agreement with <a href="#dsy028-B6"> <sup>6</sup> </a> <sup>,</sup> <a href="#dsy028-B44"> <sup>44</sup> </a> <sup>,</sup> <a href="#dsy028-B45"> <sup>45</sup> </a> that found that the main monoterpenes in some salvia plant species are formed and accumulated in very young leaves epidermal glands. Because, the formation of most epidermal glands and the accumulation of the monoterpenes, take very short time in young leave tissues. And our <i>S. guaranitica</i> plant has limited number from epidermal gland trichomes on old leave, young leaves and stem. Moreover, Sesquiterpene synthase (FPPS) gene recorded the highest expression levels in the root followed by flower, bud flower, young leave, old leave and stem. On the other hand, these results were not similar with GC-MS analysis data that showed that the main group of sesquiterpenes was mostly accumulated in young leaves. Which have five compounds followed by old leaves have 12 compounds, roots have four compounds, flowers have two and bud flower has one compound ( <a href="#dsy028-T3">Table 3</a>). Besides, from our study, we found a correlation and linkage between the β-Caryophyllene product and β-Caryophyllene synthase genes expression level in different tissues. For instance, the highest of the β-Caryophyllene synthase gene product and expression level presented in the young leaves followed by old leaves, roots then flowers ( <a href="#dsy028-T3">Table 3</a> and <a href="#dsy028-F1">Fig. 1</a>). Also, we found a correlation and linkage between the (-)-Germacrene D, Germacrene-A product and Germacrene-D synthase (TPS-6), Germacrene-A synthase (TPS-1) genes expression level in different tissues. Such as the highest of (-)-Germacrene-D, Germacrene-A gene product and expression level present in the young leave followed by old leave. Some of our results are in agreement with those of the previous studies <a href="#dsy028-B6"> <sup>6</sup> </a> <sup>,</sup> <a href="#dsy028-B44"> <sup>44–53</sup> </a> that reported that the terpene quantity levels are thought to be mainly controlled transcriptionally thought producing the different TPS enzymes. (+)-Neomenthol dehydrogenase-1,-2,-3, TPS-3-Selinene synthase, Linalool synthase-1,-2, (E, E)-geranyl linalool synthase, Geraniol isomerase synthase, TPS-Valencene synthase and Farnesol dehydrogenase genes that were detected in the Illumina HiSeq 2000 reads and QRT-PCR but was not detected in the GC-MS analysis data. We suggest that this could be due to the cyclic expression of terpene synthases is under circadian control. Although, changes in transcript levels may not directly determine protein levels or enzyme activities due to possible posttranscriptional, post-translational or enzyme-regulatory mechanisms, the positive correlation between transcript levels and volatile emission suggests that changes in transcript level are an important determinant of scent production. Furthermore, the different rates of protein synthesis and proteolytic turnover and/or differences in protein modifications. And the secondary modification of monoterpene olefins (e.g. oxidation/glycosylation) or sequestration also could contribute to the monoterpene emission profile. <a href="#dsy028-B54"> <sup>54</sup> </a> The combination of the analysed data reads from Illumina HiSeq 2000, qRT-PCR and the GC-MS will pave the way to understand the complex mechanisms of controlling and regulating the diversity of terpene compound production. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 5.</span> </a> </div>   <p>Quantitative RT-PCR validation of expression of terpene synthase genes selected from the DGE analysis in <i>S. guaranitica.</i> Total RNAs were extracted from old leaves, young leaves, stem, flower, bud flower and root samples and the expression of <i>SgGPPS</i>, <i>SgFPPS</i>, <i>SgHUMS</i>, <i>SgNEOD-1</i>, <i>SgNEOD-2</i>, <i>SgNEOD-3</i>, <i>SgTPS-1</i>, <i>SgTPS-3</i>, <i>SgTPS-6</i>, <i>SgLINS-1</i>, <i>SgLINS-2</i>, <i>SgGLNS</i>, <i>SgGERIS</i>, <i>SgTPS-V</i> and <i>SgFARD</i> genes were analysed using quantitative real-time. <i>SgACTIN</i> was used as the internal reference. The values are means ± SE of three biological replicates. </p>   </div> </div> <div class="3.9.functionalcharacterizationofterpenesynthasegenesintransgenica.thalianaleaves" title="sec"> <div class="title" tagx="title" title="title">
3.9. Functional characterization of terpene synthase genes in transgenic <i>A. thaliana</i> leaves </div> <p>To test <i>A. thaliana</i> in a transgenic expression system for the production of <i>Salvia</i> terpenes, the following genes were selected from <i>S. guaranitica</i>: farnesyl pyrophosphate synthases (FPPS), geranyl diphosphate synthases (GPPS) and (3S)-linalool synthase (LINS) encoded by <i>SgFPPS</i>, <i>SgGPPS</i> and <i>SgLINS</i>, respectively. Transgenic <i>A. thaliana</i> was carried out by using the Agrobacterium-mediated floral dip method of <i>A. thaliana</i> flowers using <i>A. tumefaciens</i> strain EHA105 carrying pB2GW7-FPPS, pB2GW7-GPPS and pB2GW7-LINS under the control of 35S promoter vector. Fully mature leaves from fifteen 35-day-old putative transgenic plants and wild type plant ( <a href="#dsy028-F6">Fig. 6A</a>), were collected for semiquantitative RT-PCR to analyse the positive transgenic <i>A. thaliana</i> and assessed the expression levels of terpene genes from the different samples ( <a href="#dsy028-F6">Fig. 6B</a>). The terpenes were extracted with hexane and analysed by GC-MS. The mono-, sesqui- and di-terpene peaks were clearly detected, and the type and amount of compounds represented by the percentage of peak area (% peak area). Compounds were identified by comparing their mass spectra the compounds with mass spectra libraries. The detected components were also confirmed by comparing them with the published references and extracts of wild-type Arabidopsis which produce different types and amounts of terpenes. Overexpression of the <i>SgFPPS</i>, <i>SgGPPS</i>, and <i>SoLINS</i> genes produced different amounts from mono-, sesqui- and di-terpenes and other terpenoids. Moreover, from the results shown in <a href="#dsy028-T4">Table 4</a> and <a href="#sup1">Supplementary Fig. S5</a>, we found that the transient expression of the different TPS genes from Salvia produced different types and amounts of mono-, sesqui- and di-terpenes and other terpenoid compounds. <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 4.</span>  </div></p><p>The major chemical compositions in transgenic <i>A. thaliana</i> leaves over-expressing of <i>SgFPPS</i>, <i>SgGPPS and SgLINS</i> </p>  <table> <div class="colgroup_UNKNOWN"> <div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div> <div class="colgroup_UNKNOWN"><div class="col_UNKNOWN" /></div>  <thead> <tr> <th> <i>N</i> </th> <th>Compound name</th> <th>Retention time (min.)</th> <th>Retention time index</th> <th>Formula</th> <th>Molecular mass(g mol <sup>−1)</sup> </th> <th>Terpene type</th> <th>W.T <hr /> </th> <th> <i>SgFPPS</i> <hr /> </th> <th> <i>SgGPPS</i> <hr /> </th> <th> <i>SgLINS</i> <hr /> </th> </tr> <tr> <th>% Peak area</th> <th>% Peak area</th> <th>% Peak area</th> <th>% Peak area</th> </tr> </thead> <tbody> <tr> <td>1</td> <td>alpha-Pinene</td> <td>5.942</td> <td>936</td> <td>C10H16</td> <td>136.24</td> <td>Mono</td> <td>–</td> <td>–</td> <td>27.63</td> <td>–</td> </tr> <tr> <td>2</td> <td>beta-Pinene</td> <td>7.945</td> <td>974</td> <td>C10H16</td> <td>136.24</td> <td>Mono</td> <td>–</td> <td>–</td> <td>5.77</td> <td>–</td> </tr> <tr> <td>3</td> <td>Menthol</td> <td>20.812</td> <td>1167</td> <td>C10H20O</td> <td>156.2652</td> <td>Mono</td> <td>2.75</td> <td>–</td> <td>–</td> <td>–</td> </tr> <tr> <td>4</td> <td>Thiourea, tetramethyl-</td> <td>22.326</td> <td>1872</td> <td>C5H12N2S</td> <td>132.227</td> <td /> <td>1.56</td> <td>–</td> <td>–</td> <td>–</td> </tr> <tr> <td>5</td> <td>Cadina-1, 4-diene</td> <td>26.844</td> <td>1533</td> <td>C15H24</td> <td>204.3511</td> <td>Sesqui</td> <td>–</td> <td>1.94</td> <td>–</td> <td>–</td> </tr> <tr> <td>6</td> <td>β-Elemene</td> <td>27.314</td> <td>1386</td> <td>C15H24</td> <td>204.3511</td> <td>Sesqui</td> <td>–</td> <td>2.14</td> <td>–</td> <td>–</td> </tr> <tr> <td>7</td> <td>β-Caryophyllene</td> <td>28.406</td> <td>1417</td> <td>C15H24</td> <td>204.3511</td> <td>Sesqui</td> <td>–</td> <td>17.71</td> <td>15.4</td> <td>4.64</td> </tr> <tr> <td>8</td> <td>Cycloheptasiloxane, tetradecamethyl</td> <td>29.621</td> <td>1519</td> <td>C14H42O7Si7</td> <td>519.0776</td> <td /> <td>–</td> <td>1.91</td> <td>–</td> <td>–</td> </tr> <tr> <td>9</td> <td>(Z)-α-Bisabolene</td> <td>29.628</td> <td>1503</td> <td>C15H24</td> <td>204.3511</td> <td>Sesqui</td> <td>–</td> <td>–</td> <td>6.76</td> <td>–</td> </tr> <tr> <td>10</td> <td>Germacrene D</td> <td>30.516</td> <td>1482</td> <td>C15H24</td> <td>204.3511</td> <td>Sesqui</td> <td>–</td> <td>71.39</td> <td>6.29</td> <td>10.73</td> </tr> <tr> <td>11</td> <td>Norcarane</td> <td>31.33</td> <td>796</td> <td>C7H12</td> <td>96.1702</td> <td /> <td>–</td> <td>–</td> <td>–</td> <td>4.06</td> </tr> <tr> <td>12</td> <td>Germacrene- <div class="sc_UNKNOWN">
d</div>-4-ol </td> <td>33.499</td> <td>1576</td> <td>C15H26O</td> <td>222.372</td> <td>Sesqui</td> <td>–</td> <td>1.33</td> <td>–</td> <td>–</td> </tr> <tr> <td>13</td> <td>Cyclooctasiloxane, hexadecamethyl-</td> <td>34.487</td> <td>1688</td> <td>C16H48O8Si8</td> <td>593.2315</td> <td /> <td>–</td> <td>1.17</td> <td>–</td> <td>–</td> </tr> <tr> <td>14</td> <td>Heneicosane</td> <td>35.592</td> <td>2100</td> <td>C21H44</td> <td>296.5741</td> <td /> <td>1.83</td> <td>–</td> <td>–</td> <td>–</td> </tr> <tr> <td>15</td> <td>Allethrin</td> <td>38.072</td> <td>2034</td> <td>C19H26O3</td> <td>302.4079</td> <td /> <td>2.6</td> <td>–</td> <td>–</td> <td>–</td> </tr> <tr> <td>16</td> <td>Nonadecane</td> <td>38.456</td> <td>1900</td> <td>C19H40</td> <td>268.5209</td> <td /> <td>1.92</td> <td>–</td> <td>–</td> <td>–</td> </tr> <tr> <td>17</td> <td>Cyclohexasiloxane, dodecamethyl-</td> <td>38.648</td> <td>1342</td> <td>C12H36O6Si6</td> <td>444.9236</td> <td /> <td>–</td> <td>0.93</td> <td>–</td> <td>–</td> </tr> <tr> <td>18</td> <td>Stearic acid</td> <td>40.237</td> <td>2178</td> <td>C18H36O2</td> <td>284.4772</td> <td /> <td>2</td> <td>–</td> <td>–</td> <td>–</td> </tr> <tr> <td>19</td> <td>Phytane</td> <td>41.196</td> <td>1800</td> <td>C20H42</td> <td>282.5475</td> <td>Diter</td> <td>2.56</td> <td>–</td> <td>–</td> <td /> </tr> <tr> <td>20</td> <td>1-Monolinoleoylglycerol trimethylsilyl ether</td> <td>42.337</td> <td>2780</td> <td>C27H54O4Si2</td> <td>498.89</td> <td /> <td>–</td> <td>0.8</td> <td>–</td> <td>–</td> </tr> <tr> <td>21</td> <td>Undecane, 4, 8-dimethyl-</td> <td>43.781</td> <td>1214</td> <td>C13H28</td> <td>184.3614</td> <td /> <td>3.26</td> <td>–</td> <td>–</td> <td>–</td> </tr> <tr> <td>22</td> <td>Oleic acid</td> <td>44.818</td> <td>2141</td> <td>C18H34O2</td> <td>282.468</td> <td /> <td>2.5</td> <td>–</td> <td>–</td> <td>–</td> </tr> <tr> <td>23</td> <td>Palmitic acid</td> <td>45.31</td> <td>2010</td> <td>C16H32O2</td> <td>256.4241</td> <td /> <td>22.06</td> <td>–</td> <td>–</td> <td>13.31</td> </tr> <tr> <td>24</td> <td>Palmitic acid, trimethylsilyl ester</td> <td>46.044</td> <td>2015</td> <td>C19H40O2Si</td> <td>328.6052</td> <td /> <td /> <td>–</td> <td>–</td> <td>–</td> </tr> <tr> <td>25</td> <td>Octadecane</td> <td>46.248</td> <td>1792</td> <td>C18H38</td> <td>254.4943</td> <td /> <td>3.17</td> <td>–</td> <td>–</td> <td>–</td> </tr> <tr> <td>26</td> <td>1-Butanol, 4-butoxy-</td> <td>46.518</td> <td>1705</td> <td>C8H18O2</td> <td>146.2273</td> <td /> <td>–</td> <td>–</td> <td>6.4</td> <td>–</td> </tr> <tr> <td>27</td> <td>Trimethylsilyl hexadecanoate</td> <td>47.316</td> <td>2015</td> <td>C19H40O2Si</td> <td>328.6052</td> <td /> <td>3.81</td> <td>–</td> <td>–</td> <td>–</td> </tr> <tr> <td>28</td> <td>Phytol</td> <td>47.772</td> <td>2115</td> <td>C20H40O</td> <td>296.531</td> <td>Diter</td> <td>–</td> <td>–</td> <td>–</td> <td>11.08</td> </tr> <tr> <td>29</td> <td>N-Hexacosane</td> <td>48.608</td> <td>2598</td> <td>C26H54</td> <td>366.707</td> <td /> <td>3.67</td> <td>–</td> <td>–</td> <td>–</td> </tr> <tr> <td>30</td> <td>dl-Methyltryptamine</td> <td>48.695</td> <td>1770</td> <td>C11H14N2</td> <td>174.24</td> <td /> <td>–</td> <td>–</td> <td>–</td> <td>7.59</td> </tr> <tr> <td>31</td> <td> <i>trans</i>-Elaidic acid </td> <td>49.469</td> <td>2123</td> <td>C18H34O2</td> <td>282.4614</td> <td /> <td>35.41</td> <td>–</td> <td>–</td> <td>–</td> </tr> <tr> <td>32</td> <td>Linoleic acid</td> <td>49.842</td> <td>2152</td> <td>C21H40O2Si</td> <td>352.6266</td> <td /> <td>–</td> <td>–</td> <td>–</td> <td>11.52</td> </tr> <tr> <td>33</td> <td>Hexacos-9-ene</td> <td>49.993</td> <td>2566</td> <td>C26H52</td> <td>364.6911</td> <td /> <td>–</td> <td>–</td> <td>9.79</td> <td>–</td> </tr> <tr> <td>34</td> <td>Heptadecane</td> <td>50.911</td> <td>1700</td> <td>C17H36</td> <td>240.4677</td> <td /> <td>3.03</td> <td>–</td> <td>–</td> <td>–</td> </tr> <tr> <td>35</td> <td>1-chloroeicosane</td> <td>51.916</td> <td>2264</td> <td>C20H41Cl</td> <td>316.993</td> <td /> <td>–</td> <td>–</td> <td>4.6</td> <td>–</td> </tr> <tr> <td>36</td> <td>Tetradeca-1, 13-diene</td> <td>52.855</td> <td>1385</td> <td>C14H26</td> <td>194.356</td> <td /> <td>–</td> <td>–</td> <td>4.34</td> <td>–</td> </tr> <tr> <td>37</td> <td>cis-4-tetradecene</td> <td>55.024</td> <td>1389</td> <td>C14H28</td> <td>196.378</td> <td /> <td>–</td> <td>–</td> <td>2.4</td> <td>–</td> </tr> <tr> <td>38</td> <td>Diisooctyl phthalate</td> <td>59.952</td> <td>2545</td> <td>C24H38O4</td> <td>390.564</td> <td /> <td>–</td> <td>–</td> <td>10.62</td> <td>21.73</td> </tr> <tr> <td>39</td> <td>17β-Estradiol, 3-deoxy</td> <td>63.773</td> <td>2300</td> <td>C18H24O</td> <td>256.3826</td> <td /> <td>–</td> <td>–</td> <td>–</td> <td>4.9</td> </tr> <tr> <td>40</td> <td>Trichloroacetic acid</td> <td>69.121</td> <td>1390</td> <td>Cl3CCOOH</td> <td>163.39</td> <td /> <td>–</td> <td>–</td> <td>–</td> <td>3.96</td> </tr> <tr> <td>41</td> <td>Cholestanol (5α-cholestan-3β-ol), TMS</td> <td>74.956</td> <td>3169</td> <td>C30H56OSi</td> <td>460.8505</td> <td /> <td>–</td> <td>–</td> <td>–</td> <td>2.91</td> </tr> <tr> <td>42</td> <td>1, 2-Epoxyhexane</td> <td>77.382</td> <td>768</td> <td>C6H12O</td> <td>100.1589</td> <td /> <td>–</td> <td>–</td> <td>–</td> <td>3.57</td> </tr> <tr> <td>43</td> <td>13, 23, 27-trimethylhenpentacontane</td> <td>78.973</td> <td>5164</td> <td>C54H110</td> <td>759.4512</td> <td /> <td>7.87</td> <td>–</td> <td>–</td> <td>–</td> </tr> <tr> <td /> <td>Total % peak area</td> <td /> <td /> <td /> <td /> <td /> <td>% 100</td> <td>% 100</td> <td>% 100</td> <td>% 100</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p>Mono, monoterpene; Sesqui, sesquiterpene; Dit, diterpene; –, the terpene and other compounds not detected. </p> </div> </div>  <p /> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 6.</span> </a> </div>   <p>Overexpression of three <i>S. guaranitica</i> terpenoid genes in transgenic Arabidopsis. (A) Comparison of the phenotypes of the transgenic <i>A. thaliana</i> and wild type <i>A. thaliana.</i> (B) Semiquantitative RT-PCR to confirm the expression of terpenoid genes. </p>   </div> <p>The putative functions of TPS genes isolated from <i>S. guaranitica</i> were initially predicted according to the conserved motifs using the InterPro protein sequence analysis and classification ( <a href="http://www.ebi.ac.uk/interpro/">http://www.ebi.ac.uk/interpro/</a>) database. The <i>SgGPPS</i> protein with a 418-aa length has a metal-binding domain (IPR005630) from 74-418 aa; inside this domain are two motifs: both are DDxxD motif (DDVLD) one motif starting at 177 aa, and the other one is starting at 304 aa. Additionally, the <i>SgFPPS</i> protein is 349-aa length has a metal-binding domain (IPR005630) from 6-349 aa; inside this domain are two motifs: both are DDxxD motif one is (DDIMD) starting at 100 aa, and the other one is a (DDYLD) starting at 239 aa. On the other hand, the <i>SgLINS</i> protein is 541-aa in length, this protein has an N-terminal domain (IPR001906) from 69-279 aa and a metal-binding domain (IPR005630) from 270-540 aa, and inside the latter domain are DDxxD conserved motifs (DDIFD) starting at 347 aa <a href="#sup1">Supplementary Fig. S6</a>. Finally, the protein sequences contaned one or two of this domain belong to the terpene synthase family. </p> <p>Furthermore, Croteau and coworkers shed light on the carbocationic reaction mechanism for all monoterpene synthases by reporting that the reaction was initiated by the divalent metal ion-dependent ionization of the substrate. The resulting cationic intermediate undergoes a series of hydride shifts or other rearrangements and cyclizations until the reaction was terminated by the addition of a nucleophile or proton loss. Croteau and coworkers illustrated this reaction mechanism by studying the native enzymes with substrate inhibitors, analogues and intermediates. <a href="#dsy028-B55"> <sup>55</sup> </a> <sup>,</sup> <a href="#dsy028-B56"> <sup>56</sup> </a> Moreover, Rodney Croteau et al. <sup>56</sup> elucidated the preliminary conversion of the geranyl cation to the tertiary linalyl cation to facilitate cyclization to a six-membered ring. Afterward, the linalyl cation provides the cyclic α-terpinyl cation; this is an important branching point intermediate in the formation of all cyclic monoterpenes because multiple terpene products can be obtained through electrophilic attack of C1 on the C6–C7 linalyl cation double bond and from the α-terpinyl cation. From the previous discussion, the reaction mechanisms of monoterpene synthases are highly reticulate. The individual intermediate may have multiple fates, which suggests the explanation for the ability of terpene enzymes to make various terpene products. <a href="#dsy028-B57"> <sup>57–60</sup> </a> On the other hand, the carbocationic reaction mechanism that uses sesquiterpene synthase to form sesquiterpenes by catalysing FPP recycling is similar to the reaction mechanism by those monoterpene synthases. Moreover, the larger carbon skeleton of FPP and the presence of three double bonds instead of two suggest a rationale for increases of the structural diversity of the sesquiterpene products. Furthermore, the initial cyclization reactions for sesquiterpene synthases can be divided into two types. Type one involves cyclization of the initially formed farnesyl cation to yield 11-membered ((E)-humulyl cation) rings of large size and a C2–C3 double bond (this type has no barrier to cyclization). The second type involves cyclization that proceeds after the tertiary nerolidyl cation produced from preliminary isomerization of the C2–C3 double bond. This isomerization mechanism is directly analogous to the isomerization of GPP to yield a linalyl cation in monoterpene synthesis. The nerolidyl cation is considered an intermediate in the sesquiterpene synthase mechanism. <a href="#dsy028-B61"> <sup>61–65</sup> </a> Collectively, we can state that the ability of TPS genes to convert a prenyl diphosphate substrate into diverse products during different reaction cycles is one of the unique traits of this type of enzyme. As described above, this property is found in the majority of all characterized monoterpene and sesquiterpene synthases. However, some monoterpene and sesquiterpene synthases can catalyse substrates into a single product, and the proteins may have specific methods for multiple product formations. For example, γ-humulene synthase from <i>A. grandis</i> has two DDxxD motifs located on opposite sides and can generate 52 different sesquiterpenes. This protein is able to bind substrates with two different conformations and resulting in different sets of products. <a href="#dsy028-B66"> <sup>66</sup> </a> In another example regarding the first monoterpene synthase cloned from <i>Salvia officinalis</i>, (+)-sabinene synthase produces 63% (+)-sabinene but also 21% γ-terpinene, 7.0% terpinolene, 6.5% limonene and 2.5% myrcene in <i>in vitro</i> assays. <a href="#dsy028-B67"> <sup>67</sup> </a> These additional monoterpene products or their immediate metabolites are also found in the monoterpene-rich essential oil of the <i>S. guaranitica</i> plant. </p> </div> </div> <div class="4.conclusion" title="sec"> <div class="title" tagx="title" title="title">
4. Conclusion</div> <p>In this study, a large, high-quality transcriptome database was established for <i>S. guaranitica</i> leaves using NGS technology to characterize and identify genes that are related to the terpenoid biosynthesis pathway. Using <i>de novo</i> sequencing and analysis of the <i>S. guaranitica</i> transcriptome data via the Illumina HiSeq 2000 system, we identified many genes that encode enzymes involved in terpenoid biosynthesis. The purpose of identifying these genes is not only to facilitate functional studies but also to develop biotechnology for improving the production of medicinal ingredients through metabolic engineering. We profiled terpenoids from six tissues of <i>S. guaranitica</i> and used qRT-PCR to determine the correlation between the expression levels of TPS genes and the end-products. By combining the transcriptome and metabolome analysis with RNA-Seq or qRT-PCR with GC-MS approaches, this study paves the way for understanding the complex metabolic genes for the production of the diverse terpene compounds in blue anise sage. The results from our study will allow us to understand the specific activities of TPS in <i>S. guaranitica</i> for the production of interesting compounds and to develop new technology for utilization. </p> <p>To our knowledge, this is the first study used Illumina HiSeq 2000 PE sequencing technology to investigate the global transcriptome of <i>S. guaranitica</i>. The valuable genetic resource in <i>salvia</i> will provide the foundation for future genetic and functional genomic research on <i>S. guaranitica</i> or closely related species. We further studied the functions of various <i>S. guaranitica</i> TPS genes, including <i>SgFPPS</i>, <i>SgGPPS</i> and <i>SoLINS</i>, by expressing these genes in <i>A. thaliana</i> transgenic plants. <i>SgFPPS</i>, <i>SgGPPS</i> and <i>SoLINS</i> were successfully expressed in the leaves of <i>A. thaliana</i>, and these transgenes altered the levels of terpenoids, as confirmed by GC-MS analysis of extracted transgenic <i>A. thaliana</i> leaves. The GC-MS analysis revealed that these <i>S. guaranitica</i> terpenoid synthases isolated from <i>S. guaranitica</i> can convert a prenyl diphosphate substrate into diverse products, which is one of the unique traits of this type of enzyme. Our study provides new insights into our understanding of plant terpenoid biosynthesis and the potential for biotechnology application. </p> </div> <div class="acknowledgements" title="sec"> <div class="title" tagx="title" title="title">
Acknowledgements</div> <p>The authors thank Associate Professor. Hazem Abdelnabby, Dr Jin Huanan, Dr Mohammed ayaad, Dr El sayed Elnishawy and Dr Mahmoud Kalil for proof-reading the manuscript. Also, the authors thank Dr Hongbo Lin for assistance for running the GC-Ms and data analysis. We also owe thanks to Prof. Zhang Yan sheng to give us the <i>S. guaranitica</i> seedlings. Also, the authors thank Dr Mohamed Hamdy Amar and Dr Wael Moussa, for their constructive comments and help. We also owe thanks to all Soybean Molecular Genetic lab and Rapeseed Lipid Biology lab members for facilitating the practical work at all levels of the experiments; the first author would like to thank the China Scholarship Council (CSC) and Huazhong Agricultural University for the scholarship. </p> </div> <div class="accessionnumbers" title="sec"> <div class="title" tagx="title" title="title">
Accession numbers</div> <p>KX869088, KX869089, KX869090, KX869091, KX869092, KX869093, KX869094, KX869095, KX869096, KX869097, KX869098, KX869099, KX869100, KX869101, KX869102, KX869103, KX869104, KX869105, KX869106, KX869107, KX869108, KX869109, KX869110, KX869111, KX869112, KX869113, KX869114, KX869115, KX869116, KX869117, KX869118, KX869119, KX869120, KX869121, KX869122, KX869123, KX869124, KX869125, KX893913, KX893914, KX893915, KX893916, KX893917, KX893918, KX893925, KX893926, KX893919, KX893920, KX893921, KX893927, KX893922, KX893923, KX893924, KX893928, KX893929, KX893930, KX893931, KX893932, KX893933, KX893934, KX893935, KX893936, KX893937, KX893938, KX893939, KX893940, KX893941, KX893942, KX893943, KX893944, KX893945, KX893946, KX893947, KX893948, KX893949, KX893950, KX893951, KX893952, KX893953, KX893954, KX893955, KX893956, KX893957, KX893958, KX893959, KX893960, KX893961, KX893962, KX893963, KX893964, KX893965, KX893966, KX893967, KX893968, KX893969, KX893970, KX893971, KX893972, KX893973, KX893974, KX893975, KX893976, KX893977, KX893978, KX893979, KX893980, KX893981, KX893982, KX893983, KX893984, KX893985, KX893986, KX893987, KX893988, KX893989, KX893990, KX893991, KX893992, KX893993, KX893994, KX893995, KX893996, KX893997, KX893998, KX893999, KX894000, KX894001, KX894002, KX894003, KX894004, KX894005, KX894006, KX894007, KX894008, KX894009, KX894010, KX894011, KX894012, KX894013, KX894014, KX894015, KX894016, KX894017 </p> </div> <div class="ethicsapprovalandconsenttoparticipate" title="sec"> <div class="title" tagx="title" title="title">
Ethics approval and consent to participate</div> <p>No investigations were undertaken using humans/human samples in this study. No experimental animals were used to conduct any of the experiments reported in this manuscript. Our study did not involve endangered or protected species. No specific permits were required from Wuhan Botanical Garden, China, for obtaining the seedlings of <i>Salvia guaranitica</i> L and Prof. Qingfeng Wang and Zhang Yan sheng should be contacted for future permissions. </p> </div> <div class="funding" title="sec"> <div class="title" tagx="title" title="title">
Funding</div> <p>M.A. was supported by the China Scholarship Council (CSC). This study was supported by the National Basic Research Development Program of China (973 Program, Grant 2013CB127001), and collaboration fund between Huazhong Agricultural University and Anhui Agricultural University. </p> </div> <div class="data-availability" title="data-availability"> <div class="title" tagx="title" title="title">
Availability of data and materials</div> <p>All data supporting my findings can be available and found in the <a href="#sup1">Supplementary data</a>. </p> </div> <div class="conflictofinterest" title="sec"> <div class="title" tagx="title" title="title">
Conflict of interest</div> <p>None declared.</p> </div> <div class="supplementary-material" title="supplementary-material"> <div class="title" tagx="title" title="title">
Supplementary Material</div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Supplementary Figures</span> <div class="media" title="media"> <a href="dsy028_supplementary_figure_s1-s6.ppt">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> <div class="local-data" title="local-data"> <span class="label" tagx="label" title="label">Supplementary Tables</span> <div class="media" title="media"> <a href="dsy028_supplementary_tables.doc">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> </div> </div> <div class="back" title="back"> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"> <a name="dsy028-B1" /> <span class="label" tagx="label" title="label">1</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Alziar</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span> </span> <span class="year" tagx="year" title="year">1988–1993</span>, Catalogue synonymique des Salvia L. dumonde (Lamiaceae). <i>I.–VI.</i> Biocosme Mesogéen., 5 (3–4): 87–136; 6(1–2, 4): 79–115, 163–204; 7(1–2): 59–109; 9(2–3): 413–497; 10(3–4): 33–117. </span> </li> <li tag="ref"> <a name="dsy028-B2" /> <span class="label" tagx="label" title="label">2</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Takano</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Okada</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> </span> <span class="year" tagx="year" title="year">2011</span>, <span class="mixed-article-title" title="mixed-article-title">Phylogenetic relationships among subgenera, species, and varieties of Japanese Salvia L. (Lamiaceae) </span>, <span class="source" tagx="source" title="source">J. Plant Res</span>., <span class="volume" tagx="volume" title="volume">124</span>, <span class="fpage" tagx="fpage" title="fpage">245</span>– <span class="lpage" tagx="lpage" title="lpage">52</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20628783">20628783</a> </span> </span> </li> <li tag="ref"> <a name="dsy028-B3" /> <span class="label" tagx="label" title="label">3</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Carretero-Paulet</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ahumada</span> <span class="given-names" tagx="given-names" title="given-names">I.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cunillera</span> <span class="given-names" tagx="given-names" title="given-names">N. M.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rodríguez</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ferrer</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Boronat</span> <span class="given-names" tagx="given-names" title="given-names">N.</span> </span> </span> <span class="year" tagx="year" title="year">2002</span>, <span class="mixed-article-title" title="mixed-article-title">Campos, expression and molecular analysis of the Arabidopsis DXR gene encoding 1-deoxy-D-xylulose-5-phosphate reductoisomerase, the first committed enzyme of the 2-C-methyl-D-erythritol-4-phosphate pathway </span>, <span class="source" tagx="source" title="source">Plant Physiol</span>., <span class="volume" tagx="volume" title="volume">129</span>, <span class="fpage" tagx="fpage" title="fpage">1581</span>– <span class="lpage" tagx="lpage" title="lpage">91</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12177470">12177470</a> </span> </span> </li> <li tag="ref"> <a name="dsy028-B4" /> <span class="label" tagx="label" title="label">4</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhao</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lawrence</span> <span class="given-names" tagx="given-names" title="given-names">C. D.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Robert</span> <span class="given-names" tagx="given-names" title="given-names">V.</span> </span> </span> <span class="year" tagx="year" title="year">2005</span>, <span class="mixed-article-title" title="mixed-article-title">Elicitor signal transduction leading to production of plant secondary metabolites</span>, <span class="source" tagx="source" title="source">Biotechnol. Adv</span>., <span class="volume" tagx="volume" title="volume">23</span>, <span class="fpage" tagx="fpage" title="fpage">283</span>– <span class="lpage" tagx="lpage" title="lpage">333</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15848039">15848039</a> </span> </span> </li> <li tag="ref"> <a name="dsy028-B5" /> <span class="label" tagx="label" title="label">5</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ward</span> <span class="given-names" tagx="given-names" title="given-names">J. A.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ponnala</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Weber</span> <span class="given-names" tagx="given-names" title="given-names">C. A.</span> </span> </span> <span class="year" tagx="year" title="year">2012</span>, <span class="mixed-article-title" title="mixed-article-title">Strategies for transcriptome analysis in nonmodel plants</span>, <span class="source" tagx="source" title="source">Am. J. Bot</span>., <span class="volume" tagx="volume" title="volume">2</span>, <span class="fpage" tagx="fpage" title="fpage">267</span>– <span class="lpage" tagx="lpage" title="lpage">76</span>. </span> </li> <li tag="ref"> <a name="dsy028-B6" /> <span class="label" tagx="label" title="label">6</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mohammed</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Penghui</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Guangbiao</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Daofu</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xiaochun</span> <span class="given-names" tagx="given-names" title="given-names">W.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jian</span> <span class="given-names" tagx="given-names" title="given-names">Z.</span> </span> </span> <span class="year" tagx="year" title="year">2017</span>, <span class="mixed-article-title" title="mixed-article-title">Transcriptome and metabolite analyses reveal the complex metabolic genes involved in volatile terpenoid biosynthesis in garden sage ( <i>Salvia officinalis</i>) </span>, <span class="source" tagx="source" title="source">Sci. Rep</span>., <span class="volume" tagx="volume" title="volume">7</span>, <span class="fpage" tagx="fpage" title="fpage">16074</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/29167468">29167468</a> </span> </span> </li> <li tag="ref"> <a name="dsy028-B7" /> <span class="label" tagx="label" title="label">7</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shubhra</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Seema</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ankita</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span> </span> <span class="year" tagx="year" title="year">2014</span>, <span class="mixed-article-title" title="mixed-article-title"> <i>De novo</i> sequencing and comparative analysis of holy and sweet basil transcriptomes </span>, <span class="source" tagx="source" title="source">BMC Genomics</span>, <span class="volume" tagx="volume" title="volume">15</span>, <span class="fpage" tagx="fpage" title="fpage">588</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25015319">25015319</a> </span> </span> </li> <li tag="ref"> <a name="dsy028-B8" /> <span class="label" tagx="label" title="label">8</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hua</span> <span class="given-names" tagx="given-names" title="given-names">W. P.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Song</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhao</span> <span class="given-names" tagx="given-names" title="given-names">L. J.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">Z. Z.</span> </span> </span> <span class="year" tagx="year" title="year">2011</span>, <span class="mixed-article-title" title="mixed-article-title"> <i>De novo</i> transcriptome sequencing in <i>Salvia miltiorrhiza</i> to identify genes involved in the biosynthesis of active ingredients </span>, <span class="source" tagx="source" title="source">Genomics</span>, <span class="volume" tagx="volume" title="volume">98</span>, <span class="fpage" tagx="fpage" title="fpage">272</span>– <span class="lpage" tagx="lpage" title="lpage">9</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21473906">21473906</a> </span> </span> </li> <li tag="ref"> <a name="dsy028-B9" /> <span class="label" tagx="label" title="label">9</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Meena</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kumar</span> <span class="given-names" tagx="given-names" title="given-names">S. R.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Venkata Rao</span> <span class="given-names" tagx="given-names" title="given-names">D. K.</span> </span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="year" tagx="year" title="year">2016</span>, <span class="mixed-article-title" title="mixed-article-title"> <i>De novo</i> sequencing and analysis of lemongrass transcriptome provide first insights into the essential oil biosynthesis of aromatic grasses </span>, <span class="source" tagx="source" title="source">Front. Plant Sci</span>., <span class="volume" tagx="volume" title="volume">7</span>, <span class="fpage" tagx="fpage" title="fpage">1129</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/27516768">27516768</a> </span> </span> </li> <li tag="ref"> <a name="dsy028-B10" /> <span class="label" tagx="label" title="label">10</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hyun</span> <span class="given-names" tagx="given-names" title="given-names">T. K.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rim</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jang</span> <span class="given-names" tagx="given-names" title="given-names">H.-J.</span> </span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="year" tagx="year" title="year">2012</span>, <span class="mixed-article-title" title="mixed-article-title"> <i>De novo</i> transcriptome sequencing of <i>Momordica cochinchinensis</i> to identify genes involved in the carotenoid biosynthesis </span>, <span class="source" tagx="source" title="source">Plant Mol. Biol</span>., <span class="volume" tagx="volume" title="volume">79</span>, <span class="fpage" tagx="fpage" title="fpage">413</span>– <span class="lpage" tagx="lpage" title="lpage">27</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22580955">22580955</a> </span> </span> </li> <li tag="ref"> <a name="dsy028-B11" /> <span class="label" tagx="label" title="label">11</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Huang</span> <span class="given-names" tagx="given-names" title="given-names">H. H.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xu</span> <span class="given-names" tagx="given-names" title="given-names">L. L.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tong</span> <span class="given-names" tagx="given-names" title="given-names">Z. K.</span> </span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="year" tagx="year" title="year">2012</span>, <span class="mixed-article-title" title="mixed-article-title"> <i>De novo</i> characterization of the Chinese fir ( <i>Cunninghamia lanceolata</i>) transcriptome and analysis of candidate genes involved in cellulose and lignin biosynthesis </span>, <span class="source" tagx="source" title="source">BMC Genomics</span>, <span class="volume" tagx="volume" title="volume">13</span>, <span class="fpage" tagx="fpage" title="fpage">648</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23171398">23171398</a> </span> </span> </li> <li tag="ref"> <a name="dsy028-B12" /> <span class="label" tagx="label" title="label">12</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shi</span> <span class="given-names" tagx="given-names" title="given-names">C. Y.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yang</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wei</span> <span class="given-names" tagx="given-names" title="given-names">C. L.</span> </span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="year" tagx="year" title="year">2011</span>, <span class="mixed-article-title" title="mixed-article-title">Deep sequencing of the <i>Camellia sinens</i> is transcriptome revealed candidate genes for major metabolic pathways of tea-specific compounds </span>, <span class="source" tagx="source" title="source">BMC Genomics</span>, <span class="volume" tagx="volume" title="volume">12</span>, <span class="fpage" tagx="fpage" title="fpage">131</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21356090">21356090</a> </span> </span> </li> <li tag="ref"> <a name="dsy028-B13" /> <span class="label" tagx="label" title="label">13</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Monica</span> <span class="given-names" tagx="given-names" title="given-names">R. L.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Federica</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rosa</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="year" tagx="year" title="year">2010</span>, <span class="mixed-article-title" title="mixed-article-title">Comparative chemical composition and antiproliferative activity of aerial parts of <i>Salvia leriifolia</i> Benth. and <i>Salvia acetabulosa</i> L. Essential oils against human tumor cell <i>in vitro</i> models </span>, <span class="source" tagx="source" title="source">J. Med. Food</span>., <span class="volume" tagx="volume" title="volume">13</span>, <span class="fpage" tagx="fpage" title="fpage">62</span>– <span class="lpage" tagx="lpage" title="lpage">9</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20136437">20136437</a> </span> </span> </li> <li tag="ref"> <a name="dsy028-B14" /> <span class="label" tagx="label" title="label">14</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Aziz</span> <span class="given-names" tagx="given-names" title="given-names">R. A.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hamed</span> <span class="given-names" tagx="given-names" title="given-names">F. k.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Abdulah</span> <span class="given-names" tagx="given-names" title="given-names">N. A.</span> </span> </span> <span class="year" tagx="year" title="year">2008</span>, <span class="mixed-article-title" title="mixed-article-title">Determination of the main components of the essential oil extracted from <i>Salvia fruticosa</i> by sing GC and GC-MS DAMASCUS </span>, <span class="source" tagx="source" title="source">J. Agric. Sci</span>., <span class="volume" tagx="volume" title="volume">24</span>, <span class="fpage" tagx="fpage" title="fpage">223</span>– <span class="lpage" tagx="lpage" title="lpage">36</span>. </span> </li> <li tag="ref"> <a name="dsy028-B15" /> <span class="label" tagx="label" title="label">15</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Su-Fang</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zeti-Azura</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Roohaida</span> <span class="given-names" tagx="given-names" title="given-names">O.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Noor</span> <span class="given-names" tagx="given-names" title="given-names">A. S.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ismanizan</span> <span class="given-names" tagx="given-names" title="given-names">I.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zamri</span> <span class="given-names" tagx="given-names" title="given-names">Z.</span> </span> </span> <span class="year" tagx="year" title="year">2014</span>, <span class="mixed-article-title" title="mixed-article-title">Functional characterization of sesquiterpene synthase from polygonum minus</span>, <span class="source" tagx="source" title="source">Sci. World J</span>., doi: 10.1155/2014/840592. </span> </li> <li tag="ref"> <a name="dsy028-B16" /> <span class="label" tagx="label" title="label">16</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Aharoni</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Giri</span> <span class="given-names" tagx="given-names" title="given-names">A. P.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Deuerlein</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="year" tagx="year" title="year">2003</span>, <span class="mixed-article-title" title="mixed-article-title">Terpenoid metabolism in wild-type and transgenic Arabidopsis plants,</span> <span class="source" tagx="source" title="source">Plant Cell</span>, <span class="volume" tagx="volume" title="volume">15</span>, <span class="fpage" tagx="fpage" title="fpage">2866</span>– <span class="lpage" tagx="lpage" title="lpage">84</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/14630967">14630967</a> </span> </span> </li> <li tag="ref"> <a name="dsy028-B17" /> <span class="label" tagx="label" title="label">17</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kim</span> <span class="given-names" tagx="given-names" title="given-names">H. A.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lim</span> <span class="given-names" tagx="given-names" title="given-names">C. J.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kim</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="year" tagx="year" title="year">2014</span>, <span class="mixed-article-title" title="mixed-article-title">High-throughput sequencing and <i>de novo</i> assembly of <i>Brassica oleracea</i> var. capitata L. for transcriptome analysis </span>, <span class="source" tagx="source" title="source">PLoS One</span>, <span class="volume" tagx="volume" title="volume">9</span>, <span class="fpage" tagx="fpage" title="fpage">e92087</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24682075">24682075</a> </span> </span> </li> <li tag="ref"> <a name="dsy028-B18" /> <span class="label" tagx="label" title="label">18</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Grabherr</span> <span class="given-names" tagx="given-names" title="given-names">M. G.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Haas</span> <span class="given-names" tagx="given-names" title="given-names">B. J.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yassour</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Levin</span> <span class="given-names" tagx="given-names" title="given-names">J. Z.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Thompson</span> <span class="given-names" tagx="given-names" title="given-names">D. A.</span> </span> </span> <span class="year" tagx="year" title="year">2011</span>, <span class="mixed-article-title" title="mixed-article-title">Full-length transcriptome assembly from RNA-Seq data without a reference genome</span>, <span class="source" tagx="source" title="source">Nat. Biotechnol</span>., <span class="volume" tagx="volume" title="volume">29</span>, <span class="fpage" tagx="fpage" title="fpage">644</span>– <span class="lpage" tagx="lpage" title="lpage">52</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21572440">21572440</a> </span> </span> </li> <li tag="ref"> <a name="dsy028-B19" /> <span class="label" tagx="label" title="label">19</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Anders</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Huber</span> <span class="given-names" tagx="given-names" title="given-names">W.</span> </span> </span> <span class="year" tagx="year" title="year">2010</span>, <span class="mixed-article-title" title="mixed-article-title">Differential expression analysis for sequence count data</span>, <span class="source" tagx="source" title="source">Genome Biol</span>., <span class="volume" tagx="volume" title="volume">11</span>, <span class="fpage" tagx="fpage" title="fpage">R106</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20979621">20979621</a> </span> </span> </li> <li tag="ref"> <a name="dsy028-B20" /> <span class="label" tagx="label" title="label">20</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Livak</span> <span class="given-names" tagx="given-names" title="given-names">K. J.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schmittgen</span> <span class="given-names" tagx="given-names" title="given-names">T. D.</span> </span> </span> <span class="year" tagx="year" title="year">2001</span>, <span class="mixed-article-title" title="mixed-article-title">Analysis of relative gene expression data using real-time quantitative PCR and the 2 <sup>−ΔΔCT</sup> method </span>, <span class="source" tagx="source" title="source">Methods</span>, <span class="volume" tagx="volume" title="volume">25</span>, <span class="fpage" tagx="fpage" title="fpage">402</span>– <span class="lpage" tagx="lpage" title="lpage">8</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11846609">11846609</a> </span> </span> </li> <li tag="ref"> <a name="dsy028-B21" /> <span class="label" tagx="label" title="label">21</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hongmei</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Luo</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhu</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="year" tagx="year" title="year">2014</span>, <span class="mixed-article-title" title="mixed-article-title">Transcriptional data mining of <i>Salvia miltiorrhiza</i> in response to methyl jasmonate to examine the mechanism of bioactive compound biosynthesis and regulation </span>, <span class="source" tagx="source" title="source">Physiol. Plant</span>, <span class="volume" tagx="volume" title="volume">152</span>, <span class="fpage" tagx="fpage" title="fpage">241</span>– <span class="lpage" tagx="lpage" title="lpage">55</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24660670">24660670</a> </span> </span> </li> <li tag="ref"> <a name="dsy028-B22" /> <span class="label" tagx="label" title="label">22</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fateme</span> <span class="given-names" tagx="given-names" title="given-names">A. M.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mohammad</span> <span class="given-names" tagx="given-names" title="given-names">H. F.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Abdolhossein</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ali</span> <span class="given-names" tagx="given-names" title="given-names">Z.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Maryam</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> </span> <span class="year" tagx="year" title="year">2013</span>, <span class="mixed-article-title" title="mixed-article-title">Volatile constituents of <i>Salvia compressa</i> and <i>Logochilus macranthus</i>, two labiatae herbs growing wild in Iran </span>, <span class="source" tagx="source" title="source">Res. J. Recent Sci</span>., <span class="volume" tagx="volume" title="volume">2</span>, <span class="fpage" tagx="fpage" title="fpage">66</span>– <span class="lpage" tagx="lpage" title="lpage">8</span>. </span> </li> <li tag="ref"> <a name="dsy028-B23" /> <span class="label" tagx="label" title="label">23</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Daniel</span> <span class="given-names" tagx="given-names" title="given-names">J. S.</span> </span> </span> <span class="year" tagx="year" title="year">2004</span>, <span class="mixed-article-title" title="mixed-article-title">Localization of salvinorin A and related compounds in glandular trichomes of the psychoactive sage <i>Salvia divinorum</i> </span>, <span class="source" tagx="source" title="source">Ann. Bot</span>., <span class="volume" tagx="volume" title="volume">93</span>, <span class="fpage" tagx="fpage" title="fpage">763</span>– <span class="lpage" tagx="lpage" title="lpage">71</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15087301">15087301</a> </span> </span> </li> <li tag="ref"> <a name="dsy028-B24" /> <span class="label" tagx="label" title="label">24</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Takano</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Okada</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> </span> <span class="year" tagx="year" title="year">2014</span>, <span class="mixed-article-title" title="mixed-article-title">Volatile profiling of aromatic traditional medicinal plant, <i>Polygonum minus</i> in different tissues and its biological activities </span>, <span class="source" tagx="source" title="source">Molecules</span>, <span class="volume" tagx="volume" title="volume">19</span>, <span class="fpage" tagx="fpage" title="fpage">19220</span>– <span class="lpage" tagx="lpage" title="lpage">42</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25420073">25420073</a> </span> </span> </li> <li tag="ref"> <a name="dsy028-B25" /> <span class="label" tagx="label" title="label">25</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">Z.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Fang</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="year" tagx="year" title="year">2010</span>, <span class="mixed-article-title" title="mixed-article-title"> <i>De novo</i> assembly and characterization of root transcriptome using Illumina paired-end sequencing and development of ISSR markers in sweet potato <i>(Ipomoea batatas</i>) </span>, <span class="source" tagx="source" title="source">BMC Genomics</span>, <span class="volume" tagx="volume" title="volume">11</span>, <span class="fpage" tagx="fpage" title="fpage">726</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21182800">21182800</a> </span> </span> </li> <li tag="ref"> <a name="dsy028-B26" /> <span class="label" tagx="label" title="label">26</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liang</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">X.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yiu</span> <span class="given-names" tagx="given-names" title="given-names">S.-M.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lim</span> <span class="given-names" tagx="given-names" title="given-names">B. L.</span> </span> </span> <span class="year" tagx="year" title="year">2013</span>, <span class="mixed-article-title" title="mixed-article-title"> <i>De novo</i> assembly and characterization of <i>Camelina sativa</i> transcriptome by paired-end sequencing </span>, <span class="source" tagx="source" title="source">BMC Genomics</span>, <span class="volume" tagx="volume" title="volume">14</span>( <span class="issue" tagx="issue" title="issue">1</span>), <span class="fpage" tagx="fpage" title="fpage">146</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23496985">23496985</a> </span> </span> </li> <li tag="ref"> <a name="dsy028-B27" /> <span class="label" tagx="label" title="label">27</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Annadurai</span> <span class="given-names" tagx="given-names" title="given-names">R. S.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Neethiraj</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jayakumar</span> <span class="given-names" tagx="given-names" title="given-names">V.</span> </span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="year" tagx="year" title="year">2013</span>, <span class="mixed-article-title" title="mixed-article-title"> <i>De Novo</i> transcriptome assembly (NGS) of <i>Curcuma longa</i> L. rhizome reveals novel transcripts related to anticancer and antimalarial terpenoids </span>, <span class="source" tagx="source" title="source">PLoS One</span>, <span class="volume" tagx="volume" title="volume">8</span>, <span class="fpage" tagx="fpage" title="fpage">e56217</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23468859">23468859</a> </span> </span> </li> <li tag="ref"> <a name="dsy028-B28" /> <span class="label" tagx="label" title="label">28</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">An</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shen</span> <span class="given-names" tagx="given-names" title="given-names">X.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ma</span> <span class="given-names" tagx="given-names" title="given-names">Q.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yang</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chen</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> </span> <span class="year" tagx="year" title="year">2014</span>, <span class="mixed-article-title" title="mixed-article-title">Transcriptome profiling to discover putative genes associated with paraquat resistance in goosegrass ( <i>Eleusine indica</i> L.) </span>, <span class="source" tagx="source" title="source">PLoS One</span>, <span class="volume" tagx="volume" title="volume">9</span>, <span class="fpage" tagx="fpage" title="fpage">e99940</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24927422">24927422</a> </span> </span> </li> <li tag="ref"> <a name="dsy028-B29" /> <span class="label" tagx="label" title="label">29</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Huang</span> <span class="given-names" tagx="given-names" title="given-names">L. L.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yang</span> <span class="given-names" tagx="given-names" title="given-names">X.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sun</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tong</span> <span class="given-names" tagx="given-names" title="given-names">W.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hu</span> <span class="given-names" tagx="given-names" title="given-names">S. Q.</span> </span> </span> <span class="year" tagx="year" title="year">2012</span>, <span class="mixed-article-title" title="mixed-article-title">The first Illumina-based <i>de novo</i> transcriptome sequencing and analysis of safflower flowers </span>, <span class="source" tagx="source" title="source">PLoS One</span>, <span class="volume" tagx="volume" title="volume">7</span>, <span class="fpage" tagx="fpage" title="fpage">e38653</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22723874">22723874</a> </span> </span> </li> <li tag="ref"> <a name="dsy028-B30" /> <span class="label" tagx="label" title="label">30</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yan</span> <span class="given-names" tagx="given-names" title="given-names">W.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yan</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhe</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="year" tagx="year" title="year">2013</span>, <span class="mixed-article-title" title="mixed-article-title"> <i>De novo</i> transcriptome sequencing of radish ( <i>Raphanus sativus</i> L.) and analysis of major genes involved in glucosinolate metabolism </span>, <span class="source" tagx="source" title="source">BMC Genomics</span>, <span class="volume" tagx="volume" title="volume">14</span>, <span class="fpage" tagx="fpage" title="fpage">836</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24279309">24279309</a> </span> </span> </li> <li tag="ref"> <a name="dsy028-B31" /> <span class="label" tagx="label" title="label">31</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gahlan</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Singh</span> <span class="given-names" tagx="given-names" title="given-names">H. R.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shankar</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="year" tagx="year" title="year">2012</span>, <span class="mixed-article-title" title="mixed-article-title"> <i>De novo</i> sequencing and characterization of <i>Picrorhiza kurrooa</i> transcriptome at two temperatures showed major transcriptome adjustments </span>, <span class="source" tagx="source" title="source">BMC Genomics</span>, <span class="volume" tagx="volume" title="volume">13</span>, <span class="fpage" tagx="fpage" title="fpage">126</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22462805">22462805</a> </span> </span> </li> <li tag="ref"> <a name="dsy028-B32" /> <span class="label" tagx="label" title="label">32</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yang</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ding</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lin</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="year" tagx="year" title="year">2013</span>, <span class="mixed-article-title" title="mixed-article-title">Transcriptome analysis of medicinal plant <i>Salvia miltiorrhiza</i> and identification of genes related to tanshinone biosynthesis </span>, <span class="source" tagx="source" title="source">PLoS One</span>, <span class="volume" tagx="volume" title="volume">8</span>, <span class="fpage" tagx="fpage" title="fpage">e80464</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24260395">24260395</a> </span> </span> </li> <li tag="ref"> <a name="dsy028-B33" /> <span class="label" tagx="label" title="label">33</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xie</span> <span class="given-names" tagx="given-names" title="given-names">F.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Burklew</span> <span class="given-names" tagx="given-names" title="given-names">C. E.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yang</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="year" tagx="year" title="year">2012</span>, <span class="mixed-article-title" title="mixed-article-title"> <i>De novo</i> sequencing and a comprehensive analysis of purple sweet potato ( <i>Impomoea batatas</i> L.) transcriptome </span>, <span class="source" tagx="source" title="source">Planta</span>, <span class="volume" tagx="volume" title="volume">236</span>, <span class="fpage" tagx="fpage" title="fpage">101</span>– <span class="lpage" tagx="lpage" title="lpage">13</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22270559">22270559</a> </span> </span> </li> <li tag="ref"> <a name="dsy028-B34" /> <span class="label" tagx="label" title="label">34</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hao</span> <span class="given-names" tagx="given-names" title="given-names">D. C.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">MA</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mu</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="year" tagx="year" title="year">2012</span>, <span class="mixed-article-title" title="mixed-article-title"> <i>De novo</i> characterization of the root transcriptome of a traditional Chinese medicinal plant <i>Polygonum</i> cuspidatum </span>, <span class="source" tagx="source" title="source">Sci. China Life Sci</span>., <span class="volume" tagx="volume" title="volume">55</span>, <span class="fpage" tagx="fpage" title="fpage">452</span>– <span class="lpage" tagx="lpage" title="lpage">66</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22645089">22645089</a> </span> </span> </li> <li tag="ref"> <a name="dsy028-B35" /> <span class="label" tagx="label" title="label">35</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kanehisa</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Goto</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> </span> <span class="year" tagx="year" title="year">2000</span>, <span class="mixed-article-title" title="mixed-article-title">KEGG: Kyoto Encyclopedia of Genes and Genomes</span>, <span class="source" tagx="source" title="source">Nucleic Acids Res</span>, <span class="volume" tagx="volume" title="volume">28</span>, <span class="fpage" tagx="fpage" title="fpage">27</span>– <span class="lpage" tagx="lpage" title="lpage">30</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10592173">10592173</a> </span> </span> </li> <li tag="ref"> <a name="dsy028-B36" /> <span class="label" tagx="label" title="label">36</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Virginie</span> <span class="given-names" tagx="given-names" title="given-names">V. D.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Germaine</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yamina</span> <span class="given-names" tagx="given-names" title="given-names">O.</span> </span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="year" tagx="year" title="year">2001</span>, <span class="mixed-article-title" title="mixed-article-title">Crystal structure of isopentenyl diphosphate: dimethylallyl diphosphate isomerase</span>, <span class="source" tagx="source" title="source">EMBO J</span>., <span class="volume" tagx="volume" title="volume">20</span>, <span class="fpage" tagx="fpage" title="fpage">1530</span>– <span class="lpage" tagx="lpage" title="lpage">7</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/11285217">11285217</a> </span> </span> </li> <li tag="ref"> <a name="dsy028-B37" /> <span class="label" tagx="label" title="label">37</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dorothea</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> </span> <span class="year" tagx="year" title="year">2006</span>, <span class="mixed-article-title" title="mixed-article-title">Terpene synthases and the regulation, diversity and biological roles of terpene metabolism</span>, <span class="source" tagx="source" title="source">Curr. Opin. Plant Biol</span>., <span class="volume" tagx="volume" title="volume">9</span>, <span class="fpage" tagx="fpage" title="fpage">297</span>– <span class="lpage" tagx="lpage" title="lpage">304</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16600670">16600670</a> </span> </span> </li> <li tag="ref"> <a name="dsy028-B38" /> <span class="label" tagx="label" title="label">38</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Douglas</span> <span class="given-names" tagx="given-names" title="given-names">J. M.-G.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rodney</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> </span> <span class="year" tagx="year" title="year">1995</span>, <span class="mixed-article-title" title="mixed-article-title">Terpenoid metabolism</span>, <span class="source" tagx="source" title="source">Plant Cell</span>, <span class="volume" tagx="volume" title="volume">7</span>, <span class="fpage" tagx="fpage" title="fpage">1015</span>– <span class="lpage" tagx="lpage" title="lpage">26</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/7640522">7640522</a> </span> </span> </li> <li tag="ref"> <a name="dsy028-B39" /> <span class="label" tagx="label" title="label">39</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Nagegowda</span> <span class="given-names" tagx="given-names" title="given-names">D. A.</span> </span> </span> <span class="year" tagx="year" title="year">2010</span>, <span class="mixed-article-title" title="mixed-article-title">Plant volatile terpenoid metabolism: biosynthetic genes, transcriptional regulation and subcellular compartmentation </span>, <span class="source" tagx="source" title="source">FEBS Lett</span>., <span class="volume" tagx="volume" title="volume">584</span>, <span class="fpage" tagx="fpage" title="fpage">2965</span>– <span class="lpage" tagx="lpage" title="lpage">73</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20553718">20553718</a> </span> </span> </li> <li tag="ref"> <a name="dsy028-B40" /> <span class="label" tagx="label" title="label">40</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Razborsek</span> <span class="given-names" tagx="given-names" title="given-names">M. I.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Voncina</span> <span class="given-names" tagx="given-names" title="given-names">D. B.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dolecek</span> <span class="given-names" tagx="given-names" title="given-names">V.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Voncina</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span> </span> <span class="year" tagx="year" title="year">2008</span>, <span class="mixed-article-title" title="mixed-article-title">Determination of oleanolic, betulinic and ursolic acid in lamiaceae and mass spectral fragmentation of their trimethylsilylated derivatives </span>, <span class="source" tagx="source" title="source">Chromatographia</span>, <span class="volume" tagx="volume" title="volume">67</span>, doi: 10.1365/s10337-008-0533-6. </span> </li> <li tag="ref"> <a name="dsy028-B41" /> <span class="label" tagx="label" title="label">41</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Misra</span> <span class="given-names" tagx="given-names" title="given-names">R. C.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Maiti</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chanotiya</span> <span class="given-names" tagx="given-names" title="given-names">C. S.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shanker</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ghosh</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> </span> <span class="year" tagx="year" title="year">2014</span>, <span class="mixed-article-title" title="mixed-article-title">Methyl jasmonate-elicited transcriptional responses and pentacyclic triterpenoid biosynthesis in sweet basil </span>, <span class="source" tagx="source" title="source">Plant Physiol</span>., <span class="volume" tagx="volume" title="volume">164</span>, <span class="fpage" tagx="fpage" title="fpage">1028</span>– <span class="lpage" tagx="lpage" title="lpage">1044</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/24367017">24367017</a> </span> </span> </li> <li tag="ref"> <a name="dsy028-B42" /> <span class="label" tagx="label" title="label">42</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Huang</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ye</span> <span class="given-names" tagx="given-names" title="given-names">H.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Li</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="year" tagx="year" title="year">2012</span>, <span class="mixed-article-title" title="mixed-article-title">Molecular characterization of the pentacyclic triterpenoid biosynthetic pathway in <i>Catharanthus roseus</i> </span>, <span class="source" tagx="source" title="source">Planta</span>, <span class="volume" tagx="volume" title="volume">236</span>, <span class="fpage" tagx="fpage" title="fpage">1571</span>– <span class="lpage" tagx="lpage" title="lpage">81</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/22837051">22837051</a> </span> </span> </li> <li tag="ref"> <a name="dsy028-B43" /> <span class="label" tagx="label" title="label">43</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Verma</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Shah</span> <span class="given-names" tagx="given-names" title="given-names">N.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bhatia</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> </span> <span class="year" tagx="year" title="year">2013</span>, <span class="mixed-article-title" title="mixed-article-title">Development of an expressed gene catalogue and molecular markers from the <i>de novo</i> assembly of short sequence reads of the lentil ( <i>Lens culinaris Medik</i>.) transcriptome </span>, <span class="source" tagx="source" title="source">Plant Biotechnol. J</span>., <span class="volume" tagx="volume" title="volume">11</span>, <span class="fpage" tagx="fpage" title="fpage">894</span>– <span class="lpage" tagx="lpage" title="lpage">905</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/23759076">23759076</a> </span> </span> </li> <li tag="ref"> <a name="dsy028-B44" /> <span class="label" tagx="label" title="label">44</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sabine</span> <span class="given-names" tagx="given-names" title="given-names">G.-G.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Corinna</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ralf</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Johannes</span> <span class="given-names" tagx="given-names" title="given-names">N.</span> </span> </span> <span class="year" tagx="year" title="year">2012</span>, <span class="mixed-article-title" title="mixed-article-title">Seasonal influence on gene expression of monoterpene synthases in <i>Salvia officinalis</i> (Lamiaceae) </span>, <span class="source" tagx="source" title="source">J. Plant Physiolol</span>., <span class="volume" tagx="volume" title="volume">169</span>, <span class="fpage" tagx="fpage" title="fpage">353</span>– <span class="lpage" tagx="lpage" title="lpage">9</span>., </span> </li> <li tag="ref"> <a name="dsy028-B45" /> <span class="label" tagx="label" title="label">45</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Croteau</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Felton</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Karp</span> <span class="given-names" tagx="given-names" title="given-names">F.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kjonaas</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> </span> <span class="year" tagx="year" title="year">1981</span>, <span class="mixed-article-title" title="mixed-article-title">Relationship of camphor biosynthesis to leaf development in sage <i>Salvia officinalis</i> </span>, <span class="source" tagx="source" title="source">Plant Physiol</span>., <span class="volume" tagx="volume" title="volume">67</span>, <span class="fpage" tagx="fpage" title="fpage">820</span>– <span class="lpage" tagx="lpage" title="lpage">4</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16661761">16661761</a> </span> </span> </li> <li tag="ref"> <a name="dsy028-B46" /> <span class="label" tagx="label" title="label">46</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dudareva</span> <span class="given-names" tagx="given-names" title="given-names">N.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cseke</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Blanc</span> <span class="given-names" tagx="given-names" title="given-names">V. M.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pichersky</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span> </span> <span class="year" tagx="year" title="year">1996</span>, <span class="mixed-article-title" title="mixed-article-title">Evolution of floral scent in Clarkia: novel patterns of S-linalool synthase gene expression in the <i>C. breweri</i> flower </span>, <span class="source" tagx="source" title="source">Plant Cell</span>., <span class="volume" tagx="volume" title="volume">8</span>, <span class="fpage" tagx="fpage" title="fpage">1137</span>– <span class="lpage" tagx="lpage" title="lpage">48</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/8768373">8768373</a> </span> </span> </li> <li tag="ref"> <a name="dsy028-B47" /> <span class="label" tagx="label" title="label">47</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">McConkey</span> <span class="given-names" tagx="given-names" title="given-names">M. E.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gershenzon</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Croteau</span> <span class="given-names" tagx="given-names" title="given-names">R. B.</span> </span> </span> <span class="year" tagx="year" title="year">2000</span>, <span class="mixed-article-title" title="mixed-article-title">Developmental regulation of monoterpene biosynthesis in the glandular trichomes of peppermint </span>, <span class="source" tagx="source" title="source">Plant Physiol</span>., <span class="volume" tagx="volume" title="volume">122</span>, <span class="fpage" tagx="fpage" title="fpage">215</span>– <span class="lpage" tagx="lpage" title="lpage">24</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/10631265">10631265</a> </span> </span> </li> <li tag="ref"> <a name="dsy028-B48" /> <span class="label" tagx="label" title="label">48</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mahmoud</span> <span class="given-names" tagx="given-names" title="given-names">S. S.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Croteau</span> <span class="given-names" tagx="given-names" title="given-names">R. B.</span> </span> </span> <span class="year" tagx="year" title="year">2003</span>, <span class="mixed-article-title" title="mixed-article-title">Menthofuran regulates essential oil biosynthesis in peppermint by controlling a downstream monoterpene reductase </span>, <span class="source" tagx="source" title="source">Proc. Natl. Acad. Sci. U.S.A</span>., <span class="volume" tagx="volume" title="volume">100</span>, <span class="fpage" tagx="fpage" title="fpage">14481</span>– <span class="lpage" tagx="lpage" title="lpage">6</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/14623962">14623962</a> </span> </span> </li> <li tag="ref"> <a name="dsy028-B49" /> <span class="label" tagx="label" title="label">49</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mahmoud</span> <span class="given-names" tagx="given-names" title="given-names">S. S.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Williams</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Croteau</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> </span> <span class="year" tagx="year" title="year">2004</span>, <span class="mixed-article-title" title="mixed-article-title">Cosuppression of limonene-3-hydroxylase in peppermint promotes accumulation of limonene in the essential oil </span>, <span class="source" tagx="source" title="source">Phytochemistry</span>, <span class="volume" tagx="volume" title="volume">65</span>, <span class="fpage" tagx="fpage" title="fpage">547</span>– <span class="lpage" tagx="lpage" title="lpage">54</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15003417">15003417</a> </span> </span> </li> <li tag="ref"> <a name="dsy028-B50" /> <span class="label" tagx="label" title="label">50</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xie</span> <span class="given-names" tagx="given-names" title="given-names">Z.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kapteyn</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gang</span> <span class="given-names" tagx="given-names" title="given-names">D. R.</span> </span> </span> <span class="year" tagx="year" title="year">2008</span>, <span class="mixed-article-title" title="mixed-article-title">A systems biology investigation of the MEP/terpenoid and shikimate/phenylpropanoid pathways points to multiple levels of metabolic control in sweet basil glandular trichomes </span>, <span class="source" tagx="source" title="source">Plant J</span>., <span class="volume" tagx="volume" title="volume">54</span>, <span class="fpage" tagx="fpage" title="fpage">349</span>– <span class="lpage" tagx="lpage" title="lpage">61</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18248593">18248593</a> </span> </span> </li> <li tag="ref"> <a name="dsy028-B51" /> <span class="label" tagx="label" title="label">51</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lane</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Boecklemann</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Woronuk</span> <span class="given-names" tagx="given-names" title="given-names">G. N.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sarker</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mahmoud</span> <span class="given-names" tagx="given-names" title="given-names">S. S.</span> </span> </span> <span class="year" tagx="year" title="year">2010</span>, <span class="mixed-article-title" title="mixed-article-title">A genomics resource for investigating regulation of essential oil production in <i>Lavandula angustifolia</i> </span>, <span class="source" tagx="source" title="source">Planta</span>, <span class="volume" tagx="volume" title="volume">231</span>, <span class="fpage" tagx="fpage" title="fpage">835</span>– <span class="lpage" tagx="lpage" title="lpage">45</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20043174">20043174</a> </span> </span> </li> <li tag="ref"> <a name="dsy028-B52" /> <span class="label" tagx="label" title="label">52</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schmiderer</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Grausgruber-Gröger</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Grassi</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Steinborn</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Novak</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> </span> <span class="year" tagx="year" title="year">2010</span>, <span class="mixed-article-title" title="mixed-article-title">Influence of gibberellin and daminozide on the expression of terpene synthases in common sage ( <i>Salvia officinalis</i>) </span>, <span class="source" tagx="source" title="source">J. Plant Physiol</span>., <span class="volume" tagx="volume" title="volume">167</span>, <span class="fpage" tagx="fpage" title="fpage">779</span>– <span class="lpage" tagx="lpage" title="lpage">86</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/20163890">20163890</a> </span> </span> </li> <li tag="ref"> <a name="dsy028-B53" /> <span class="label" tagx="label" title="label">53</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kampranis</span> <span class="given-names" tagx="given-names" title="given-names">S. C.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ioannidis</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Purvis</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="year" tagx="year" title="year">2007</span>, <span class="mixed-article-title" title="mixed-article-title">Rational conversion of substrate and product specificity in a Salvia monoterpene synthase: structural insights into the evolution of terpene synthase function </span>, <span class="source" tagx="source" title="source">Plant Cell</span>, <span class="volume" tagx="volume" title="volume">19</span>, <span class="fpage" tagx="fpage" title="fpage">1994</span>– <span class="lpage" tagx="lpage" title="lpage">2005</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17557809">17557809</a> </span> </span> </li> <li tag="ref"> <a name="dsy028-B54" /> <span class="label" tagx="label" title="label">54</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dudareva</span> <span class="given-names" tagx="given-names" title="given-names">N.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Martin</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kish</span> <span class="given-names" tagx="given-names" title="given-names">C. M.</span> </span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="year" tagx="year" title="year">2003</span>, <span class="mixed-article-title" title="mixed-article-title">( <i>E</i>)-β-Ocimene and myrcene synthase genes of floral scent biosynthesis in snapdragon </span>, <span class="source" tagx="source" title="source">Plant Cell</span>, <span class="volume" tagx="volume" title="volume">15</span>, <span class="fpage" tagx="fpage" title="fpage">1227</span>– <span class="lpage" tagx="lpage" title="lpage">41</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12724546">12724546</a> </span> </span> </li> <li tag="ref"> <a name="dsy028-B55" /> <span class="label" tagx="label" title="label">55</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rodney</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mark</span> <span class="given-names" tagx="given-names" title="given-names">F.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Robert</span> <span class="given-names" tagx="given-names" title="given-names">C. R.</span> </span> </span> <span class="year" tagx="year" title="year">1980</span>, <span class="mixed-article-title" title="mixed-article-title">Biosynthesis of monoterpenes – conversion of the acyclic precursor’s geranyl pyrophosphate and nerylpyrophosphate to the rearranged monoterpenes fenchol and fenchone by a soluble enzyme preparation from fennel ( <i>Foeniculum vulgare</i>) </span>, <span class="source" tagx="source" title="source">Arch. Biochem. Biophys</span>., <span class="volume" tagx="volume" title="volume">200</span>, <span class="fpage" tagx="fpage" title="fpage">524</span>– <span class="lpage" tagx="lpage" title="lpage">33</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/7436420">7436420</a> </span> </span> </li> <li tag="ref"> <a name="dsy028-B56" /> <span class="label" tagx="label" title="label">56</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rodney</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> </span> <span class="year" tagx="year" title="year">1987</span>, <span class="mixed-article-title" title="mixed-article-title">Biosynthesis and catabolism of monoterpenoids</span>, <span class="source" tagx="source" title="source">Chem. Rev</span>., <span class="volume" tagx="volume" title="volume">87</span>, <span class="fpage" tagx="fpage" title="fpage">929</span>– <span class="lpage" tagx="lpage" title="lpage">54</span>. </span> </li> <li tag="ref"> <a name="dsy028-B57" /> <span class="label" tagx="label" title="label">57</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wise</span> <span class="given-names" tagx="given-names" title="given-names">M. L.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rodney</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> </span> <span class="year" tagx="year" title="year">1999</span>, <span class="source" tagx="source" title="source">Comprehensive Natural Products Chemistry, Isoprenoids Including Caroteinoids and Steroids</span>, vol. 2, pp. <span class="fpage" tagx="fpage" title="fpage">97</span>– <span class="lpage" tagx="lpage" title="lpage">135</span>. <span class="publisher-name" tagx="publisher-name" title="publisher-name">Elsevier</span>: <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Amsterdam</span>. </span> </li> <li tag="ref"> <a name="dsy028-B58" /> <span class="label" tagx="label" title="label">58</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lücker</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">El-Tamer</span> <span class="given-names" tagx="given-names" title="given-names">M. K.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schwab</span> <span class="given-names" tagx="given-names" title="given-names">W.</span> </span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="year" tagx="year" title="year">2002</span>, <span class="mixed-article-title" title="mixed-article-title">Monoterpene biosynthesis in lemon (Citrus Limon) – cDNA isolation and functional analysis of four monoterpene synthases </span>, <span class="source" tagx="source" title="source">Eur. J. Biochem</span>., <span class="volume" tagx="volume" title="volume">269</span>, <span class="fpage" tagx="fpage" title="fpage">3160</span>– <span class="lpage" tagx="lpage" title="lpage">71</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12084056">12084056</a> </span> </span> </li> <li tag="ref"> <a name="dsy028-B59" /> <span class="label" tagx="label" title="label">59</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Takehiko</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tomoko</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hiroshi</span> <span class="given-names" tagx="given-names" title="given-names">F.</span> </span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="year" tagx="year" title="year">2004</span>, <span class="mixed-article-title" title="mixed-article-title">Molecular cloning and functional characterization of four monoterpene synthase genes from Citrus unshiu Marc </span>, <span class="source" tagx="source" title="source">Plant Sci</span>., <span class="volume" tagx="volume" title="volume">166</span>, <span class="fpage" tagx="fpage" title="fpage">49</span>– <span class="lpage" tagx="lpage" title="lpage">58</span>. </span> </li> <li tag="ref"> <a name="dsy028-B60" /> <span class="label" tagx="label" title="label">60</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dezene</span> <span class="given-names" tagx="given-names" title="given-names">P. W. H.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ryan</span> <span class="given-names" tagx="given-names" title="given-names">N. P.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kimberley-Ann</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rona</span> <span class="given-names" tagx="given-names" title="given-names">N. S.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jörg</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span> </span> <span class="year" tagx="year" title="year">2005</span>, <span class="mixed-article-title" title="mixed-article-title">Characterization of four terpene synthase cDNAs from methyl jasmonateinduced Douglas-fir, <i>Pseudotsuga menziesii</i> </span>, <span class="source" tagx="source" title="source">Phytochemistry</span>, <span class="volume" tagx="volume" title="volume">66</span>, <span class="fpage" tagx="fpage" title="fpage">1427</span>– <span class="lpage" tagx="lpage" title="lpage">39</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15921711">15921711</a> </span> </span> </li> <li tag="ref"> <a name="dsy028-B61" /> <span class="label" tagx="label" title="label">61</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Martin</span> <span class="given-names" tagx="given-names" title="given-names">D. M.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bohlmann</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> </span> <span class="year" tagx="year" title="year">2004</span>, <span class="mixed-article-title" title="mixed-article-title">Identification of <i>Vitis vinifera</i> (-)-alpha-terpineol synthase by in silico screening of full-length cDNA ESTs and functional characterization of recombinant terpene synthase </span>, <span class="source" tagx="source" title="source">Phytochemistry</span>, <span class="volume" tagx="volume" title="volume">65</span>, <span class="fpage" tagx="fpage" title="fpage">1223</span>– <span class="lpage" tagx="lpage" title="lpage">9</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/15184006">15184006</a> </span> </span> </li> <li tag="ref"> <a name="dsy028-B62" /> <span class="label" tagx="label" title="label">62</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">David</span> <span class="given-names" tagx="given-names" title="given-names">E. C.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stephen</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pushpalatha</span> <span class="given-names" tagx="given-names" title="given-names">P. N. M.</span> </span> </span> <span class="year" tagx="year" title="year">1981</span>, <span class="mixed-article-title" title="mixed-article-title">Trichodiene biosynthesis and the enzymatic cyclization of farnesyl pyrophosphate</span>, <span class="source" tagx="source" title="source">J. Am. Chem. Soc</span>., <span class="volume" tagx="volume" title="volume">103</span>, <span class="fpage" tagx="fpage" title="fpage">2136</span>– <span class="lpage" tagx="lpage" title="lpage">8</span>. </span> </li> <li tag="ref"> <a name="dsy028-B63" /> <span class="label" tagx="label" title="label">63</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">David</span> <span class="given-names" tagx="given-names" title="given-names">E. C.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Guohan</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> </span> <span class="year" tagx="year" title="year">1994</span>, <span class="mixed-article-title" title="mixed-article-title">Trichodiene synthase – stereochemical studies of the cryptic allylic diphosphate isomerase activity using an anomalous substrate </span>, <span class="source" tagx="source" title="source">J. Org. Chem</span>., <span class="volume" tagx="volume" title="volume">59</span>, <span class="fpage" tagx="fpage" title="fpage">5794</span>– <span class="lpage" tagx="lpage" title="lpage">8</span>. </span> </li> <li tag="ref"> <a name="dsy028-B64" /> <span class="label" tagx="label" title="label">64</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">David</span> <span class="given-names" tagx="given-names" title="given-names">E. C.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Manish</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> </span> <span class="year" tagx="year" title="year">1995</span>, <span class="mixed-article-title" title="mixed-article-title">Epicubenol synthase and the stereochemistry of the enzymatic cyclization of farnesyl and nerolidyl diphosphate </span>, <span class="source" tagx="source" title="source">J. Am. Chem. Soc</span>., <span class="volume" tagx="volume" title="volume">117</span>, <span class="fpage" tagx="fpage" title="fpage">5602</span>– <span class="lpage" tagx="lpage" title="lpage">3</span>. </span> </li> <li tag="ref"> <a name="dsy028-B65" /> <span class="label" tagx="label" title="label">65</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Alchanati</span> <span class="given-names" tagx="given-names" title="given-names">I.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Patel</span> <span class="given-names" tagx="given-names" title="given-names">J. A. A.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> </span>, <span class="etal" title="etal"> <i>et al.</i> </span> <span class="year" tagx="year" title="year">1998</span>, <span class="mixed-article-title" title="mixed-article-title">The enzymatic cyclization of nerolidyl diphosphate by delta cadinene synthase from cotton stele tissue infected with Verticillium dahlia </span>, <span class="source" tagx="source" title="source">Phytochemistry</span>, <span class="volume" tagx="volume" title="volume">47</span>, <span class="fpage" tagx="fpage" title="fpage">961</span>– <span class="lpage" tagx="lpage" title="lpage">7</span>. </span> </li> <li tag="ref"> <a name="dsy028-B66" /> <span class="label" tagx="label" title="label">66</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Steele</span> <span class="given-names" tagx="given-names" title="given-names">C. L.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Crock</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bohlmann</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Croteau</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> </span> <span class="year" tagx="year" title="year">1998</span>, <span class="mixed-article-title" title="mixed-article-title">Sesquiterpene synthases from grand fir ( <i>Abies grandis</i>) – Comparison of constitutive and wound-induced activities, and cDNA isolation, characterization and bacterial expression of delta-selinene synthase and gamma-humulene synthase </span>, <span class="source" tagx="source" title="source">J. Biol. Chem</span>., <span class="volume" tagx="volume" title="volume">273</span>, <span class="fpage" tagx="fpage" title="fpage">2078</span>– <span class="lpage" tagx="lpage" title="lpage">89</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9442047">9442047</a> </span> </span> </li> <li tag="ref"> <a name="dsy028-B67" /> <span class="label" tagx="label" title="label">67</span> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wise</span> <span class="given-names" tagx="given-names" title="given-names">M. L.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Savage</span> <span class="given-names" tagx="given-names" title="given-names">T. J.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Katahira</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span>, <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Croteau</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> </span> <span class="year" tagx="year" title="year">1998</span>, <span class="mixed-article-title" title="mixed-article-title">Monoterpene synthases from common sage ( <i>Salvia officinalis</i>) – cDNA isolation, characterization, and functional expression of (+)-sabinene synthase, 1, 8-cineole synthase, and (+)- bornyl diphosphate synthase </span>, <span class="source" tagx="source" title="source">J. Biol. Chem</span>., <span class="volume" tagx="volume" title="volume">273</span>, <span class="fpage" tagx="fpage" title="fpage">14891</span>– <span class="lpage" tagx="lpage" title="lpage">9</span>. <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/9614092">9614092</a> </span> </span> </li> </ul> </div> </div>  </body></html>