<html xmlns="http://www.w3.org/1999/xhtml"><head> <style type="text/css">
         	     		 
         	    a                          {background : #ffffff; }
         	    article                    {border-style : dotted; border-width : 2px; }
         		 	
         		 	div                        {background : #ffffcc;}
         		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
         		 	div.ack                    {border-style : solid ; border-color : red; margin
         : 2em; }
         		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.boxed-text             {margin : 5em; border-style : solid;}
         		 	div.contrib-group          {margin : 1em; }
         		 	div.fig                    {border-style : solid; border-width : 2px; margin :
         2em; }
         		 	div.funding                {font-weight : bold ; font-size : 16pt;}
         		 	div.given-names            {font-style : italic;}
         		 	div.intro                  {border-style : inset; margin : 5px;}
         		 	div.introduction           {border-style : inset; margin : 5px;}
         		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.journal-title-group    {background : #ffeeee;}
         		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
         		 	div.kwd                    {font-style : italic}
         		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
         		 	div.name                   {font-weight : bold;}
         		 	div.alt-title              {font-style : italic; font-size : 12px;}
         		 	
         		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
         		 	div.title                  {font-family : courier; font-weight : bold;
         		 	                            font-size : 16pt; margin : 5px;}
         		 	
         		 	div.materials_methods      {border-style : double; margin : 5px; }
         		 	div.methods                {border-style : double; margin : 5px; }
         		 	
         		 	div.results                {border-style : solid; margin : 5px;}
         		 	div.background             {border-style : dotted; margin : 5px;}
         		 	
         		 	div.discussion             {border-style : groove; margin : 5px;}
         		 	div.conclusion             {border-style : ridge; margin : 5px;}
         		 	div.conclusions             {border-style : ridge; margin : 5px;}
         		 	div.supplementary-material {border-style : inset; margin : 5px;}
         		 	div.abbreviations          {border-style : double; border-color : red; margin
         : 5px;}
         		 	div.competinginterests     {border-style : double; border-color : blue; margin
         : 5px;}
         		 	div.acknowledgements       {border-style : double; border-color : green; margin
         : 5px;}
         		 	div.authors_contributions   {border-style : double; border-color : purple; margin
         : 5px;}
         		 	
         		 	div.publisher              {border-style : outset; margin : 5px;}
         		 	div.fn-type-conflict       {background : #f88; }
         		 	div.fn-type-con            {background : #ddf; }
         		 	div.fn-type-other          {background : #ddd; }
         		 	
         		 	div.unknown                {background : #ffd;
         									 	  border-style : solid;
         									 	  border-width : 1px;
         									 	  padding : 2 px;}
         		 	  
         	    table                      {background : #ffffdd;}
         		 	tr                         {background : #ddddff; padding : 1px;}
         		 	
         		 	span                       {background : #ffcccc;}
         		 	
         		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
         		 	span.collab                {background : #ddffff; }
         		 	span.comment               {font-family : courier; font-size : 6px; background
         : #ffaaff;}
         		 	span.contrib               {background : #ffffff;}
         		 	span.corresp               {background : #ddffdd; }
         		 	span.doi                   {background : #ffffff;}
         		 	span.email                 {font-family : courier; }
         		 	span.etal                  {font-style : italic;}
         		 	span.fpage                 {font-family : courier;}
         		 	span.given-names           {background : #ffffff;}
         		 	span.iso-abbrev            {background : #ffffff;}
         		 	span.issn-epub             {background : #ffffff;}
         		 	span.issn-ppub             {background : #ffffff;}
         		 	span.journal-title         {background : #ffffff;}
         		 	span.lpage                 {font-family : courier;}
         		 	span.mixed-article-title   {font-style : italic ;}
         		 	span.nlm-ta                {background : #ffffff;}
         		 	span.pmc                   {background : #ffffff;}
         		 	span.pmcid                 {background : #ffffff;}
         		 	span.pmid                  {background : #ffffff;}
         		 	span.publisher             {background : #ffffff;}
         		 	span.publisher-id          {background : #ffffff;}
         		 	span.publisher-name        {background : #ffffff;}
         		 	span.source                {background : #ffffff;}
         		 	span.subject               {background : #ffffff;}
         		 	span.surname               {background : #ffffff;}
         		 	span.volume                {font-family : courier; font-weight : bold;}
         		 	span.year                  {font-family : courier ; font-style : italic;}
         			
      </style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"> <span class="nlm-ta" title="nlm-ta">Front Plant Sci</span> <span class="iso-abbrev" title="iso-abbrev">Front Plant Sci</span> <span class="publisher-id" title="publisher-id">Front. Plant Sci.</span> <div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"> <span class="journal-title" tagx="journal-title" title="journal-title">Frontiers in Plant Science</span> </div> <span class="issn-epub" tagx="issn" title="issn-epub">1664-462X</span> <div class="publisher" tagx="publisher" title="publisher"> <span class="publisher-name" tagx="publisher-name" title="publisher-name">Frontiers Media S.A.</span> </div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"> <span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/6099104">6099104</a> </span> <span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.3389/fpls.2018.01081">10.3389/fpls.2018.01081</a> </span> <div class="article-categories" title="article-categories"> <div class="sub-group" title="subj-group">
: <span class="subject" title="subject">Plant Science</span>: <span class="subject" title="subject">Technology Report</span> </div> </div> <div class="title-group"> <div class="article-title" title="article-title">
AromaDb: A Database of Medicinal and Aromatic Plantâ€™s Aroma Molecules With Phytochemistry and Therapeutic Potentials </div> </div> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kumar</span> <span class="given-names" tagx="given-names" title="given-names">Yogesh</span> </span> <a href="#aff1"> <sup>1</sup> </a> <span class="uri" title="uri"> <a href="http://loop.frontiersin.org/people/501832/overview" /> </span> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Prakash</span> <span class="given-names" tagx="given-names" title="given-names">Om</span> </span> <a href="#aff1"> <sup>1</sup> </a> <span class="uri" title="uri"> <a href="http://loop.frontiersin.org/people/543106/overview" /> </span> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tripathi</span> <span class="given-names" tagx="given-names" title="given-names">Himanshu</span> </span> <a href="#aff1"> <sup>1</sup> </a> <span class="uri" title="uri"> <a href="http://loop.frontiersin.org/people/588013/overview" /> </span> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tandon</span> <span class="given-names" tagx="given-names" title="given-names">Sudeep</span> </span> <a href="#aff2"> <sup>2</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gupta</span> <span class="given-names" tagx="given-names" title="given-names">Madan M.</span> </span> <a href="#aff3"> <sup>3</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rahman</span> <span class="given-names" tagx="given-names" title="given-names">Laiq-Ur</span> </span> <a href="#aff4"> <sup>4</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lal</span> <span class="given-names" tagx="given-names" title="given-names">Raj K.</span> </span> <a href="#aff5"> <sup>5</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Semwal</span> <span class="given-names" tagx="given-names" title="given-names">Manoj</span> </span> <a href="#aff6"> <sup>6</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Darokar</span> <span class="given-names" tagx="given-names" title="given-names">Mahendra Pandurang</span> </span> <a href="#aff7"> <sup>7</sup> </a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Khan</span> <span class="given-names" tagx="given-names" title="given-names">Feroz</span> </span> <a href="#aff1"> <sup>1</sup> </a> <a href="#aff8"> <sup>8</sup> </a> <a href="#c001"> <sup>*</sup> </a> <span class="uri" title="uri"> <a href="http://loop.frontiersin.org/people/562406/overview" /> </span> </span> </div> <span class="citation_author_institution" id="aff1">[], <sup>1</sup> <span class="institution" title="institution">Department of Metabolic and Structural Biology, CSIR-Central Institute of Medicinal and Aromatic Plants </span> <span class="addr-line" title="addr-line">Lucknow</span> <span class="country" tagx="country" title="country">India</span> </span> <span class="citation_author_institution" id="aff2">[], <sup>2</sup> <span class="institution" title="institution">Department of Process Chemistry and Chemical Engineering, CSIR-Central Institute of Medicinal and Aromatic Plants </span> <span class="addr-line" title="addr-line">Lucknow</span> <span class="country" tagx="country" title="country">India</span> </span> <span class="citation_author_institution" id="aff3">[], <sup>3</sup> <span class="institution" title="institution">Department of Analytical Chemistry, CSIR-Central Institute of Medicinal and Aromatic Plants </span> <span class="addr-line" title="addr-line">Lucknow</span> <span class="country" tagx="country" title="country">India</span> </span> <span class="citation_author_institution" id="aff4">[], <sup>4</sup> <span class="institution" title="institution">Department of Plant Biotechnology, CSIR-Central Institute of Medicinal and Aromatic Plants </span> <span class="addr-line" title="addr-line">Lucknow</span> <span class="country" tagx="country" title="country">India</span> </span> <span class="citation_author_institution" id="aff5">[], <sup>5</sup> <span class="institution" title="institution">Department of Plant Breeding, CSIR-Central Institute of Medicinal and Aromatic Plants</span> <span class="addr-line" title="addr-line">Lucknow</span> <span class="country" tagx="country" title="country">India</span> </span> <span class="citation_author_institution" id="aff6">[], <sup>6</sup> <span class="institution" title="institution">Information and Communication Technology Lab, CSIR-Central Institute of Medicinal and Aromatic Plants </span> <span class="addr-line" title="addr-line">Lucknow</span> <span class="country" tagx="country" title="country">India</span> </span> <span class="citation_author_institution" id="aff7">[], <sup>7</sup> <span class="institution" title="institution">Department of Molecular Bioprospection, CSIR-Central Institute of Medicinal and Aromatic Plants </span> <span class="addr-line" title="addr-line">Lucknow</span> <span class="country" tagx="country" title="country">India</span> </span> <span class="citation_author_institution" id="aff8">[], <sup>8</sup> <span class="institution" title="institution">Skaggs School of Pharmacy and Pharmaceutical Sciences, University of California, San Diego </span> <span class="addr-line" title="addr-line">San Diego, CA</span> <span class="country" tagx="country" title="country">United States</span> </span> <div class="author-notes" title="author-notes"> <div class="fn-type-edited-by" title="edited-by"> <p>Edited by: Yusuf Akhter, Babasaheb Bhimrao Ambedkar University, India</p> </div> <div class="fn-type-edited-by" title="edited-by"> <p>Reviewed by: Harinder Singh, J. Craig Venter Institute, United States; Hifzur Rahman Ansari, King Abdullah International Medical Research Center KAIMRC, Saudi Arabia; Aparoy Polamarasetty, Central University of Himachal Pradesh, India </p> </div> <div class="corresp" title="corresp">
*Correspondence: Feroz Khan, <span class="email" tagx="email" title="email">f.khan@cimap.res.in</span> </div> <div class="fn-type-other" title="other"> <p>This article was submitted to Bioinformatics and Computational Biology, a section of the journal Frontiers in Plant Science </p> </div> </div> <span class="pub-date-epub" title="pub-date-epub">epub: <span>2018-8-8</span> </span> <span class="pub-date-collection" title="pub-date-collection">collection: <span>2018</span> </span> <span class="volume" tagx="volume" title="volume">9</span> <span class="elocation-id" tagx="elocation-id" title="elocation-id">1081</span> <span class="history" title="history"> <span class="received" title="received">received: 2018-3-24</span> <span class="accepted" title="accepted">accepted: 2018-7-04</span> </span> <div class="permissions"> <span class="copyright" title="copyright">(C) <span class="copyright-holder" tagx="copyright-holder" title="copyright-holder">Kumar, Prakash, Tripathi, Tandon, Gupta, Rahman, Lal, Semwal, Darokar and Khan</span>, <span class="copyright-year" tagx="copyright-year" title="copyright-year">2018</span> </span> <span class="license" title="license"> <span class="license-p" title="license-p">This is an open-access article distributed under the terms of the Creative Commons Attribution License (CC BY). The use, distribution or reproduction in other forums is permitted, provided the original author(s) and the copyright owner(s) are credited and that the original publication in this journal is cited, in accordance with accepted academic practice. No use, distribution or reproduction is permitted which does not comply with these terms. </span> </span> </div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>In traditional, herbal medicine, and aromatherapy, use of essential oils and their aroma compounds have been known since long, for the management of various human diseases. The essential oil is a mixture of highly complex, naturally occurring volatile aroma compounds synthesized by medicinal and aromatic plants as secondary metabolites. Essential oils widely used in pharmaceutical, cosmetic, sanitary, food industry and agriculture for their antibacterial, antiviral, antifungal, antiparasitic, insecticidal, anticancer, neuroprotective, psychophysiological, and anti-aging activities. Moreover, volatile aroma compounds comprise a chemically diverse class of low molecular weight organic compounds with significant vapor pressure. However, aroma compounds produced by plants, mainly attract pollinators, seed dispersers and provide defense against pests or pathogens. However, in humans, about 300 active olfactory receptor genes are involved to detect thousands of different aroma compounds and modulates expression of different metabolic genes regulating human psychophysiological activity, brain function, pharmacological signaling, and therapeutic potential. Keeping in mind this importance, present database, namely, AromaDb ( <a href="http://bioinfo.cimap.res.in/aromadb/">http://bioinfo.cimap.res.in/aromadb/</a>) covers information of plant varieties/chemotypes, essential oils, chemical constituents, GC-MS profile, yield variations due to agro-morphological parameters, trade data, aroma compounds, fragrance type, and bioactivity details. The database includes 1,321 aroma chemical structures, bioactivities of essential oil/aroma compounds, 357 fragrance type, 166 commercially used plants, and their high yielding 148 varieties/chemotypes. Also includes calculated cheminformatics properties related to identification, physico-chemical properties, pharmacokinetics, toxicological, and ecological information. Also comprises interacted human genes affecting various diseases related cell signaling pathways correlating the use of aromatherapy. This database could be a useful resource to the plantâ€™s growers/producers, an aroma/fragrance industrialist, health professionals, and researchers exploring the potential of essential oils and aroma compounds in the development of novel formulations against human diseases. </p> </div> <div class="kwd-group"> <div class="kwd" title="kwd">
medicinal</div> <div class="kwd" title="kwd">
aromatic</div> <div class="kwd" title="kwd">
plants</div> <div class="kwd" title="kwd">
essential oils</div> <div class="kwd" title="kwd">
chemotypes</div> <div class="kwd" title="kwd">
aroma</div> <div class="kwd" title="kwd">
molecule</div> <div class="kwd" title="kwd">
bioactivity</div> </div> <div class="funding-group" title="funding-group"> <div class="funding" title="funding">
Funding</div> <div class="award-group" title="award-group"> <div class="funding-source" title="funding-source">
Central Institute of Medicinal and Aromatic Plants <span class="named-content" title="named-content">10.13039/501100007854</span> </div> <span class="award-id" tagx="award-id" title="award-id">CSIR Aroma Mission (HCP0007)</span> <span class="award-id" tagx="award-id" title="award-id">CSIR GENESIS (BSC0121)</span> <span class="award-id" tagx="award-id" title="award-id">CSIR CHEMBIO (BSC0203)</span> <span class="award-id" tagx="award-id" title="award-id">CSIR CIMAP OLP07</span> </div> </div> <div class="counts" title="counts"> <span class="fig-count" title="fig-count">fig-count: </span> <span class="table-count" title="table-count">table-count: </span> <span class="equation-count" title="equation-count">equation-count: </span> <span class="ref-count" title="ref-count">ref-count: </span> <span class="page-count" title="page-count">page-count: </span> <span class="word-count" title="word-count">word-count: </span> </div> </div> </div> <div class="body" title="body"> <div class="introduction" title="sec"> <div class="title" tagx="title" title="title">
Introduction</div> <p>Both flowers and leaves of plants emit aroma compound that moved through the air and is detected by the olfactory system of animals. The complexity of aroma is still challenging because smell of any flower or plant is not due to a single chemical compound. Although plants and flowers consist of many chemical compounds, all of them do not contribute in the aroma. As the rose scent majorly influenced by major constituent compound (â€“) - <i>cis</i>-rose oxide and minor constituent compounds, namely, beta-damascenone (family â€“ rose ketones) and beta-ionone of the plantâ€™s essential oil, while other compounds like geraniol, nerol, (â€“) -citronellol, farnesol, and linalool contributions as minor. </p> <p>Aroma is a mixture of volatile compounds with a molecular weight &amp;lt;300 and high vapor pressure, but the complete group of volatile compounds comprises thousands of inorganic and organic compounds stemming from major pathways of secondary metabolism ( <a href="#B4">Frauendorfer and Schieberle, 2006</a>). There are three significant pathways which involved in the biosynthesis process of the main aroma components in plants, such as the shikimic acid pathway by which eugenol (cloves) biosynthesized, degradation of lipids for the formation of short-chain alcohols and aldehydes and terpenoid pathway by which geraniol (Rose) and menthol (Peppermint) are synthesized. These aroma molecules act as semiochemical (mixture or chemical that carries a message for the purpose of communication), pheromones, defense mechanism, allow animals to recognize and detect individuals. The pheromone aroma molecules are essential for mating choices, sexual behavior, fertilization, and nursing and to warn kin in situations of danger (alarm pheromones) and to defend against predators. Aroma molecules are also involved in communication and interaction like plantâ€“plant interaction ( <a href="#B2">Das et al., 2013</a>) and plantâ€“animal interactions ( <a href="#B5">Herrera and Pellmyr, 2002</a>). These interactions and communications were performed through pollination ( <a href="#B9">Schaefer et al., 2004</a>), while another property of aroma molecule is a plantâ€™s defense response against herbivores ( <a href="#B6">Glaum and Kessler, 2017</a>). Bacteria also emit a wealth of aroma molecules with an influence on plants, fungi, animals, and bacteria ( <a href="#B10">Vet and Dicke, 1992</a>; <a href="#B7">Heil and Silva Bueno, 2007</a>). Aroma compounds can also be found in food, spices, perfumes, wine, fragrance oils, and essential oils. These molecules form biochemically during ripening of fruits and other crops. Aroma fragrance molecules are of great commercial interest, resulting in many applications of volatile aroma molecules in research, health, food, cosmetic, and health industries. </p> <p>A comprehensive information of plants based aroma molecules (2D and 3D structures), aroma or fragrance type, essential oils, respective commercial plants varieties available for cultivation in India and abroad, therapeutic potential in terms of biological activities, physicochemical, stereo, and pharmacokinetics (ADMET) properties of aroma molecules, interacted human genes and corresponding essential oilâ€™s export/import trade data trend information are not publicly available for both scientific research and industrial use. It was, therefore, our goal to develop a database AromaDb which cover basic and advance information of aroma molecules and provide a platform for comparative analysis and quantitative structure aroma relationship studies (QSAR). </p> <p>The AromaDb database would be helpful to answers the queries of researchers, industries, and growers related to aroma compounds and essential oils. However, a variety of database resources of, essential oils, scent, fragrance and flavor (synthetic) components are already reported but still limited at some extent focused on certain subgroups, for example, SuperScent ( <a href="#B3">Dunkel et al., 2009</a>), OdorDB, Pherobase, EssOilDB ( <a href="#B8">Kumari et al., 2014</a>); ScentBase <sup> <a href="#fn01">1</a> </sup>, AroChemBase <sup> <a href="#fn02">2</a> </sup>, and Flavornet <sup> <a href="#fn03">3</a> </sup> ( <b>Table <a href="#T1">1</a> </b>). Consequently, these databases are good but useful for special purposes, and therefor there is a need for a comprehensive listing of commercially important volatile aroma molecules found in plantâ€™s essential oils from different local and globally grown aromatic plants with information of chemotype-specific varieties, type of fragrance (aroma), physicochemical properties of aroma molecule, chemical identification, chemical structures (2D &amp;amp; 3D) for free downloads, Pharmacokinetic properties (ADMET), effect of aroma on human genes, their vapor pressure and logP," and export and import trade data trends in terms of global demand and business turnover in India and abroad, so that to guide the future research directions for researchers, growers, farmers, and industries. </p> <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 1</span>  <p>Summary of existing databases related to aroma, odor, scent, essential oil, flavor, toxic compounds, and volatile molecules with its website addresses and descriptions. </p>  <table> <thead> <tr> <th>S.No.</th> <th>Database</th> <th>Description</th> <th>Website address</th> </tr> </thead> <tbody> <tr> <td>1.</td> <td>SuperScent</td> <td>A database focusses on scent molecules (including synthetic compounds) related to flavors and scents. </td> <td> <a href="http://bioinf-applied.charite.de/superscent/">http://bioinf-applied.charite.de/superscent/</a> </td> </tr> <tr> <td>2.</td> <td>OdorDB</td> <td>A database focusses on a database of odor molecules related to the olfactory receptor database. </td> <td> <a href="https://senselab.med.yale.edu/OdorDB/">https://senselab.med.yale.edu/OdorDB/</a> </td> </tr> <tr> <td>3.</td> <td>Pherobase</td> <td>A database focusses on pheromones.</td> <td> <a href="http://www.pherobase.com/">http://www.pherobase.com/</a> </td> </tr> <tr> <td>4.</td> <td>EssOilDB</td> <td>A database focusses on plantâ€™s essential oils reflecting terpene composition, GC/MS data of essential oil, and variability in the plant kingdom. </td> <td> <a href="http://nipgr.res.in/Essoildb/">http://nipgr.res.in/Essoildb/</a> </td> </tr> <tr> <td>5.</td> <td>ScentBase</td> <td>A database focusses on a compilation of floral scent components.</td> <td> <a href="http://www2.dpes.gu.se/SCENTbase.html">http://www2.dpes.gu.se/SCENTbase.html</a> </td> </tr> <tr> <td>6.</td> <td>AroChemBase</td> <td>A database focusses on compounds for aroma and chemical analysis using gas chromatography and Kovats index calculations. </td> <td> <a href="http://www.alpha-mos.com/">http://www.alpha-mos.com/</a> </td> </tr> <tr> <td>7.</td> <td>Flavornet</td> <td>A database of volatile compounds found in the human olfactory perception space.</td> <td> <a href="http://www.flavornet.org/flavornet.html">http://www.flavornet.org/flavornet.html</a> </td> </tr> <tr> <td>8.</td> <td>SuperToxic</td> <td>A comprehensive database of toxic compounds.</td> <td> <a href="http://bioinformatics.charite.de/supertoxic">http://bioinformatics.charite.de/supertoxic</a> </td> </tr> <tr> <td>9.</td> <td>mVOC</td> <td>A database of microbial volatiles.</td> <td> <a href="http://bioinformatics.charite.de/mvoc">http://bioinformatics.charite.de/mvoc</a> </td> </tr> <tr> <td>10.</td> <td>KNApSAcK Metabolite Ecology</td> <td>A database covering information on the relationships between volatile organic compounds (VOCs) and their emitting organisms. </td> <td> <a href="http://kanaya.naist.jp/MetaboliteEcology/top.jsp">http://kanaya.naist.jp/MetaboliteEcology/top.jsp</a> </td> </tr> </tbody> </table> </div> </div> <div class="materials_methods" title="materials|methods"> <div class="title" tagx="title" title="title">
Materials and Methods</div> <div class="dataselectionandresources" title="sec"> <div class="title" tagx="title" title="title">
Data Selection and Resources</div> <p>Data were retrieved from the national and international literature and various web database resources or papers published in national and international journals. The database includes records from more than 100 scientific journals related to aroma (fragrance) and flavor. Following web resources were used to select the required data, such as selection of floral scents/fragrance based on medicinal and aromatic plants volatile ( <a href="#B1">Baldwin et al., 2006</a>) ( <b>Table <a href="#T2">2</a> </b>). The abstracts were screened searched against chemical names and synonyms of chemical compounds on given literature search engines. The information regarding adverse effect or allergic responses (skin irritation toxicity) by some aroma molecules is covered in the AromaDb database under fields "physical and ADMET properties" which was calculated through TOPKAT module of Discovery Studio v3.5 software (Accelrys, San Diego, CA, United States). Compounds human genes interaction data was retrieved from the EPA (United States Environmental Protection Agency) <sup> <a href="#fn04">4</a> </sup> and ACToR <sup> <a href="#fn05">5</a> </sup> information portal by searching all compounds using CAS ID. </p> <div class="table-wrap_UNKNOWN"> <span class="label" tagx="label" title="label">Table 2</span>  <p>Details of supporting databases and web resources used in AromaDb database development.</p>  <table> <thead> <tr> <th>S.No.</th> <th>Database description</th> <th>Website address</th> </tr> </thead> <tbody> <tr> <td>1.</td> <td>Natural products from Record of Natural Products (RNP)</td> <td> <a href="http://www.acgpubs.org/RNP/">http://www.acgpubs.org/RNP/</a> </td> </tr> <tr> <td>2.</td> <td>Human genes interaction with essential oil compounds through Human Gene Database (Gene cards) </td> <td> <a href="http://www.genecards.org/">http://www.genecards.org/</a> </td> </tr> <tr> <td>3.</td> <td>Clinical trials data from ClinicalTrials.gov</td> <td> <a href="https://clinicaltrials.gov/ct2/home">https://clinicaltrials.gov/ct2/home</a> </td> </tr> <tr> <td>4.</td> <td>Aromatic plants molecules fragrance information retrieved from Food and Agriculture Organization of the United Nations (FAO-UN) </td> <td> <a href="http://www.fao.org/home/en/">http://www.fao.org/home/en/</a> </td> </tr> <tr> <td>5.</td> <td>Trade data of commercially important aromatic and medicinal crops from United Nations Commodity Trade Statistics Database (UN Comtrade) </td> <td> <a href="http://comtrade.un.org/db/">http://comtrade.un.org/db/</a> </td> </tr> <tr> <td>6.</td> <td>National trade data of plants essential oils through Department of Commerce, Ministry of Commerce &amp;amp; Industry (Govt. of India), India </td> <td> <a href="http://commerce.gov.in/">http://commerce.gov.in/</a> </td> </tr> <tr> <td>7.</td> <td>PubChem compounds structures (2D and 3D)</td> <td> <a href="http://pubchem.ncbi.nlm.nih.gov/">http://pubchem.ncbi.nlm.nih.gov/</a> </td> </tr> <tr> <td>8.</td> <td>PubMed literature search database</td> <td> <a href="http://www.ncbi.nlm.nih.gov/pubmed">www.ncbi.nlm.nih.gov/pubmed</a> </td> </tr> <tr> <td>9.</td> <td>Google Scholar literature search database</td> <td> <a href="https://scholar.google.co.in/">https://scholar.google.co.in/</a> </td> </tr> <tr> <td>10.</td> <td>TOPKAT module of Discovery Studio v3.5 molecular modeling software (Accelrys, San Diego, CA, United States) </td> <td> <a href="http://accelrys.com/">http://accelrys.com/</a> </td> </tr> <tr> <td>11.</td> <td>CSIR-CIMAP Annual Reports (2009â€“2015), India</td> <td> <a href="http://www.cimap.res.in/english/index.php/annual-report">http://www.cimap.res.in/english/index.php/annual-report</a> </td> </tr> <tr> <td>12.</td> <td>Improved Varieties â€“ Medicinal and Aromatic Plants and JMAPs data, CSIR-CIMAP, India</td> <td> <a href="http://intranet.cimap.res.in/cimvariety/">http://intranet.cimap.res.in/cimvariety/</a> </td> </tr> </tbody> </table> </div> </div> <div class="databasedevelopmentbackendinformation" title="sec"> <div class="title" tagx="title" title="title">
Database Development Backend Information</div> <p>AromaDb database is developed on Apache HTTP server, which is platform independent and available as open-source software. The database is developed on MySQL for storing the information in the backend. The database website front end is developed in PHP, HTML, CSS, and JavaScript. AromaDb comprises basic and advances molecular information retrieved from different resources as shown in <b>Figures <a href="#F1">1A,B</a> </b>. </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">FIGURE 1</span> </a> </div>   <p>Details of the AromaDb database <b>(A)</b> architecture, and <b>(B)</b> entity-relationship diagram. </p>   </div> </div> <div class="structuresimilaritysearchtoolapplicationandstructureeditor" title="sec"> <div class="title" tagx="title" title="title">
Structure Similarity Search Tool Application and Structure Editor</div> <p>Structure editor and structure similarity search tools in AromaDb for structural similarity within the database and from the extended database, i.e., ChemAxon SMILES based search option was used, for displaying 3D structures, JSmol was used which is available as open-source JavaScript viewer for chemical structures in 3D <sup> <a href="#fn06">6</a> </sup>. JSmol is the extension of the Java-based molecular visualization applet Jmol <sup> <a href="#fn07">7</a> </sup> as an HTML5 JavaScript-only web app. Similarly, JSME (JavaScript Molecular Editor) was applied for the built-in molecule editor, which allows the user to screen with self-edited molecules. JSME is a free molecule editor written in JavaScript <sup> <a href="#fn08">8</a> </sup>. </p> </div> </div> <div class="results" title="sec"> <div class="title" tagx="title" title="title">
Results</div> <div class="databasecontent" title="sec"> <div class="title" tagx="title" title="title">
Database Content</div> <p>The major parameters in the database were plants, variety, essential oils, chemical molecules, chemical group, trade data, structural data, and biological activity of essential oil, compounds and their interacting genes, etc. For better performance, all the data is kept distributed in several interrelated tables logically. For wide complex searching search engine was used to check each link of the database website and accordingly showed the matched results ( <b>Figure <a href="#F1">1B</a> </b>). All the information in AromaDb database can be categorized into three categories: primary, secondary, and tertiary information. The database information was manually curated and selected from various sources such as published international literature, CSIR-CIMAP, Lucknow <sup> <a href="#fn09">9</a> </sup> essential oils monograph, web link annual reports, journal (JMAPS; <i>Journal of Medicinal and Aromatic Plant Sciences</i>; <a href="#B1">Baldwin et al., 2006</a>), newsletters, and books. The primary information has been retrieved from the literature, these informationâ€™s consists of the following major fields namely: (i) plant details (ii) essential oil name, and (iii) plant variety. </p> <p>The secondary information, which was derived from the plant essential oil (primary information) includes the essential oil description details, chemical constituents, major minor compound details, content percentage, and exportâ€“import essential oil trade data. The tertiary information is further derived from secondary information which comprises detailed information about the essential oil compounds as followed: IUPAC name, Chemical class biochemical classes (e.g., terpene hydrocarbons and oxygenated compounds: phenols, alcohols, aldehydes, ketones, esters, lactones, coumarins, ethers, and oxides), Fragrance type, Physical and chemical properties, Absorption and Metabolism information, Toxicological information, Ecological information, Hazards information, and compound bioactivity data for therapeutic use or drug formulation development. This information provides the user a comprehensive aroma molecules database, together with substantial options, such as chemical compounds search based on structural similarity using compound Canonical SMILES within databases. <b>Figure <a href="#F2">2</a> </b> summarizes snapshots of database home page ( <b>Figure <a href="#F2">2A</a> </b>), plant and varieties details ( <b>Figure <a href="#F2">2B</a> </b>), and essential oil details ( <b>Figure <a href="#F2">2C</a> </b>). </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">FIGURE 2</span> </a> </div>   <p>Snapshots showing details of the AromaDbdatabase <b>(A)</b> homepage, <b>(B)</b> Indian plant varieties, and <b>(C)</b> essential oil constituents. </p>   </div> </div> <div class="resultsofstructureeditorandstructuresimilaritysearchtools" title="sec"> <div class="title" tagx="title" title="title">
Results of Structure Editor and Structure Similarity Search Tools</div> <p>The AromaDb database provides the diverse and commercially important aroma molecules, fragrance types, essential oils, aromatic plants, trade data, and other industrially important information as required while formulating any herbal product formulation for human uses. We have incorporated various tools for making the database easy and more convenient for users. The database contents could be accessible with different search tools, options, e.g., simple and advance search tools. The simple search tool is powered by Google search engine, which search data from within the AromaDb database and also from public databases, where another mining tool search the query within the database, the Advance search tool option further divided into two types: (i) search based on structural similarity (within AromaDb database) and (ii) search based on structural physicochemical and pharmacokinetics (ADMET) properties. The database search fields enable the user to look for compounds using physicochemical properties, plants, varieties, chemotype, essential oil, aroma molecules, chemical classification, biological activity, etc. Database allows the user to choose certain functional groups, species or range of molecular weights, which search whole entries and retrieve the user required results. To mine the database user require a 2D molecular structure or structure canonical SMILES code (Simplified Molecular Input Line Entry System) or a MOL-file of interested aroma compounds. Structure drawing/editing option is provided in the AromaDb database with the help of JSME (JavaScript Molecular Editor; JSME Homepage <sup> <a href="#fn010">10</a> </sup>). With the help of JSME user can draw either full structure or part of it. The most similar database entries are listed in the order of structural similarity. For each compound, details of plant, plant name, variety, essential oil constituents, chemical class, ADMET properties, 2D/3D chemical structures, trade data, and the SMILES based similarity search percentage are presented. Furthermore, a similarity search to find the most similar compounds analogs are provided. Additionally, similar information can be viewed separately by accessing database header fields or the menu. </p> </div> <div class="browsinginformationinaromadb" title="sec"> <div class="title" tagx="title" title="title">
Browsing Information in AromaDb</div> <p>The user can find IUPAC name, synonyms or common names, CAS registry number, chemical classification, functional groups, molecular weight, pharmacokinetics data, and commercially cultivated aromatic plants and their varieties in which the aroma compounds have been found, together with the authentic references and bibliography, as well as in-house published contributed data (CSIR-CIMAP, Lucknow, India <sup> <a href="#fn011">11</a> </sup>). Furthermore, the aroma compounds were classified according to their structures, chemical features, quality of fragrance (aroma type), essential oil, yield, major and minor constituents, (Gas Chromatography-Mass Spectrometry) GCâ€“MS spectra, last 18 years trade data (1996â€“2014) of the most commercially used 34 aromatic plants in India and other industrial research purpose information. <b>Figure <a href="#F3">3</a> </b> represents the snapshots of chemical constituents of essential oil, GCâ€“MS analysis data and months wise variations in oil yield ( <b>Figure <a href="#F3">3A</a> </b>), essential oil trade data and comparative graphical plots ( <b>Figure <a href="#F3">3B</a> </b>), and aroma molecule description, a 3D structure for downloads ( <b>Figure <a href="#F3">3C</a> </b>). </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">FIGURE 3</span> </a> </div>   <p>Snapshots showing <b>(A)</b> constituents, GCâ€“MS data and monthâ€™s wise variations in oil yield, <b>(B)</b> essential oil trade data and comparative graphical plots, and <b>(C)</b> aroma compound description and 3D structures for downloads. </p>   </div> </div> <div class="informationaboutputativetherapeuticfreehumantargetsorgenesinaromadb" title="sec"> <div class="title" tagx="title" title="title">
Information About Putative Therapeutic Free Human Targets or Genes in AromaDb</div> <p>The other useful feature of the AromaDb database is the information of aroma molecules and related interacting human therapeutic genes or proteins involved to modulate the biological system biological pathways and for unraveling the underlying mechanism of action and therapeutic potential of these aroma molecules along with supporting references. Also enlisted the therapeutic important aroma molecules along with the option to see the related aromatic plants, essential oils, export and import trade data, and other information. </p> </div> <div class="displayofessentialoiltradedatainformation" title="sec"> <div class="title" tagx="title" title="title">
Display of Essential Oil Trade Data Information</div> <p>The graphical analysis of trade data suggests the consumption and demand data trend in India and the world or own country (India) as a guideline or forecasting tool to the aroma or essential oil industries and growers (farmers or producers) to streamline their resources, economy, and time according to expected demand of respective essential oils in the world and therefore indirectly helps to improve the socio-economical condition of farmers and producers. Here, the database entries have been clustered according to the quality of their aroma properties. A manually verified upload option through email allows the scientific community to contribute to the database. Here, the user can import a MOL-file together with corresponding information of the compound. The AromaDb database will be updated on a quarterly basis. </p> </div> <div class="comparativedisplayofphysicalpropertiesandsafetypredictiondata" title="sec"> <div class="title" tagx="title" title="title">
Comparative Display of Physical Properties and Safety Prediction Data</div> <p>The database features the properties related to aroma molecule, hazard identification, exposure controls and personal protection, physical and chemical properties, toxicological and ecological information. Compound identification properties include chemical name, IUPAC name and chemical class. Hazards identification includes properties related to physical hazards, health hazards such as skin corrosion and ocular irritancy property. Toxicological properties such as irritation, absorption level, aqueous solubility level, LogP (octanol/water partition coefficient), polar surface area (PSA), and bloodâ€“brain barrier level indicate the precautionary properties related to exposure controls and personal protection (eye/face protection, skin protection, respiratory protection, and thermal hazards). Physical and chemical properties indicated by molecular weight, molecular formula, fragrance type, LogP ( <i>n</i>-octanol/water), H-bond donor and H-bond acceptor, rotatable bond, topological polar surface area (TPSA), IUPAC name, InChIKey, aqueous solubility level, vapor pressure, PubChem (NCBI, United States) ( <a href="#B11">Wang et al., 2009</a>) database ID and SMILES information. Toxicological information includes properties related to information on likely routes of exposure such as inhalation, skin contact, eye contact, ingestion, symptoms related to the physical, chemical and toxicological characteristics, information on toxicological effects, e.g., acute toxicity (LD <sub>50</sub> â€“ dermal/oral, rabbit/rat, mg/kg), skin corrosion/irritation, serious eye damage/eye irritation, respiratory or skin sensitization, germ cell mutagenicity (genotoxic), carcinogenicity, reproductive toxicity, specific target organ toxicity (single exposure/repeated exposure), and aspiration hazard. However, the ecological information includes properties related to eco-toxicity (environmentally hazardous), persistence, and degradability. Since these aroma molecules used in aroma based natural therapy (aromatherapy) in traditional herbal medicine system of India (Ayurveda and Unani) and other East Asian countries, therefore, database covered properties related to pharmacokinetics such as absorption, distribution, metabolism, excretion, and toxicity (ADMET). These properties indicate drug-likeness properties, ADME compliance, and toxicity risk assessment data. Moreover, the database also includes information related to the molecular interaction of aroma molecules/essential oils with human proteins (genes) directly or indirectly affecting the metabolic processes and therefore causing their useful biological activity or responses. This information is supported with cross-references or proper evidence based on reported publications and therefore offer a possibility for biological interpretation of these aroma molecules. Moreover, an investigation of medical effects is also possible by browsing the "biological activity" field for each essential oils/aroma molecules. The comparative trends or pattern plots shows the emitted aroma compounds of the chosen species compared with all other plants species, which emit these compounds. The comparative data analysis through graph plots shows the unique pattern of aroma molecules (fingerprints), essential oil, yield, major constituent percentage (chemotype), trade analysis trend, and others based on changes in different agro-morphological parameters, e.g., soil type, stress conditions, temperature, weather type, months wise oil yield, etc., which are important feature for distinguishing between more or less useful features/parameters during agriculture practices and/or cultivation by farmers, or industrial growers. As an example, a snapshot represents these calculated properties of aroma showing molecule, chemical identification, description and 3D structure for download snapshot represented in the given figure ( <b>Figure <a href="#F3">3C</a> </b>). </p> </div> </div> <div class="discussion" title="sec"> <div class="title" tagx="title" title="title">
Discussion</div> <p>It was difficult to explore such information about plants, its variety, essential oil constituents, and essential oil trade data, on a single platform in addition to the aroma molecules information about its physicochemical properties, absorption and metabolism information, toxicological information, ecological information, Hazards identification, and compound biological activity with its fragrance and interacting human genes information. </p> <div class="browsingandsearchingdatabasecontents" title="sec"> <div class="title" tagx="title" title="title">
Browsing and Searching Database Contents</div> <p>Database contents can be seen in two ways: (i) browsing different data fields available on the header menu on the home page such as, plants, essential oils, aroma molecules, chemical classes of aroma molecules, biological activity, and trade data, and (ii) searching database contents through advanced search option and wild search through Googleâ€™s custom search option. Users can search any text within the database based on text similarity concept through Googleâ€™s custom search option (wild search or complex search) available in the header of the database. A similar analysis with the help of representing snapshots of the database showing database home page search parameters ( <b>Figure <a href="#F2">2A</a> </b>), plant and varieties details ( <b>Figure <a href="#F2">2B</a> </b>), and essential oil details ( <b>Figure <a href="#F2">2C</a> </b>). </p> </div> <div class="propertiessearchbyenteringvalues" title="sec"> <div class="title" tagx="title" title="title">
Properties Search by Entering Values</div> <p>Besides, users can also limit searches using ranges of physiochemical data, e.g., maximum and minimum values of TPSA (Topological Polar Surface Area) ( <b>Figures <a href="#F3">3A</a>â€“ <a href="#F3">C</a> </b>). Database snapshots of these properties and parameters showing chemical constituents of essential oil, GCâ€“MS data and months wise variations in oil yield ( <b>Figure <a href="#F3">3A</a> </b>), essential oil trade data and comparative graphical plots ( <b>Figure <a href="#F3">3B</a> </b>), and aroma molecule description and 3D structure for downloads ( <b>Figure <a href="#F3">3C</a> </b>). </p> </div> <div class="similarstructuressearchanddownloadingofaromamolecules" title="sec"> <div class="title" tagx="title" title="title">
Similar Structures Search and Downloading of Aroma Molecules</div> <p>Users can search and download interested aroma molecules (2D and 3D chemical structures) through structure search option. In this option, users can draw or edit any structure of their interest and convert it to SMILES and MOL file format and subsequently search whole AromaDb database aroma molecules and resulted most similar matched structures available in the database and can easily see details of these matched small molecules or free downloads the structures. For example, if the user draws the phenyl ring in the JME editor and enter the option either "get SMILES" or get MOL fileâ€™ resulted in SMILES or MOL file data would be shown on the other side of JME editor and subsequently enter the key similarity search within database, results of matched structures showed in the new web page, with no exact match aroma molecule found, and enlisted the similar matched compounds, e.g., ( <i>E</i>)-cinnamyl acetate, 1,1-diisobutoxy-2-phenylethane, 1,1-dimethoxy-2-benzylideneheptane, etc. All these matched structures have phenyl ring in common. Moreover, the user can see further details of each matched compound by entering key on "details" button. Details of aroma compound include calculated properties related to aroma molecule identification, hazards, physical and chemical properties, pharmacokinetic properties, toxicological information, and ecological information. The database also represents the 2D structure, 3D structure visualization in 3D structure viewer window with spin-on/off option so that to see the structural conformations quickly with free download option. The compounds search outputs are oriented to explore the matched compounds by following the "details" hyperlink to the table of physical, ADMET and safety properties described earlier. <b>Figure <a href="#F4">4</a> </b> showing snapshots of advanced search options based on different properties, fragrance type and toxicological parameters of aroma molecule ( <b>Figure <a href="#F4">4A</a> </b>), drawing tool for structure-based search option as well as SMILES or MOL file basis ( <b>Figure <a href="#F4">4B</a> </b>), and results of structure-based search based on SMILES basis structural similarity (exact match and similar hits; <b>Figure <a href="#F4">4C</a> </b>). </p> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">FIGURE 4</span> </a> </div>   <p>AromaDb database snapshots showing <b>(A)</b> advance search options based on different properties, fragrance type and toxicological parameters of aroma molecule, <b>(B)</b> drawing tool for structure-based search, and SMILES or MOL file basis search options, and <b>(C)</b> results of structure-based search based on SMILES similarity. </p>   </div> </div> <div class="searchbyselectingdatafields" title="sec"> <div class="title" tagx="title" title="title">
Search by Selecting Data Fields</div> <p>Similarly, naive users can directly see and select the interesting data through search menu buttons provided at the end of the home page of aroma database and the further user may browse more information without wasting time on thinking about aroma (fragrance), aroma molecules name, essential oils, plant varieties, and plants. This type of search display option will help the fresh researchers, scholars, and students. For example, a new user can directly browse the home page header menu field "Plants," which will display the list of all aromatic plants with their common and scientific names and a number of their available plant varieties or highly yielding chemotypes. Users can see more details of these plants by clicking on the button "View Detail." For example, if the user wishes to search for Menthol mint plant ( <i>Mentha arvensis</i>), which included at present 11 mint plant varieties or chemotypes based on different aroma molecules constituentâ€™s ratio variations. Users can see more details such as brief introduction, family, localization, uses, essential oils types, and the name of 11 mint varieties used by Indian growers or farmers. Users have the option to print or save this data. In this page, users have two options to see further details; (i) essential oil types and (ii) variety details. For example, if the user searches for <i>M. arvensis</i> MAS-1 essential oil details, database will display brief introduction, major constituent Menthol with 84% with multiple minor components such as Menthone (5.8%), etc., comparative graph plot showing percentage ratio of different chemical constituents of <i>M. arvensis</i> MAS-1 essential oil ( <b>Figure <a href="#F2">2C</a> </b>), and the Indian scientists or researchers contribution, if any, showed in the references. Users have the option to print (or save) this data and graph plot. Also, the user can directly move to see details of major or minor compounds and yield percentage by simply enter on the name of compounds. </p> <p>Moreover, if the user enters on <i>M. arvensis</i> variety Gomti button, the database will display a tabular text data showing brief details of plant, variety name, major constituents of given variety such as Menthol 74% in ratio, menthone 12.6%, isomenthone 3.7%, and methyl acetate 2.9%. Beside this database show details of essential oil, GCâ€“MS graph (if available) of essential oil, major compounds peak in GC data, compound property, year of plant variety release and complete reference ( <b>Figures <a href="#F3">3A</a>â€“ <a href="#F3">C</a> </b>). </p> </div> <div class="agronomicparametersbasedgaphicaldataanalysis" title="sec"> <div class="title" tagx="title" title="title">
Agronomic Parameters Based Gaphical Data Analysis</div> <p>At the same time, the database also covers data related to different agronomic parameters based and provide an option to the user to see either in tabular or graphical forms. For example, in the case of <i>M. arvensis</i> Gomti variety, the database covers five observation readings revealing variations in Z-Linalool oxide compound percentage ranges from 2.2 to 2.7% from January to May 2015. This data suggest that the highest yield of Z-Linalool oxide was 2.7 February. Likewise, the user can see other parameter based graph plot analysis and get benefited from future cultivation, for example, if the user sees the Himalaya variety of <i>M. arvensis</i>, there are five readings based on the age of the plant (in days) and menthol compound yield percentage. This data showed that the study was performed for 30 days old plants to 150 days old plant (menthol mint) and menthol yield ranges from 70.66 to 82.18%. Graphical analysis revealed that highest yield obtained at 120 days old plant ( <b>Figure <a href="#F2">2</a> </b>). A similar analysis with the help of representing snapshots of the database showing database home page search parameters ( <b>Figure <a href="#F2">2A</a> </b>), plant and varieties details ( <b>Figure <a href="#F2">2B</a> </b>), and essential oil details ( <b>Figure <a href="#F2">2C</a> </b>). </p> </div> </div> <div class="conclusion" title="sec"> <div class="title" tagx="title" title="title">
Conclusion</div> <p>The AromaDb database is a useful tool to retrieve information about aroma molecules, aroma or fragrance types, essential oils, plants varieties, bioactivity of essential oils or aroma molecules, toxicological and ecological data, and trade data. The database provides, 3D structures of aroma compounds for free downloads and option to see the essential oil yields or constituents percentage variation trends at different agro-morphological conditions during plant growth. The included data on aroma molecules along with a focus on associated plants and their essential oils chemotype (varieties) will enable systematic experimental approaches on the relation between structural similarities, essential oils, and aroma (fragrance) type and aroma chemical classes. Besides, last 18 years global export and import trade data of plants essential oils will educate the growers or farmers to prioritize the cultivation of aromatic plants based on expected global demand. Furthermore, structure comparisons of self-edited molecules with the database aroma molecules as well as the external database may allow a first rough estimation of the potential aroma of new chemicals. The AromaDb database is a free resource with embedded screening functions for aroma molecule based on molecular weight, plants, varieties, essential oils, fragrance or aroma type, toxicological and ecological information (allergic or toxic responses). </p> </div> <div class="availabilityandimplementation" title="sec"> <div class="title" tagx="title" title="title">
Availability and Implementation</div> <p>The database is available at URL: <a href="http://bioinfo.cimap.res.in/aromadb/">http://bioinfo.cimap.res.in/aromadb/</a>. </p> </div> <div class="authorcontributions" title="sec"> <div class="title" tagx="title" title="title">
Author Contributions</div> <p>YK carried out the database entry, data searching and retrieval, download, designed 2D and 3D structures, participated in the data analysis, and manuscript writing. OP developed the preliminary database offline using PHP and MySQL for initial data entry. HT add the trade data of essential oil based plants commodities and economic profile in Indian currency. ST provided 25 aroma molecules to the database and other properties. MG also provided small molecules and corrected chemical classes of aroma molecules and corrected the chemistry part. L-UR provided data related to Indian plants varieties, essential oils details, constituents, and contributed in the design of the database. RL guided to add published in-house data related to aroma molecules, essential oils, aroma plants, and bioactivity. MS coordinated in database web hosting and IT support. FK conceived the study, contributed to its design, ER relationship, and drafted the manuscript. MD Addition of updated bioactivity ( <i>in vitro</i>) data for essential oil/aroma molecules with their cros references. All authors read and approved the final manuscript. </p> </div> <div class="conflictofintereststatement" title="sec"> <div class="title" tagx="title" title="title">
Conflict of Interest Statement</div> <p>The authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest. </p> </div> </div> <div class="back" title="back"> <div class="fn-group" title="fn-group"> <div class="fn-type-financial-disclosure" title="financial-disclosure"> <p> <b>Funding.</b> We acknowledge the Council of Scientific &amp;amp; Industrial Research (CSIR), New Delhi, India for financial support through CHEMBIO (BSC0203), GENESIS (BSC0121), OLP-07, and CSIR Aroma Mission (HCP0007). </p> </div> </div> <div class="ack" title="ack"> <p>We are thankful to the Director, CSIR-CIMAP, Lucknow, India for rendering essential research facilities and support. YK, OP, and HT acknowledge the UGC, CSIR, and ICMR, New Delhi, India for the fellowship support through SRF (RGNF; UGC), PA-III (BSC-0121; GENESIS), and SRF (ICMR), respectively. We also acknowledge the CSIR, New Delhi, India for financial support to the Bioinformatics lab through networking project BSC-0121 (GENESIS). We also acknowledge the team ICT Division, CSIR-CIMAP, Lucknow for constant IT support. The CIMAP communication number is CIMAP/PUB/2015/13. We acknowledge the Bito Technologies Pvt. Ltd., Lucknow for database web interface development. </p> </div> <div class="fn-group" title="fn-group"> <div class="fn-type-" title=""> <span class="label" tagx="label" title="label">1</span> <p> <a href="https://scentbase.com/">https://scentbase.com/</a> </p> </div> <div class="fn-type-" title=""> <span class="label" tagx="label" title="label">2</span> <p> <a href="https://www.alpha-mos.com">www.alpha-mos.com</a> </p> </div> <div class="fn-type-" title=""> <span class="label" tagx="label" title="label">3</span> <p> <a href="http://www.flavornet.org/">http://www.flavornet.org/</a> </p> </div> <div class="fn-type-" title=""> <span class="label" tagx="label" title="label">4</span> <p> <a href="https://www.epa.gov/">https://www.epa.gov/</a> </p> </div> <div class="fn-type-" title=""> <span class="label" tagx="label" title="label">5</span> <p> <a href="https://actor.epa.gov/actor/home.xhtml">https://actor.epa.gov/actor/home.xhtml</a> </p> </div> <div class="fn-type-" title=""> <span class="label" tagx="label" title="label">6</span> <p> <a href="http://sourceforge.net/projects/jsmol/">http://sourceforge.net/projects/jsmol/</a> </p> </div> <div class="fn-type-" title=""> <span class="label" tagx="label" title="label">7</span> <p> <a href="http://jmol.sourceforge.net/">http://jmol.sourceforge.net/</a> </p> </div> <div class="fn-type-" title=""> <span class="label" tagx="label" title="label">8</span> <p> <a href="http://peter-ertl.com/jsme/">http://peter-ertl.com/jsme/</a> </p> </div> <div class="fn-type-" title=""> <span class="label" tagx="label" title="label">9</span> <p> <a href="http://www.cimap.res.in">www.cimap.res.in</a> </p> </div> <div class="fn-type-" title=""> <span class="label" tagx="label" title="label">10</span> <p> <a href="http://peter-ertl.com/jsme/">http://peter-ertl.com/jsme/</a> </p> </div> <div class="fn-type-" title=""> <span class="label" tagx="label" title="label">11</span> <p> <a href="http://www.cimap.res.in">www.cimap.res.in</a> </p> </div> </div> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"> <a name="B1" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Baldwin</span> <span class="given-names" tagx="given-names" title="given-names">I. T.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Halitschke</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Paschold</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Von Dahl</span> <span class="given-names" tagx="given-names" title="given-names">C. C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Preston</span> <span class="given-names" tagx="given-names" title="given-names">C. A.</span> </span> </span> ( <span class="year" tagx="year" title="year">2006</span>). <span class="mixed-article-title" title="mixed-article-title">Volatile signaling in plant-plant interactions: "Talking trees" in the genomics era.</span> <span class="source" tagx="source" title="source"> <i>Science</i> </span> <span class="volume" tagx="volume" title="volume">311</span> <span class="fpage" tagx="fpage" title="fpage">812</span>â€“ <span class="lpage" tagx="lpage" title="lpage">815</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1126/science.1118446">10.1126/science.1118446</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16469918">16469918</a> </span> </span> </li> <li tag="ref"> <a name="B2" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Das</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lee</span> <span class="given-names" tagx="given-names" title="given-names">S. H.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hyun</span> <span class="given-names" tagx="given-names" title="given-names">T. K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kim</span> <span class="given-names" tagx="given-names" title="given-names">S. W.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kim</span> <span class="given-names" tagx="given-names" title="given-names">J. Y.</span> </span> </span> ( <span class="year" tagx="year" title="year">2013</span>). <span class="mixed-article-title" title="mixed-article-title">Plant volatiles as method of communication.</span> <span class="source" tagx="source" title="source"> <i>Plant Biotechnol. Rep.</i> </span> <span class="volume" tagx="volume" title="volume">7</span> <span class="fpage" tagx="fpage" title="fpage">9</span>â€“ <span class="lpage" tagx="lpage" title="lpage">26</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1007/s11816-012-0236-1">10.1007/s11816-012-0236-1</a> </span> </span> </li> <li tag="ref"> <a name="B3" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dunkel</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schmidt</span> <span class="given-names" tagx="given-names" title="given-names">U.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Struck</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Berger</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gruening</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hossbach</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> ( <span class="year" tagx="year" title="year">2009</span>). <span class="mixed-article-title" title="mixed-article-title">SuperScent â€“ A database of flavors and scents.</span> <span class="source" tagx="source" title="source"> <i>Nucleic Acids Res.</i> </span> <span class="volume" tagx="volume" title="volume">37</span> <span class="fpage" tagx="fpage" title="fpage">D291</span>â€“ <span class="lpage" tagx="lpage" title="lpage">D294</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gkn695">10.1093/nar/gkn695</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18931377">18931377</a> </span> </span> </li> <li tag="ref"> <a name="B4" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Frauendorfer</span> <span class="given-names" tagx="given-names" title="given-names">F.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schieberle</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> </span> ( <span class="year" tagx="year" title="year">2006</span>). <span class="mixed-article-title" title="mixed-article-title">Identification of the key aroma compounds in cocoa powder based on molecular sensory correlations. </span> <span class="source" tagx="source" title="source"> <i>J. Agric. Food. Chem.</i> </span> <span class="volume" tagx="volume" title="volume">54</span> <span class="fpage" tagx="fpage" title="fpage">5521</span>â€“ <span class="lpage" tagx="lpage" title="lpage">5529</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1021/jf060728k">10.1021/jf060728k</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16848541">16848541</a> </span> </span> </li> <li tag="ref"> <a name="B5" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Herrera</span> <span class="given-names" tagx="given-names" title="given-names">C. M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pellmyr</span> <span class="given-names" tagx="given-names" title="given-names">O.</span> </span> </span> ( <span class="year" tagx="year" title="year">2002</span>). <span class="source" tagx="source" title="source"> <i>Plant-Animal Interactions: An Evolutionary Approach</i>. </span> <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Hoboken</span>: <span class="publisher-name" tagx="publisher-name" title="publisher-name">Wiley</span>. </span> </li> <li tag="ref"> <a name="B6" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Glaum</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kessler</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> </span> ( <span class="year" tagx="year" title="year">2017</span>). <span class="mixed-article-title" title="mixed-article-title">Functional reduction in pollination through herbivore-induced pollinator limitation and its potential in mutualist communities. </span> <span class="source" tagx="source" title="source"> <i>Nat Commun.</i> </span> <span class="volume" tagx="volume" title="volume">8</span>: <span class="issue" tagx="issue" title="issue">2031</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1038/s41467-017-02072-4">10.1038/s41467-017-02072-4</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/29229901">29229901</a> </span> </span> </li> <li tag="ref"> <a name="B7" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Heil</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Silva Bueno</span> <span class="given-names" tagx="given-names" title="given-names">J. C.</span> </span> </span> ( <span class="year" tagx="year" title="year">2007</span>). <span class="mixed-article-title" title="mixed-article-title">Within-plant signaling by volatiles leads to induction and priming of an indirect plant defense in nature. </span> <span class="source" tagx="source" title="source"> <i>Proc. Natl. Acad. Sci. U.S.A.</i> </span> <span class="volume" tagx="volume" title="volume">104</span> <span class="fpage" tagx="fpage" title="fpage">5467</span>â€“ <span class="lpage" tagx="lpage" title="lpage">5472</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1073/pnas.0610266104">10.1073/pnas.0610266104</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/17360371">17360371</a> </span> </span> </li> <li tag="ref"> <a name="B8" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kumari</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Pundhir</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Priya</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jeena</span> <span class="given-names" tagx="given-names" title="given-names">G.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Punetha</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Chawla</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> ( <span class="year" tagx="year" title="year">2014</span>). <span class="mixed-article-title" title="mixed-article-title">EssOilDB: a database of essential oils reflecting terpene composition and variability in the plant kingdom. </span> <span class="source" tagx="source" title="source"> <i>Database</i> </span> <span class="volume" tagx="volume" title="volume">2014</span>: <span class="issue" tagx="issue" title="issue">bau120</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1093/database/bau120">10.1093/database/bau120</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25534749">25534749</a> </span> </span> </li> <li tag="ref"> <a name="B9" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schaefer</span> <span class="given-names" tagx="given-names" title="given-names">H. M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schaefer</span> <span class="given-names" tagx="given-names" title="given-names">V.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Levey</span> <span class="given-names" tagx="given-names" title="given-names">D. J.</span> </span> </span> ( <span class="year" tagx="year" title="year">2004</span>). <span class="mixed-article-title" title="mixed-article-title">How plant-animal interactions signal new insights in communication.</span> <span class="source" tagx="source" title="source"> <i>Trends Ecol. Evol.</i> </span> <span class="volume" tagx="volume" title="volume">19</span> <span class="fpage" tagx="fpage" title="fpage">577</span>â€“ <span class="lpage" tagx="lpage" title="lpage">584</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.tree.2004.08.003">10.1016/j.tree.2004.08.003</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18031220">18031220</a> </span> </span> </li> <li tag="ref"> <a name="B10" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Vet</span> <span class="given-names" tagx="given-names" title="given-names">L. E. M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Dicke</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> </span> ( <span class="year" tagx="year" title="year">1992</span>). <span class="mixed-article-title" title="mixed-article-title">Ecology of infochemical use by natural enemies in a tritrophic context.</span> <span class="source" tagx="source" title="source"> <i>Annu. Rev. Entomol.</i> </span> <span class="volume" tagx="volume" title="volume">37</span> <span class="fpage" tagx="fpage" title="fpage">141</span>â€“ <span class="lpage" tagx="lpage" title="lpage">172</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1146/annurev.en.37.010192.001041">10.1146/annurev.en.37.010192.001041</a> </span> </span> </li> <li tag="ref"> <a name="B11" /> <span class="mixed-citation" tagx="mixed-citation" title="mixed-citation"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Xiao</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Suzek</span> <span class="given-names" tagx="given-names" title="given-names">T. O.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bryant</span> <span class="given-names" tagx="given-names" title="given-names">S. H.</span> </span> </span> ( <span class="year" tagx="year" title="year">2009</span>). <span class="mixed-article-title" title="mixed-article-title">PubChem: a public information system for analyzing bioactivities of small molecules.</span> <span class="source" tagx="source" title="source"> <i>Nucleic Acids Res.</i> </span> <span class="volume" tagx="volume" title="volume">37</span> <span class="fpage" tagx="fpage" title="fpage">W623</span>â€“ <span class="lpage" tagx="lpage" title="lpage">W633</span>. <span class="pub-id"> <a href="https://dx.doi.org/10.1093/nar/gkp456">10.1093/nar/gkp456</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19498078">19498078</a> </span> </span> </li> </ul> </div> </div>  </body></html>